% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{book}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={A New Instrument for Microbial Epidemiology},
  pdfauthor={Matthijs S. Berends},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\usepackage{longtable,booktabs,array}
\usepackage{calc} % for calculating minipage widths
% Correct order of tables after \paragraph or \subparagraph
\usepackage{etoolbox}
\makeatletter
\patchcmd\longtable{\par}{\if@noskipsec\mbox{}\fi\par}{}{}
\makeatother
% Allow footnotes in longtable head/foot
\IfFileExists{footnotehyper.sty}{\usepackage{footnotehyper}}{\usepackage{footnote}}
\makesavenoteenv{longtable}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage{booktabs}
\usepackage{amsthm}
\makeatletter
\def\thm@space@setup{%
  \thm@preskip=8pt plus 2pt minus 4pt
  \thm@postskip=\thm@preskip
}
\makeatother
\ifluatex
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\usepackage[]{natbib}
\bibliographystyle{apalike}

\title{A New Instrument for Microbial Epidemiology}
\usepackage{etoolbox}
\makeatletter
\providecommand{\subtitle}[1]{% add subtitle to \maketitle
  \apptocmd{\@title}{\par {\large #1 \par}}{}{}
}
\makeatother
\subtitle{Empowering Antimicrobial Resistance Data Analysis}
\author{Matthijs S. Berends}
\date{25 August 2021}

\begin{document}
\maketitle

{
\setcounter{tocdepth}{1}
\tableofcontents
}
\hypertarget{preamble}{%
\chapter*{Preamble}\label{preamble}}
\addcontentsline{toc}{chapter}{Preamble}

This is the integral PhD thesis `A New Instrument for Microbial Epidemiology' (DOI \href{https://doi.org/10.33612/diss.177417131}{10.33612/diss.177417131}) by \href{https://www.rug.nl/staff/m.s.berends}{Matthijs S. Berends}, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021.

All texts were copied from the printed version `as is'; no modifications were made, although non-essential parts were left out (such as the personal acknowledgements and the curriculum vitae).

The shortened URL of the online version of this PhD thesis (an R Markdown project) is \href{https://git.io/PhDthesisAMR}{git.io/PhDthesisAMR} (case-sensitive).

\hypertarget{short-summary-250-words}{%
\subsection*{Short summary (250 words)}\label{short-summary-250-words}}
\addcontentsline{toc}{subsection}{Short summary (250 words)}

Treating infectious diseases requires insights into the microorganisms causing infectious diseases. Antimicrobial resistance (AMR) in microorganisms limits treatment possibilities and poses an enormous healthcare problem worldwide. The spread and AMR patterns of microorganisms, risk factors for infection, and preventive and control measures of infectious disease are studied within the field of Microbial Epidemiology, a cross-over field between Epidemiology and Clinical Microbiology. For analysing the spread and AMR patterns of microorganisms, however, no standardised method previously existed. This thesis showcases the development and applied use of a new instrument to analyse AMR data: the AMR package for R. From multiple viewpoints, the AMR package and its advantages are put into perspective: from a technical viewpoint, from an infection management viewpoint and from a clinical viewpoint. These combined provide a common ground for comprehending what the AMR package could yield in the field and how it can set a new empowered starting point for future applications of microbial epidemiology, in clinical and research settings alike. This thesis subsequently elaborates on these multiple viewpoints by illustrating the use of this new instrument in epidemiological research projects in the Dutch-German cross-border region to better understand the occurrence and AMR patterns of microorganisms on a (eu)regional level. In conclusion, this thesis shows the added value of a consistent data-analytical instrument to prepare and analyse AMR data in a full-region approach, that can also be used in clinical settings to obtain novel insights on AMR patterns.

\hypertarget{colophon}{%
\chapter*{Colophon}\label{colophon}}
\addcontentsline{toc}{chapter}{Colophon}

Cover design: Matthijs Berends (images used with permission)\\
Layout: Matthijs Berends\\
Printing: Gildeprint -- www.gildeprint.nl

The work described within this thesis was supported by (1) the Certe Medical Diagnostics and Advice Foundation, (2) the INTERREG V A (202085) funded project EurHealth-1Health (\url{http://www.eurhealth1health.eu}), part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport, the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia, and the Ministry for National and European Affairs and Regional Development of the German Federal State of Lower Saxony, (3) the European Union's Horizon 2020 Research and Innovation Programme under the Marie Skłodowska-Curie Grant Agreement 713660 (MSCA-COFUND-2015-DP ``Pronkjewail''), and (4) the European Society for Clinical Microbiology and Infectious Diseases (ESCMID) through the ESCMID Study Group for Antimicrobial Stewardship (ESGAP).

Printing of this thesis was financially supported by the Certe Medical Diagnostics and Advice Foundation. This support is greatly appreciated.

\textbf{Copyright © 2021 by Matthias Simeon Berends.} All rights reserved. Any unauthorised reprint or use of this material is prohibited. No parts of this thesis may be reproduced, stored, or transmitted in any form or by any means, without written permission of the author or, when appropriate, the publishers of the publications.

\hypertarget{contents}{%
\chapter*{Contents}\label{contents}}
\addcontentsline{toc}{chapter}{Contents}

\textbf{Section I}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\item
  \textbf{General Introduction}
\item
  \textbf{Diagnostic Stewardship: Sense or Nonsense?!}\\
  Berends MS*, Luz CF*, Wouthuyzen-Bakker M, Märtson AG, Alffenaar JW, Dik JWH, Glasner C, Sinha BNM\\
  \emph{Dutch Journal of Clinical Microbiology (2018) 26;3}
\item
  \textbf{Introducing a New, Free, and Independent Method for Standardised, Reproducible and Reliable Analyses of Antimicrobial Resistance Data}\\
  Berends MS, Luz CF, Sinha BNM, Glasner C\textsuperscript{‡}, Friedrich AW\textsuperscript{‡}\\
  \emph{In preparation}
\end{enumerate}

\textbf{Section II}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{3}
\item
  \textbf{AMR - An R Package for Working with Antimicrobial Resistance Data}\\
  Berends MS*, Luz CF*, Friedrich AW, Sinha BNM, Albers CJ, Glasner C\\
  \emph{Journal of Statistical Software (2021), ahead of print}
\item
  \textbf{Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship}\\
  Luz CF, Berends MS, Dik JWH, Lokate M, Pulcini C, Glasner C, Sinha BNM\\
  \emph{Journal of Medical Internet Research (2019) 21;6, e12843}
\item
  \textbf{Better Antimicrobial Resistance Data Analysis and Reporting in Less Time}\\
  Berends MS*, Luz CF*, Zhou X, Friedrich AW, Lokate ML, Sinha BNM\textsuperscript{‡}, Glasner C\textsuperscript{‡}\\
  \emph{medRxiv {[}preprint{]} (2021), 21257599}
\end{enumerate}

\textbf{Section III}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{6}
\item
  \textbf{Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019}\\
  Berends MS, Luz CF, Ott A, Andriesse GI, Becker K, Glasner C\textsuperscript{‡}, Friedrich AW\textsuperscript{‡}\\
  \emph{In preparation}
\item
  \textbf{Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region-Impact of National Guidelines}\\
  Köck R, Siemer P, Esser J, Kampmeier S, Berends MS, Glasner C, Arends JP, Becker K, Friedrich AW\\
  \emph{Microorganisms (2018) 6;1}
\item
  \textbf{Changing Epidemiology of Methicillin-Resistant \emph{Staphylococcus aureus} in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow-Policy}\\
  Jurke A, Daniels-Haardt I, Silvis W, Berends MS, Glasner C, Becker K, Köck R, Friedrich AW\\
  \emph{Eurosurveillance (2019) 24;15}
\item
  \textbf{A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures}\\
  Berends MS*, Glasner C*, Becker K, Esser J, Gieffers J, Jurke A, Kampinga G, Kampmeier S, Klont R, Köck R, Al Naemi N, Ott A, Ruis G, Saris K, Tami A, Van Zeijl J, Von Müller L, Voss A, Waar K, Friedrich AW\\
  \emph{Eurosurveillance (2021), ahead of print}
\end{enumerate}

\textbf{Section IV}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\setcounter{enumi}{10}
\tightlist
\item
  \textbf{Summary and Future Perspectives}
\end{enumerate}

\textbf{Gearfetting} yn Frysk\\
\textbf{Samenvatting} in het Nederlands\\
\textbf{Zusammenfassung} auf Deutsch\\
\textbf{Alphabetical list of published work}\\
\textbf{Alphabetical list of related presentations}\\
\textbf{Acknowledgements} / Tankwurd / Dankwoord / Danksagung\\
\textbf{Curriculum Vitae}

\begin{center}\rule{0.5\linewidth}{0.5pt}\end{center}

* Equal contribution\\
\textsuperscript{‡} Equal contribution

\hypertarget{ch01-introduction}{%
\chapter{General Introduction}\label{ch01-introduction}}

\hypertarget{microbial-epidemiology}{%
\section{Microbial epidemiology}\label{microbial-epidemiology}}

Epidemiology is the medical scientific field that investigates all the factors that determine the presence or absence of diseases and disorders. While many subspecialties within this field exist nowadays, such as veterinary epidemiology and cardiovascular epidemiology, its development started with an infectious disease. Between 1846 and 1860, the world endured the third cholera pandemic, taking assumably millions of lives \textsuperscript{{[}1{]}}. The year 1854 was considered the worst year, when 23,000 people died in the United Kingdom, out of 16 million inhabitants (0.14\%) \textsuperscript{{[}2{]}}. As a side note, this is still quite less than the 146,000 UK deaths due to COVID-19 out of 56 million inhabitants (0.26\%) until March 2021 \textsuperscript{{[}3{]}}. But 1854 was also the year that the basis was laid for the field of epidemiology by John Snow, an English physician and hygiene specialist.

At the time of a local cholera outbreak at the Broad Street in London in that year, Snow did not know the exact source of cholera and called it `cholera poison' in a book he published in 1856 \textsuperscript{{[}4{]}}. Interestingly, the Italian Filippo Pacini managed to isolate the bacterium causing cholera, \emph{Vibrio cholerae}, in 1854 -- the same year that Snow investigated the outbreak \textsuperscript{{[}5{]}}. Although it was not until 1884 that \emph{V. cholerae} was formally given its name by the German Robert Koch \textsuperscript{{[}6{]}}.

In his book about the `cholera poison' , Snow famously wrote \textsuperscript{{[}4{]}}:

\begin{quote}
There is no doubt that the mortality was much diminished, as I said before, by the flight of the population, which commenced soon after the outbreak; but the attacks had so far diminished before the use of the water was stopped, that it is impossible to decide whether the well still contained the cholera poison in an active state, or whether, from some cause, the water had become free from it.
\end{quote}

For this reason, Snow hypothesised that the local outbreak was caused by poisoned water coming from a water pump. To investigate the number of cases, he drew one of the most well-known data visualisations in epidemiology, Figure \ref{fig:fig1-1} (top). In this then-novel form of data visualisation, he counted the cases per household and denoted them as stacked rectangles. This resulted in his conclusion that there had been no particular outbreak or prevalence of cholera in that part of London except among the persons who were in the habit of drinking the water of one specific water pump: the one on Broad Street. The handle of the pump was removed on the day following his briefing to the local government, leading to an end of the outbreak.

With the advancements in information technology, heatmaps would nowadays be a more effective way to visualise geographic trends, Figure \ref{fig:fig1-1} (bottom). Using modern map data as illustrated, the incredible accuracy of Snow's drawing of London from 167 years ago is also highlighted. The type of investigating geographic trends in health and disease is nowadays known as \emph{spatial} epidemiology.

\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{images/01-01a} 

}

\end{figure}
\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{images/01-01b} 

}

\caption{Visualisations of the ‘Broad Street cholera outbreak’ in London in 1854. Top: original map as drawn by John Snow. Bottom: Snow’s original map with a self-made heatmap visualisation overlay, based on the geographic position of the cases. The blue circles (n = 13) indicate the location of the water pumps.}\label{fig:fig1-1}
\end{figure}

Spatial epidemiology is one example of the many different specialities in the field of epidemiology. Another example is the direct consequence of Snow's work: infectious disease epidemiology, which has developed widely since the nineteenth century and has become the de facto standard for researching diseases and their health effects caused by pathogens (i.e., bacteria, viruses and fungi). Since this speciality concerns pathogens, it is a domain shared by the fields of epidemiology and clinical microbiology (Figure \ref{fig:fig1-2}). Moreover, infectious disease epidemiology can be split into two subspecialties: clinical (infectious disease) epidemiology and microbial epidemiology. The former focuses on the properties of the disease (such as the burden of disease caused by infection, or the disease-related mental and financial costs), while the latter focuses on the properties of the pathogen (such as the credibility of its source, antimicrobial resistance and pathogenicity).

Applying microbial epidemiology was barely possible in the days of John Snow, for the lack of scientific knowledge about pathogens and the lack of advancement in information technology. Antibiotics were not discovered yet, the cause of cholera was undetermined, and scientists had no clue about the infectivity and pathogenicity of different bacteria. However, what John Snow did in 1854 `clinical epidemiologically', is in essence quite equal to what we currently do on a large scale during the COVID-19 pandemic. Information technology required to attain this large scale has brought us not only the possibilities to look beyond regional, national and international borders but to observe, analyse and understand pandemics in real-time. Methods we develop and use today can be implemented on the other side of the world tomorrow. This is an important advantage in modern infectious disease epidemiology, as is also illustrated in this thesis.

Microbial epidemiology has an important focus on observing and analysing (1) the microorganisms that cause infections and the human site of origin, (2) the intrinsic or acquired antimicrobial resistance they manifest, and (3) their infectivity and pathogenicity. As any type of microorganism -- bacteria, viruses and fungi (including yeasts) -- can cause infections in humans, microbial epidemiology is not limited to a certain type of microorganism. Nonetheless, there tends to be a stronger focus on bacteria and fungi, which are more easily isolated at a clinical microbiology laboratory than viruses and can be tested for phenotypical antimicrobial resistance in a routine diagnostic setting. Based on these diagnostic findings, treatment guidelines are developed and evaluated. This in itself urges microbial epidemiology to be employed in a routine setting as well, to make sure that treatment guideline development continually has a solid epidemiological basis.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/01-02} 

}

\caption{Overview of the diverse sections and subspecialties of epidemiology and clinical microbiology and their common field: infectious disease epidemiology. Microbial epidemiology can be considered to be a subspecialty of infectious disease epidemiology.}\label{fig:fig1-2}
\end{figure}

\hypertarget{antimicrobial-resistance-in-microorganisms}{%
\section{Antimicrobial resistance in microorganisms}\label{antimicrobial-resistance-in-microorganisms}}

The antimicrobial resistance (AMR) that manifests in bacteria and fungi, is central within the diverse field of microbial epidemiology. It occurs when microorganisms develop mechanisms that protect them from the effects of antimicrobial agents, such as antibiotics \textsuperscript{{[}7{]}}. AMR occurring specifically in bacteria is often termed antibiotic resistance (ABR). An important distinction should be made between intrinsic AMR (that is, AMR inherently present in certain microbial species as a distinctive property of that species) and acquired AMR (that is, AMR present in some strains of a certain microbial species induced by the presence of an antimicrobial agent). Infections caused by microorganisms that are resistant to one or more antimicrobial agents cannot be treated with those antimicrobial agents anymore.

AMR is a global health problem and of great concern for human medicine, veterinary medicine, and the environment alike. It is associated with significant burdens to both patients and health care systems. Current estimates show the immense dimensions we are already facing, such as claiming at least 50,000 lives due to AMR each year across Europe and the US alone \textsuperscript{{[}8{]}}. Although estimates for the burden through AMR and their predictions are disputed by some, the rising trend is undeniable, thus calling for worldwide efforts to tackle this problem \textsuperscript{{[}9,10{]}}. For this reason, laboratory diagnostics are of utmost importance for generating AMR results that can be used to acquire new or improved AMR insights by conducting microbial epidemiology.

\hypertarget{laboratory-diagnostics}{%
\subsection{Laboratory diagnostics}\label{laboratory-diagnostics}}

From clinical illness alone (such as fever, redness, swelling, pain, and loss of function), it is impossible to determine whether the microorganism causing the infection is drug-resistant; it requires laboratory diagnostics to measure AMR. For decades, clinical microbiological laboratories have been using techniques where a defined amount of a microbial isolate is brought unto the medium of an agar plate \textsuperscript{{[}11{]}}. This technique is called the `disk diffusion test' and was first used by Dutch botanist Martinus Beijerinck in 1889 to study the effect of auxins (a class of plant hormones) on bacterial growth \textsuperscript{{[}11,12{]}}. The technique has been further developed and refined by the American microbiologists William Kirby and Alfred Bauer in 1959 and 1966, leading to this test technique sometimes being referred to as the `Kirby-Bauer test' or `KB test' \textsuperscript{{[}13,14{]}}. To perform the test, small filter paper disks containing a specified concentration of different antimicrobial agents are laid on the agar medium containing the microorganism, which is subsequently incubated for 18 to 24 hours at a specified temperature. During the incubation, the antimicrobial agent (antibiotic or antifungal) will radially diffuse over the agar, leading to high antimicrobial concentrations near the disk and low antimicrobial concentrations away from the disk. A disk typically has a diameter of 6 millimetres. After the incubation, the growth inhibition zone around the disk can be measured with a ruler. The wider the growth inhibition zone, the lower antimicrobial concentrations are required for the microorganism to inhibit growth. The narrower the growth inhibition zone, the higher antimicrobial concentrations are required for the microorganism to inhibit growth. The range of a disk diffusion test result is typically 6 to 50 millimetres.

Although disk diffusion tests is being widely used in many areas, some laboratories have replaced them with an automated incubator allowing colourimetric detection of CO2 produced by growing microorganisms in the presence of antimicrobial agents \textsuperscript{{[}15--17{]}}. Growth is subsequently optically measured for different concentrations and different antimicrobial agents. The concentration that inhibits at least 99.99\% growth of the microorganism, is denoted the minimum inhibitory concentration (MIC) and is typically expressed in milligrams per litre (mg/L). These incubators are referred to as antimicrobial susceptibility testing (AST) devices. AST devices allow for timely and reproducible results. Yet, the cartridges used for this type of instrument have a limited number of wells to test different manufacturer-set concentrations and types of antimicrobial agents. Since this limitation thus disallows testing for any desired concentration, MICs are often capped at a minimum or maximum value. For example, an actual MIC could be 128 mg/L, although the highest available concentration on a cartridge could be 32 mg/L. In such cases, the MIC will be reported as ≥ 32 mg/L. This is a technical limitation of colourimetric detection of CO2 production as a test technique, which brings important disadvantages for microbial epidemiological analyses. Capped values (such as ≤ 0.0125 mg/L and ≥ 32 mg/L) hinder comparison with previous findings or findings from other laboratories as they might conceal the true MICs. Furthermore, different cartridges may be used for bacteria isolated from different specimen types (such as urine or blood), which can yield different ranges of the resulting MICs. For example, an isolate of Staphylococcus aureus from a urinary tract infection could be tested for many concentrations of only a few orally available antibiotics using cartridge A, while an isolate of S. aureus from a complex surgical wound could be tested for only a few concentrations of many intravenously available antibiotics using cartridge B. Consequently, the MIC of e.g., ciprofloxacin could be reported as ≤ 0.0625 mg/L using cartridge A, while it could be reported as ≤ 0.125 mg/L using cartridge B, even when the S. aureus isolates are identical. This makes it hard to compare results in epidemiological data analyses as the data availability can (unknowingly) be unequal, potentially affecting the outcome of any AMR data analysis.

\hypertarget{interpretation-of-raw-results}{%
\subsection{Interpretation of raw results}\label{interpretation-of-raw-results}}

When raw AMR testing results are available, they are not yet suitable for reporting back to clinicians. The growth inhibition zones of disk diffusion tests and the MICs from the colourimetric detection tests need interpretation to consider an antimicrobial agent suitable for treatment. Typically, AMR is interpreted and reported as either (a tri-form abbreviated as `RSI'):

\begin{itemize}
\item
  R = resistant. A microorganism is categorised as `resistant' when there is a high likelihood of therapeutic failure even when there is increased exposure.\\
  Exposure is a function of how the mode of administration, dose, dosing interval, infusion time, as well as distribution and excretion of the antimicrobial agent will influence the infecting organism at the site of infection.
\item
  S = susceptible. A microorganism is categorised as `susceptible' when there is a high likelihood of therapeutic success using a standard dosing regimen of the agent.
\item
  I\\
  (according to CLSI) I = intermediate. A microorganism is categorised as `intermediate' when there is an unsure likelihood of therapeutic success. Additionally, CLSI considers a susceptible dose-dependent (SDD) category for certain drug and organism combinations, for which the susceptibility of an isolate depends on the dosing regimen used.\\
  (according to EUCAST) I = Susceptible, increased exposure. A microorganism is categorised as such when there is a high likelihood of therapeutic success because exposure to the agent is increased by adjusting the dosing regimen or by its concentration at the site of infection.
\end{itemize}

For this interpretation of raw AMR test results, international guidelines exist. The most often applied guidelines are supplied by the Clinical and Laboratory Standards Institute (CLSI) and the European Committee on Antimicrobial Susceptibility Testing (EUCAST) \textsuperscript{{[}18,19{]}}. In Europe, an increasing number of clinical laboratories apply EUCAST guidelines, as it was shown that the coverage of EUCAST guidelines among these laboratories was 73.2\% in 2013, and only a few European countries did not use the EUCAST methodology in 2019 \textsuperscript{{[}20,21{]}}. According to the World Health Organisation (WHO), guidelines from CLSI and EUCAST are adopted by 94\% of all countries reporting AMR to the Global Antimicrobial Resistance Surveillance System (GLASS) of the WHO \textsuperscript{{[}22{]}}.

Generally, AMR is defined as the proportion of resistant microorganisms (R) among all tested microorganisms of the same species (R + S + I). The CLSI and EUCAST guidelines define the interpretations for the most common combinations of pathogenic microorganisms and antimicrobial agents. For example, the EUCAST 2021 guideline considers ciprofloxacin against Escherichia coli to be susceptible when either the MIC is at most 0.25 mg/L or when a diffusion disk with 5 µg has a growth inhibition zone of at least 25 millimetres (Figure \ref{fig:fig1-3}).

In 2017, EUCAST implemented the area of technical uncertainty (ATU) for certain microbial species/antibiotic combinations, to warn laboratory staff that the interpretation of routine susceptibility testing is uncertain \textsuperscript{{[}23{]}}. For example, disk diffusion results from the combination of any species in the order of Enterobacterales with amoxicillin/clavulanic acid are considered unreliable for a zone diameter of 19-20 mm in the latest EUCAST interpretation guideline \textsuperscript{{[}24{]}}. EUCAST advises to rerun the test, perform an additional test, or to report this uncertainty with a clear warning \textsuperscript{{[}23{]}}.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/01-03} 

}

\caption{Interpretation of 100 random minimum inhibitory concentrations (top) and 100 random disk diffusion growth inhibition zones (bottom) of ciprofloxacin in *Escherichia coli*, interpreted using colours according to the EUCAST 2021 guideline. These plots were generated with the AMR package for R.}\label{fig:fig1-3}
\end{figure}

To mitigate the risks of laboratories reporting erroneous susceptibility results, CLSI and EUCAST guidelines are also provided as ``expert rules'' in the previously mentioned AST devices, which helps to ensure compliance with guidelines and standards, increasing the quality of AMR data \textsuperscript{{[}25{]}}.

Analysing AMR data, such as raw MICs and antimicrobial interpretations (`RSI'), is tedious and complex, especially when evaluating cumulative AMR reports \textsuperscript{{[}26{]}}. Nonetheless, it is essential to monitor up-and-coming AMR trends at the local and regional level to support clinical decision-making, infection control interventions, and AMR containment strategies \textsuperscript{{[}27,28{]}}. AMR data analysis has been challenged by poor comparability of antimicrobial susceptibility statistics between institutions because of the diversity of calculation methods \textsuperscript{{[}26{]}}. Moreover, many laboratories have used simplistic calculation approaches, with a strong tendency to overestimate drug resistance rates \textsuperscript{{[}26{]}}. In the first ten years of this century, it was shown that this was primarily attributed to the lack of correction for duplicate isolates \textsuperscript{{[}29--31{]}}.

In an attempt to overcome this, CLSI started in 2002 with developing guidelines to recommend epidemiologically sound workflows for the analysis and presentation of AMR results and trends, with their fourth and currently latest version released in 2014 \textsuperscript{{[}32{]}}. These guidelines comprise advice on the inclusion of a minimum number of isolates, the choice of antimicrobial agents to analyse, and the presenting of numbers and percentages of AMR. In 2007, Hindler \emph{et al.} evaluated the then-latest version of this guideline \textsuperscript{{[}26{]}}. They concluded that although CLSI provided a comprehensive collection of suggestions, only a few publications had implemented these practical recommendations. Nevertheless, it continuously provides a theoretical basis for microbial epidemiological analyses but lacks suggestions of how these theoretical recommendations can be implemented practically or what kind of software would be suitable to analyse AMR data and, more specific, AMR data about multi-drug resistant organisms.

\hypertarget{multi-drug-resistant-organisms}{%
\subsection{Multi-drug resistant organisms}\label{multi-drug-resistant-organisms}}

Multi-drug resistant organisms (MDROs) are microorganisms that acquired AMR to at least one antimicrobial agent in multiple antimicrobial categories. Because of MDROs, there are countries in many parts of the world where antimicrobial treatment is ineffective in more than half of all patients \textsuperscript{{[}33{]}}. Common MDROs include vancomycin-resistant enterococci (VRE), methicillin-resistant \emph{Staphylococcus aureus} (MRSA), extended-spectrum β-lactamase (ESBL) producing Gram-negative bacteria such as \emph{E. coli} and \emph{Klebsiella pneumoniae}, carbapenemase-producing Gram-negative bacteria, third-generation cephalosporin (3GC) resistant Gram-negative bacteria and carbapenemase-producing Gram-negative bacteria.

In 2012, MDROs were formally categorised into different degrees of severity in favour of international comparison purposes \textsuperscript{{[}34{]}}. Multi-drug resistance (MDR) was defined as acquired AMR to three or more antimicrobial categories, extensive drug resistance (XDR) was defined as acquired AMR to all antimicrobial agents except in two or fewer antimicrobial categories, and pan-drug resistance (PDR) was defined as acquired AMR to all antimicrobial agents in all antimicrobial categories \textsuperscript{{[}34{]}}. MDR among microorganisms is very common, PDR is very uncommon \textsuperscript{{[}7,33,35{]}}. In 2014, the WHO published a report in which they performed five systematic reviews involving 221 studies with a special focus on MDR bacteria (defined as MRSA, 3GC/fluoroquinolone-resistant E. coli, and 3GC/carbapenem-resistant K. pneumoniae) \textsuperscript{{[}36{]}}. The outcomes of this report underlined the increasing necessity of surveillance programs.

\hypertarget{surveillance-programs}{%
\subsection{Surveillance programs}\label{surveillance-programs}}

With the current WHO surveillance program GLASS, the overall coverage of AMR is continuously being monitored for most countries of the world \textsuperscript{{[}37{]}}. For Europe, the prevalence of AMR on the country level is monitored by national surveillance programs that share their data with the European Centre for Disease Prevention and Control (ECDC), an agency of the European Union \textsuperscript{{[}38{]}}. Their surveillance program European Antimicrobial Resistance Surveillance Network (EARS-Net) is the largest publicly funded system for AMR surveillance in Europe. Public access to descriptive data (maps, graphs and tables) are available through the ECDC Surveillance Atlas of Infectious Diseases \textsuperscript{{[}38{]}}, which was also consulted for multiple studies in this thesis. While the ECDC estimated in 2009 that bacterial infections caused by MDROs were responsible for 25,000 extra deaths per year \textsuperscript{{[}39{]}}, others found that there is a large discrepancy between the real count of deaths attributable to MDROs and the subsequent alarmist predictions, based on data from over 500 studies \textsuperscript{{[}35{]}}.

Although surveillance programs allow for signalling significant differences and shifts in AMR rates, additional AMR data analyses and AMR surveillance studies are strict requirements to fully understand the continuous development in AMR rates as there is no ``ideal'' surveillance system covering all aspects \textsuperscript{{[}28{]}}. Nonetheless, the desire to continuously monitor, analyse, model and predict AMR, has led to the increased development and use of local, regional, national and international surveillance systems \textsuperscript{{[}27{]}}. Critchley \emph{et al.} have inventoried the requirement set by different types of users (Table 1).

On the local level, clinical microbiology laboratories should communicate AMR surveillance data to healthcare providers in an understandable manner. Since MDROs can migrate between healthcare institutions, countries and continents by migrating people, local healthcare providers should be aware of local, regional, national and international surveillance program implementations and their ensuing results on AMR. On the other hand, such surveillance program implementations should be well-designed, well-maintained, longitudinal, and involve an appropriate collaboration with local laboratories over time \textsuperscript{{[}27{]}}.

Table 1. Uses of antibiotic resistance surveillance system data by hospitals, university researchers, pharmaceutical companies and governments. From Critchley \emph{et al.}, 2004 \textsuperscript{{[}27{]}}.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/01-t01} 

}

\end{figure}

As an example, ISIS-AR (Infectious disease Surveillance Information System for Antibiotic Resistance) is a Dutch national surveillance program, for which a large number of the Dutch clinical microbiology laboratories provide anonymised data on AMR to the National Institute for Public Health and the Environment (Rijksinstituut voor Volksgezondheid en Milieu, RIVM) \textsuperscript{{[}40{]}}. In Germany, ARS (Antibiotic Resistance Surveillance) is a similar laboratory-based national surveillance program, that attempts to enable differential statements according to structural characteristics of health care and regions \textsuperscript{{[}41,42{]}}. Both these national surveillance programs provide data for EARS-Net and GLASS of the WHO \textsuperscript{{[}37,43{]}}.

\hypertarget{data-analysis-using-r}{%
\section{Data analysis using R}\label{data-analysis-using-r}}

In academia, the free and open-source statistical language R is an increasingly popular tool for analysing study results and developing new scientific methods, especially in medical fields such as human genetics, health decision sciences, and proteomics \textsuperscript{{[}44--47{]}}. Even more so, a new type of study seems to currently arise where researchers from different medical fields publish tutorials on how to acquire new insights using R as a programming language \textsuperscript{{[}48--50{]}}. In 2020, R ranked 8th in the TIOBE index, a global initiative to measure the popularity of programming languages, while it ranked 73rd in 2008 \textsuperscript{{[}51{]}}.

R was developed for statistical computing and graphics supported by the R Foundation for Statistical Computing \textsuperscript{{[}52,53{]}}. It is freely available under the GNU General Public License v2, meaning that it may be used for both private and commercial purposes in any way, but not for patent purposes. As a statistical package, it is comparable to the proprietary software programs Stata, SAS and SPSS \textsuperscript{{[}54{]}}. However, as opposed to these proprietary software programs, R has an open file format and can read data from any source, including files from other software programs, and websites. Moreover, the `base' functions of R are extendible by users who develop so-called packages for R. The Comprehensive R Archive Network (CRAN) that hosts and maintains R through the R Foundation for Statistical Computing, accepts package submissions from users and subjects users to a peer-review submission process and a strict repository policy \textsuperscript{{[}53,55{]}}. As of May 2021, the CRAN package repository features 17,671 available packages.

Not only the popularity of using R has increased over the last decade. The number of developed packages has also increased strongly over the last years, especially since 2016 (Figure \ref{fig:fig1-4}). This is probably attributed to a rather new integrated desktop environment (IDE) to use R, called RStudio \textsuperscript{{[}56{]}}. RStudio is also the name of the corporation that developed the RStudio IDE and authored the so-called tidyverse, a collection of R packages (such as dplyr and ggplot2) that are specifically designed to ease data importing, tidying, manipulating, visualising, and programming, as well as to improve code reading \textsuperscript{{[}57--59{]}}. The tidyverse can be used for most data analytical tasks and has been the method of choice for numerous (clinical) studies, including those presented in this thesis.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/01-04} 

}

\caption{The number of R packages by date of the last update over the last ten years. Every bar represents one month. Every R package occurs once in this figure.}\label{fig:fig1-4}
\end{figure}

For microbial epidemiology, no particular R packages were available to analyse phenotypic AMR test results as of 2017. One R package that provides approaches to work with disk diffusion zone diameters and MICs from environment samples started development in 2018, but still has no released version as of May 2021 \textsuperscript{{[}60{]}}. For `non-microbial' infectious disease epidemiology, however, outbreaks and epidemics could already be analysed with dedicated packages in R \textsuperscript{{[}61--65{]}}. Most of these packages were developed within RECON, the R Epidemics Consortium, that gathers experts in data science, modelling methodology, public health, and software development to create the next generation of analytics tools in R for informing the response to disease outbreaks, health emergencies and humanitarian crises. Their R package EpiEstim is being used worldwide for calculating and presenting reproduction rates of SARS-CoV-2 during the ongoing COVID-19 pandemic, also by the Dutch National Institute for Public Health and the Environment (RIVM) \textsuperscript{{[}65,66{]}}.

\hypertarget{setting-for-this-thesis}{%
\section{Setting for this thesis}\label{setting-for-this-thesis}}

Studies within this thesis were geographically organised or initiated in the Northern cross-border region of the Netherlands and Germany, Figure \ref{fig:fig1-5}. According to the German philosopher Liessmann, there are only national borders defined by humans, but no natural borders \textsuperscript{{[}67{]}}. He explained that borders as man-made conventions are never absolute, but that it is always possible to cross them. Despite the existing territorial border, there are many similarities in the Netherlands and Germany today, but just as many and clear differences, especially concerning the healthcare sector. A German patient can become a patient in the Netherlands just as quickly as a Dutch patient can in Germany. Since pathogens know no borders, patient protection and infection prevention must not stop at borders \textsuperscript{{[}68{]}}. The Netherlands and Germany have, among many other matters, apparent differences within the healthcare system in general and in terms of AMR, especially concerning MDRO definitions and infection prevention guidelines. To study these differences, INTERREG programs enable cross-border, transnational and interregional cooperation. INTERREG is one of the central instruments in European cohesion and regional policy, with which the development differences between the European countries in the border regions should be reduced and economic cohesion strengthened. It aims to ensure that national borders are not an obstacle to the balanced development and integration of the European territory \textsuperscript{{[}69{]}}. One of its programs, EurHealth-1Health, was a large research project that aimed to facilitate working together in battling AMR and MDROs and to empower sustainable collaborations across the border.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/01-05} 

}

\caption{Geographic overview of three Euregio’s that make up most of the Dutch-German cross-border region.}\label{fig:fig1-5}
\end{figure}

In the Northern Netherlands, five clinical microbiological laboratories together conduct the microbiological diagnostics for more than two million Dutch inhabitants in primary care, secondary care (non-university hospitals) and tertiary care (university hospital). Three of these five are regional non-profit laboratories: Izore in Leeuwarden (Friesland), Certe in Groningen (Groningen) and LabMicTA in Hengelo (Overijssel). The other two laboratories are hospital departments of the Isala hospital in Zwolle (Overijssel) and the University Medical Center Groningen. On the other side of the border in Germany, laboratories are more numerous, more centralised, often privatised, and organised on a different scale than in the Netherlands. This is largely due to a higher number of small hospitals in Germany compared to the Netherlands, which is inherent to the different healthcare structures. In 2018, Germany had 2.33 hospitals per 100,000 inhabitants (1 hospital per 43,010 inhabitants), while in the Netherlands this was 0.68 hospitals per 100,000 inhabitants (1 hospital per 148,113 inhabitants), almost 3.5 times less \textsuperscript{{[}70--73{]}}.

These differences posed important reasons to research the effects of having different national guidelines regarding AMR (and MDRO interpretations) and screening guidelines, as is investigated in this thesis.

\hypertarget{aim-of-this-thesis-and-introduction-to-its-chapters}{%
\section{Aim of this thesis and introduction to its chapters}\label{aim-of-this-thesis-and-introduction-to-its-chapters}}

This thesis aims to present the development of a new instrument for microbial epidemiology -- a new and open method for standardised AMR data analysis -- while also providing applied examples of how this new instrument has empowered AMR data analysis in regional and euregional studies.

This thesis is presented in four sections.

\textbf{SECTION I} opens with a broad introduction to the usefulness and necessity of having timely diagnostic information in \textbf{chapter 2}. Diagnostic stewardship programs (DSP) are a requirement to gain answers instead of results, including those from a clinical microbiology laboratory. DSP is a multidisciplinary approach to gain the most benefit for the patient by democratising different medical specialities. In \textbf{chapter 3}, the usefulness and necessity of having a dedicated tool for microbial epidemiology are introduced, through the AMR package for R as a new instrument. It is explained why microbial epidemiology and its effects are hindering efforts to dispose of AMR trends and how the AMR package for R can compensate for this. This chapter was primarily intended for non-data-technical professionals who work in the field of infectious diseases, such as clinical microbiologists and infectiologists.

\textbf{SECTION II} outlines the working and implementation of the AMR package for R. It starts with explaining this newly developed instrument in \textbf{chapter 4}. In this methodological and technical paper, the working mechanisms of the AMR package for R are thoroughly described. It is demonstrated that the AMR package enables standardised and reproducible AMR data analyses, including the application of evidence-based rules, determination of first isolates, translation of various codes for microorganisms and antimicrobial agents, determination of (multi-drug) resistant microorganisms, and calculation of antimicrobial resistance, prevalence and future trends. This chapter was primarily intended for data-technical professionals who work in the field of microbiology, such as (infectious disease) epidemiologists and biostatisticians. For \textbf{chapter 5}, the AMR package was implemented in a newly developed web application to present the design, development, and testing of RadaR (Rapid analysis of diagnostic and antimicrobial patterns in R), a software app for infection management, and to ascertain whether RadaR can facilitate user-friendly, intuitive, and interactive analyses of large datasets in the absence of prior in-depth software or programming knowledge. Subsequently, in \textbf{chapter 6}, we aimed at demonstrating and studying the usability of our developed approach and its impact on clinicians' workflows in a typical scenario. By comparing traditional software methods such as Excel and SPSS with an online implementation of our new instrument, we tried to establish the benefit of using dedicated tools in a clinical situation.

\textbf{SECTION III} provides real-life examples of how the new instrument was used in studies that focus on AMR data analysis, in the Northern Dutch region as well as the Northern cross-border region of the Netherlands and Germany. \textbf{Chapter 7} brings a thorough analysis of the occurrence and antibiotic resistance of coagulase-negative staphylococci (CoNS) in the Northern three provinces of the Netherlands, by analysing almost 20,000 antibiograms. Since 2013, all regional clinical microbiological laboratories make use of matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry to identify microbial isolates to the species level. Using the AMR package for R, all relevant antibiotic results could be analysed for all different CoNS species that were found during the study period (2013-2019). In \textbf{chapter 8}, country-specific guidelines for determining MDROs in the Netherlands and Germany were compared in this border region. This was done by interpreting all isolates found on both sides of the border with the national guidelines from both countries. Major differences were observed, which also imply a strong challenge for healthcare personnel working in the border region. Isolate selection and MDRO determination on the Dutch side of the border was carried out using the AMR package. \textbf{Chapter 9} outlines the euregional epidemiology of methicillin-resistant Staphylococcus aureus (MRSA) by analysing results from 42 hospitals. MRSA colonisation, infection and bacteraemia rate trends were described from the Dutch-German border region hospitals between 2012 and 2016. Although measures for MRSA cases were similar in both countries, defining patients at risk for MRSA differed. For \textbf{chapter 10}, twenty-three hospitals in the Dutch-German border region participated in a prospective screening study for the determination of the carriage of multi-drug resistance on admission to intensive care units (ICU), including more than 3,000 patients. The screening compliance, hospital and ICU sizes, and outcome of AMR data analysis were compared between both sides of the border.

\textbf{SECTION IV} summarises the presented work and provides future perspectives.

\hypertarget{references}{%
\section*{References}\label{references}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Hays JN. Epidemics and pandemics: their impacts on human history. Santa Barbara, Calif.; 2005.
\item
  Paneth N, Vinten-Johansen P, Brody H, Rip M. A rivalry of foulness: official and unofficial investigations of the London cholera epidemic of 1854. Am J Public Health 1998;88:1545--53. \url{doi:10.2105/AJPH.88.10.1545}.
\item
  Office for National Statistics. Deaths with COVID-19 on the death certificate. 5 March 2021 2021. \url{https://coronavirus.data.gov.uk/details/deaths} (accessed March 21, 2021).
\item
  Snow J. On the Mode of Communication of Cholera. Edinb Med J 1856;1:668--70.
\item
  Pacini F. Osservazioni microscopiche e deduzioni patologiche sul cholera asiatico. Gazz Medica Ital Toscana 1854;4:397--401.
\item
  Howard-Jones N. Robert Koch and the cholera vibrio: a centenary. BMJ 1984;288:379--81. \url{doi:10.1136/bmj.288.6414.379}.
\item
  World Health Organization. Antimicrobial resistance Fact sheet N°194. April 2014 2014. \url{https://www.who.int/mediacentre/factsheets/fs194/en/} (accessed March 21, 2021).
\item
  O'Neill J. Antimicrobial Resistance: Tackling a Crisis for the Health and Wealth of Nations. Rev Antimicrob Resist 2014:1--16.
\item
  de Kraker MEA, Stewardson AJ, Harbarth S. Will 10 Million People Die a Year due to Antimicrobial Resistance by 2050? PLOS Med 2016;13:e1002184. \url{doi:10.1371/journal.pmed.1002184}.
\item
  Centers for Disease Control and Prevention (CDC). AR Threats Report: Antibiotic Resistance Threats In The United States. 2019.
\item
  Humphries RM, Kircher S, Ferrell A, Krause KM, Malherbe R, Hsiung A, \emph{et al.} The Continued Value of Disk Diffusion for Assessing Antimicrobial Susceptibility in Clinical Laboratories: Report from the Clinical and Laboratory Standards Institute Methods Development and Standardization Working Group. J Clin Microbiol 2018;56. \url{doi:10.1128/JCM.00437-18}.
\item
  Wheat PF. History and development of antimicrobial susceptibility testing methodology. J Antimicrob Chemother 2001;48:1--4. \url{doi:10.1093/jac/48.suppl_1.1}.
\item
  Bauer AW, Kirby WMM, Sherris JC, Turck M. Antibiotic Susceptibility Testing by a Standardized Single Disk Method. Am J Clin Pathol 1966;45:493--6. \url{doi:10.1093/ajcp/45.4_ts.493}.
\item
  BAUER AW. Single-Disk Antibiotic-Sensitivity Testing of Staphylococci. AMA Arch Intern Med 1959;104:208. \url{doi:10.1001/archinte.1959.00270080034004}.
\item
  Sakoulas G, Gold HS, Venkataraman L, DeGirolami PC, Eliopoulos GM, Qian Q. Methicillin-Resistant Staphylococcus aureus: Comparison of Susceptibility Testing Methods and Analysis of mecA-Positive Susceptible Strains. J Clin Microbiol 2001;39:3946--51. \url{doi:10.1128/JCM.39.11.3946-3951.2001}.
\item
  Pérez-Vázquez M, Oliver A, Sánchez del Saz B, Loza E, Baquero F, Cantón R. Performance of the VITEK2 system for identification and susceptibility testing of routine Enterobacteriaceae clinical isolates. Int J Antimicrob Agents 2001;17:371--6. \url{doi:10.1016/S0924-8579(01)00318-1}.
\item
  Stürenburg E, Sobottka I, Feucht H-H, Mack D, Laufs R. Comparison of BDPhoenix and VITEK2 automated antimicrobial susceptibility test systems for extended-spectrum beta-lactamase detection in Escherichia coli and Klebsiella species clinical isolates. Diagn Microbiol Infect Dis 2003;45:29--34. \url{doi:10.1016/S0732-8893(02)00481-9}.
\item
  Clinical and Laboratory Standards Institute (CLSI) 2021. \url{https://clsi.org} (accessed March 21, 2021).
\item
  European Committee on Antimicrobial Susceptibility Testing (EUCAST) 2021. \url{https://eucast.org} (accessed March 21, 2021).
\item
  Brown D, Cantón R, Dubreuil L, Gatermann S, Giske C, MacGowan A, \emph{et al.} Widespread implementation of EUCAST breakpoints for antibacterial susceptibility testing in Europe. Eurosurveillance 2015;20. \url{doi:10.2807/1560-7917.ES2015.20.2.21008}.
\item
  European Centre for Disease Prevention and Control. Antimicrobial resistance in the EU/EEA (EARS-Net): Annual Epidemiological Report for 2019. 2019.
\item
  World Health Organization. Global Antimicrobial Resistance Surveillance System (GLASS) Report: Early Implementation 2017-2018. 2018.
\item
  EUCAST. Area of Technical Uncertainty (ATU) in antimicrobial susceptibility testing 2019. \url{https://www.eucast.org/fileadmin/src/media/PDFs/EUCAST_files/Disk_test_documents/ATU/Area_of_Technical_Uncertainty_-_guidance_2019.pdf} (accessed July 7, 2021).
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Breakpoint tables for interpretation of MICs and zone diameters. Version 11.0. 2021.
\item
  Clinical and Laboratory Standards Institute. Performance standards for antimicrobial susceptibility testing; approved standard - 28th ed M100. Wayne (Pennsylvania): 2018.
\item
  Hindler JF, Stelling J. Analysis and Presentation of Cumulative Antibiograms: A New Consensus Guideline from the Clinical and Laboratory Standards Institute. Clin Infect Dis 2007;44:867--73. \url{doi:10.1086/511864}.
\item
  Critchley IA, Karlowsky JA. Optimal use of antibiotic resistance surveillance systems. Clin Microbiol Infect 2004;10:502--11. \url{doi:10.1111/j.1469-0691.2004.00911.x}.
\item
  Bax R, Bywater R, Cornaglia G, Goossens H, Hunter P, Isham V, \emph{et al.} Surveillance of antimicrobial resistance --- what, how and whither? Clin Microbiol Infect 2001;7:316--25. \url{doi:10.1046/j.1198-743x.2001.00239.x}.
\item
  Bosso JA, Mauldin PD, Steed LL. Consequences of Combining Cystic Fibrosis--- and Non-Cystic Fibrosis-Derived Pseudomonas aeruginosa Antibiotic Susceptibility Results in Hospital Antibiograms. Ann Pharmacother 2006;40:1946--9. \url{doi:10.1345/aph.1H377}.
\item
  Horvat RT, Klutman NE, Lacy MK, Grauer D, Wilson M. Effect of Duplicate Isolates of Methicillin-Susceptible and Methicillin-Resistant Staphylococcus aureus on Antibiogram Data. J Clin Microbiol 2003;41:4611--6. \url{doi:10.1128/JCM.41.10.4611-4616.2003}.
\item
  Cebrián L, Rodríguez JC, Escribano I, Cascales E, López-Lozano JM, Royo G. Influence of various criteria for elimination of duplicates when calculating the prevalence and antibiotic susceptibility of microorganisms associated with urinary infections. Int J Antimicrob Agents 2005;25:173--6. \url{doi:10.1016/j.ijantimicag.2004.09.017}.
\item
  Clinical and Laboratory Standards Institute (CLSI). M39-A4 Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 2014.
\item
  World Health Organization. Antimicrobial resistance 2021. \url{https://www.who.int/news-room/fact-sheets/detail/antimicrobial-resistance} (accessed March 24, 2021).
\item
  Magiorakos A-P, Srinivasan A, Carey RB, Carmeli Y, Falagas ME, Giske CG, \emph{et al.} Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: an international expert proposal for interim standard definitions for acquired resistance. Clin Microbiol Infect 2012;18:268--81. \url{doi:10.1111/j.1469-0691.2011.03570.x}.
\item
  Abat C, Fournier P-E, Jimeno M-T, Rolain J-M, Raoult D. Extremely and pandrug-resistant bacteria extra-deaths: myth or reality? Eur J Clin Microbiol Infect Dis 2018;37:1687--97. \url{doi:10.1007/s10096-018-3300-0}.
\item
  World Health Organization. Antimicrobial resistance: global report on surveillance. 2014.
\item
  World Health Organization. Global Antimicrobial Resistance Surveillance System (GLASS) 2021. \url{https://www.who.int/glass/en/} (accessed March 25, 2021).
\item
  European Centre for Disease Prevention and Control. ECDC Surveillance Atlas of Infectious Diseases 2021. \url{http://atlas.ecdc.europa.eu} (accessed March 24, 2021).
\item
  European Centre for Disease Prevention and Control, European Medicines Agency. The bacterial challenge: time to react (Joint Technical Report, 2009). 2009.
\item
  Rijksinstituut voor Volksgezondheid en Milieu. Infectieziekten Surveillance Informatie Systeem-Antibiotica Resistentie (ISIS-AR) 2021. \url{https://www.rivm.nl/isis-ar} (accessed March 25, 2021).
\item
  Schweickert B, Noll I, Feig M, Claus H, Krause G, Velasco E, \emph{et al.} MRSA-surveillance in Germany: data from the Antibiotic Resistance Surveillance System (ARS) and the mandatory surveillance of MRSA in blood. Eur J Clin Microbiol Infect Dis 2012;31:1855--65. \url{doi:10.1007/s10096-011-1511-8}.
\item
  Robert Koch Institut. Antibiotika-Resistenz-Surveillance (ARS) 2021. \url{https://ars.rki.de} (accessed March 25, 2021).
\item
  European Centre for Disease Prevention and Control. European Antimicrobial Resistance Surveillance Network (EARS-Net) 2021. \url{https://www.ecdc.europa.eu/en/about-us/partnerships-and-networks/disease-and-laboratory-networks/ears-net} (accessed March 25, 2021).
\item
  Tippmann S. Programming tools: Adventures with R. Nature 2015;517:109--10. \url{doi:10.1038/517109a}.
\item
  Jalal H, Pechlivanoglou P, Krijkamp E, Alarid-Escudero F, Enns E, Hunink MGM. An Overview of R in Health Decision Sciences. Med Decis Mak 2017;37:735--46. \url{doi:10.1177/0272989X16686559}.
\item
  Gatto L, Christoforou A. Using R and Bioconductor for proteomics data analysis. Biochim Biophys Acta - Proteins Proteomics 2014;1844:42--51. \url{doi:10.1016/j.bbapap.2013.04.032}.
\item
  Chan BKC. Data Analysis Using R Programming. Adv. Exp. Med. Biol., vol.~1082, 2018, p.~47--122. \url{doi:10.1007/978-3-319-93791-5_2}.
\item
  Herber R, Kaiser A, Grählert X, Range U, Raiskup F, Pillunat LE, \emph{et al.} Statistische Auswertung korrelierter Messdaten in der Augenheilkunde. Der Ophthalmol 2020;117:27--35. \url{doi:10.1007/s00347-019-0904-4}.
\item
  Balduzzi S, Rücker G, Schwarzer G. How to perform a meta-analysis with R: a practical tutorial. Evid Based Ment Heal 2019;22:153--60. \url{doi:10.1136/ebmental-2019-300117}.
\item
  Muschelli J, Gherman A, Fortin J-P, Avants B, Whitcher B, Clayden JD, \emph{et al.} Neuroconductor: an R platform for medical imaging analysis. Biostatistics 2019;20:218--39. \url{doi:10.1093/biostatistics/kxx068}.
\item
  The Software Quality Company. TIOBE Index 2020. \url{https://www.tiobe.com/tiobe-index/}.
\item
  The Comprehensive R Archive Network, Hornik K. What is the R Foundation? 2017. \url{https://cran.r-project.org/doc/FAQ/R-FAQ.html\#What-is-the-R-Foundation_003f} (accessed August 6, 2018).
\item
  The Comprehensive R Archive Network, Hornik K. What is R? 2017. \url{https://cran.r-project.org/doc/FAQ/R-FAQ.html\#What-is-R_003f} (accessed August 6, 2018).
\item
  Burns P. R Relative to Statistical Packages: Comment 1 on Technical Report Number 1 (Version 1.0) Strategically using General Purpose Statistics Packages: A Look at Stata, SAS and SPSS. Stat Consult Gr UCLA Acad Technol Serv 2006.
\item
  The Comprehensive R Archive Network. CRAN Repository Policy 2021. \url{https://cran.r-project.org/web/packages/policies.html} (accessed March 25, 2021).
\item
  RStudio PBC. RStudio 2021. \url{https://www.rstudio.com/products/rstudio/} (accessed March 25, 2021).
\item
  Wickham H, Averick M, Bryan J, Chang W, McGowan L, François R, \emph{et al.} Welcome to the Tidyverse. J Open Source Softw 2019;4:1686. \url{doi:10.21105/joss.01686}.
\item
  Wickham H, François R, Henry L, Müller K. dplyr: A Grammar of Data Manipulation 2018.
\item
  Wickham H, Winston C, RStudio. ggplot2: Create Elegant Data Visualisations Using the Grammar of Graphics. CRAN 2016.
\item
  Petzoldt T. antibioticR: Analysis of Antibiotic Resistance Data 2021. \url{https://github.com/tpetzoldt/antibioticR} (accessed April 19, 2021).
\item
  Kamvar ZN, Cai J, Pulliam JRC, Schumacher J, Jombart T. Epidemic curves made easy using the R package incidence. F1000Research 2019;8:139. \url{doi:10.12688/f1000research.18002.1}.
\item
  Jombart T, Kamvar ZN, Cai J, Pulliam J, Chisholm S, FitzJohn R, \emph{et al.} reconhub/incidence: Incidence version 1.6.0 2019. \url{doi:10.5281/zenodo.2584018}.
\item
  Nagraj VP, Jombart T, Randhawa N, Sudre B, Campbell F, Crellen T. epicontacts: Handling, Visualisation and Analysis of Epidemiological Contacts 2017.
\item
  Cori A. EpiEstim: Estimate Time Varying Reproduction Numbers from Epidemic Curves 2021.
\item
  Cori A, Ferguson NM, Fraser C, Cauchemez S. A New Framework and Software to Estimate Time-Varying Reproduction Numbers During Epidemics. Am J Epidemiol 2013;178:1505--12. \url{doi:10.1093/aje/kwt133}.
\item
  Rijksinstituut voor Volksgezondheid en Milieu. Rekenmodellen openbaar en toegankelijk 2021. \url{https://www.rivm.nl/coronavirus-covid-19/hoe-berekeningen-bijdragen-aan-bestrijding-van-virus/rekenmodellen} (accessed March 25, 2021).
\item
  Link O. ``Ohne Grenzen könnten wir nicht leben'': Konrad Paul Liessmann im Interview 2013. \url{https://www.brandeins.de/magazine/brand-eins-wirtschaftsmagazin/2013/grenzen/ohne-grenzen-koennten-wir-nicht-leben} (accessed April 19, 2021).
\item
  Glasner C, Rocker D, Köck R, Pulz M, Jurke A, Smollich M, \emph{et al.} Deutschland -- Niederlande: Grenzenloser Schutz der Gesundheit. Umweltmed - Hyg - Arbeitsmed 2017;33:313--23.
\item
  European Commission. Budget of the European Cohesion policy 2020. \url{http://ec.europa.eu/regional_policy/en/funding/available-budget/} (accessed March 31, 2021).
\item
  Statista. Anzahl der Krankenhäuser in Deutschland 2018. \url{https://de.statista.com/statistik/daten/studie/2617/umfrage/anzahl-der-krankenhaeuser-in-deutschland-seit-2000/} (accessed March 31, 2021).
\item
  Statistisches Bundesamt. Bevölkerung nach Nationalität und Geschlecht 2018. \url{https://www.destatis.de/DE/Themen/Gesellschaft-Umwelt/Bevoelkerung/Bevoelkerungsstand/Tabellen/zensus-geschlecht-staatsangehoerigkeit-2018.html} (accessed March 31, 2021).
\item
  Volksgezondheidenzorg.info. Aantal instellingen voor medisch specialistische zorg 2019. \url{https://www.volksgezondheidenzorg.info/onderwerp/ziekenhuiszorg/cijfers-context/aanbod\#node-aantal-instellingen-voor-medisch-specialistische-zorg} (accessed March 31, 2021).
\item
  Statistics Netherlands {[}Centraal Bureau voor de Statistiek; CBS{]}. Bevolking; kerncijfers 2018. \url{https://opendata.cbs.nl/statline/\#/CBS/nl/dataset/37296ned/table?ts=1617225065173} (accessed March 31, 2021).
\end{enumerate}

\hypertarget{ch02-diagnostic-stewardship}{%
\chapter{Diagnostic Stewardship: Sense or Nonsense?!}\label{ch02-diagnostic-stewardship}}

Published in Dutch Journal of Clinical Microbiology, 2019 Sep 27, 26:3\\
(Nederlands Tijdschrift voor Medische Microbiologie; original work in Dutch)

Berends MS \textsuperscript{1,2}*, Luz CF \textsuperscript{2}*, Wouthuyzen-Bakker M \textsuperscript{2}, Märtson AG \textsuperscript{3}, Alffenaar JW \textsuperscript{3}, Dik JWH \textsuperscript{2}, Glasner C \textsuperscript{2}, Sinha BNM \textsuperscript{2}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics \& Advice Foundation, Groningen, Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Control, Groningen, Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Clinical Pharmacy and Pharmacology, Groningen, Netherlands
\end{enumerate}

* These authors contributed equally

\hypertarget{abstract}{%
\section*{Abstract}\label{abstract}}
\addcontentsline{toc}{section}{Abstract}

The right test at the right time for the right patient to answer the right questions and start the right treatment - many important decisions have to be made involving multiple medical specialists. The importance of appropriate and timely diagnostics guide this process (stewardship) can be obvious but is still often neglected in classic stewardship concepts of infection management. We describe the approach of a multidisciplinary, intertwined stewardship concept with a focus on diagnostics, where medical specialists in general and microbiologists in particular closely interact for optimal quality of care and patient safety in successful infection management. Diagnostics in medical microbiology laboratories are advancing fast with regards to new technologies and improved workflows. Yet, diagnostics in infection management is broader than this and covers many clinical areas where communication and interaction are the key to make the best use of knowledge and expertise that all specialisms can contribute to patient care. These aspects are demonstrated in two cases of patients with prosthetic joint infections with two very different outcomes.

\hypertarget{introduction}{%
\section{Introduction}\label{introduction}}

Diagnostic stewardship or diagnostic stewardship programme (DSP), a trending topic in the field of medical microbiology and beyond. But what is this concept about, is it really so new and how is it incorporated into infection management? The term diagnostic stewardship was used in an opinion piece by Dik \emph{et al.} which described various facets of infection management, the so-called integrated stewardship \textsuperscript{{[}1{]}}. We want to highlight the diagnostic side of this model and describe its concept; diagnostics as a multidisciplinary bigger picture from admission to discharge.

Although the term DSP was first mentioned in an indexed PubMed article in 2016, articles on antimicrobial stewardship (ASP) have been appearing for 15 years (Figure \ref{fig:fig2-1}).

\begin{figure}

{\centering \includegraphics[width=0.8\linewidth]{images/02-01} 

}

\caption{The increase of articles indexed in PubMed. Search strategies: 'antimicrobial stewardship'[Title/Abstract]; 'diagnostic stewardship'[Title/Abstract]; 'antimicrobial resistance'[Title/Abstract]. Source: https://www.ncbi.nlm.nih.gov/pubmed/ (assessed: 2018-05-31). * Extrapolation based on count from 2018-01-01 to 2018-05-31.}\label{fig:fig2-1}
\end{figure}

Nevertheless, the concept of DSP is neither intended to replace other stewardship concepts (in particular ASP) nor to be an alternative. DSP concerns decision making and goes beyond microbiological diagnostics alone. Kahneman \emph{et al.} \textsuperscript{{[}2{]}} said about decision making:

\begin{quote}
We think, each of us, that we're much more rational than we are. And we think that we make our decisions because we have good reasons to make them. Even when it's the other way around. We believe in the reasons, because we've already made the decision. \textsuperscript{{[}2{]}}
\end{quote}

Adequate diagnostics should help us to prevent this kind of situation in medicine by providing a basis to make well-informed decisions. Defining a proper diagnosis is a complex process with several aspects. We believe that DSP is a concept that requires collaboration between different medical specialties for optimal infection management and quality of care. This can include reduced morbidity and/or mortality, unnecessary interventions or treatments, complications, and length of stay. We want to point out why and how DSP affects the entire diagnostic process and that it involves more than just results or turnaround times of microbiological tests. By comparing different patient cases, we want to demonstrate how DSP serves the most important purpose: improved patient care. This involves process optimisation as a basis as well as medical questions and decisions on the individual patient level.

This entire diagnostic process requires multiple decisions along the way of patient care. Guidance and communication on this path are essential because:

\begin{quote}
Intuitive diagnosis is reliable when people have a lot of relevant feedback. But people are very often willing to make intuitive diagnoses even when they're very likely to be wrong. \textsuperscript{{[}3{]}}
\end{quote}

Modern medicine is centred around evidence-based actions and tries to minimise the chance of mistakes while trying to keep the balance between the quality of care and the outcome on one hand and preventing collateral damage and costs on the other hand. In infection management stewardship activities can provide support and guidance in diagnosis and therapy. Physicians can be supported at the bedside to choose the right diagnostic test at the right time for the right patient. The same applies to therapeutic choices: the right treatment at the right time for the right patient in order to achieve the most optimal result. Naturally, these approaches to diagnostic and therapeutic support go hand in hand.

We outline two different case studies - fictitious but nevertheless realistic - of a patient with a prosthetic joint infection (PJI) in different scenarios and different outcomes. These examples underline how interdisciplinary stewardship can lead to a successful outcome for the patient and the physician.

\hypertarget{case-1}{%
\subsection{Case 1}\label{case-1}}

A 70-year-old woman was seen by the orthopaedic surgeon because of chronic pain in her hip prosthesis placed 3 years earlier. An X-ray showed signs of loosening of the prosthesis - an indication for revision surgery. C-reactive protein (CRP) was low (6 mg/L). The diagnosis of aseptic loosening was made, and the patient underwent revision surgery. To rule out low-grade infection, antibiotic prophylaxis was administered only after intraoperative tissue biopsies had been taken for culturing and histology. Cutibacterium acnes (formerly Propionibacterium acnes) was isolated from one out of five tissue biopsies (semi-quantitative \textless1+). Histology showed no indication of inflammation. The positive culture was considered contamination by the attending clinical microbiologist and the patient was discharged without further antibiotic therapy. However, during outpatient follow-up, the patient complained about persistent stiffness of her hip. Three years later, the patient presented again with recurrent loosening of the prosthesis and the presence of a fistula around the surgical site. A second revision intervention was necessary. Due to poor bone quality and poor soft tissue, multiple revisions were needed. Multiple intraoperative tissue biopsies revealed Cutibacterium acnes with the same antibiogram as three years earlier together with a methicillin-sensitive Staphylococcus hominis. The patient was given a cement spacer which made her temporarily immobile and was treated with a high dose of flucloxacillin intravenously. She was discharged with clindamycin per os and re-admitted several months later for reimplantation of the definitive prosthesis. After eight months of revalidation the functional result was poor. The patient permanently walks with support of a cane.

Figure \ref{fig:fig2-2} shows the course of the disease of this patient in which the decision moments are shown in circles. The potential stewardship zone shows the moments when a different action could/should have been taken.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/02-02} 

}

\caption{The first case.}\label{fig:fig2-2}
\end{figure}

The outcome for this patient was certainly not optimal. To illustrate how infection management with stewardship elements can improve the quality of care, a second case of the same patient with a PJI follows. Several additional diagnostic steps were performed (shown in bold) underlining the need for collaboration in stewardship activities including antimicrobial stewardship, of course, and how this affects clinical outcome and hospitalisation.

\hypertarget{case-2}{%
\subsection{Case 2}\label{case-2}}

A 70-year-old woman was seen by the orthopaedic surgeon because of chronic pain in her hip prosthesis placed 3 years earlier. An X-ray showed signs of loosening of the prosthesis - an indication for revision surgery. C-reactive protein (CRP) was low (6 mg/L). The radiologist was consulted to reassess the X-ray taken a year earlier. This image already showed subtle signs of radiolucency around the head and neck of the prosthesis making a mechanical cause of detachment less likely. Synovial fluid was punctured to rule out septic loosening of the prosthesis. The synovial fluid culture remained negative and the leukocyte count was only slightly increased, but several biomarkers were positive suggesting infection (450 mg/L calprotectin and positive alpha-defensin). Subsequently, prior to revision surgery, several tissue biopsies were taken by the orthopaedic surgeon in a sterile environment. Cutibacterium acnes (formerly Propionibacterium acnes) was isolated from one out of five tissue biopsies (5-10 CFU/ml). Histology showed no indication of inflammation. During revision surgery, antibiotic prophylaxis was given prior to surgical incision and several tissue samples were taken for culturing (including sonication) of the prosthesis. Empirical treatment was initiated with high doses of amoxicillin. Due to the previous positive culture with Cutibacterium acnes, all intraoperative cultures were incubated for 14 days on the advice of the clinical microbiologist. C. acnes was found again in two of five tissue biopsies and also in the sonication fluid. These isolates showed the same antibiogram as the isolates from before revision surgery. The patient was then discharged and treated at home with 10 weeks of amoxicillin per os. She fully recovered within a few weeks.

Figure \ref{fig:fig2-3} shows the additional decisions compared to Figure \ref{fig:fig2-2}. These lead to a better outcome for the patient through the implementation of stewardships. The differences with Figure \ref{fig:fig2-2} are shown in red.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/02-03} 

}

\caption{The second case.}\label{fig:fig2-3}
\end{figure}

\hypertarget{the-general-concept}{%
\section{The general concept}\label{the-general-concept}}

\hypertarget{diagnostics}{%
\subsection{`Diagnostics'}\label{diagnostics}}

The term diagnostics seems simple, but its various aspects are very diverse, as the cases above demonstrate. The second case emphasises the importance of stewardships and centres around facilitating an optimal care process through communication, crossing the boundaries of specialisms, and increasing awareness of the integral nature of successful infection management and optimal quality of care. Different physicians (involved in infection management) and their perceptions are reflected in this view on diagnostics. While some think of the entire process of diagnosing a disease, others think purely of the technical aspect in the lab as diagnostics (of their own speciality). This diversity underlines the importance of communication and collaboration across the boundaries of different medical specialties. The concept of stewardship is widely used to facilitate communication (and clinical decision making). Multiple attempts have been made to establish a clear definition of stewardship, but this has proved challenging \textsuperscript{{[}3,4{]}}. Overall, most of these attempts have been made in the light of antimicrobial stewardship programmes (ASP) and are accompanied by terms such as responsibility, balance, due diligence, and management \textsuperscript{{[}3,4{]}}.

\hypertarget{dsp-in-the-microbiological-laboratory}{%
\subsection{DSP in the microbiological laboratory}\label{dsp-in-the-microbiological-laboratory}}

A medical laboratory usually only has added value if, in addition to the reporting and advice, the range of tests and the test technique meet the requirements of the applicant. The technical aspect of the medical microbiology laboratories has seen tremendous technological advances in recent years. Advanced developments such as sequencing as part the routine to identify isolate properties (e.g., resistance genes) and Matrix-Assisted Laser Desorption/Ionization Time of Flight (MALDI-TOF) mass spectrometry methods have recently revolutionized the laboratories \textsuperscript{{[}5-7{]}}. In addition, many new and fast diagnostic assays such as point-of-care test (POCT) and molecular rapid diagnostic test (mRDT) have entered the market \textsuperscript{{[}8{]}}. The progress is undeniable although integration into workflow, quality control, data storage and availability, added value, and clinical impact often still need to be evaluated.

We embrace these developments but there are two aspects that are really essential for optimal quality of care. Both these aspects can be achieved through stewardship. Firstly, stewardship provides guidance for the appropriate choice of a customised diagnostic strategy for individual patients and patient groups in a specific setting. Guidelines and protocols for diagnostic and appropriate therapeutic choices are key elements in the development of this guidance or steering. A stewardship framework can form the basis for personalised decisions in individual patient care. It has already been demonstrated that new tests such as the aforementioned mRDT are most cost-effective for the diagnosis of bacteraemia when combined with an antimicrobial stewardship programme \textsuperscript{{[}9{]}}. In addition, mRDT is associated with a significant reduction in mortality risk for septic patients but only when combined with ASP \textsuperscript{{[}10{]}}.

Secondly, it is important to consider the entire information loop in a process-oriented way and not just focus on the time-to-result. Stewardship covers this loop and starts making choices at the bedside. In addition, the interpretation of test results and timely feedback are equally important in order to be able to make good, evidence-based, and rapid therapy adjustments when needed. For example, physicians considering starting non-prophylactic intravenous antimicrobial treatment should (almost) always take blood cultures before starting. Although this is standard care and described in international guidelines \textsuperscript{{[}11{]}}, compliance is only 30 to 50\% {[}12, 13, Luz \emph{et al.}; unpublished data{]}. Only through complete `loops', from bedside to bedside, can better technology and improved work processes in microbiology laboratories be extended and made to work to their full potential.

\hypertarget{dsp-as-process-optimisation}{%
\subsection{DSP as process optimisation}\label{dsp-as-process-optimisation}}

Turnaround times (TAT) are a commonly used but poorly defined term in many areas. In a systematic review, a total of 61 different TAT definitions (out of a total of 151) were found to be used in several clinical areas \textsuperscript{{[}14{]}}. Of those, only 10 definitions cover the time from test order placement to the time at which the results are being viewed by the ordering physician (Figure \ref{fig:fig2-4}).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/02-04} 

}

\caption{Time points mentioned in TAT definitions.}\label{fig:fig2-4}
\end{figure}

Nevertheless, even the order of a test is a decision within a diagnostic loop and should be taken into account when time is measured. We are convinced that infection management can help to understand the importance of a full loop from moment of choice to moment of choice, from the bedside to a diagnostic result and back. This implies the time from the moment when the need for diagnostics becomes clear, to the time when it can be acted upon based on its results. We call this time to action which is indicated by a red arrow in Figure \ref{fig:fig2-4}.

\hypertarget{multidisciplinary-aspects-of-dsp-and-infection-management}{%
\subsection{Multidisciplinary aspects of DSP and infection management}\label{multidisciplinary-aspects-of-dsp-and-infection-management}}

It is essential to realise that the information needed to assess this time to action does not come only from microbiological laboratories. Communication and collaboration in the stewardship zone (Figures 2 and 3) are key and this applies to all specialities. But what would be the effect on the patient if microbiological diagnostics were not led by DSP when there is already good communication and cooperation in place? Would DSP no longer be necessary? Or is good cooperation equivalent to DSP?

DSP can significantly reduce the time to action by making proper use of each other's expertise to make optimal decisions for the patient. In practice, information from one diagnostic discipline can help to steer the diagnostic process of another diagnostic discipline. One reason for this is that during the diagnostic process of many disciplines, such as medical microbiology and imaging, an intrinsic amount of interpretation takes place. The clinical course is no less important here. We always need DSP, because together we try to act as optimally as possible in the interest of the patient, in which diagnosis is an important tool. DSP is not specific to medical microbiology, as demonstrated by the relevance of its collaboration with radiology in case 2. Nor is it specific to any other speciality. DSP is not intended as a reactive ad hoc solution but rather as a proactive, structural approach. DSP should be seen as guiding the entire diagnostic process, not only on the basis of antibiotics, but also on the basis of extensive imaging (such as for endocarditis), biomarkers (such as leukocytes and CRP, or procalcitonin for de-escalation of treatment), or by therapeutic drug monitoring (TDM) modelling the optimal dosage from the start of (empirical) treatment for individual patients and patient groups. One form of diagnostics is relevant to monitor trends, the other to directly answer a clinical question. This does not mean that one is less important than the other or that we should look at the value of an antibiogram differently from the value of a therapeutic drug monitoring. A pharmacist is also part of DSP.

As an example, in Dutch hospitals we are used to having a hospital pharmacist in house, providing clinical pharmaceutical services. Consultations are typically performed via e-mail, telephone, or an electronic prescription system. On the other hand, in countries such as the United Kingdom, these pharmacists work in infection management in the clinical (nursing) departments on a daily basis in collaboration with other specialists. This supports the most safe, appropriate, and cost-effective antimicrobial treatment \textsuperscript{{[}15{]}}. In addition, as mentioned earlier, the guidance of antimicrobial therapy by TDM is another important aspect. Hospital pharmacists can make suggestions on sample timing for TDM, inform about early prediction of attainable levels and dose adjustments to achieve adequate exposure and reduce toxicity as quickly as possible, and interpret results \textsuperscript{{[}16{]}}. As a result, they are an integral part of the stewardship concept. We are convinced that the different stewardship terms and concepts form synergy for the best infection management \textsuperscript{{[}1,17{]}}. Infection management has different aspects (such as ASP) and stewardship refers to guidance provided by focused experts \textsuperscript{{[}18{]}}.

Empirical antimicrobial therapy is a good example to illustrate how these aspects are linked. The working diagnosis (see also cases 1 and 2), based on an appropriate differential diagnosis, forms the basis for an appropriate empirical therapy that takes into account the most relevant pathogens, their anticipated susceptibility, the source of infection (taking into account the compartment), and underlying patient factors. Adequate initial diagnostic initiatives (such as deep focus puncture, see case 2) may simultaneously be therapeutic (such as surgical/interventional drainage for source control). Vice versa, the clinical course under therapy can be diagnostic in itself, for example, if diagnostics for the working diagnosis are correct and complete. Ultimately, the treatment of patients with complex infections almost always requires targeted treatment. This, in turn, requires adequate initial and ongoing diagnostics for optimal treatment. Figure \ref{fig:fig2-5} shows the decision moments and different specialisms that can be involved in this whole process.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/02-05} 

}

\caption{Stewardship in infection management.}\label{fig:fig2-5}
\end{figure}

\hypertarget{conclusion}{%
\section{Conclusion}\label{conclusion}}

The answer to the question from the title (Diagnostic stewardship - sentence or nonsense?!) is: both. It is nonsense to debate terminology and the discussion about differences between diagnostic stewardship and infection management is only of semantic nature. Diagnostic stewardship makes sense in the concept discussed above. It can guide specialists (physician-microbiologist/medical-molecular microbiologists and experts from other fields, such as hospital pharmacists, radiologists, nuclear medicine, etc.) to the area of the stewardship zone of interaction and communication (Fig. 5), where they can bring in their expertise to complex clinical decision-making. Clinical information, including a patient's clinical development, is extremely important for correctly interpreting diagnostic results and steering the process. It can also help leading clinicians and other clinicians to understand the full potential (and limitations) of diagnostics and how important they are for evidence-based decision-making. We follow an integrated stewardship model that adds different perspectives (antimicrobial, infection prevention, and diagnostic stewardship - AID) to the ultimate goal of all stewardship intentions - the best quality care for the individual patient \textsuperscript{{[}1{]}}.

Stewardship consists largely of translation and communication during the decision-making process. Diagnostics are essential in this. But there is no need for a new name. Diagnostic stewardship as a name may be without added value and more and more use of stewardship-like terms could lead to confusion. The aim of all efforts and experts in infection management is the same: to improve quality of care and patient outcomes. We see with our own eyes how DSP guidelines are adhered to and realise how important it is that we continue to emphasise the often-underexposed diagnostic aspects of infection management. Multidisciplinary management based on diagnostics builds the basis for optimal outcomes for patients with infections.

\hypertarget{financing}{%
\section*{Financing}\label{financing}}
\addcontentsline{toc}{section}{Financing}

This study was partly supported by the INTERREG V A (202085) funded project EurHealth-1Health (\url{http://www.eurhealth1health.eu}), part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport, the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the Ministry for National and European Affairs and Regional Development of Lower Saxony.

In addition, this study was part of a project funded by the European Union's Horizon 2020 research and innovation programme under the Marie Sklodowska-Curie grant agreement 713660 (MSCA-COFUND-2015-DP ``Pronkjewail'').

\hypertarget{references-1}{%
\section*{References}\label{references-1}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Dik J-WH, Poelman R, Friedrich AW, Panday PN, Lo-Ten-Foe JR, van Assen S, \emph{et al.} An integrated stewardship model: antimicrobial, infection prevention and diagnostic (AID). Future Microbiol. 2016;11(1):93--102.
\item
  Kahneman D. Thinking, fast and slow. Macmillan; 2011.
\item
  Dyar OJ, Huttner B, Schouten J, Pulcini C, ESGAP (ESCMID Study Group for Antimicrobial stewardshiP). What is antimicrobial stewardship? Clin Microbiol Infect. 2017 Nov;23(11):793--8.
\item
  Mendelson M, Balasegaram M, Jinks T, Pulcini C, Sharland M. Antibiotic resistance has a language problem. Nature. 2017 May 3;545(7652):23--5.
\item
  Greub G, Moran-Gilad J, Rossen J, Egli A, ESCMID Study Group for Genomic and Molecular Diagnostics (ESGMD). ESCMID postgraduate education course: applications of MALDI-TOF mass spectrometry in clinical microbiology. Microbes Infect. 2017 Sep;19(9-10):433--42.
\item
  Didelot X, Bowden R, Wilson DJ, Peto TEA, Crook DW. Transforming clinical microbiology with bacterial genome sequencing. Nat Rev Genet. 2012 Sep;13(9):601--12.
\item
  Greninger AL. The challenge of diagnostic metagenomics. Expert Rev Mol Diagn. 2018 Jun 18;1--11.
\item
  Kozel TR, Burnham-Marusich AR. Point-of-Care Testing for Infectious Diseases: Past, Present, and Future. J Clin Microbiol. 2017 Aug;55(8):2313--20.
\item
  Pliakos EE, Andreatos N, Shehadeh F, Ziakas PD, Mylonakis E. The Cost-Effectiveness of Rapid Diagnostic Testing for the Diagnosis of Bloodstream Infections with or without Antimicrobial Stewardship. Clin Microbiol Rev.~2018 Jul;31(3).
\item
  Timbrook TT, Morton JB, McConeghy KW, Caffrey AR, Mylonakis E, LaPlante KL. The Effect of Molecular Rapid Diagnostic Testing on Clinical Outcomes in Bloodstream Infections: A Systematic Review and Meta-analysis. Clin Infect Dis. 2017 Jan 1;64(1):15--23.
\item
  Rhodes A, Evans LE, Alhazzani W, Levy MM, Antonelli M, Ferrer R, \emph{et al.} Surviving Sepsis Campaign: International Guidelines for Management of Sepsis and Septic Shock: 2016. Crit Care Med. 2017 Mar;45(3):486--552.
\item
  Reissig A, Mempel C, Schumacher U, Copetti R, Gross F, Aliberti S. Microbiological diagnosis and antibiotic therapy in patients with community-acquired pneumonia and acute COPD exacerbation in daily clinical practice: comparison to current guidelines. Lung. 2013 Jun;191(3):239--46.
\item
  Shallcross LJ, Freemantle N, Nisar S, Ray D. A cross-sectional study of blood cultures and antibiotic use in patients admitted from the Emergency Department: missed opportunities for antimicrobial stewardship. BMC Infect Dis. 2016 Apr 18;16:166.
\item
  Breil B, Fritz F, Thiemann V, Dugas M. Mapping turnaround times (TAT) to a generic timeline: a systematic review of TAT definitions in clinical domains. BMC Med Inform Decis Mak. 2011 May 24;11:34.
\item
  Wickens HJ, Jacklin A. Impact of the Hospital Pharmacy Initiative for promoting prudent use of antibiotics in hospitals in England. J Antimicrob Chemother. 2006 Dec;58(6):1230--7.
\item
  van Wanrooy MJP, Rodgers MGG, Span LFR, Zijlstra JG, Uges DRA, Kosterink JGW, \emph{et al.} Voriconazole Therapeutic Drug Monitoring Practices in Intensive Care. Ther Drug Monit. 2016 Jun;38(3):313--8.
\item
  Pulcini C, Binda F, Lamkang AS, Trett A, Charani E, Goff DA, \emph{et al.} Developing core elements and checklist items for global hospital antimicrobial stewardship programmes: a consensus approach. Clin Microbiol Infect \textsuperscript{{[}Internet{]}}. 2018 Apr 3; Available from: \url{http://dx.doi.org/10.1016/j.cmi.2018.03.033}
\item
  British Society for Antimicrobial Chemotherapy. Antimicrobial Stewardship: From Principal to Practice \textsuperscript{{[}Internet{]}}. Birmingham, United Kingdom: British Society for Antimicrobial Chemotherapy; 2018. Available from: \url{http://bsac.org.uk/antimicrobial-stewardship-from-principles-to-practice-e-book/}
\end{enumerate}

\hypertarget{ch03-introducing-new-method}{%
\chapter{Introducing a New, Free, and Independent Method for Standardised, Reproducible and Reliable Analyses of Antimicrobial Resistance Data}\label{ch03-introducing-new-method}}

In preparation

(as of date of PhD defence: 25 August 2021)

Berends MS \textsuperscript{1,2}, Luz CF \textsuperscript{2}, Sinha BNM \textsuperscript{2}, Glasner C \textsuperscript{2‡}, Friedrich AW \textsuperscript{2‡}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics \& Advice Foundation, Groningen, the Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology \& Infection Control, Groningen, the Netherlands
\end{enumerate}

\textsuperscript{‡} These authors contributed equally

\hypertarget{abstract-1}{%
\section*{Abstract}\label{abstract-1}}
\addcontentsline{toc}{section}{Abstract}

As the burden of antimicrobial resistance (AMR) is continuously increasing, reliable and reproducible data and data analysis are of utmost importance. Conducting AMR data analysis is challenging since it requires (1) a thorough understanding of (clinical) epidemiology; (2) expertise in (clinical) microbiology and infectious diseases; (3) experience in microbiological data analysis; (4) availability of reference data, such as the biological taxonomy of microorganisms and defined daily doses (DDD) for antimicrobials; and (5) availability of (inter-)national guidelines and software methods to apply them. Furthermore, data stored in laboratory information systems lack the right structure, (inter-) national guidelines for interpreting raw laboratory test results cannot be easily applied, and scientifically reliable reference data about microorganisms and antimicrobial agents are not readily available. To fill this gap, we developed a free, independent, and open-source software solution to cover all those aspects of working with AMR data. The AMR package for R enables AMR data analysis for research and clinical workflows alike. Through an online survey package users reported more reproducibility of analysis results (83\%), more reliable outcomes of AMR analyses (72\%), and new or improved insight into AMR patterns (61\%). The AMR package was also used to support clinical decision-making (44\%) and for clinical research (28\%). Our first insights into the usage and the usability of the AMR package confirm that this package is fulfilling its intended aim, as regional, national, and international organisations already use the package to support clinical decision-making in infection management. The flexible open-source design also enables rapid integration of updated guidelines (e.g., new EUCAST breakpoints) and setting-specific adaptations are encouraged. Together, the AMR package for R can thus empower any specialist in the field working with AMR data by providing a comprehensive toolbox of solutions for AMR data analyses.

\hypertarget{background}{%
\section{Background}\label{background}}

As the burden of antimicrobial resistance (AMR) is continuously increasing, surveillance programs with reliable and reproducible data and data analysis methods are of utmost importance for controlling and streamlining efforts to curb AMR \textsuperscript{{[}1,2{]}}. To guide these efforts and to support clinical decision-making and infection-control interventions, AMR data analysis has to be conducted in a clinically and epidemiologically sensible way \textsuperscript{{[}3{]}}. Conducting AMR data analysis is challenging since it requires (1) a thorough understanding of (clinical) epidemiology; (2) expertise in (clinical) microbiology and infectious diseases; (3) experience in microbiological data analysis; (4) availability of reference data, such as the biological taxonomy of microorganisms and defined daily doses (DDD) for antimicrobials; and (5) availability of (inter-)national guidelines and software methods to apply them.

Moreover, AMR data analysis is often also hindered by three key aspects. Firstly, data stored in microbiological laboratory information systems (LIS) are typically not readily suitable for (epidemiological) data analyses. LIS were initially designed to fit result registration and billing purposes rather than AMR data analysis. Consequently, fundamental requirements for (epidemiological) data analyses are often lacking, such as isolate selection criteria, phenotypic determination of (multi-)drug resistance, and the ability to extract data for analysis in an automated, structured, fast, and reliable way. Moreover, data analyses that require data from multiple LIS sources (e.g., in multi-centre studies) face major barriers in data aggregation which, to the best of our knowledge, cannot be solved by currently available commercial software solutions. Besides, as applications of artificial intelligence are expected of being increasingly developed in the coming years, also in clinical microbiology, microbiological data technologies and structures need to become compatible for these future applications.

Secondly, AMR data analysis depends on (inter-)national standards and guidelines for the interpretation of raw laboratory measurements and the reporting of AMR results. In Europe, guidelines from the European Committee on Antimicrobial Susceptibility Testing (EUCAST) are the predominantly implemented set of rules in clinical microbiological laboratories \textsuperscript{{[}4,5{]}}. LIS need to be well-maintained to be able to integrate continuous guideline updates. In our experience, this maintenance can often not be guaranteed and depends on the availability of local or external software support services. This is further hindered by the current distribution of manually formatted guidelines in Microsoft Excel and Portable Document Format (PDF) formats that are not often readily machine-readable. LIS maintainers, in collaboration with clinical staff, are therefore forced to manually implement updated guidelines which can be time-consuming and error-prone

Thirdly, reliable AMR data analysis depends on taxonomic reference data to interpret raw LIS data using AMR interpretation guidelines, such as EUCAST Expert Rules and EUCAST Clinical Breakpoints \textsuperscript{{[}5,6{]}}. Unfortunately, typical LIS contain local, static taxonomic data. We found that these data are often poorly maintained. We collected the taxonomic names of bacteria used in clinical reports from seven different public health institutions in the Netherlands which cover microbiological diagnostics in hospitals and primary care for 15\% of the total Dutch population. The taxonomic names were compared to publicly available and authoritative reference databases; the Catalogue of Life and the List of Prokaryotic names with Standing in Nomenclature (LPSN, previously known as the Deutsche Sammlung von Mikroorganismen und Zellkulturen, DSMZ) \textsuperscript{{[}7,8{]}}. We found that all participating institutions reported taxonomic names in clinical reports that did not match current taxonomic standards according to reference databases. For example, \emph{Enterobacter aerogenes} and \emph{Enterobacter massiliensis} were renamed \emph{Klebsiella aerogenes} and \emph{Metakosakonia massiliensis} respectively in 2017 \textsuperscript{{[}9,10{]}}. LIS that are not kept up to date are consequently not entirely compatible with recent interpretation guidelines. Given that AMR guidelines are strongly based on the microbial taxonomy (some rules only apply to a specific genus, other rules apply to a specific family) it is crucial that this information is correct and kept up to date. In the studied institutions, the lag between the reported taxonomic names and the taxonomic standard was up to 41 years as of March 2021.

\hypertarget{standardising-amr-data-analysis}{%
\section{Standardising AMR data analysis}\label{standardising-amr-data-analysis}}

Previously, no dedicated software solution was available to address all aforementioned aspects. To fill this gap, we developed a free, independent, and open-source software solution to cover all those aspects of working with AMR data. The AMR package for R \textsuperscript{{[}11{]}} provides functionalities that enable standardised and reproducible workflows from any raw LIS data to results ready to publish, for research and clinical workflows alike. The AMR package for R was developed with a team of contributors from 12 public health organisations in seven countries aiming to be used in any research or clinical setting where (epidemiological) data analysis of microorganisms, AMR, or antimicrobial agents is required. It is independent of any other software solution and was designed to work in any setting, including those with limited computational and financial resources.

With this AMR package, we aimed at providing: (1) tools to simplify AMR data cleaning, transformation, and analysis; (2) methods to easily incorporate (inter)national guidelines; and (3) scientifically reliable reference data, including the aforementioned aspects. The AMR package enables standardised and reproducible AMR data analysis with the application of evidence-based rules (e.g., EUCAST expert rules for intrinsic resistance), the selection of first isolates, the translation of various codes for microorganisms and antimicrobial agents, determination of (multi-)drug-resistant microorganisms, and the calculation of antimicrobial resistance rates, prevalence, and future trends. The AMR package supports all EUCAST MIC/disk diffusion interpretation guidelines from 2011 until 2021 and EUCAST Expert rules versions 3.1 (2016) and 3.2 (2020) \textsuperscript{{[}12,13{]}} In addition, the AMR package supports all CLSI MIC/disk diffusion interpretation guidelines from 2011 until 2019 (non-veterinary only). For all mentioned guidelines, files readable for LIS are provided for easy implementation.

As of 30 April 2021, the AMR package for R has been downloaded from 162 countries since its first release in early 2018 (Figure \ref{fig:fig3-1}), according to data from a popular public repository where users can download R packages. After 19 releases, the median number of downloads per release is 2,548 (range: 269-5,050).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/03-01} 

}

\caption{Countries (grey, n = 162) with registered downloads of the AMR package for R between March 2018 and April 2021. Sources: cran.rstudio.org and cloud.r-project.org.}\label{fig:fig3-1}
\end{figure}

A technical validation of the AMR package has been accepted for publication \textsuperscript{{[}11{]}}. Additionally, it has been clinically and epidemiologically validated in a tertiary care hospital and across seven clinical microbiology laboratories in the Netherlands {[}Berends \emph{et al.}, unpublished, see chapter 6 and 7 of this thesis{]}. Moreover, the AMR package has already been used in several scientific publications that focused on different aspects in the field of AMR \textsuperscript{{[}14--17{]}}.

\hypertarget{comparison-with-existing-software-methods}{%
\section{Comparison with existing software methods}\label{comparison-with-existing-software-methods}}

Popular statistical software such as SPSS, Stata and SAS, focus on a broad implementation of statistical functions but are proprietary software, disallowing users to freely use, modify, or share the software. This also prohibits extending the software by unaffiliated developers. Since R is free, open software and extendible, users and developers can contribute to the software, to which end the AMR package is a practical example.

Other free software alternatives for AMR data analysis exist, for example WHONET, a free microbiology laboratory database software supported by the WHO \textsuperscript{{[}18{]}}. WHONET allows manual data entry from LIS reports and provides AMR interpretation using recent CLSI and EUCAST guidelines with a particular focus on AMR surveillance. Results from WHONET can also be shared to surveillance programs such as the European Antimicrobial Resistance Surveillance Network (EARS-Net) and the WHO Global Antimicrobial Resistance Surveillance System (GLASS). Yet, the latest release, WHONET 2020, does not provide tools for cleaning and transforming data and relies on outdated EUCAST guidelines. Furthermore, we found a lag between the included taxonomic database and the current taxonomic standard of up to 59 years (median 7 years). Another alternative of a free software program is Epi Info which is provided by the United States Centers for Disease Control and Prevention (CDC) and aims at public health practitioners and researchers \textsuperscript{{[}19{]}}. While Epi Info provides statistical and epidemiological methods for analysing data, it does not offer tools nor reference data for working with AMR test results or antimicrobial drugs, thus, ruling out the option for dedicated AMR data analysis. With the AMR package for R, an open and dedicated software solution is available that covers all aspects of working with AMR data.

\hypertarget{user-feedback}{%
\section{User feedback}\label{user-feedback}}

In July 2020, we published a survey on the website created for this package (\url{https://msberends.github.io/AMR}) to seek voluntary feedback from package users about user backgrounds and usage of the AMR package. Until December 2020, 18 participants completed the survey. Participants have used the AMR package in Australia, Colombia, Egypt, France, Germany, Haiti, India, Mali, Mexico, the Netherlands, Nigeria, Philippines, Spain, Sweden, and the United Kingdom.

Participants were asked to rate their experience in the statistical programming language R and in using the AMR package on a scale from 1 (not experienced/useful) to 10 (very experienced/useful). The overall experience in R was reported with a median of 7 (range: 4-9)., whereas Ssuit ability for AMR analyses using the AMR package was rated with a median of 9 (range: 6-9). The participants rated the usefulness of the AMR package for their work with a median of 9 (range: 5-9). The convenience of the included software functions was rated with a median of 8 (range: 6-9) and the documentation of the AMR package was rated with a median of 8.5 (range: 7-10). Of all participants, 83\% reported more reproducibility of analysis results and, 72\% reported more reliable outcomes of AMR analyses (Figure \ref{fig:fig3-2}). Notably, 61\% reported new or improved insight into AMR for their institution or region. The AMR package was also used to support clinical decision-making (44\%) and for clinical research (28\%). Furthermore, 66\% reported a faster and streamlined analysis workflow and 39\% reported improved communicating analysis results. In 33\%, participants started using R more often because of the capabilities that the AMR package provides.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/03-02} 

}

\caption{The outcome of the survey amongst 18 participants. MIC: minimal inhibitory concentration, MDRO: multidrug-resistant organism, SNOMED: Systematised Nomenclature of Medicine.}\label{fig:fig3-2}
\end{figure}

Aside from AMR data analysis, most participants (78\%) used the AMR package as a reference for the taxonomy of microorganisms. It was also regularly used for interpreting raw MIC and disk diffusion values (56\%) and applying EUCAST expert rules (67\%). This is in line with the original aims of the AMR package development.

\hypertarget{conclusion-1}{%
\section{Conclusion}\label{conclusion-1}}

AMR data analysis is dependent on (inter-)national guidelines and reliable (reference) data on the one hand but constrained by diverse and often inadequate data analysis tools and poor data quality on the other. We aimed to address these dependencies and constraints by introducing the AMR package for R for standardised and reproducible AMR data analyses. Our first insights into the usage and the usability of the AMR package confirm that this package is fulfilling its intended aim. Regional, national, and international organisations already use the AMR package to support clinical decision-making in infection management by gaining new or improved insights into resistance levels. We invite others to make use of our open-source approach and adapt it to their needs. The advantages of sharing open-source software such as the AMR package allow for a collaborative, transparent use and further development that can lead to more standardised analysis processes for AMR data. The flexible open-source design also enables rapid integration of updated guidelines (e.g., new EUCAST breakpoints), and setting-specific adaptations are encouraged. Together, the AMR package for R can thus empower any specialist in the field working with AMR data by providing a comprehensive toolbox of solutions for AMR data analysis.

\hypertarget{references-2}{%
\section*{References}\label{references-2}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Limmathurotsakul D, Dunachie S, Fukuda K, Feasey NA, Okeke IN, Holmes AH, \emph{et al.} Improving the estimation of the global burden of antimicrobial resistant infections. Lancet Infect Dis 2019;3099:1--7. \url{doi:10.1016/S1473-3099(19)30276-2}.
\item
  OECD. Stemming the Superbug Tide. Paris: OECD; 2018. \url{doi:10.1787/9789264307599-en}.
\item
  Hindler JF, Stelling J. Analysis and Presentation of Cumulative Antibiograms: A New Consensus Guideline from the Clinical and Laboratory Standards Institute. Clin Infect Dis 2007;44:867--73. \url{doi:10.1086/511864}.
\item
  Brown D, Canton R, Dubreuil L, Gatermann S, Giske C, MacGowan A, \emph{et al.} Widespread implementation of EUCAST breakpoints for antibacterial susceptibility testing in Europe. Euro Surveill 2015;20. \url{doi:10.2807/1560-7917.es2015.20.2.21008}.
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Breakpoint tables for interpretation of MICs and zone diameters. Version 10.0. 2020.
\item
  Kassim A, Omuse G, Premji Z, Revathi G. Comparison of Clinical Laboratory Standards Institute and European Committee on Antimicrobial Susceptibility Testing guidelines for the interpretation of antibiotic susceptibility at a University teaching hospital in Nairobi, Kenya: a cross-sectional stud. Ann Clin Microbiol Antimicrob 2016;15:21. \url{doi:10.1186/s12941-016-0135-3}.
\item
  Kassim A, Pflüger V, Premji Z, Daubenberger C, Revathi G. Comparison of biomarker based Matrix Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS) and conventional methods in the identification of clinically relevant bacteria and yeast. BMC Microbiol 2017;17:128. \url{doi:10.1186/s12866-017-1037-z}.
\item
  Parte AC, Sardà Carbasse J, Meier-Kolthoff JP, Reimer LC, Göker M. List of Prokaryotic names with Standing in Nomenclature (LPSN) moves to the DSMZ. Int J Syst Evol Microbiol 2020;70:5607--12. \url{doi:10.1099/ijsem.0.004332}.
\item
  Tindall BJ, Sutton G, Garrity GM. \emph{Enterobacter aerogenes} Hormaeche and Edwards 1960 (Approved Lists 1980) and \emph{Klebsiella mobilis} Bascomb \emph{et al.} 1971 (Approved Lists 1980) share the same nomenclatural type (ATCC 13048) on the Approved Lists and are homotypic synonyms, with consequences for. Int J Syst Evol Microbiol 2017;67:502--4. \url{doi:10.1099/ijsem.0.001572}.
\item
  Alnajar S, Gupta RS. Phylogenomics and comparative genomic studies delineate six main clades within the family \emph{Enterobacteriaceae} and support the reclassification of several polyphyletic members of the family. Infect Genet Evol 2017;54:108--27. \url{doi:10.1016/j.meegid.2017.06.024}.
\item
  Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR - An R Package for Working with Antimicrobial Resistance Data. J Stat Softw 2021;(in press). \url{doi:https://doi.org/10.1101/810622}.
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Intrinsic Resistance and Exceptional Phenotypes. Version 3.1. 2016.
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Intrinsic Resistance and Exceptional Phenotypes. Version 3.2. 2020.
\item
  Le Guern R, Titécat M, Loïez C, Duployez C, Wallet F, Dessein R. Comparison of time-to-positivity between two blood culture systems: a detailed analysis down to the genus-level. Eur J Clin Microbiol Infect Dis 2021. \url{doi:10.1007/s10096-021-04175-9}.
\item
  Dutey-Magni PF, Gill MJ, McNulty D, Sohal G, Hayward A, Shallcross L, \emph{et al.} Feasibility study of hospital antimicrobial stewardship analytics using electronic health records. JAC-Antimicrobial Resist 2021;3. \url{doi:10.1093/jacamr/dlab018}.
\item
  N. Tenea G, Jarrin-V P, Yepez L. Microbiota of Wild Fruits from the Amazon Region of Ecuador: Linking Diversity and Functional Potential of Lactic Acid Bacteria with Their Origin. Ecosyst. Biodivers. Amaz., IntechOpen; 2021. \url{doi:10.5772/intechopen.94179}.
\item
  Kim S, Yoo SJ, Chang J. Importance of Susceptibility Rate of `the First' Isolate: Evidence of Real-World Data. Medicina (B Aires) 2020;56:507. \url{doi:10.3390/medicina56100507}.
\item
  World Health Organization. WHONET 2020. \url{https://whonet.org} (accessed May 20, 2021).
\item
  Centers for Disease Control and Prevention (CDC). Epi Info (TM) 2020. \url{https://www.cdc.gov/epiinfo/index.html} (accessed May 20, 2021).
\end{enumerate}

\hypertarget{ch04-amr-r-package}{%
\chapter{\texorpdfstring{\texttt{AMR} - An \texttt{R} Package for Working with Antimicrobial Resistance Data}{AMR - An R Package for Working with Antimicrobial Resistance Data}}\label{ch04-amr-r-package}}

Accepted in Journal of Statistical Software (ahead of print)

(as of date of PhD defence: 25 August 2021)

Berends MS \textsuperscript{1,2}*, Luz CF \textsuperscript{2}*, Friedrich AW \textsuperscript{2}, Sinha BNM \textsuperscript{2}, Albers CJ \textsuperscript{3}, Glasner C \textsuperscript{2}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, the Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, the Netherlands
\item
  University of Groningen, Heymans Institute for Psychological Research, Groningen, the Netherlands
\end{enumerate}

* These authors contributed equally

\hypertarget{abstract-2}{%
\section*{Abstract}\label{abstract-2}}
\addcontentsline{toc}{section}{Abstract}

Antimicrobial resistance is an increasing threat to global health. Evidence for this trend is generated in microbiological laboratories through testing microorganisms for resistance against antimicrobial agents. International standards and guidelines are in place for this process as well as for reporting data on (inter-)national levels. However, there is a gap in the availability of standardised and reproducible tools for working with laboratory data to produce the required reports. It is known that extensive efforts in data cleaning and validation are required when working with data from laboratory information systems. Furthermore, the global spread and relevance of antimicrobial resistance demands to incorporate international reference data in the analysis process. In this paper, we introduce the \texttt{AMR} package for\texttt{R} that aims at closing this gap by providing tools to simplify antimicrobial resistance data cleaning and analysis, while incorporating international guidelines and scientifically reliable reference data. The \texttt{AMR} package enables standardised and reproducible antimicrobial resistance analyses, including the application of evidence-based rules, determination of first isolates, translation of various codes for microorganisms and antimicrobial agents, determination of (multi-drug) resistant microorganisms, and calculation of antimicrobial resistance, prevalence and future trends. The \texttt{AMR} package works independently of any laboratory information system and provides several functions to integrate into international workflows (e.g., WHONET software provided by the World Health Organization).

\hypertarget{introduction-1}{%
\section{Introduction}\label{introduction-1}}

Antimicrobial resistance is a global health problem and of great concern for human medicine, veterinary medicine, and the environment alike. It is associated with significant burdens to both patients and health care systems. Current estimates show the immense dimensions we are already facing, such as claiming at least 50,000 lives due to antimicrobial resistance each year across Europe and the US alone \textsuperscript{{[}1{]}}. Although estimates for the burden through antimicrobial resistance and their predictions are disputed \textsuperscript{{[}2{]}} the rising trend is undeniable \textsuperscript{{[}3{]}}, thus calling for worldwide efforts on tackling this problem.

Surveillance programs and reliable data are key for controlling and streamlining these efforts. Surveillance data of antimicrobial resistance at higher levels (national or international) usually comprise aggregated numbers. The basis of this information is generated and stored at local microbiological laboratories where isolated microorganisms are tested for their susceptibility to a whole range of antimicrobial agents. The efficacy of these agents against microorganisms is nowadays interpreted as follows \textsuperscript{{[}4{]}}:

\begin{itemize}
\tightlist
\item
  R (``resistant'') - there is a high likelihood of therapeutic failure;
\item
  S (``susceptible, standard dosing regimen'') - there is a high likelihood of therapeutic success using a standard dosing regimen of an antimicrobial agent;
\item
  I (``susceptible, increased exposure'') - there is a high likelihood of therapeutic success, but only when exposure to an antimicrobial agent is increased by adjusting the dosing regimen or its concentration at the site of infection.
\end{itemize}

Generally, antimicrobial resistance is defined as the proportion of resistant microorganisms (R) among all tested microorganisms of the same species (R + S + I). Today, the two major guideline institutes to define the international standards on antimicrobial resistance are the European Committee on Antimicrobial Susceptibility Testing (EUCAST) \textsuperscript{{[}5{]}} and the Clinical and Laboratory Standards Institute (CLSI) \textsuperscript{{[}6{]}}. The guidelines from these two institutes are adopted by 94\% of all countries reporting antimicrobial resistance to the WHO \textsuperscript{{[}7{]}}.

Although these standardised guidelines are in place on the laboratory level for the data generation process, stored data in laboratory information systems are often not yet suitable for data analysis. Laboratory information systems are often designed to fit billing purposes rather than epidemiological data analysis. Furthermore, (inter-)national surveillance is hindered by inadequate standardisation of epidemiological definitions, different types of samples and data collection, settings included, microbiological testing methods (including susceptibility testing), and data sharing policies \textsuperscript{{[}8{]}}. The necessity of accurate data analysis in the field of antimicrobial resistance has just recently been further underlined \textsuperscript{{[}9{]}}. Antimicrobial resistance analyses require a thorough understanding of microbiological tests and their results, the biological taxonomy of microorganisms, the clinical and epidemiological relevance of the results, their pharmaceutical implications, and (inter-)national standards and guidelines for working with and reporting antimicrobial resistance.

Here, we describe the \texttt{AMR} package for\texttt{R} \textsuperscript{{[}10{]}}, which has been developed to standardise clean and reproducible antimicrobial resistance data analyses using international standardised recommendations \textsuperscript{{[}5,6{]}} while incorporating scientifically reliable reference data about valid laboratory outcome, antimicrobial agents, and the complete biological taxonomy of microorganisms. The \texttt{AMR} package provides solutions and support for these aspects while being independent of underlying laboratory information systems, thereby democratising the analysis process. Developed in\texttt{R} and available on the Comprehensive\texttt{R} Archive Network (CRAN) since February 22nd 2018 \textsuperscript{{[}11{]}}, the \texttt{AMR} package enables reproducible workflows as described in other fields, such as environmental science \textsuperscript{{[}12{]}}. The \texttt{AMR} package provides a new technical instrument to aid in curbing the global threat of antimicrobial resistance. Furthermore, local, and regional data in the laboratories can now become relevant in any setting for public health.

While no other packages\texttt{R} package with the purpose of dealing with antimicrobial resistance data are available on CRAN or Bioconductor, the \texttt{AMR} package may be integrated in workflows of related packages. For example, the\texttt{R} Epidemics Consortium (RECON) provides high-quality packages for data analysis in infectious disease outbreaks or epidemics (for example incidence and epicontacts) \textsuperscript{{[}13,14{]}}. In addition, on the laboratory side the antibioticR package provides approaches to work with disc diffusion zone diameter and minimum inhibitory concentration data from environment samples \textsuperscript{{[}15{]}}. We aim at providing a comprehensive and standardised toolbox for antimicrobial resistance data processing and analysis, with a focus on microbiological, clinical, and epidemiological purposes that was yet missing.

The following sections describe the functionality of the \texttt{AMR} package according to its core functionalities for transforming, enhancing, and analysing antimicrobial resistance data using scientifically reliable reference data.

\hypertarget{antimicrobial-resistance-data}{%
\section{Antimicrobial resistance data}\label{antimicrobial-resistance-data}}

Microbiological tests can be performed on different specimens, such as blood or urine samples or nasal swabs. After arrival at the microbiological laboratory, the specimens are traditionally cultured on specific media, such as blood agar. If a microorganism can be isolated from these media, it is tested against several antimicrobial agents. Based on the minimal inhibitory concentration (MIC) of the respective agent and interpretation guidelines, such as guidelines by EUCAST \textsuperscript{{[}5{]}} and CLSI \textsuperscript{{[}6{]}}, test results are reported as ``resistant'' (R), ``susceptible'' (S) or ``susceptible, increased exposure'' (I). A typical data structure is illustrated in Table 1 \textsuperscript{{[}5{]}}.

Table 1. Example of an antimicrobial resistance report.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t01} 

}

\end{figure}

Table 2. Example of an antimicrobial resistance report.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t02} 

}

\end{figure}

The \texttt{AMR} package aims at providing a standardised and automated way of cleaning, transforming, and enhancing these typical data structures (Table 1 and 2), independent of the underlying data source. Processed data would be similar to Table 3 that highlights several package functionalities in the sections below.

Table 3. Enhanced antimicrobial resistance report example.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t03} 

}

\end{figure}

\hypertarget{antimicrobial-resistance-data-transformation}{%
\section{Antimicrobial resistance data transformation}\label{antimicrobial-resistance-data-transformation}}

\hypertarget{working-with-taxonomically-valid-microorganism-names}{%
\subsection{Working with taxonomically valid microorganism names}\label{working-with-taxonomically-valid-microorganism-names}}

Coercing is a computational process of forcing output based on an input. For microorganism names, coercing user input to taxonomically valid microorganism names is crucial to ensure correct interpretation and to enable grouping based on taxonomic properties. To this end, the \texttt{AMR} package includes all microbial entries from The Catalogue of Life (\url{http://www.catalogueoflife.org}), the most comprehensive and authoritative global index of species currently available \textsuperscript{{[}16{]}}. It holds essential information on the names, relationships, and distributions of more than 1.9 million species. The integration of it into the \texttt{AMR} package is described in Appendix A.

The \texttt{as.mo()} function makes use of this underlying data to transform a vector of characters to a new class `\texttt{‘mo’} of taxonomically valid microorganism name. The resulting values are microbial IDs, which are human-readable for the trained eye and contain information about the taxonomic kingdom, genus, species, and subspecies (Figure 1).

\begin{figure}

{\centering \includegraphics[width=0.66\linewidth]{images/04-01} 

}

\caption{The structure of a typical microbial ID as used in the `AMR` package. An ID consists of two to four elements, separated by an underscore. The first element is the abbreviation of the taxonomic kingdom. The remaining elements consist of abbreviations of the lowest taxonomic levels of every microorganism: genus, species (if available) and subspecies (if available). Abbreviations used for the microbial IDs of microorganism names were created using the base` R` function `abbreviate()`.}\label{fig:fig4-1}
\end{figure}

The \texttt{as.mo()} function compares the user input with taxonomically valid microorganism names, rates the matching with a score and returns results based on the highest score. This matching score (\(m\)), ranging from \(0\) to \(1\), is calculated using the following equation:

\[m_{(x,n)} = \frac{l_{n} - 0.5 \cdot \min\{ l_n, \operatorname{lev}(x,n) \} }{l_{n} \cdot p_{n} \cdot k_{n}}\]

where:

\begin{itemize}
\tightlist
\item
  \(x\) is the user input;
\item
  \(n\) is a taxonomic name (genus, species, and subspecies);
\item
  \(l_n\) is the length of \(n\);
\item
  lev is the Levenshtein distance function \textsuperscript{{[}17{]}}, which counts any insertion, deletion and substitution as \(1\) that is needed to change \(x\) into \(n\);
\item
  \(p_n\) is the human pathogenic prevalence group of \(n\), as described below;
\item
  \(k_n\) is the taxonomic kingdom of \(n\), set as Bacteria = 1, Fungi = 2, Protozoa = 3, Archaea = 4, others = 5.
\end{itemize}

The grouping into human pathogenic prevalence (\(p\)) is based on experience from several microbiological laboratories in the Netherlands in conjunction with international reports on pathogen prevalence \textsuperscript{{[}7,18,19{]}}. \textbf{Group 1} (most prevalent microorganisms) consists of all microorganisms where the taxonomic class is Gammaproteobacteria or where the taxonomic genus is \emph{Enterococcus}, \emph{Staphylococcus} or \emph{Streptococcus}. This group consequently contains all common Gram-negative bacteria, such as \emph{Pseudomonas} and \emph{Legionella} and all species within the order Enterobacterales. \textbf{Group 2} consists of all microorganisms where the taxonomic phylum is Proteobacteria, Firmicutes, Actinobacteria or Sarcomastigophora, or where the taxonomic genus is \emph{Absidia}, \emph{Acremonium}, \emph{Actinotignum}, \emph{Alternaria}, \emph{Anaerosalibacter}, \emph{Apophysomyces}, \emph{Arachnia}, \emph{Aspergillus}, \emph{Aureobacterium}, \emph{Aureobasidium}, \emph{Bacteroides}, \emph{Basidiobolus}, \emph{Beauveria}, \emph{Blastocystis}, \emph{Branhamella}, \emph{Calymmatobacterium}, \emph{Candida}, \emph{Capnocytophaga}, \emph{Catabacter}, \emph{Chaetomium}, \emph{Chryseobacterium}, \emph{Chryseomonas}, \emph{Chrysonilia}, \emph{Cladophialophora}, \emph{Cladosporium}, \emph{Conidiobolus}, \emph{Cryptococcus}, \emph{Curvularia}, \emph{Exophiala}, \emph{Exserohilum}, \emph{Flavobacterium}, \emph{Fonsecaea}, \emph{Fusarium}, \emph{Fusobacterium}, \emph{Hendersonula}, \emph{Hypomyces}, \emph{Koserella}, \emph{Lelliottia}, \emph{Leptosphaeria}, \emph{Leptotrichia}, \emph{Malassezia}, \emph{Malbranchea}, \emph{Mortierella}, \emph{Mucor}, \emph{Mycocentrospora}, \emph{Mycoplasma}, \emph{Nectria}, \emph{Ochroconis}, \emph{Oidiodendron}, \emph{Phoma}, \emph{Piedraia}, \emph{Pithomyces}, \emph{Pityrosporum}, \emph{Prevotella}, \emph{Pseudallescheria}, \emph{Rhizomucor}, \emph{Rhizopus}, \emph{Rhodotorula}, \emph{Scolecobasidium}, \emph{Scopulariopsis}, \emph{Scytalidium}, \emph{Sporobolomyces}, \emph{Stachybotrys}, \emph{Stomatococcus}, \emph{Treponema}, \emph{Trichoderma}, \emph{Trichophyton}, \emph{Trichosporon}, \emph{Tritirachium} or \emph{Ureaplasma}. \textbf{Group 3} consists of all other microorganisms.

This will lead to the effect that e.g., \texttt{"E.\ coli"} will return the microbial ID of \emph{Escherichia coli} (\(m = 0.688\), a highly prevalent microorganism found in humans) and not \emph{Entamoeba coli} (\(m = 0.079\), a less prevalent microorganism in humans), although the latter would alphabetically come first. The matching score function is for users available as \texttt{mo\_matching\_score()}.

If any coercion rules are applied, a warning is printed to the console and scores can be reviewed by calling \texttt{mo\_uncertainties()}, that prints all other matches with their matching scores. Users can furthermore control the coercion rules by setting the allow\_uncertain argument in the \texttt{as.mo()} function. The following values or levels can be used:

\begin{itemize}
\tightlist
\item
  \texttt{0}: no additional rules are applied;
\item
  \texttt{1}: allow previously accepted (but now invalid) taxonomic names and minor spelling errors;
\item
  \texttt{2}: allow all of 1, strip values between brackets, inverse the words of the input, strip off text elements from the end keeping at least two elements;
\item
  \texttt{3}: allow all of level 1 and 2, strip off text elements from the end, allow any part of a taxonomic name;
\item
  \texttt{TRUE} (default): equivalent to \texttt{2};
\item
  \texttt{FALSE}: equivalent to \texttt{0}.
\end{itemize}

To support organisation specific microbial IDs, users can specify a custom reference \texttt{‘data.frame’}, by using \texttt{as.mo(...,\ reference\_df\ =\ ...)}. This process can also be automated by users with the \texttt{set\_mo\_source()} function.

\hypertarget{properties-of-microorganisms}{%
\subsubsection{Properties of microorganisms}\label{properties-of-microorganisms}}

The package contains functions to return a specific (taxonomic) property of a microorganism from the \texttt{microorganisms} data set (see Appendix A). Functions that start with \texttt{mo\_*} can be used to retrieve the most recently defined taxonomic properties of any microorganism quickly and conveniently. These functions rely on the \texttt{as.mo()} function internally: \texttt{mo\_name()}, \texttt{mo\_fullname()}, \texttt{mo\_shortname()}, \texttt{mo\_subspecies()}, \texttt{mo\_species()}, \texttt{mo\_genus()}, \texttt{mo\_family()}, \texttt{mo\_order()}, \texttt{mo\_class()}, \texttt{mo\_phylum()}, \texttt{mo\_kingdom()}, \texttt{mo\_type()}, \texttt{mo\_gramstain()}, \texttt{mo\_ref()}, \texttt{mo\_authors()}, \texttt{mo\_year()}, \texttt{mo\_rank()}, \texttt{mo\_taxonomy()}, \texttt{mo\_synonyms()}, \texttt{mo\_info()} and \texttt{mo\_url()}. Determination of the Gram stain, by using \texttt{mo\_gramstain()}, is based on the taxonomic subkingdom and phylum. According to Cavalier-Smith \textsuperscript{{[}20{]}}, who defined the subkingdoms Negibacteria and Posibacteria, only the following phyla are Posibacteria: Actinobacteria, Chloroflexi, Firmicutes and Tenericutes. Bacteria from these phyla are considered Gram-positive - all other bacteria are considered Gram-negative. Gram stains are only relevant for species within the kingdom of Bacteria. For species outside this kingdom, \texttt{mo\_gramstain()} will return \texttt{NA}.

\hypertarget{working-with-antimicrobial-names-or-codes}{%
\subsection{Working with antimicrobial names or codes}\label{working-with-antimicrobial-names-or-codes}}

The \texttt{AMR} package includes the \texttt{antibiotics} data set, which comprises common laboratory information system codes, official names, ATC (Anatomical Therapeutic Chemical) codes, defined daily doses (DDD) and more than 5,000 trade names of 456 antimicrobial agents (see Appendix A). The ATC code system and the reference list for DDDs have been developed and made available by the World Health Organization Collaborating Centre for Drug Statistics Methodology (WHOCC) to standardise pharmaceutical classifications \textsuperscript{{[}21{]}}. All agents in the \texttt{antibiotics} data set have a unique antimicrobial ID, which is based on abbreviations used by the European Antimicrobial Resistance Surveillance Network (EARS-Net), the largest publicly funded system for antimicrobial resistance surveillance in Europe \textsuperscript{{[}22{]}}. Furthermore, the \texttt{AMR} package includes the antivirals data seta containing antiviral agents, which is also described in Appendix A.

\hypertarget{properties-of-antimicrobial-agents}{%
\subsubsection{Properties of antimicrobial agents}\label{properties-of-antimicrobial-agents}}

It is a common task in microbiological data analyses (and other clinical or epidemiological fields) to work with different antimicrobial agents. The \texttt{AMR} package provides several functions to translate inputs such as ATC codes, abbreviations, or names in any direction. Using \texttt{as.ab()}, any input will be transformed to an antimicrobial ID of class \texttt{‘ab’}. Helper functions are available to get specific properties of antimicrobial IDs, such as \texttt{ab\_name()} for getting the official name, \texttt{ab\_atc()} for the ATC code, or \texttt{ab\_cid()} for the CID (Compound ID) used by PubChem \textsuperscript{{[}23{]}}. Trade names can be also used as input. For example, the input values ``Amoxil'', ``dispermox'', ``amox'' and ``J01CA04'' all return the ID of amoxicillin (AMX):

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{as.ab}\NormalTok{(}\StringTok{"Amoxicillin"}\NormalTok{)}
\CommentTok{\#\textgreater{} Class \textless{}ab\textgreater{}}
\CommentTok{\#\textgreater{} [1] AMX}

\FunctionTok{as.ab}\NormalTok{(}\FunctionTok{c}\NormalTok{(}\StringTok{"Amoxil"}\NormalTok{, }\StringTok{"dispermox"}\NormalTok{, }\StringTok{"amox"}\NormalTok{, }\StringTok{"J01CA04"}\NormalTok{))}
\CommentTok{\#\textgreater{} Class \textless{}ab\textgreater{}}
\CommentTok{\#\textgreater{} [1] AMX AMX AMX AMX}

\FunctionTok{ab\_name}\NormalTok{(}\StringTok{"Amoxil"}\NormalTok{)}
\CommentTok{\#\textgreater{} [1] "Amoxicillin" }

\FunctionTok{ab\_atc}\NormalTok{(}\StringTok{"amox"}\NormalTok{)}
\CommentTok{\#\textgreater{} [1] "J01CA04"}

\FunctionTok{ab\_name}\NormalTok{(}\StringTok{"J01CA04"}\NormalTok{)}
\CommentTok{\#\textgreater{} [1] "Amoxicillin"}
\end{Highlighting}
\end{Shaded}

If more than one antimicrobial agent is found in the input string, a warning with the additional findings is printed to the console.

\hypertarget{filtering-data-based-on-classes-of-antimicrobial-agents}{%
\subsubsection{Filtering data based on classes of antimicrobial agents}\label{filtering-data-based-on-classes-of-antimicrobial-agents}}

The application of the ATC classification system also enables grouping of antimicrobial agents for data analyses. Data sets with microbial isolates can be filtered on isolates with specific results for tested antimicrobial agents in a specific antimicrobial class. For example, using \texttt{filter\_carbapenems(result\ =\ "R")} returns data of all isolates with tested resistance to any of the 14 available antimicrobial agents in in the group of carbapenems according to the \texttt{antibiotics} data set.

\hypertarget{working-with-antimicrobial-susceptibility-test-results}{%
\subsection{Working with antimicrobial susceptibility test results}\label{working-with-antimicrobial-susceptibility-test-results}}

Minimal inhibitory concentrations (MIC) are susceptibility test results measured by microbiological laboratory equipment to determine at which minimum antimicrobial drug concentration 99.9\% of a microorganism is inhibited in growth. These concentrations are often capped at a minimum and maximum, for example ≤0.02 µg/ml and ≥32 µg/ml, respectively. The \texttt{‘mic’} class, an ordered \texttt{‘factor’} containing valid MIC values, keeps these operators while still ordering all possible outcomes correctly so that e.g., ``\textless=0.02'' will be considered lower than ``0.04''.

Another susceptibility testing method is the use of drug diffusion disks, which are small tablets containing a specified concentration of an antimicrobial agent. These disks are applied onto a solid growth medium or a specific agar plate. After 24 hours of incubation, the diameter of the growth inhibition around a disk can be measured in millimetres with a ruler. The \texttt{‘disk’} class can be used to clean these kinds of measurements, since they should always be valid numeric values between 6 and 50. The supported minima and maxima of valid values for both classes, \texttt{‘mic’} and \texttt{‘disk’}, are displayed in Table 4.

Table 4. Antimicrobial susceptibility test classes.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t04} 

}

\end{figure}

The higher the MIC or the smaller the growth inhibition diameter, the more active substance of an antimicrobial agent is needed to inhibit cell growth, i.e.~the higher the antimicrobial resistance against the tested antimicrobial agent. At high MICs and small diameters, guidelines interpret the microorganism as ``resistant'' (R) to the tested antimicrobial agent. At low MICs and wide diameters, guidelines interpret the microorganism as ``susceptible'' (S) to the tested antimicrobial agent. In between, the microorganism is classified as ``susceptible, increased exposure'' (I). For these three interpretations the \texttt{‘rsi’} class has been developed. When using \texttt{as.rsi()} on MIC values (of class \texttt{‘mic’}) or disk diffusion diameters (of class \texttt{‘disk’}), the values will be interpreted according to the guidelines from the CLSI or EUCAST (all guidelines between 2011 and 2020 are included in the \texttt{AMR} package) \textsuperscript{{[}24,25{]}}. Guidelines can be changed by setting the guidelines argument.

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{\# Low MIC value}
\FunctionTok{as.rsi}\NormalTok{(}\FunctionTok{as.mic}\NormalTok{(}\DecValTok{2}\NormalTok{), }\StringTok{"E. coli"}\NormalTok{, }\StringTok{"ampicillin"}\NormalTok{, }\AttributeTok{guideline =} \StringTok{"EUCAST 2020"}\NormalTok{)}
\CommentTok{\#\textgreater{} Class \textless{}rsi\textgreater{}}
\CommentTok{\#\textgreater{} [1] S}

\CommentTok{\# High MIC value}
\FunctionTok{as.rsi}\NormalTok{(}\FunctionTok{as.mic}\NormalTok{(}\DecValTok{32}\NormalTok{), }\StringTok{"E. coli"}\NormalTok{, }\StringTok{"ampicillin"}\NormalTok{, }\AttributeTok{guideline =} \StringTok{"EUCAST 2020"}\NormalTok{)}
\CommentTok{\#\textgreater{} Class \textless{}rsi\textgreater{}}
\CommentTok{\#\textgreater{} [1] R}
\end{Highlighting}
\end{Shaded}

When using the \texttt{as.rsi()} function on existing antimicrobial interpretations, it tries to coerce the input to the values ``R'', ``S'' or ``I''. These values can in turn be used to calculate the proportion of antimicrobial resistance.

\hypertarget{interpretative-rules-by-eucast}{%
\subsection{Interpretative rules by EUCAST}\label{interpretative-rules-by-eucast}}

Next to supplying guidelines to interpret raw MIC values, EUCAST has developed a set of expert rules to assist clinical microbiologists in the interpretation and reporting of antimicrobial susceptibility tests \textsuperscript{{[}5{]}}. The rules comprise assistance on intrinsic resistance, exceptional phenotypes, and interpretive rules. The \texttt{AMR} package covers intrinsic resistant and interpretive rules for data transformation and standardisation purposes. The first prevents false susceptibility reporting by providing a list of organisms with known intrinsic resistance to specific antimicrobial agents (e.g., cephalosporin resistance of all enterococci). Interpretative rules apply inference from established resistance mechanisms \textsuperscript{{[}26-29{]}}. Both groups of rules are based on classic IF THEN statements (e.g., IF Enterococcus spp. resistant to ampicillin THEN also report as resistant to imipenem). Some rules provide assistance for further actions when certain resistance has been detected, i.e., performing additional testing of the isolated microorganism. The \texttt{AMR} package function \texttt{eucast\_rules()} can apply all EUCAST rules that do not rely on additional clinical information, such as additional information on patients' diagnoses. Table 2 and 3 highlight the transformation for the reporting of AMX = S in patient\_id = 000003 to the correct report according to EUCAST rules of AMX = R. Of note, however, EUCAST rules overwrite original data to correct for the difference in how antimicrobial agents affect the tested microorganism in vitro (in the laboratory) and in vivo (in the human body). This requires users to closely collaborate with the data source provider to ensure correct versioning, backward compatibility, reproducibility, and taking into account specific local regulation for resistance reporting. Typical scenarios where changes to the original data points apply include in vitro test results indicating susceptibility when resistance in vivo is known. The changes are based on scientific consensus to ensure reliable high-quality reporting of antimicrobial susceptibility results. All changes to the data are printed to the console and can also be reviewed in detail by setting the argument \texttt{eucast\_rules(...,\ verbose\ =\ TRUE)}.

EUCAST rules are subject to regular updates which are implemented into the \texttt{AMR} package by the \texttt{AMR} maintenance team shortly after publication. The \texttt{eucast\_rules()} function supports versioning of the rules. The arguments version\_breakpoints and version\_expertrules can be set to current or previous versions. By default, the \texttt{eucast\_rules()} function uses the latest implemented version.

\hypertarget{working-with-defined-daily-doses-ddd}{%
\subsection{Working with defined daily doses (DDD)}\label{working-with-defined-daily-doses-ddd}}

DDDs are essential for standardising antimicrobial consumption analysis, for inter-institutional or international comparison. The official DDDs are published by the WHOCC \textsuperscript{{[}36{]}}. Updates to the official publication are monitored by the \texttt{AMR} maintenance team and implemented in the \texttt{antibiotics} data set included in the \texttt{AMR} package. Other metrics exist such as the recommended daily dose (RDD) or the prescribed daily dose (PDD). However, DDDs are the only metric that is independent of a patient's disease and therapeutic choices and thus suitable for the \texttt{AMR} package.

Functions from the \texttt{atc\_online\_*()} family take any text as input that can be coerced with \texttt{as.ab()} (i.e., to class \texttt{‘ab’}). Next, the functions access the WHOCC online registry \textsuperscript{{[}30{]}} (internet connection required) and download the property defined in the arguments (e.g., administration = ``O'' or administration = ``P'' for oral or parenteral administration and property = ``ddd'' or property = ``groups'' to get DDD or the group of the selected antimicrobial defined by its ATC code).

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{atc\_online\_ddd}\NormalTok{(}\StringTok{"amoxicillin"}\NormalTok{, }\AttributeTok{administration =} \StringTok{"O"}\NormalTok{)}
\CommentTok{\#\textgreater{} [1] 1.5}

\FunctionTok{atc\_online\_groups}\NormalTok{(}\StringTok{"amoxicillin"}\NormalTok{)}
\CommentTok{\#\textgreater{} [1] "ANTIINFECTIVES FOR SYSTEMIC USE"}
\CommentTok{\#\textgreater{} [2] "ANTIBACTERIALS FOR SYSTEMIC USE"}
\CommentTok{\#\textgreater{} [3] "BETA{-}LACTAM ANTIBACTERIALS, PENICILLINS"}
\CommentTok{\#\textgreater{} [4] "Penicillins with extended spectrum"}
\end{Highlighting}
\end{Shaded}

\hypertarget{enhancing-antimicrobial-resistance-data}{%
\section{Enhancing antimicrobial resistance data}\label{enhancing-antimicrobial-resistance-data}}

\hypertarget{determining-first-isolates}{%
\subsection{Determining first isolates}\label{determining-first-isolates}}

Determining antimicrobial resistance or susceptibility can be done for a single agent (mono- therapy) or multiple agents (combination therapy). The calculation of antimicrobial resistance statistics is dependent on two prerequisites: the data should only comprise the first isolates and a minimum required number of 30 isolates should be met for every stratum in further analysis \textsuperscript{{[}6{]}}.

An isolate is a microorganism strain cultivated on specified growth media in a laboratory, so its phenotype can be determined. First isolates are isolates of any species found first in a patient per episode, regardless of the body site or the type of specimen (such as blood or urine) \textsuperscript{{[}6{]}}. The selection on first isolates (using function \texttt{first\_isolate()}) is important to prevent selection bias, as it would lead to overestimated or underestimated resistance to an antimicrobial agent. For example, if a patient is admitted with a multi-drug resistant microorganism and that microorganism is found in five different blood cultures the following week, it would overestimate resistance if all isolates were to be included in the analysis.

The episode in days can be set with the argument episode\_days, which defaults to 365 as suggested by the CLSI guideline \textsuperscript{{[}6{]}}.

\hypertarget{determining-multi-drug-resistant-organisms-mdro}{%
\subsection{Determining multi-drug resistant organisms (MDRO)}\label{determining-multi-drug-resistant-organisms-mdro}}

Definitions of multi-drug resistant organisms (MDRO) are regulated by national and international expert groups and differ between nations. The \texttt{AMR} package provides the functionality to quickly identify MDROs in a data set using the \texttt{mdro()} function. Guidelines can be set with the argument guideline. At default, it applies the guideline as proposed by Magiorakos \emph{et al.} \textsuperscript{{[}31{]}}. Their work describes the definitions for bacteria being \texttt{‘MDR’} (multi-drug-resistant), \texttt{‘XDR’} (extensively drug-resistant) or \texttt{‘PDR’} (pan-drug-resistant). These definitions are widely adopted \textsuperscript{{[}32{]}} and known in the field of medical microbiology.

Other guidelines currently supported are the international EUCAST guideline (guideline = ``EUCAST'' \textsuperscript{{[}33{]}}), the international WHO guideline on the management of drug-resistant tuberculosis (guideline = ``TB'' \textsuperscript{{[}34{]}}), and the national guidelines of The Netherlands (guideline = ``NL'' \textsuperscript{{[}35{]}}), and Germany (guideline = ``DE'' \textsuperscript{{[}36{]}}).

Some guidelines require a minimum availability of tested antimicrobial agents per isolate. This is needed to prevent false-negatives, since no reliable determination can be performed on only a few test results. This required minimum defaults to 50\%, but can be set by the user with the pct\_minimum\_classes. Isolates that do not meet this requirement will be skipped for determination and will return \texttt{NA} (not applicable), with an informative warning printed to the console.

The rules are applied per row of the data. The \texttt{mdro()} function automatically identifies the variables containing the microorganism codes and antimicrobial agents based on the \texttt{guess\_ab\_col()} function. Following the guideline set by the user, it analyses the specific antimicrobial resistance of a microorganism and flags that microorganism accordingly. The outcome is demonstrated in Table 5, where the first row is an MDRO according to the Dutch guidelines \textsuperscript{{[}35{]}}.

Table 5. Example of a multi-drug resistant organism (MDRO) in a data set identified by applying Dutch guidelines.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t05} 

}

\end{figure}

\hypertarget{multi-drug-resistant-tuberculosis}{%
\subsubsection{Multi-drug resistant tuberculosis}\label{multi-drug-resistant-tuberculosis}}

Tuberculosis is a major threat to global health caused by Mycobacterium tuberculosis (MTB) and is one of the top ten causes of death worldwide \textsuperscript{{[}37{]}}. Exceptional antimicrobial resistance in MTB is therefore of special interest. To this end, the international WHO guideline for the classification of drug resistance in MTB \textsuperscript{{[}34{]}} is included in the \texttt{AMR} package. The \texttt{mdr\_tb()} function is a convenient wrapper around \texttt{mdro(...,\ guideline\ =\ "TB")}, which returns an other ordered \texttt{‘factor’} than other \texttt{mdro()} functions. The output will contain the \texttt{‘factor’} levels \texttt{‘Negative’} \textless{} \texttt{‘Mono-resistant’} \textless{} \texttt{‘Poly-resistant’} \textless{} \texttt{‘Multi-drug-resistant’} \textless{} \texttt{‘Extensively\ drug-resistant’}, following the WHO guideline.

\hypertarget{analysing-antimicrobial-resistance-data}{%
\section{Analysing antimicrobial resistance data}\label{analysing-antimicrobial-resistance-data}}

\hypertarget{calculation-of-antimicrobial-resistance}{%
\subsection{Calculation of antimicrobial resistance}\label{calculation-of-antimicrobial-resistance}}

The \texttt{AMR} package contains several functions for fast and simple resistance calculations of bacterial or fungal isolates. A minimum number of available isolates is needed for the reliability of the outcome. The CLSI guideline suggests a minimum of 30 available first isolates irrespective of the type of statistical analysis \textsuperscript{{[}6{]}}. Therefore, this number is used as the default setting for any function in the package that calculates antimicrobial resistance or susceptibility, which can be changed with the minimum argument in all applicable functions.

\hypertarget{counts}{%
\subsubsection{Counts}\label{counts}}

The \texttt{AMR} package relies on the concept of tidy data \textsuperscript{{[}38{]}}, although not strictly following its rules (one row per test rather than one row per observation). Function names to calculate the number of available isolates follow these general resistance interpretation standards with \texttt{count\_S()}, \texttt{count\_I()}, and \texttt{count\_R()} respectively. Combinations of antimicrobial resistance interpretations can be counted with \texttt{count\_SI()} and \texttt{count\_IR()}. All these functions take a vector of interpretations of the class \texttt{‘rsi’} (as discussed above) or are internally transformed with \texttt{as.rsi()}. The returned value is the sum of the respective interpretation in the selected test column. All \texttt{count\_*()} functions support quasi-quotation with pipes, grouped variables, and can be used with \texttt{dplyr::summarise()} \textsuperscript{{[}39{]}}.

\hypertarget{proportions}{%
\subsubsection{Proportions}\label{proportions}}

Calculation of antimicrobial resistance is carried out by counting the number of first resistant isolates (interpretation of ``R'') and dividing it by the number of all first isolates, see Equation 2. This is implemented in the \texttt{proportion\_R()} function. To calculate antimicrobial susceptibility, the number of susceptible first isolates (interpretation of ``S'' and ``I'') has to be counted and divided by the number of all first isolates, which is implemented in the \texttt{proportion\_SI()} function. For convenience, the \texttt{resistance()} function is an alias of the \texttt{proportion\_R()} function, and the \texttt{susceptibility()} function is an alias of the \texttt{proportion\_SI()} function.

The functions \texttt{proportion\_R()}, \texttt{proportion\_IR()}, \texttt{proportion\_I()}, \texttt{proportion\_SI()}, and \texttt{proportion\_S()} follow the same logic as the \texttt{count\_*()} functions and all return a vector of class \texttt{‘double’} with a value between 0 and 1. The argument \texttt{minimum} defines the minimal allowed number of available (tested) isolates (default: \texttt{minimum} = 30). Any number below the set minimum will return \texttt{NA} with a warning.

For calculating the proportion (\(P\)) of antimicrobial resistance or susceptibility to one antimicrobial agent, the following equation is used:

\[P_{(x, o)} = \frac{\sum_{i=1}^k [x_i \in o]}{\sum_{i=1}^k [x_i \in \{R,S,I\}]}\]

where \(P\) is the proportion of outcome \(o\) (that is either ``R'', ``S'', ``I'', or a combination of two of them), where \(x\) is a character vector of length \(k\) only consisting of values ``R'', ``S'', or ``I'' and \([x_i \in o]\) is the indicator function, returning \(1\) if the indicator function is true and \(0\) otherwise. The denominator must include the collection \(\{R,S,I\}\) so that '\texttt{wrong’} elements in \(x\) (i.e., elements not being ``R'', ``S'', or ``I'') will not be counted. Thus, the theoretical antimicrobial susceptibility of the vector \(x = \{S,S,I,R,R\}\) is:

\[P_{(x, o = \{S, I\})} = \frac{3}{5} = 0.6\]

For the proportion of empiric susceptibility (\(s\)) for more than one antimicrobial agent, the calculation can be carried out in two ways (Table 6).

Table 6. Example calculation for determining empiric susceptibility (\%SI) for more than one antimicrobial agent.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/04-t06} 

}

\end{figure}

The first method is to count the total number of first isolates where at least one agent was tested as ``S'' or ``I'' and divide it by the number of first isolates tested where any of the agents was tested (Equation 4). This method will be used when setting \texttt{only\_all\_tested\ =\ FALSE} in the \texttt{susceptibility()} function:

\[s_{(x, y)} = \frac{\sum_{i=1}^k [x_i \in \{S,I\} \lor y_i \in \{S,I\}]}{\sum_{i=1}^k [x_i \in \{R,S,I\} \lor y_i \in \{R,S,I\}]}\]

where \(x\) is a character vector only consisting of values ``R'', ``S'', or ``I'' (i.e., '\texttt{agent\ A’}) and \(y\) is another character vector only consisting of values ``R'', ``S'', or ``I'' (i.e., '\texttt{agent\ B’}).

The second method is to count the total number of first isolates where at least one agent was tested as ``S'' or ``I'' and where all agents were tested divided by the number of first isolates tested where all of the agents were tested (Equation 5). This method will be used when setting \texttt{only\_all\_tested\ =\ TRUE} in the \texttt{susceptibility()} function:

\[s'_{(x, y)} = \frac{\sum_{i=1}^k [(x_i \in \{S,I\} \lor y_i \in \{S,I\}) \, \land x_i \in \{R,S,I\} \land y_i \in \{R,S,I\}]}{\sum_{i=1}^k [x_i \in \{R,S,I\} \land y_i \in \{R,S,I\}]}\]

Based on Equation 2, the overall resistance and susceptibility of antimicrobial agents like gentamicin (GEN) and amoxicillin (AMX) can be calculated using the following syntax. The \texttt{example\_isolates} data set is an example data set included in the \texttt{AMR} package, see Appendix A. The \texttt{n\_rsi()} function is analogous to the \texttt{n()} function of the \texttt{dplyr} package. It counts the number of available isolates, but only includes observations with valid antimicrobial results (i.e., ``R'', ``S'', or ``I''):

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{library}\NormalTok{(}\StringTok{"dplyr"}\NormalTok{)}
\NormalTok{example\_isolates }\SpecialCharTok{\%\textgreater{}\%}
   \FunctionTok{summarise}\NormalTok{(}\AttributeTok{r\_gen =} \FunctionTok{proportion\_R}\NormalTok{(GEN),}
             \AttributeTok{r\_amx =} \FunctionTok{proportion\_R}\NormalTok{(AMX),}
             \AttributeTok{n\_gen =} \FunctionTok{n\_rsi}\NormalTok{(GEN),}
             \AttributeTok{n\_amx =} \FunctionTok{n\_rsi}\NormalTok{(AMX),}
             \AttributeTok{n\_total =} \FunctionTok{n}\NormalTok{())}
\CommentTok{\#\textgreater{}         r\_gen     r\_amx n\_gen n\_amx n\_total}
\CommentTok{\#\textgreater{} [1] 0.2458221 0.5955556  1855  1350    2000}
\end{Highlighting}
\end{Shaded}

This output reads: the resistance to gentamicin of all isolates in the \texttt{example\_isolates} data set is \(P{(x = GEN, o = \{R\})} = 24.6\%\), based on \(1855\) out of \(2000\) available isolates. This means that the susceptibility is \(P{(x = GEN, o = \{S,I\})} = 75.4\%\). The susceptibility to amoxicillin is \(P{(x = AMX, o = \{S,I\})} = 40.4\%\) based on \(1350\) isolates.

To calculate the effect of combination therapy, i.e., treating patients with multiple agents at the same time, all \texttt{proportion\_*()} functions can handle multiple variables as arguments as defined in Equation 4 and 5. For example, to calculate the empiric susceptibility of a combination therapy comprising gentamicin (GEN) and amoxicillin (AMX):

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{example\_isolates }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{summarise}\NormalTok{(}\AttributeTok{si\_gen\_amx =} \FunctionTok{proportion\_SI}\NormalTok{(GEN, AMX),}
            \AttributeTok{n\_gen\_amx =} \FunctionTok{n\_rsi}\NormalTok{(GEN, AMX),}
            \AttributeTok{n\_total =} \FunctionTok{n}\NormalTok{())}
\CommentTok{\#\textgreater{}     si\_gen\_amx n\_gen\_amx n\_total}
\CommentTok{\#\textgreater{} [1]   0.931843      1921    2000}
\end{Highlighting}
\end{Shaded}

This leads to the conclusion that combining gentamicin with amoxicillin would cover \(s{(x = GEN, y = AMX)} = 93.2\%\) based on \(1921\) out of \(2000\) available isolates, which is \(17.8\%\) more than when treating with gentamicin alone (\(P{(x = GEN, o = \{S,I\})} = 75.4\%\)). With these functions, exact calculations can be done to evaluate the empiric success of treating infections with one or more antimicrobial agents.

\hypertarget{design-decisions}{%
\section{Design decisions}\label{design-decisions}}

The \texttt{AMR} package follows the rationale of tidyverse packages as authored by Wickham \emph{et al.} \textsuperscript{{[}40{]}}. Most functions take a \texttt{‘data.frame’} or \texttt{‘tibble’} as input, support piping (\%\textgreater\%) operations, can work with quasi-quotations, and can be integrated into \texttt{dplyr} workflows, such as \texttt{mutate()} to create new variables and \texttt{group\_by()} to group by variables. Although the \texttt{AMR} package integrates well into tidyverse workflows, it can also be used with base \texttt{R}only. To this extent, the \texttt{AMR} package was developed to be independent of any other \texttt{R}package to ensure and maintain sustainability.

The \texttt{AMR} package supports multiple languages. Currently supported languages are English, Dutch, French, German, Italian, Portuguese, and Spanish. The system language will be used if the language is supported but can be overwritten with \texttt{options(AMR\_locale\ =\ ...)}. Multi-language support affects language-dependent output of functions such as \texttt{mo\_name()}, \texttt{mo\_gramstain()}, \texttt{mo\_type()}, and \texttt{ab\_name()}.

The \texttt{AMR} package uses S3 classes, object oriented (OO) systems available in R. They allow different types of output based on the user input. The \texttt{AMR} package introduces 5 S3 classes (\texttt{‘mo’}, \texttt{‘ab’}, \texttt{‘rsi’}, \texttt{‘mic’}, and \texttt{‘disk’}) to increase the convenience when working with antimicrobial susceptibility data.

\hypertarget{reproducible-example}{%
\section{Reproducible example}\label{reproducible-example}}

We consider the problem of working with antimicrobial resistance data from three different hospitals between 2011-01-01 and 2020-01-01. After loading the \texttt{AMR} package and additional tidyverse packages to allow transformation and plotting, we load the \texttt{example\_isolates}\_unclean example data from the \texttt{AMR} package into the global environment and assign it a new name.

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{library}\NormalTok{(}\StringTok{"dplyr"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"tidyr"}\NormalTok{)}
\FunctionTok{library}\NormalTok{(}\StringTok{"AMR"}\NormalTok{)}

\FunctionTok{options}\NormalTok{(}\AttributeTok{AMR\_locale =} \StringTok{"en"}\NormalTok{)}

\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ example\_isolates\_unclean}

\FunctionTok{glimpse}\NormalTok{(data)}
\CommentTok{\#\textgreater{} Rows: 3,000}
\CommentTok{\#\textgreater{} Columns: 8}
\CommentTok{\#\textgreater{} $ patient\_id \textless{}chr\textgreater{} "J3", "R7", "P3", "P10", "B7", "W3", "J8", "M3",…}
\CommentTok{\#\textgreater{} $ hospital   \textless{}chr\textgreater{} "A", "A", "A", "A", "A", "A", "A", "A", "A", "A"…}
\CommentTok{\#\textgreater{} $ date       \textless{}date\textgreater{} 2012{-}11{-}21, 2018{-}04{-}03, 2014{-}09{-}19, 2015{-}12{-}10,…}
\CommentTok{\#\textgreater{} $ bacteria   \textless{}chr\textgreater{} "E. coli", "K. pneumoniae", "E. coli", "E. coli"…}
\CommentTok{\#\textgreater{} $ AMX        \textless{}chr\textgreater{} "R", "R", "R", "S", "S", "R", "R", "R", "S", "S"…}
\CommentTok{\#\textgreater{} $ AMC        \textless{}chr\textgreater{} "I", "I", "S", "I", "S", "S", "S", "S", "S", "S"…}
\CommentTok{\#\textgreater{} $ CIP        \textless{}chr\textgreater{} "S", "S", "S", "S", "S", "R", "S", "S", "S", "S"…}
\CommentTok{\#\textgreater{} $ GEN        \textless{}chr\textgreater{} "S", "S", "S", "S", "S", "S", "S", "S", "S", "S"…}

\FunctionTok{unique}\NormalTok{(data}\SpecialCharTok{$}\NormalTok{hospital)}
\CommentTok{\#\textgreater{} [1] "A" "B" "C"}

\FunctionTok{unique}\NormalTok{(data}\SpecialCharTok{$}\NormalTok{bacteria)}
\CommentTok{\#\textgreater{}  [1] "E. coli"                  "K. pneumoniae"           }
\CommentTok{\#\textgreater{}  [3] "S. aureus"                "S. pneumoniae"           }
\CommentTok{\#\textgreater{}  [5] "klepne"                   "strpne"                  }
\CommentTok{\#\textgreater{}  [7] "esccol"                   "staaur"                  }
\CommentTok{\#\textgreater{}  [9] "Escherichia coli"         "Staphylococcus aureus"   }
\CommentTok{\#\textgreater{} [11] "Streptococcus pneumoniae" "Klebsiella pneumoniae"    }

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(bacteria)}
\CommentTok{\#\textgreater{}                    bacteria   n}
\CommentTok{\#\textgreater{} 1                   E. coli 494}
\CommentTok{\#\textgreater{} 2                    esccol 508}
\CommentTok{\#\textgreater{} 3          Escherichia coli 516}
\CommentTok{\#\textgreater{} 4             K. pneumoniae 108}
\CommentTok{\#\textgreater{} 5     Klebsiella pneumoniae 102}
\CommentTok{\#\textgreater{} 6                    klepne 116}
\CommentTok{\#\textgreater{} 7                 S. aureus 247}
\CommentTok{\#\textgreater{} 8             S. pneumoniae 151}
\CommentTok{\#\textgreater{} 9                    staaur 240}
\CommentTok{\#\textgreater{} 10    Staphylococcus aureus 243}
\CommentTok{\#\textgreater{} 11 Streptococcus pneumoniae 139}
\CommentTok{\#\textgreater{} 12                   strpne 136}
\end{Highlighting}
\end{Shaded}

The data contains 3,000 observations of 8 variables from 3 hospitals. The ``bacteria'' variable comprises 12 unique elements. However, they appear to encode the same information in different formats ('\texttt{E.\ coli’}, '\texttt{K.\ pneumoniae’}, '\texttt{S.\ aureus’}, '\texttt{S.\ pneumoniae’}, '\texttt{klepne’}, '\texttt{strpne’}, '\texttt{esccol’}, '\texttt{staaur’}, '\texttt{Escherichia\ coli’}, '\texttt{Staphylococcus\ aureus’}, '\texttt{Streptococcus\ pneumoniae’}, '\texttt{Klebsiella\ pneumoniae’}). We can use the \texttt{as.mo()} function to standardise the bacterial codes and add a variable with the official scientific name. The correct transformation of the bacterial codes can be reviewed by calling the \texttt{mo\_uncertainties()} function.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{mutate}\NormalTok{(}\AttributeTok{bacteria =} \FunctionTok{as.mo}\NormalTok{(bacteria),}
         \AttributeTok{bacteria\_name =} \FunctionTok{mo\_name}\NormalTok{(bacteria))}

\FunctionTok{mo\_uncertainties}\NormalTok{()}
\CommentTok{\#\textgreater{} "E. coli" {-}\textgreater{} Escherichia coli (B\_ESCHR\_COLI, matching score =}
\CommentTok{\#\textgreater{}              0.688)}
\CommentTok{\#\textgreater{}              Also matched: Entamoeba coli (0.079)}
\CommentTok{\#\textgreater{} "K. pneumoniae" {-}\textgreater{} Klebsiella pneumoniae (B\_KLBSL\_PNMN, matching}
\CommentTok{\#\textgreater{}                    score = 0.786)}
\CommentTok{\#\textgreater{}                    Also matched: Klebsiella pneumoniae ozaenae}
\CommentTok{\#\textgreater{}                    (0.707), Klebsiella pneumoniae rhinoscleromatis}
\CommentTok{\#\textgreater{}                    (0.658)}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} "S. aureus" {-}\textgreater{} Staphylococcus aureus (B\_STPHY\_AURS, matching score}
\CommentTok{\#\textgreater{}                = 0.690)}
\CommentTok{\#\textgreater{}                Also matched: Staphylococcus aureus anaerobius}
\CommentTok{\#\textgreater{}                (0.625), Streptomyces aureus (0.355), Stentor aureus}
\CommentTok{\#\textgreater{}                (0.052)}

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(bacteria, bacteria\_name)}
\CommentTok{\#\textgreater{}       bacteria            bacteria\_name    n}
\CommentTok{\#\textgreater{} 1 B\_ESCHR\_COLI         Escherichia coli 1518}
\CommentTok{\#\textgreater{} 2 B\_KLBSL\_PNMN    Klebsiella pneumoniae  326}
\CommentTok{\#\textgreater{} 3 B\_STPHY\_AURS    Staphylococcus aureus  730}
\CommentTok{\#\textgreater{} 4 B\_STRPT\_PNMN Streptococcus pneumoniae  426}
\end{Highlighting}
\end{Shaded}

In a next step, we can further enrich the data with additional microbial taxonomic data based on the ``bacteria'' variable, such as Gram-stain and microorganism family.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{mutate}\NormalTok{(}\AttributeTok{gram\_stain =} \FunctionTok{mo\_gramstain}\NormalTok{(bacteria),}
         \AttributeTok{family =} \FunctionTok{mo\_family}\NormalTok{(bacteria))}

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(gram\_stain)}
\CommentTok{\#\textgreater{}      gram\_stain    n}
\CommentTok{\#\textgreater{} 1 Gram{-}negative 1844}
\CommentTok{\#\textgreater{} 2 Gram{-}positive 1156}

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(family)}
\CommentTok{\#\textgreater{}               family    n}
\CommentTok{\#\textgreater{} 1 Enterobacteriaceae 1844}
\CommentTok{\#\textgreater{} 2 Staphylococcaceae   730}
\CommentTok{\#\textgreater{} 3 Streptococcaceae    426}
\end{Highlighting}
\end{Shaded}

The variables ``AMX'', ``AMC'', ``CIP'', and ``GEN'' contain antimicrobial susceptibility test results. The abbreviations stand for the tested antimicrobial agent. The official names and additional information about the antimicrobial agents can be checked with the \texttt{ab\_info()} function from the \texttt{AMR} package.

\begin{Shaded}
\begin{Highlighting}[]
\FunctionTok{ab\_info}\NormalTok{(}\StringTok{"AMX"}\NormalTok{)}
\CommentTok{\#\textgreater{} $ab}
\CommentTok{\#\textgreater{} [1] "AMX"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $atc}
\CommentTok{\#\textgreater{} [1] "J01CA04"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $cid}
\CommentTok{\#\textgreater{} [1] 33613}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $name}
\CommentTok{\#\textgreater{} [1] "Amoxicillin"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $group}
\CommentTok{\#\textgreater{} [1] "Beta{-}lactams/penicillins"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $atc\_group1}
\CommentTok{\#\textgreater{} [1] "Beta{-}lactam antibacterials, penicillins"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $atc\_group2}
\CommentTok{\#\textgreater{} [1] "Penicillins with extended spectrum"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $tradenames}
\CommentTok{\#\textgreater{} [1] "actimoxi" "amoclen" "amolin" }
\CommentTok{\#\textgreater{} [4] "amopen" "amopenixin" "amoxibiotic" }
\CommentTok{\#\textgreater{} [7] "amoxicaps" "amoxicilina" "amoxicillin" }
\CommentTok{\#\textgreater{} [10] "amoxicilline" "amoxicillinum" "amoxiden" }
\CommentTok{\#\textgreater{} [13] "amoxil" "amoxivet" "amoxy" }
\CommentTok{\#\textgreater{} [16] "amoxycillin" "anemolin" "aspenil" }
\CommentTok{\#\textgreater{} [19] "biomox" "bristamox" "cemoxin" }
\CommentTok{\#\textgreater{} [22] "clamoxyl" "delacillin" "dispermox" }
\CommentTok{\#\textgreater{} [25] "efpenix" "flemoxin" "hiconcil" }
\CommentTok{\#\textgreater{} [28] "histocillin" "hydroxyampicillin" "ibiamox" }
\CommentTok{\#\textgreater{} [31] "imacillin" "lamoxy" "metafarma capsules"}
\CommentTok{\#\textgreater{} [34] "metifarma capsules" "moxacin" "moxatag" }
\CommentTok{\#\textgreater{} [37] "ospamox" "pamoxicillin" "piramox" }
\CommentTok{\#\textgreater{} [40] "robamox" "sawamox pm" "tolodina" }
\CommentTok{\#\textgreater{} [43] "unicillin" "utimox" "vetramox" }
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $ddd}
\CommentTok{\#\textgreater{} $ddd$oral}
\CommentTok{\#\textgreater{} $ddd$oral$amount}
\CommentTok{\#\textgreater{} [1] 1.5}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $ddd$oral$units}
\CommentTok{\#\textgreater{} [1] "g"}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $ddd$iv}
\CommentTok{\#\textgreater{} $ddd$iv$amount}
\CommentTok{\#\textgreater{} [1] 3}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} $ddd$iv$units}
\CommentTok{\#\textgreater{} [1] "g"}
\end{Highlighting}
\end{Shaded}

In a data set containing antimicrobial names or codes (e.g., antimicrobial prescription data), the \texttt{as.ab()} function can be used to transform all values to valid antimicrobial codes. Extra columns with the official name and the defined daily dose (DDD) for intravenous administration could be added using \texttt{ab\_name()} and \texttt{ab\_ddd()}.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{antimicrobial\_example }\OtherTok{\textless{}{-}} \FunctionTok{data.frame}\NormalTok{(}\AttributeTok{agents =} \FunctionTok{c}\NormalTok{(}\StringTok{"AMX"}\NormalTok{,}
                                               \StringTok{"Ceftriaxon"}\NormalTok{,}
                                               \StringTok{"Cipro"}\NormalTok{))}

\NormalTok{antimicrobial\_example }\SpecialCharTok{\%\textgreater{}\%} 
  \FunctionTok{mutate}\NormalTok{(}\AttributeTok{agents =} \FunctionTok{as.ab}\NormalTok{(agents), }
         \AttributeTok{agent\_names =} \FunctionTok{ab\_name}\NormalTok{(agents),}
         \AttributeTok{ddd\_iv =} \FunctionTok{ab\_ddd}\NormalTok{(agents, }\AttributeTok{administration =} \StringTok{"iv"}\NormalTok{))}
\CommentTok{\#\textgreater{} agents agent\_names ddd\_iv}
\CommentTok{\#\textgreater{} 1 AMX Amoxicillin 3.0}
\CommentTok{\#\textgreater{} 2 CRO Ceftriaxone 2.0}
\CommentTok{\#\textgreater{} 3 CIP Ciprofloxacin 0.8}
\end{Highlighting}
\end{Shaded}

Coming back to the cleaning of the data, the columns for the antimicrobial susceptibility test results (``AMX'', ``AMC'', ``CIP'', ``GEN'') need to be checked to contain only standard values (``R'', ``S'', ``I'').

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{select}\NormalTok{(AMX}\SpecialCharTok{:}\NormalTok{GEN) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{pivot\_longer}\NormalTok{(}\FunctionTok{everything}\NormalTok{(),}
               \AttributeTok{names\_to =} \StringTok{"antimicrobials"}\NormalTok{,}
               \AttributeTok{values\_to =} \StringTok{"interpretation"}\NormalTok{) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(interpretation)}
\CommentTok{\#\textgreater{} \# A tibble: 4 x 2}
\CommentTok{\#\textgreater{}   interpretation     n}
\CommentTok{\#\textgreater{}   \textless{}chr\textgreater{}          \textless{}int\textgreater{}}
\CommentTok{\#\textgreater{} 1 \textless{} 0.5 S          143}
\CommentTok{\#\textgreater{} 2 I               1105}
\CommentTok{\#\textgreater{} 3 R               4607}
\CommentTok{\#\textgreater{} 4 S               6145}
\end{Highlighting}
\end{Shaded}

The values contain some mixed values. The \texttt{as.rsi()} function can be used to clean these values and to assign a new class (\texttt{‘rsi’}) for further use of AMR functions.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{mutate\_at}\NormalTok{(}\FunctionTok{vars}\NormalTok{(AMX}\SpecialCharTok{:}\NormalTok{GEN), as.rsi)}

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{select}\NormalTok{(AMX}\SpecialCharTok{:}\NormalTok{GEN) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{pivot\_longer}\NormalTok{(}\FunctionTok{everything}\NormalTok{(),}
               \AttributeTok{names\_to =} \StringTok{"antimicrobials"}\NormalTok{,}
               \AttributeTok{values\_to =} \StringTok{"interpretation"}\NormalTok{) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(interpretation)}
\CommentTok{\#\textgreater{} \# A tibble: 3 x 2}
\CommentTok{\#\textgreater{}   interpretation     n}
\CommentTok{\#\textgreater{}   \textless{}rsi\textgreater{}          \textless{}int\textgreater{}}
\CommentTok{\#\textgreater{} 1   S             6288}
\CommentTok{\#\textgreater{} 2   I             1105}
\CommentTok{\#\textgreater{} 3   R             4607}
\end{Highlighting}
\end{Shaded}

After this transformation, the \texttt{eucast\_rules()} function can be applied to apply the latest resistance reporting guidelines.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{eucast\_rules}\NormalTok{()}
\end{Highlighting}
\end{Shaded}

The output to the console lists the changes made to data:

\begin{Shaded}
\begin{Highlighting}[]
\CommentTok{\#\textgreater{} The rules affected 508 out of 3,000 rows, making a total of 657 edits}
\CommentTok{\#\textgreater{} =\textgreater{} added 0 test results}
\CommentTok{\#\textgreater{} }
\CommentTok{\#\textgreater{} =\textgreater{} changed 657 test results}
\CommentTok{\#\textgreater{} {-} 11 test results changed from "S" to "I"}
\CommentTok{\#\textgreater{} {-} 473 test results changed from "S" to "R"}
\CommentTok{\#\textgreater{} {-} 85 test results changed from "I" to "R"}
\CommentTok{\#\textgreater{} {-} 19 test results changed from "I" to "S"}
\CommentTok{\#\textgreater{} {-} 33 test results changed from "R" to "I"}
\CommentTok{\#\textgreater{} {-} 36 test results changed from "R" to "S"}
\end{Highlighting}
\end{Shaded}

The data is now clean and ready for further analysis, for example, the identification of multi-drug resistant microorganisms. In this example, we use the Dutch guideline to determine multi-drug resistance \textsuperscript{{[}35{]}}.

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{data }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{mutate}\NormalTok{(}\AttributeTok{mdro =} \FunctionTok{mdro}\NormalTok{(., }\AttributeTok{guideline =} \StringTok{"nl"}\NormalTok{))}

\NormalTok{data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{count}\NormalTok{(bacteria\_name, mdro)}
\CommentTok{\#\textgreater{}              bacteria\_name     mdro    n}
\CommentTok{\#\textgreater{} 1         Escherichia coli Negative 1123}
\CommentTok{\#\textgreater{} 2         Escherichia coli Positive  395}
\CommentTok{\#\textgreater{} 3    Klebsiella pneumoniae Negative  237}
\CommentTok{\#\textgreater{} 4    Klebsiella pneumoniae Positive   89}
\CommentTok{\#\textgreater{} 5    Staphylococcus aureus Negative  730}
\CommentTok{\#\textgreater{} 6 Streptococcus pneumoniae Negative  426}
\end{Highlighting}
\end{Shaded}

According to the Dutch guideline, 484 (395 + 89) multi-drug resistant microorganisms were found in 3,000 tested isolates. No multi-drug resistance was found in Staphylococcus aureus and Streptococcus pneumoniae.

As described in Section 4.4.1, the identification of first isolates is essential for the reporting of resistance patterns. Using the \texttt{filter\_first\_isolate()} function and \texttt{proportion\_df()} in combination with \texttt{group\_by()}, we get a complete resistance analysis per hospital, bacteria, first isolate, and tested antimicrobial agent in one call:

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{resistance\_proportion }\OtherTok{\textless{}{-}}\NormalTok{ data }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{filter\_first\_isolate}\NormalTok{() }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{group\_by}\NormalTok{(hospital) }\SpecialCharTok{\%\textgreater{}\%}
  \FunctionTok{proportion\_df}\NormalTok{()}

\FunctionTok{head}\NormalTok{(resistance\_proportion)}
\CommentTok{\#\textgreater{}   hospital                  antibiotic interpretation     value}
\CommentTok{\#\textgreater{} 1        A                 Amoxicillin             SI 0.5773050}
\CommentTok{\#\textgreater{} 2        A                 Amoxicillin              R 0.4226950}
\CommentTok{\#\textgreater{} 3        A Amoxicillin/clavulanic acid             SI 0.8085106}
\CommentTok{\#\textgreater{} 4        A Amoxicillin/clavulanic acid              R 0.1914894}
\CommentTok{\#\textgreater{} 5        A               Ciprofloxacin             SI 0.8042553}
\CommentTok{\#\textgreater{} 6        A               Ciprofloxacin              R 0.1957447}
\end{Highlighting}
\end{Shaded}

From the console we get the information how many first isolates were identified and used in the filter.

From here on, the data is ready for further analysis with functions for plotting (e.g., the \texttt{ggplot2} package \textsuperscript{{[}41{]}}), AMR extension functions for base \texttt{R}(e.g., \texttt{summary()}, \texttt{plot()}), or \texttt{AMR} helper functions for plotting and basic modelling (e.g., \texttt{ggplot\_rsi()}, \texttt{geom\_rsi()}, \texttt{resistance\_predict()}).

\hypertarget{discussion}{%
\section{Discussion}\label{discussion}}

For the first time, a free and open-source software solution is available to cover all aspects of working with antimicrobial resistance data. The \texttt{AMR} package provides functionalities that enable standardised and reproducible workflows from raw laboratory data to publishable results, for research and clinical workflows alike. In the field of clinical microbiology and infectious diseases, research and clinical workflows are closely linked. For example, a performed research study on the prevalence of antimicrobial-resistant bacteria can have direct implications on the choice of antimicrobial agents for the treatment of patients. The \texttt{AMR} package was developed to be used in any research or clinical setting where the data analysis on microorganisms, antimicrobial resistance, antimicrobial agents is required.

Both, researchers and clinicians rely on the data from electronic laboratory information systems (LIS) where laboratory test results are processed, stored, and archived. Although some commercial solutions exist to conduct medical microbiological data analysis, these solutions are not comprehensive enough to apply antimicrobial resistance analysis for any clinical or research setting. Costs of these tools are a further constraint in resource-limited settings. Moreover, researchers and clinicians that require data from multiple LIS sources to be used in multi-center studies experience major barriers which cannot be solved by available commercial solutions.

Firstly, simple codes for microorganisms show substantial differences between different LIS and presumably correct taxonomic names are often misspelled or outdated. We analysed the taxonomic names of bacteria used in reports from seven different public health institutions that perform microbiological diagnostics in the Netherlands and compared them with an official scientific up-to-date source for microbial taxonomy, the Catalogue of Life \textsuperscript{{[}16{]}}. These institutions cover microbiological diagnostics for hospitals and primary care for 15\% of the total Dutch population. All institutions reported outdated taxonomic names with a maximum lag ranging between 34 and 41 years. Given that antimicrobial resistance guidelines are strongly based on the microbial taxonomy (some rules only apply to a specific genus, other rules apply to a specific family), it is crucial that this information is correct and timely updated. All institutions admitted that there was no standard operating procedure to maintain their taxonomic reference data. Implementing and maintaining the taxonomic data for these and other institutions has been challenging, since no common machine-readable, reliable and up-to-date resource for the microbial taxonomy was publicly available. For reliable reference data about antimicrobial agents, this also holds true. The \texttt{AMR} package provides machine-readable reference data files for the complete microbial taxonomy and for more than 500 antimicrobial agents. Using functions starting with \texttt{mo\_*} and \texttt{ab\_*}, names of microorganisms and antimicrobial agents can be translated between different LIS codes or other forms of text codes for microorganisms and consequently allows to merge data sets from different sites with little effort.

Secondly, antimicrobial resistance interpretation guidelines \textsuperscript{{[}5,6{]}} and taxonomic definitions of microorganisms are under constant change and are continually published in dedicated peer-reviewed journals. This is further complicated by differences between local, regional, and national guidelines. Yet, comparability and reproducibility across setting and time are key in research and clinics. The \texttt{AMR} package functions \texttt{eucast\_rules()} (to apply guidelines to data), \texttt{mdro()} (to check for multi-drug resistance according to guidelines), or \texttt{first\_isolate()} (to determine first isolates according to guidelines) address the needs to standardise comparability, and empower data analysts beyond the capabilities of their local LIS. The \texttt{AMR} package can be used as an extra layer of data validation when retrieving raw data from a LIS. Overall, the functionality of the \texttt{AMR} package has the potential to improve data validity in clinical settings, to ease multi-center study workflows, and to foster research reporting practices. The inherent global nature of antimicrobial resistances requires researchers, clinicians, and policy makers to reach beyond the borders of their local laboratory. The \texttt{AMR} package can build the bridge to link these sources and further encourages open science principles through its open-source approach.

The \texttt{AMR} package also has limitations. It does not introduce novel statistical tests or models, nor does it add additional analytical approaches for AMR research. The calculation of the proportion of susceptibility for more than one antimicrobial agent simultaneously (see Section 4.5.1) seems simple but is subject to unclear reporting in clinical practice \textsuperscript{{[}42,43{]}}. The lack of clearly defined algorithms can lead to the effect that co-resistance rates for more than one antimicrobial agent are dropped altogether \textsuperscript{{[}44{]}}. The inclusion of isolates that are tested for some agents (only\_all\_tested = FALSE) or only isolates tested for all agents (only\_all\_tested = TRUE) can have an imminent clinical impact on patient care, if one combination of antimicrobial agents is preferred over another. Therefore, the \texttt{AMR} package provides different algorithms to standardise this crucial calculation. Unfortunately, unambiguous methodology for determining the right algorithm is lacking in scientific literature. An analysis on the algorithms used in the \texttt{AMR} package and their clinical impact is in preparation.

Reliable information about antimicrobial resistance is vital for clinical decision-making in infectious diseases, since the outcome of local antimicrobial resistance analyses support medical professionals/clinicians in the treatment choices for their patients. Moreover, when this information can be reliably stratified by, for example, year, hospital, and type of patients, new information can lead to new insights for choosing the best antimicrobial therapy for patients suffering from infections. The \texttt{AMR} package enables this by providing all required analysis tools and can therefore empower decision-making in infectious management. The \texttt{AMR} package is already being applied to this end in six hospitals in the Netherlands. The choice of empirical antimicrobial treatment (meaning; choosing the initial therapy at a time of not knowing the infection-causing pathogen) for septic non-post-surgical patients has been altered in at least one Dutch hospital, by analysing antimicrobial resistance data with the \texttt{AMR} package. The clinical effect of this adjustment is being studied at the moment. To improve the quality of such analyses, planned future developments comprise the implementations of an imputation algorithm specifically for antimicrobial agents, and method guidance for applying prediction modelling in a health care setting based on patient-specific properties.

Since the first package release, users from different public and private settings have been suggesting additional functionalities, in particular, the incorporation of country- or time- specific guidelines (e.g., Magiorakos \emph{et al.} \textsuperscript{{[}31{]}}). This community-centred development will be continued and maintained by researchers at the University Medical Center Groningen and data scientists at Certe Medical Diagnostics and Advice, both non-profit public health organisations located in Groningen, the Netherlands. Moreover, a group of contributors from five different Dutch health care institutions has been formed at the Dutch Association for Medical Microbiology (Nederlandse Vereniging voor Medische Microbiologie - NVMM) that also peer-review major changes to the package, including the implementation of guideline updates. This way, updates required for scientific developments as well as maintaining consistent reproducibility are ensured. Updates to databases and guidelines included in the \texttt{AMR} package are incorporated on a regular and automated basis, while preserving version control. Any function making use of guidelines (e.g., \texttt{eucast\_rules()}) refers to the latest implemented version of the guideline by default.

The aim of the \texttt{AMR} package is to provide a comprehensive toolbox of solutions for antimicrobial resistance data processing and analysis on an institution- and country-independent scale for clinical practice and research that are required according to international standards, but were not available to date.

\hypertarget{computational-details}{%
\section*{Computational Details}\label{computational-details}}
\addcontentsline{toc}{section}{Computational Details}

The results in this paper were obtained using \texttt{R}4.0.2 in RStudio 1.3.1093 \textsuperscript{{[}45{]}} with the \texttt{AMR} package 1.5.0, running under macOS Catalina 10.15.

\texttt{R}itself and all packages used are available from the Comprehensive \texttt{R}Archive Network (CRAN) at \url{https://CRAN.R-project.org/}. All development versions of the \texttt{AMR} package are available at \url{https://github.com/msberends/AMR/}.

\hypertarget{acknowledgements}{%
\section*{Acknowledgements}\label{acknowledgements}}
\addcontentsline{toc}{section}{Acknowledgements}

The authors Matthijs S. Berends and Christian F. Luz contributed equally to this publication.

For their contributions to the development of the \texttt{AMR} package, we would like to thank (in alphabetical order) Judith M. Fonville, Erwin E.A. Hassing, Eric H.L.C.M. Hazenberg, Gwen Knight, Annick Lenglet, Bart C. Meijer, Sofia Ny, Rogier P. Schade, Dennis Souverein, and Anthony Underwood.

The development of the \texttt{AMR} package was partly supported by the INTERREG V A (202085) funded project EurHealth-1Health (\url{http://www.eurhealth1health.eu}), part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport, the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the Ministry for National and European Affairs and Regional Development of Lower Saxony.

Furthermore, the \texttt{AMR} package was developed as part of a project funded by the European Commission Horizon 2020 Framework Marie Skłodowska-Curie Actions (grant agreement number: 713660-PRONKJEWAIL-H2020-MSCA-COFUND-2015).

The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

\hypertarget{references-3}{%
\section*{References}\label{references-3}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  O'Neill J. Review on antimicrobial resistance: tackling a crisis for the health and wealth of nations. London: Wellcome Trust; 2014.
\item
  De Kraker MEA, Stewardson AJ, Harbarth S. Will 10 million people die a year due to antimicrobial resistance by 2050? PLoS Med 2016;13:e1002184.
\item
  Centers for Disease Control and Prevention. AR threats report: antibiotic resistance threats in the United States, 2019. Atlanta: Centers for Disease Control and Prevention; 2019.
\item
  The European Committee on Antimicrobial Susceptibility Testing. New definitions of S, I and R from 2019; 2019. \url{https://www.eucast.org/newsiandr/}.
\item
  Leclercq R, Cantón R, Brown DFJ, Giske CG, Heisig P, MacGowan AP, \emph{et al.} EUCAST Expert Rules In Antimicrobial Susceptibility Testing. Clin Microbiol Infect 2013;19:141--160.
\item
  Clinical and Laboratory Standards Institute. M39-A4, analysis and presentation of cumulative antimicrobial susceptibility test data, 4th edition. Pittsburgh: Clinical and Laboratory Standards Institute; 2014.
\item
  World Health Organization. Global antimicrobial resistance surveillance system (GLASS) report: early implementation 2017-2018. Geneva: World Health Organization; 2018.
\item
  Tacconelli E, Sifakis F, Harbarth S, Schrijver R, van Mourik M, Voss A, \emph{et al.} Surveillance for control of antimicrobial resistance. Lancet Infect Dis 2018;18:e99--e106.
\item
  Limmathurotsakul D, Dunachie S, Fukuda K, Feasey NA, Okeke IN, Holmes AH, \emph{et al.} Improving the estimation of the global burden of antimicrobial resistant infections. Lancet Infect Dis 2019;19:e392--8.
\item
  R Core Team. R: A language and environment for statistical computing. Vienna: R Foundation for Statistical Computing; 2019.
\item
  Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR: Antimicrobial resistance analysis. R package version 1.5.0; 2020. \url{https://CRAN}. R-project.org/package=AMR.
\item
  Lowndes JSS, Best BD, Scarborough C, Afflerbach JC, Frazier MR, O'Hara CC, \emph{et al.} Our path to better science in less time using open data science tools. Nat Ecol Evol 2017;1:160.
\item
  Jombart T, Kamvar ZN, FitzJohn R, Cai J, Bhatia S, Schumacher J, \emph{et al.} incidence: compute, handle, plot and model incidence of dated events. R package version 1.7.3; 2020. \url{https://doi.org/10.5281/zenodo.2584018}.
\item
  Nagraj V, Jombart T, Randhawa N, Sudre B, Campbell F, Crellen T. epicontacts: handling, visualisation and analysis of epidemiological contacts. R package version 1.1.0 2017. \url{https://CRAN.R-project.org/package=epicontacts}.
\item
  Petzoldt T. antbioticR: Analysis of Antibiotic Resistance Data. R package version 0.3.2; 2019. \url{https://github.com/tpetzoldt/antibioticR}.
\item
  Roskov Y, Ower G, Orrell T, Nicolson D, Bailly N, Kirk PM, \emph{et al.} Species 2000 \& ITIS catalogue of life; 2019. www.catalogueoflife.org/col.
\item
  Levenshtein VI.Binary codes capable of correcting deletions, insertions, and reversals. Soviet physics doklady 1966;10:707--710.
\item
  De Greeff SC, Mouton JW, Schoffelen AF, Verduin CM (2019). ``NethMap 2019: consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in the Netherlands in 2019/ MARAN 2019: monitoring of antimicrobial resistance and antibiotic usage in animals in the Netherlands in 2019. Bilthoven: Rijksinstituut voor Volksgezondheid en Milieu; 2020.
\item
  European Centre for Disease Prevention and Control. European antimicrobial resistance surveillance network (EARS-Net); 2010. \url{https://ecdc.europa.eu/en/about-us/partnerships-and-networks/disease-and-laboratory-networks/ears-net}.
\item
  Cavalier-Smith T. The neomuran origin of archaebacteria, the negibacterial root of the universal tree and bacterial megaclassification. Int J Syst Evol Microbiol 2002;52:7--76.
\item
  WHO Collaborating Centre for Drug Statistics Methodology. Guidelines for ATC classification and DDD assignment 2019. Oslo: WHO Collaborating Centre for Drug Statistics Methodology; 2019.
\item
  European Centre for Disease Prevention and Control. Antimicrobial resistance (AMR) reporting protocol 2018. European antimicrobial resistance surveillance network (EARS-Net) surveillance data for 2017. Stockholm: European Centre for Disease Prevention and Control; 2018.
\item
  Kim S, Chen J, Cheng T, Gindulyte A, He J, He S, \emph{et al.} PubChem 2019 update: improved access to chemical data. Nucleic Acids Res 2019;47:D1102--9.
\item
  Clinical and Laboratory Standards Institute. Susceptibility testing of infectious agents and evaluation of performance of antimicrobial susceptibility test devices -- part 1, 2nd edition. Pittsburgh: Clinical and Laboratory Standards Institute; 2019
\item
  The European Committee on Antimicrobial Susceptibility Testing. breakpoint tables for interpretation of mics and zone diameters, version 10.0, 2020; 2020. \url{http://www.eucast.org/clinical_breakpoints/}.
\item
  Winstanley T, Courvalin P. Expert systems in clinical microbiology. Clin Microbiol Rev 2011;24:515--556.
\item
  Courvalin P. Interpretive reading of antimicrobial susceptibility tests. Molecular analysis and therapeutic interpretation of in vitro tests to improve antibiotic therapy. ASM American Society for Microbiology News 1992;58:368--75.
\item
  Courvalin P. Interpretive reading of in vitro antibiotic susceptibility tests (the antibiogramme). Clin Microbiol Infect 1996;2 Suppl 1:S26--34.
\item
  Livermore DM, Winstanley TG, Shannon KP. Interpretative reading: recognizing the unusual and inferring resistance mechanisms from resistance phenotypes. J Antimicrob Chemother 2001;48 Suppl 1:87--102.
\item
  WHO Collaborating Center for Drug Statistics Methodology. ATC/DDD index; 2019 \url{https://www.whocc.no/atc_ddd_index/}
\item
  Magiorakos A-P, Srinivasan A, Carey RB, Carmeli Y, Falagas ME, Giske CG, \emph{et al.} Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: an international expert proposal for interim standard definitions for acquired resistance. Clin Microbiol Infect 2012;18:268--81.
\item
  Abat C, Fournier P-E, Jimeno M-T, Rolain J-M, \emph{et al.} Extremely and pandrug-resistant bacteria extra-deaths: myth or reality? Eur J Clin Microbiol Infect Dis 2018;37:1687--97.
\item
  European Committee on Antimicrobial Susceptibility Testing (2016). EUCAST expert rules, intrinsic resistance and exceptional phenotypes tables. Version 3.1, 2016; 2016. \url{http://www.eucast.org/fileadmin/src/media/PDFs/EUCAST_files/} Expert\_Rules/Expert\_rules\_intrinsic\_exceptional\_V3.1.pdf.
\item
  World Health Organization. Companion handbook to the WHO guidelines for the programmatic management of drug-resistant tuberculosis. Geneva: World Health Organization; 2014.
\item
  Werkgroep Infectiepreventie. Bijzonder resistente micro-organismen (BRMO). Leiden: Werkgroep Infectiepreventie; 2011.
\item
  Müller J, Voss A, Köck R, Sinha B, Rossen JW, Kaase M, \emph{et al.} Cross-border comparison of the Dutch and German guidelines on multidrug-resistant Gram-negative microorganisms. Antimicrob Resist Infect Control 2015;4:7.
\item
  World Health Organization. Global Tuberculosis Report 2018. Geneva: World Health Organization; 2018.
\item
  Wickham H. Tidy data. J Stat Softw 2014;59:1--23.
\item
  Wickham H, François R, Henry L, Müller K. dplyr: a grammar of data manipulation. R package version 1.0.2; 2020.
\item
  Wickham H, Averick M, Bryan J, Chang W, McGowan L, François R, \emph{et al.} Welcome to the tidyverse. J Open Source Softw 2019;4:1686.
\item
  Wickham H. ggplot2: elegant graphics for data analysis. New York: Springer; 2009.
\item
  Schechner V, Temkin E, Harbarth S, Carmeli Y, Schwaber MJ. Epidemiological interpretation of studies examining the effect of antibiotic usage on resistance. Clin Microbiol Rev 2013;26:289--307.
\item
  Ma J, Li N, Liu Y, Wang C, Liu X, Chen S, \emph{et al.} Antimicrobial resistance patterns, clinical features, and risk factors for septic shock and death of nosocomial E. coli bacteremia in adult patients with hematological disease. Medicine 2017;96:e6959.
\item
  Baur D, Gladstone BP, Burkert F, Carrara E, Foschi F, Döbele S, \emph{et al.} Effect of antibiotic stewardship on the incidence of infection and colonisation with antibiotic-resistant bacteria and Clostridium difficile infection: a systematic review and meta-analysis. Lancet Infect Dis 2017;17:990--1001.
\item
  RStudio Team. RStudio: integrated development environment for R; 2020. \url{http://www.rstudio.com/}.
\item
  WHO Collaborating Centre for Surveillance of Antimicrobial Resistance. WHONET; 2019. \url{http://www.whonet.org/}
\end{enumerate}

\hypertarget{appendix-a-included-data-sets}{%
\section*{Appendix A: Included Data Sets}\label{appendix-a-included-data-sets}}
\addcontentsline{toc}{section}{Appendix A: Included Data Sets}

\begin{itemize}
\item
  \texttt{microorganisms}

  A \texttt{‘data.frame’} containing 67,151 (sub)species with 16 columns comprising their complete microbial taxonomy according to the Catalogue of Life \textsuperscript{{[}16{]}}. Included microorganisms and their complete taxonomic tree of all included (sub)species from kingdom to subspecies with year of scientific publication and responsible author(s):

  \begin{itemize}
  \tightlist
  \item
    All 55,415 (sub)species from the kingdoms of Archaea, Bacteria, Chromista and Protozoa;
  \item
    All 9,582 (sub)species from these orders of the kingdom of Fungi: Eurotiales, Onygenales, Pneumocystales, Saccharomycetales, Schizosaccharomycetales and Tremellales;
  \item
    All 2,153 (sub)species from 47 other relevant genera from the kingdom of Animalia (like Strongyloides and Taenia);
  \item
    All 12,708 previously accepted names of included (sub)species that have been taxonomically renamed.
  \end{itemize}

  The kingdom of Fungi is a very large taxon with almost 300,000 different (sub)species, of which most are not microbial (but rather macroscopic, such as mushrooms). Therefore, not all fungi fit the scope of the \texttt{AMR} package. By only including the aforementioned taxonomic orders, the most relevant fungi are covered (such as all species of Aspergillus, Candida, Cryptococcus, Histoplasma, Pneumocystis, Saccharomyces and Trichophyton).
\item
  \texttt{antibiotics}

  A \texttt{‘data.frame’} containing 456 antibiotic agents with 14 columns. All entries in this data set have three different identifiers: a human readable EARS-Net code (as used by ECDC \textsuperscript{{[}19{]}} and WHONET \textsuperscript{{[}46{]}} and primarily used by this package), an ATC code (as used by the WHO \textsuperscript{{[}21{]}}), and a CID code (Compound ID, as used by PubChem \textsuperscript{{[}23{]}}). The data set contains more than 5,000 official brand names from many different countries, as found in PubChem. Other properties in this data set are derived from one or more of these codes, such as official names of pharmacological and chemical subgroups, and defined daily doses (DDD).
\item
  \texttt{antivirals}

  A \texttt{‘data.frame’} containing 102 antiviral agents with 9 columns. Like the \texttt{antibiotics} data set, it contains ATC codes (as used by the WHO \textsuperscript{{[}21{]}}), and a CID code (Compound ID, as used by PubChem \textsuperscript{{[}23{]}}), as well as the official name and defined daily dose (DDD) for each antiviral agent.
\item
  \texttt{example\_isolates}

  A \texttt{‘data.frame’} containing test results of 2,000 microbial isolates. The data set reflects real patient data and can be used to practice AMR analysis. It is structured in the typical format of laboratory information systems with one row per isolate and one column per tested antimicrobial agent (i.e., an antibiogram).
\item
  \texttt{example\_isolates\_unclean}

  A \texttt{‘data.frame’} containing test results of 3,000 microbial isolates that require cleaning up before they can be used for analysis. This data set can be used to practice AMR analysis and is featured in section 4.7.
\item
  \texttt{WHONET}

  A `\texttt{‘data.frame’} containing 500 observations and 53 columns, with the exact same structure as an export file from WHONET 2019 software \textsuperscript{{[}46{]}}. Such files can be used with the \texttt{AMR} package, as this example data set demonstrates. The antibiotic test results are from the \texttt{example\_isolates} data set. All patient names are created using online surname generators and are only in place for practice purposes.
\end{itemize}

\hypertarget{ch05-radar}{%
\chapter{Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship}\label{ch05-radar}}

Published in Journal of Medical Internet Research, 2019 (21);6, e12843

Luz CF \textsuperscript{1}, Berends MS \textsuperscript{1,2}, Dik JWH \textsuperscript{1}, Lokate ML \textsuperscript{1}, Pulcini C \textsuperscript{3,4}, Glasner C \textsuperscript{1}, Sinha BNM \textsuperscript{1}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, The Netherlands
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, The Netherlands
\item
  Université de Lorraine, APEMAC, Nancy, France
\item
  Université de Lorraine, CHRU-Nancy, Infectious Diseases Department, Nancy, France
\end{enumerate}

\hypertarget{abstract-3}{%
\section*{Abstract}\label{abstract-3}}
\addcontentsline{toc}{section}{Abstract}

Analysing process and outcome measures for all patients diagnosed with an infection in a hospital, including those suspected of having an infection, requires not only processing of large datasets but also accounting for numerous patient parameters and guidelines. Substantial technical expertise is required to conduct such rapid, reproducible, and adaptable analyses; however, such analyses can yield valuable insights for infection management and antimicrobial stewardship (AMS) teams. The aim of this study was to present the design, development, and testing of RadaR (Rapid analysis of diagnostic and antimicrobial patterns in R), a software app for infection management, and to ascertain whether RadaR can facilitate user-friendly, intuitive, and interactive analyses of large datasets in the absence of prior in-depth software or programming knowledge. RadaR was built in the open-source programming language R, using Shiny, an additional package to implement web-app frameworks in R. It was developed in the context of a 1339-bed academic tertiary referral hospital to handle data of more than 180,000 admissions. RadaR enabled visualisation of analytical graphs and statistical summaries in a rapid and interactive manner. It allowed users to filter patient groups by 17 different criteria and investigate antimicrobial use, microbiological diagnostic use and results including antimicrobial resistance, and outcome in length of stay. Furthermore, with RadaR, results can be stratified and grouped to compare defined patient groups on the basis of individual patient features. AMS teams can use RadaR to identify areas within their institutions that might benefit from increased support and targeted interventions. It can be used for the assessment of diagnostic and therapeutic procedures and for visualizing and communicating analyses. RadaR demonstrated the feasibility of developing software tools for use in infection management and for AMS teams in an open-source approach, thus making it free to use and adaptable to different settings.

\hypertarget{introduction-2}{%
\section{Introduction}\label{introduction-2}}

\hypertarget{background-1}{%
\subsection{Background}\label{background-1}}

With antimicrobial resistance (AMR) on the rise, efforts are being made worldwide to focus on the preservation of antimicrobials as a precious non-renewable resource. Infection management in the form of antimicrobial stewardship (AMS) programs has emerged as an effective solution to address this global health problem in hospitals. AMS programs are defined as ``a coherent set of actions which promote using antimicrobials responsibly'' \textsuperscript{{[}1{]}}. Stewardship interventions and activities focus on individual patients (personalised medicine and consulting) as well as patient groups or clinical syndromes (guidelines, protocols, information technology infrastructure, and clinical decision support systems) while prioritizing improvement in quality of care and patient safety for any intervention. The appropriate use of antimicrobials based on accurate and timely diagnostics is integral for the successful management of infections. In doing so, the diagnostics contribute to efforts in minimizing AMR by optimizing the use of antimicrobials.

AMS setups in hospitals are often heterogeneous, but audit and feedback to assess the goals are essential parts of most programs, and they are included in international guidelines and reviews \textsuperscript{{[}2-7{]}}. Important data for AMS programs include, for example, days of therapy (DOT), daily defined doses (DDD), admission dates, length of stay (LOS), and adherence to local or national diagnostic, therapeutic, or infection management guidelines \textsuperscript{{[}1{]}}. Clinical outcomes, quality of care, or consumption of hospital resources can be measured, for example, using mortality data or surrogate parameters such as LOS. The collection of these data is facilitated by electronic health records (EHRs) and administrative local databases. Notably, administrative data have also been shown to be a reliable source for assessing clinical outcomes \textsuperscript{{[}8{]}}.

EHRs usually offer quick insights into useful infection management data on the individual patient level. However, easy access to analyse patient groups (e.g., stratified by departments or wards, specific antimicrobials, or diagnostic procedures used) is difficult to implement in daily practice. It is even more challenging to rapidly analyse larger patient populations (e.g., spread over multiple specialties) even though this information might be available. Nevertheless, this is vital for meaningful analysis, including possible confounders and pattern recognition across different populations. Moreover, when aggregated data are available, it is often not possible to trace individual patients, and analyses lack the ability to be further adjusted or stratified.

AMS teams are multidisciplinary, and they act beyond the borders of single specialties \textsuperscript{{[}9{]}}. They are usually understaffed, with limited data analysis support \textsuperscript{{[}10,11{]}}. Therefore, they need user-friendly and time-saving data analysis resources, without the need for profound technical expertise once the system is set up. Aggregating and linking data of antimicrobial use, guideline adherence, and clinical outcomes at the institutional level can build the basis for important insights for these teams. These could be used to identify areas within hospitals that might benefit most from supportive AMS interventions (e.g., subspecialties with lower guideline adherence or unusual patterns of antimicrobial use). Moreover, feedback from these data could help physicians better understand their patient population as a whole; in addition, hospital administration could allocate resources in a more targeted fashion.

Furthermore, aggregated data and simultaneous analysis of multiple areas (e.g., use of diagnostics and antimicrobials) present an extensive insight into large patient populations. This also enables the development of comprehensive and multidisciplinary approaches of infection management, combining diagnostic and therapeutic perspectives \textsuperscript{{[}1,9,12{]}}. Unfortunately, these kinds of analyses still require substantial statistical knowledge and software skills, and it is time consuming when performed.

Technology, data science, and software app development can bring solutions to complex data handling problems such as those described above. Software app development for medical and epidemiological (research) questions has found many important answers during recent years. For example, software apps at hospital emergency departments (EDs) in the form of a dashboard have been shown to improve efficiency and quality of care for patients requiring emergency admission to hospital \textsuperscript{{[}13{]}}. These software apps are used to communicate clearly defined clinical problems, such as mortality ratio, number of cardiac arrests, or readmission rate to the EDs. This has led to a decreased LOS and mortality at the EDs. Others used similar approaches to rapidly and interactively display geographical locations of tuberculosis cases without the need of technical expertise improving the understanding of transmission and detection \textsuperscript{{[}14{]}}. Furthermore, data-driven fields such as genomics are front runners in developing new, innovative software apps to handle large datasets, in close collaboration with bioinformatics \textsuperscript{{[}15{]}}.

It is important to note that all of these abovementioned software apps have been created in an open-source approach. This means that the underlying source code can be easily shared, easily modified, and freely distributed through open repositories, such as GitHub \textsuperscript{{[}16{]}}, taking open-source software license obligations into account. This facilitates collaboration, quality control through code review, and easy adaptation to many different settings and information technology systems, and this supports the use of advanced data visualisation for users with minimal experience in programming and little or no budget for professional database engineers \textsuperscript{{[}15{]}}.

In the field of medical microbiology, different approaches have been described to interactively work with microbiological diagnostics data and EHRs: electronic antibiograms, centralised resistance analysis, EHR data mining, and clinical decision support systems for AMS are great examples for innovation in the field \textsuperscript{{[}17-19{]}}. However, a full open-source approach for software apps working with combined antimicrobials use and diagnostic data of individual patients on the hospital level in the field of infection management is still lacking.

\hypertarget{objectives}{%
\subsection{Objectives}\label{objectives}}

We followed principles of open knowledge \textsuperscript{{[}20{]}} to address the need for an interactive, easy-to-use software app that allows users to investigate antimicrobial use, microbiological diagnostic use, and patient outcomes at an institutional (hospital) level. We developed an open-source, web-based software app -- Rapid analysis of diagnostic and antimicrobial patterns in \texttt{R} (RadaR) that can be used for AMS and infection management. This free software app can be run on regular computers or implemented on local or web-based servers to be accessed through standard web browsers. The focus user group of this software app is health care professionals involved in AMS (e.g., infectious disease specialists, clinical microbiologists, and pharmacists). Although some technical expertise (basic \texttt{R} knowledge) is needed for installation and implementation, the use of RadaR follows usual web browser user experiences. RadaR enables rapid and reproducible data analysis without extensive previous analysis expertise in a graphically appealing way while being adaptable to different settings. RadaR's analyses are based on datasets of individual patients. Therefore, aggregated results can also be stripped down, and additional patient features can be investigated. With this software app, we aim at supporting data-driven hospital insights and decision making for actors in the field of AMS in a free, transparent, and reproducible way.

\hypertarget{methods}{%
\section{Methods}\label{methods}}

For the development of software in an open-source environment, we used the open-source programming language \texttt{R} in conjunction with RStudio version 1.1.463 (RStudio, Inc) \textsuperscript{{[}21{]}}, an open-source integrated desktop environment for \texttt{R} \textsuperscript{{[}22{]}}. Both \texttt{R} and RStudio are free of charge, and they need to be installed for the development and implementation of RadaR. To build RadaR as a web-based software app, we used the Shiny package for \texttt{R} \textsuperscript{{[}23{]}}. Shiny allows \texttt{R} users to build interactive web apps without extensive knowledge in web design and its programming languages. The web apps can be run and hosted on the web for free \textsuperscript{{[}24{]}}, as well as on local or cloud-based servers or on personal computers.

The functionality of \texttt{R} can be easily extended by installing additional packages. All packages used for the development of RadaR are listed in Table 1. RadaR is developed in an open-source environment and licensed under GNU General Public License v2.0 \textsuperscript{{[}25{]}}, giving options to change, modify, and adapt RadaR to both personal and commercial users' needs while requiring the need to document code changes \textsuperscript{{[}25{]}}.

RadaR's calculations and data aggregation are done reactively on the basis of the selection of the user. Single observations on the patient level build the basis for any calculation. RadaR uses common CSV files as input. A total of three different data sources are read in RadaR for admission, antimicrobial, and microbiological data, which are merged and transformed upon start. A patient number or study number is used as a unique identifier. All antimicrobial and microbiological data are checked to ascertain whether they fall in the interval of admission dates.

Table 1. Required \texttt{R} packages for RadaR.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/05-t01} 

}

\end{figure}

The input data should be structured in a dataset format, where each variable is one column and each observation is one row. This follows the concept of ``tidy data,'' as defined by Hadley Wickham \textsuperscript{{[}26{]}}. Table 2 displays the set of variables underlying RadaR's functionality. In our setting for the development of RadaR, these variables originated from three different data sources: administrative data from the hospital data warehouse, microbiological data from the laboratory information system, and antimicrobial prescription data from the computerised prescriber order entry system. The data preparation and cleaning process are very specific for each data source, dependent on local data standards, and difficult to generalise. Therefore, Table 2 represents the final variables and formats for the analysis and use with RadaR, referring to the ``tidy data'' concept above and to the \texttt{R} package collection tidyverse for the preparation process \textsuperscript{{[}26,27{]}}. Additional variables are calculated and transformed using the packages lubridate and zoo for time points and intervals, and AMR for antimicrobial (group) names, microbial isolate names, first isolate identification, and resistance analysis \textsuperscript{{[}28-30{]}}. Microbiological resistance is calculated per antimicrobial substance or as co-resistance if more than one substance is selected.

Table 2. Input variables for RadaR.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/05-t02} 

}

\end{figure}

The input data should be structured in a dataset format, where each variable is one column and each observation is one row. This follows the concept of ``tidy data,'' as defined by Hadley Wickham \textsuperscript{{[}26{]}}. Table 2 displays the set of variables underlying RadaR's functionality. In our setting for the development of RadaR, these variables originated from three different data sources: administrative data from the hospital data warehouse, microbiological data from the laboratory information system, and antimicrobial prescription data from the computerised prescriber order entry system. The data preparation and cleaning process are very specific for each data source, dependent on local data standards, and difficult to generalise. Therefore, Table 2 represents the final variables and formats for the analysis and use with RadaR, referring to the ``tidy data'' concept above and to the \texttt{R} package collection tidyverse for the preparation process \textsuperscript{{[}26,27{]}}. Additional variables are calculated and transformed using the packages lubridate and zoo for time points and intervals, and AMR for antimicrobial (group) names, microbial isolate names, first isolate identification, and resistance analysis \textsuperscript{{[}28-30{]}}. Microbiological resistance is calculated per antimicrobial substance or as co-resistance if more than one substance is selected.

RadaR can be used for graphical exploratory data analysis. Differences in LOS are displayed by a Kaplan-Meier curve in conjunction with a log-rank test, using the survminer package \textsuperscript{{[}32{]}}. Time trends for number of admissions, antimicrobial consumption, and resistance counts per year, quarter, or month, are visualised in run charts using the qicharts2 package \textsuperscript{{[}33{]}}. Nonrandom variation in these run charts is tested using Anhøj's rules \textsuperscript{{[}34{]}}.

RadaR has been developed in macOS High Sierra (1.4 GHz, 4 GB RAM), and it was successfully tested in Windows 7 (3.2 GHz, 8 GB RAM) and Linux (Ubuntu 16.04.4 LTS, 3.4 GHz, 12 GB RAM). A running example version has been deployed to shinyapps.io, a publicly available web hosting service for \texttt{R} Shiny apps \textsuperscript{{[}35{]}}. The entire source code of RadaR is freely accessible on GitHub \textsuperscript{{[}36{]}}. We intend to integrate suggestions and feedback coming from its users and the \texttt{R} community. RadaR was developed using data of patients admitted to the University Medical Center Groningen, Groningen, the Netherlands. Data were collected retrospectively, and permission was granted by the ethical committee (METc 2014/530). RadaR can be used locally in protected environments or hosted on the web, provided appropriate measures have been taken to guarantee data protection, depending on national regulations.

\hypertarget{results}{%
\section{Results}\label{results}}

\hypertarget{overview}{%
\subsection{Overview}\label{overview}}

We have developed RadaR, a web-based software app providing an intuitive platform for rapid analysis of large datasets containing information about patients' admission, antimicrobial use, and results of microbiological diagnostic tests. This software app can help users (i.e., AMS team members) find answers to questions, such as ``What are the most commonly used antimicrobials at an institution/specialty/department and have they changed over time?'', ``Were adequate microbiological diagnostics performed at the start of antimicrobial treatments?'', ``What are the most frequent microorganisms found and their resistance patterns in different departments?'', and ``Can we identify priority areas within a hospital where antimicrobial or microbiological diagnostic use has the largest room for improvement?''.

\hypertarget{application-design}{%
\subsection{Application Design}\label{application-design}}

RadaR is designed in the form of a web browser--based dashboard that most users are familiar with from typical websites and web-based tools (see Figure 1). The basis of RadaR's functionality is filtering datasets and producing analytical graphs according to selection criteria defined by the user. Any calculations and data aggregation are based on single observations of individual patients. To identify and analyse groups of patients, 17 different selection criteria can be found in the sidebar (Table 3). The output of RadaR is grouped into four panels (patient, antimicrobials, diagnostics, and outcome) that each comprise three to four output boxes displaying the results.

Table 3. Selection criteria in sidebar.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/05-t03} 

}

\end{figure}

All output is based on the selection criteria defined by the user in the sidebar. Each new selection and any change need to be confirmed by clicking the confirm selection button (see Figure 1). Users can navigate among the different analysis panels by clicking the respective button.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/05-01} 

}

\caption{Application design.}\label{fig:fig5-1}
\end{figure}

Results are shown in bar charts, density plots, run charts, a bubble plot, and a Kaplan-Meier curve for LOS in hospital. Each panel further displays a table summarizing the respective data analyses. All output boxes and their content are described in Table 4. Most output boxes include modification options that can be identified by small gear icons (see Figure 1). These clickable icons allow for further specification of the generated plots and tables. Users can compare different groups (e.g., antimicrobial use by antimicrobial agent, resistance patterns per isolate, or LOS by specialty) or modify the plots (e.g., switch from count to proportion, change the chart type, or show or hide the legend). Plots and tables can be downloaded through download buttons as PNG files for plots and CSV, Excel, or PDF files for tables.

Table 4. Output boxes for analysis results.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/05-t04} 

}

\end{figure}

Finally, two datasets (antimicrobial/admission data and microbiological data) of the user-defined selection can be downloaded from the sidebar menu in a CSV-file format for further analysis (e.g., retrieving a list of patient numbers of the selected patient group).

\hypertarget{development-process}{%
\subsection{Development Process}\label{development-process}}

RadaR has been developed in close contact with the AMS team and senior consulting specialists at the University Medical Center Groningen, Groningen, the Netherlands, to meet the needs and requirements of this user group. Subsequently, all members of the European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship (ESGAP) were asked to evaluate and test the software app through a running web-based example of RadaR and by filling out a web-based survey. The ESGAP comprises around 200 members from more than 30 countries worldwide. A total of 12 members from 9 different countries took part in the evaluation. This yielded important information on user experiences with the software app, which in turn led to further improvements that are reflected in the version we presented in this report. In a next phase, RadaR will be tested in different settings of ESGAP members and other interested partners using locally available data (e.g., an 837-bed tertiary care hospital in the Netherlands and a 750-bed tertiary care hospital in Greece).

\hypertarget{workflow}{%
\subsection{Workflow}\label{workflow}}

RadaR was developed and tested with a dataset of all patients admitted to our institution, a 1339-bed academic tertiary referral hospital, within the years of 2009 to 2016, comprising over 180,000 admissions. For simulation purposes and web-based user testing, we have created a test dataset of 60,000 simulated patients. This sample dataset allows testing of RadaR's functionality, but it does not produce meaningful results.

A typical example workflow with RadaR comprises 6 steps (with examples from the test dataset). They are listed below:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Define the selection: For example, patients receiving intravenous second- or third-generation cephalosporins as first treatment for at least two days, starting within the first two days of hospital admission from any specialty in all years in the dataset.
\item
  Patients' panel: Identify the total number of patients and the subspecialties with the highest number of included patients (e.g., 537 patients selected in total, with 97 patients from internal medicine). Investigate patients' gender and age distribution.
\item
  Antimicrobials panel: Identify the total use of the initial cefuroxime treatment in DDD and DOT per 100 bed days (e.g., 4.51 and 1.5, respectively). Stratify the results by subspecialty and identify the highest number of DDD and DOT per 100 bed days (e.g., highest use by DDD and DOT in internal medicine).
\item
  Diagnostics panel: Check if the selected microbiological diagnostic test (e.g., blood culture test) has been performed on the same day as the start of the treatment (defined in the sidebar). Investigate the proportion of tests performed over the years and investigate which subspecialty performs best compared with others (e.g., paediatrics). Check which microorganisms (as first isolates) were found in the selected diagnostic specimens (the most common isolate: Escherichia coli). Investigate the proportion of isolates resistant to cefuroxime (8.9\%) and analyse the trend over time.
\item
  Outcome panel: Check for patterns of differences in LOS in the defined patient group by subspecialties or performed diagnostics (e.g., highest mean LOS of 7.8 days in Surgery).
\item
  Refine the selection: Investigate a subgroup of the original selection. For example, select only the top three subspecialties by number of patients and repeat step 2 to 5.
\end{enumerate}

\hypertarget{customisation}{%
\subsection{Customisation}\label{customisation}}

For setting up RadaR in a new environment after data preparation, users only need to perform the following four steps:

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Downloading \texttt{R} and RStudio \textsuperscript{{[}21,22{]}}, which are free to use and open-source software
\item
  Download or copy and paste RadaR's source code \textsuperscript{{[}36{]}} into three files in RStudio -- \texttt{global.R}, \texttt{server.R}, and \texttt{ui.R}
\item
  In \texttt{global.R}, manually edit the paths for the prepared datasets to be imported into RadaR
\item
  Run the app in RStudio with the calling the function \texttt{runApp()} in the console or by clicking the green run app button. This will download and install the required \texttt{R} packages needed for the app if they have not been installed previously, and this will create the final dataset for analysis. The RadaR interface will open in the RStudio viewer pane or in a new window of the standard browser of the user's operating system.
\end{enumerate}

RadaR's appearance has been customised using a cascading style sheets (CSS) script \textsuperscript{{[}37{]}} that is loaded into the app upon its start. This script needs to be saved into a subdirectory of the directory of the three main files (\texttt{global.R}, \texttt{server.R}, and \texttt{ui.R}) called \texttt{www}. We recommend RStudio's project function to create a single project for RadaR and to store all information in this project directory. Users with experience in using CSS can fully alter RadaR's design by changing the underlying CSS script.

\hypertarget{discussion-1}{%
\section{Discussion}\label{discussion-1}}

\hypertarget{principal-findings}{%
\subsection{Principal Findings}\label{principal-findings}}

We have developed a web-based software app for rapid analysis of diagnostic and antimicrobial patterns that can support AMS teams to tailor their interventions. It has been designed to enhance communication of relevant findings while being easy to use. This also applies to users without extensive prior software skills, as it follows usual web browser user experiences. Moreover, it has been developed using open-source software. It is therefore free to use and accessible for download. In our experience, this system can be adapted to new settings within one day, when the required data (Table 2) are available.

Commercial software for infection management is available (e.g., Epic Antimicrobial Stewardship Module, TREAT Steward). These offer extensive options for filtering, analysing, and visualizing EHRs with real-time connections to hospital data infrastructures and have been shown to be useful in clinical practice \textsuperscript{{[}38{]}}. However, it is difficult to compare functionalities of these tools because of their non--open-source nature. This fact, along with the required budget to purchase the software, drastically limits their use. We are convinced that transparent software development can support the adoption of data-driven developments while enhancing optimal quality of care and patient safety, which is crucial in the light of new data-driven developments of using EHRs \textsuperscript{{[}39,40{]}}.

The global nature of infections further calls to develop software tools applicable in resource-limited settings \textsuperscript{{[}41{]}}. Open-source approaches for data analysis, such as RadaR, have advantages over traditional methods, such as Excel or SPSS. Hughes et al described those in their report of a software app for RNA-sequencing data analysis \textsuperscript{{[}15{]}}. They highlight aspects that were also fundamental for the development of RadaR. First, \texttt{R} allows transparent, reproducible, and sustainable data analysis through scripts that can easily be shared and changed. This can build the basis for collaboration, and this enforces the spirit of open science (also through the strong collaborative \texttt{R} community on the web). Second, \texttt{R} is open source and free to use; therefore, it also enables use in resource-limited settings. Finally, Shiny empowers users to interact with the data, making even very large datasets quickly interpretable.

Innovative approaches used in supporting infection management by leveraging EHRs are being investigated \textsuperscript{{[}17-19{]}}. Reporting on AMR, antimicrobial use, and hospital infections (e.g., for quality assurance) is well established, but it is important to integrate these data sources in an approach that allows detailed filtering options on all input. Merely looking at antimicrobial use alone or comparing aggregated results (e.g., total amount of a specific antimicrobial substance per hospital correlated with the total count of a resistant isolate) will result in loss of information or even misleading interpretation. Detailed data and calculations on the basis of each individual patient are crucial to draw informed conclusions. Unfortunately, the abovementioned infection management approaches \textsuperscript{{[}17-19{]}} either depend on additional commercial software for data visualisation or the source code is not openly available. We want to encourage others to turn toward available open-source software solutions, such as R, for an increased potential of collaboration and transparency. However, their strength is the connection to real-time data flows. This enables the prospective use and increases their usability for daily clinical practice. RadaR is currently still limited to retrospective data analysis because of a changing hospital data infrastructure in our setting. Technically, it is feasible to connect R-based software apps such as RadaR to real-time hospital data infrastructures running with clinical data standards \textsuperscript{{[}42{]}}. For a start, access to static data extraction is often easier and faster to achieve. RadaR can be used to advocate the use of data visualisation tools and improved accessibility of hospital data sources. Until connection to real-time hospital data is established, RadaR can support users as a stand-alone option for retrospective data analysis in infection management. Next steps will involve testing in multiple settings and forming a user and research group to continue and expand the use of open-source technology and open science principles in infection management.

\hypertarget{conclusions}{%
\subsection{Conclusions}\label{conclusions}}

RadaR demonstrates the feasibility of developing software tools for infection management and AMS teams in an open-source approach, making it free to use, share, or modify according to various needs in different settings. RadaR has the potential to be a highly useful tool for infection management and AMS in daily practice.

\hypertarget{acknowledgements-1}{%
\section*{Acknowledgements}\label{acknowledgements-1}}
\addcontentsline{toc}{section}{Acknowledgements}

The authors would like to thank the ESGAP executive committee for supporting the evaluation of RadaR in the ESGAP study group and all its members for their valuable input, suggestions, and comments. Furthermore, the authors wish to thank Igor van der Weide, Jan Arends, and Prashant Nannan Panday for their great support in obtaining required data at our institution that built the basis for the development of RadaR. The authors also thank the online \texttt{R} community as well as the valuable comments, suggestions, and input from reviewers that they have received to improve RadaR.

RadaR was developed as part of a project funded by the European Commission Horizon 2020 Framework Marie Skłodowska-Curie Actions (grant agreement number: 713660-PRONKJEWAIL-H2020-MSCA-COFUND-2015).

\hypertarget{conflicts-of-interests}{%
\section*{Conflicts of interests}\label{conflicts-of-interests}}
\addcontentsline{toc}{section}{Conflicts of interests}

None declared.

\hypertarget{references-4}{%
\section*{References}\label{references-4}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Dyar OJ, Huttner B, Schouten J, Pulcini C, ESGAP (ESCMID Study Group for Antimicrobial stewardshiP). What is antimicrobial stewardship? Clin Microbiol Infect 2017 Nov;23(11):793--798. \url{PMID:28882725}
\item
  Barlam TF, Cosgrove SE, Abbo LM, MacDougall C, Schuetz AN, Septimus EJ, Srinivasan A, Dellit TH, Falck-Ytter YT, Fishman NO, Hamilton CW, Jenkins TC, Lipsett PA, Malani PN, May LS, Moran GJ, Neuhauser MM, Newland JG, Ohl CA, Samore MH, Seo SK, Trivedi KK. Implementing an antibiotic stewardship program: Guidelines by the Infectious Diseases Society of America and the Society for Healthcare Epidemiology of America. Clin Infect Dis 2016 May 15;62(10):e51--77. \url{PMID:27080992}
\item
  Davey P, Marwick CA, Scott CL, Charani E, McNeil K, Brown E, Gould IM, Ramsay CR, Michie S. Interventions to improve antibiotic prescribing practices for hospital inpatients. Cochrane Database Syst Rev 2017 Feb 9;2:CD003543. \url{PMID:28178770}
\item
  National Institute for Health and Care Excellence. Antimicrobial stewardship: systems and processes for effective antimicrobial medicine use. \url{https://www.nice.org.uk/Guidance/NG15}. Archived at: \url{http://www.webcitation.org/73ybLqgCI}
\item
  Schuts EC, Hulscher MEJL, Mouton JW, Verduin CM, Stuart JWTC, Overdiek HWPM, van der Linden PD, Natsch S, Hertogh CMPM, Wolfs TFW, Schouten JA, Kullberg BJ, Prins JM. Current evidence on hospital antimicrobial stewardship objectives: a systematic review and meta-analysis. Lancet Infect Dis 2016 Jul;16(7):847--856. \url{PMID:26947617}
\item
  Stichting Werkgroep AntibioticaBeleid. SWAB Guidelines for Antimicrobial Stewardship 2016. \url{http://www.swab.nl/swab/cms3.nsf/uploads/FAF701FBF4FCBDD6C12580EC0037D292/$FILE/20161228\%20SWAB\%20Guidelines\%20for\%20Antimicrobial\%20Stewardship\%20-definitief.pdf} . Archived at: \url{http://www.webcitation.org/73ycxKUfO}
\item
  Pulcini C, Binda F, Lamkang AS, Trett A, Charani E, Goff DA, Harbarth S, Hinrichsen SL, Levy-Hara G, Mendelson M, Nathwani D, Gunturu R, Singh S, Srinivasan A, Thamlikitkul V, Thursky K, Vlieghe E, Wertheim H, Zeng M, Gandra S, Laxminarayan R. Developing core elements and checklist items for global hospital antimicrobial stewardship programmes: a consensus approach. Clin Microbiol Infect. \url{PMID:29625170}
\item
  Sarkies MN, Bowles K-A, Skinner EH, Mitchell D, Haas R, Ho M, Salter K, May K, Markham D, O'Brien L, Plumb S, Haines TP. Data collection methods in health services research: hospital length of stay and discharge destination. Appl Clin Inform 2015 Feb 18;6(1):96--109. \url{PMID:25848416}
\item
  British Society for Antimicrobial Chemotherapy. Antimicrobial stewardship: From Principal to Practice. Birmingham, United Kingdom: British Society for Antimicrobial Chemotherapy; 2018. \url{http://bsac.org.uk/antimicrobial-stewardship-from-principles-to-practice-e-book/}. Archived at: \url{http://www.webcitation.org/73ydEbLNu}
\item
  Pulcini C, Morel CM, Tacconelli E, Beovic B, de With K, Goossens H, Harbarth S, Holmes A, Howard P, Morris AM, Nathwani D, Sharland M, Schouten J, Thursky K, Laxminarayan R, Mendelson M. Human resources estimates and funding for antibiotic stewardship teams are urgently needed. Clin Microbiol Infect 2017 Nov;23(11):785--787. \url{PMID:28778544}
\item
  Howard P, Pulcini C, Levy Hara G, West RM, Gould IM, Harbarth S, Nathwani D, ESCMID Study Group for Antimicrobial Policies (ESGAP), ISC Group on Antimicrobial Stewardship. An international cross-sectional survey of antimicrobial stewardship programmes in hospitals. J Antimicrob Chemother 2015 Apr;70(4):1245--1255. \url{PMID:25527272}
\item
  Dik J-WH, Poelman R, Friedrich AW, Panday PN, Lo-Ten-Foe JR, van Assen S, van Gemert-Pijnen JEWC, Niesters HGM, Hendrix R, Sinha B. An integrated stewardship model: antimicrobial, infection prevention and diagnostic (AID). Future Microbiol 2016;11(1):93--102. \url{PMID:26323589}
\item
  Staib A, Sullivan C, Jones M, Griffin B, Bell A, Scott I. The ED-inpatient dashboard: Uniting emergency and inpatient clinicians to improve the efficiency and quality of care for patients requiring emergency admission to hospital. Emerg Med Australas 2017 Jun;29(3):363--366. \url{PMID:27592365}
\item
  Smith CM, Hayward AC. DotMapper: an open source tool for creating interactive disease point maps. BMC Infect Dis 2016 Apr 12;16:145. \url{PMID:27066780}
\item
  Hughes LD, Lewis SA, Hughes ME. ExpressionDB: An open source platform for distributing genome-scale datasets. PLoS One 2017 Nov 2;12(11):e0187457. \url{PMID:29095940}
\item
  GitHub. \url{https://github.com} . Archived at: \url{http://www.webcitation.org/73yb4rqQT}
\item
  Simpao AF, Ahumada LM, Larru Martinez B, Cardenas AM, Metjian TA, Sullivan KV, Gálvez JA, Desai BR, Rehman MA, Gerber JS. Design and Implementation of a Visual Analytics Electronic Antibiogram within an Electronic Health Record System at a Tertiary Pediatric Hospital. Appl Clin Inform 2018 Jan;9(1):37--45. \url{PMID:29342478}
\item
  Lesho EP, Waterman PE, Chukwuma U, McAuliffe K, Neumann C, Julius MD, Crouch H, Chandrasekera R, English JF, Clifford RJ, Kester KE. The antimicrobial resistance monitoring and research (ARMoR) program: the US Department of Defense response to escalating antimicrobial resistance. Clin Infect Dis 2014 Aug 1;59(3):390--397. \url{PMID:24795331}
\item
  Simões AS, Maia MR, Gregório J, Couto I, Asfeldt AM, Simonsen GS, Póvoa P, Viveiros M, Lapão LV. Participatory-Implementation of an antibiotic stewardship programme supported by an innovative surveillance and clinical decision-support system. J Hosp Infect 2018 Jul 30; \url{PMID:30071264}
\item
  Molloy JC. The open knowledge foundation: open data means better science. PLoS Biol 2011 Dec;9(12):e1001195. \url{PMID:22162946}
\item
  RStudio. \url{https://www.rstudio.com/} . Archived at: \url{http://www.webcitation.org/73ybbedpd}
\item
  R Core Team. R: A Language and Environment for Statistical Computing. \url{https://www.R-project.org/}. Archived at: \url{http://www.webcitation.org/73ydeuUVX}
\item
  Chang W, Cheng J, Allaire JJ, Xie Y, McPherson J. shiny: web Application Framework for R. \url{https://CRAN.R-project.org/package=shiny}. Archived at: \url{http://www.webcitation.org/} 73ydh83pW
\item
  shinyapps.io. \url{https://www.shinyapps.io/}. Archived at: \url{http://www.webcitation.org/} 73ybe2prd
\item
  Stallman R. GNU General Public License, version 2. 1991. \url{https://www.gnu.org/licenses/gpl-2.0.html}. Archived at: \url{http://www.webcitation.org/73ydum67R}
\item
  Wickham H. Tidy Data. Journal of Statistical Software. Foundation for Open Access Statistics; 2014;59(10):1--23. {[}doi: 10.18637/jss.v059.i10{]}
\item
  Tidyverse. \url{https://www.tidyverse.org/}. Archived at: \url{http://www.webcitation.org/} 76WntlTmp
\item
  Grolemund G, Wickham H. Dates and Times Made Easy with lubridate. Journal of Statistical Software 2011;40(3):1--25. {[}doi: 10.18637/jss.v040.i03{]}
\item
  Zeileis A, Grothendieck G. zoo: S3 Infrastructure for Regular and Irregular Time Series. Journal of Statistical Software. 2005. p.~1--27. {[}doi: 10.18637/jss.v014.i06{]}
\item
  Berends MS, Luz CF. AMR: An R package to simplify the analysis and prediction of Antimicrobial Resistance (AMR) and work with antibiotic properties by using evidence-based methods. {[}doi: 10.5281/zenodo.1749847{]}
\item
  WHO Collaborating Centre for Drug Statistics Methodology. Guidelines for ATC classification and DDD assignment. Oslo, Norway; 2018. \url{https://www.whocc.no/filearchive/publications/} guidelines.pdf. Archived at: \url{http://www.webcitation.org/73yeFdXyM}
\item
  Kassambara A, Kosinski M. survminer: Drawing Survival Curves using ``ggplot2''. \url{https://CRAN.R-project.org/package=survminer}. Archived at: \url{http://www.webcitation.org/} 73yeLlWLx
\item
  Anhoej J. qicharts2: Quality Improvement Charts. \url{https://CRAN.R-project.org/} package=qicharts2 . Archived at: \url{http://www.webcitation.org/76nVgPWG5}
\item
  Anhøj J. Diagnostic value of run chart analysis: using likelihood ratios to compare run chart rules on simulated data series. PLoS One 2015 Mar 23;10(3):e0121349. \url{PMID:25799549}
\item
  RadaR shinyapps.io. \url{https://ceefluz.shinyapps.io/radar/}. Archived at: \url{http://www}. webcitation.org/73ybyEBHu
\item
  RadaR GitHub. \url{https://github.com/ceefluz/radar}. Archived at: \url{http://www.webcitation.org/} 73yc2PgeW
\item
  RadaR CSS. \url{https://github.com/ceefluz/radar/blob/master/www/radar_style.css}. Archived at: \url{http://www.webcitation.org/73ycOiZA5}
\item
  Pettit NN, Han Z, Choksi A, Bhagat PH, Pisano J. Using the Epic® Antimicrobial Stewardship (ASP) Module to Optimize Antimicrobial Stewardship Interventions. Open Forum Infectious Diseases 2017 Oct 1;4(suppl\_1):S263--S264. {[}doi: 10.1093/ofid/ofx163.580{]}
\item
  Xiao C, Choi E, Sun J. Opportunities and challenges in developing deep learning models using electronic health records data: a systematic review. J Am Med Inform Assoc 2018 Jun 8; \url{PMID:29893864}
\item
  Pirracchio R, Cohen MJ, Malenica I, Cohen J, Chambaz A, Cannesson M, Lee C, Resche-Rigon M, Hubbard A, ACTERREA Research Group. Big data and targeted machine learning in action to assist medical decision in the ICU. Anaesth Crit Care Pain Med 2018 Oct 16; \url{PMID:30339893}
\item
  Hahn E, Blazes D, Lewis S. Understanding How the ``Open'' of Open Source Software (OSS) Will Improve Global Health Security. Health Secur 016 Jan;14(1):13--18. \url{PMID:26889576}
\item
  Hong N, Prodduturi N, Wang C, Jiang G. Shiny FHIR: An Integrated Framework Leveraging Shiny R and HL7 FHIR to Empower Standards-Based Clinical Data Applications. Stud Health Technol Inform 2017;245:868--872. \url{PMID:29295223}
\end{enumerate}

\hypertarget{ch06-radar2}{%
\chapter{Better Antimicrobial Resistance Data Analysis and Reporting in Less Time}\label{ch06-radar2}}

medRxiv {[}preprint{]} (2021), 21257599

(as of date of PhD defence: 25 August 2021)

Berends MS \textsuperscript{1,2}*, Luz CF \textsuperscript{2}*, Zhou XW \textsuperscript{2}, Lokate ML \textsuperscript{2}, Friedrich AW \textsuperscript{2}, Sinha BNM \textsuperscript{2‡}, Glasner C \textsuperscript{2‡}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, The Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, Netherlands
\end{enumerate}

* These authors contributed equally\\
\textsuperscript{‡} These authors contributed equally

\hypertarget{abstract-4}{%
\section*{Abstract}\label{abstract-4}}
\addcontentsline{toc}{section}{Abstract}

Insights and knowledge about local antimicrobial resistance (AMR) levels and epidemiology are essential to guide optimal decision-making processes in antimicrobial use. However, dedicated tools for reliable and reproducible AMR data analysis and reporting are often lacking. In this study, we aimed at comparing the effectiveness and efficiency of traditional analysis and reporting versus a new approach for reliable and reproducible AMR data analysis in a clinical setting. Ten professionals that routinely work with AMR data were recruited and provided with one year's blood culture test results from a tertiary care hospital results including antimicrobial susceptibility test results. Participants were asked to perform a detailed AMR data analysis in a two-round process: first using their analysis software of choice and next using previously developed open-source software tools. Accuracy of the results and time spent were compared between both rounds. Finally, participants rated the usability of the tools using the systems usability scale (SUS). The mean time spent on creating a comprehensive AMR report reduced from 93.7 (SD ±21.6) minutes to 22.4 (SD ±13.7) minutes (\emph{p} \textless{} 0.001). Average task completion per round changed from 56\% (SD: ±23\%) to 96\% (SD: ±5.5\%) (\emph{p} \textless{} 0.05). The proportion of correct answers in the available results increased from 37.9\% in the first to 97.9\% in the second round (\emph{p} \textless{} 0.001). The usability of the new tools was rated with a median of 83.8 (out of 100) on the SUS. This study demonstrated the significant improvement in efficiency and accuracy in standard AMR data analysis and reporting workflows through open-source software tools in a clinical setting. Integrating these tools in clinical settings can democratise the access to fast and reliable insights about local microbial epidemiology and associated AMR levels. Thereby, our approach can support evidence-based decision-making processes in the use of antimicrobials.

\hypertarget{introduction-3}{%
\section{Introduction}\label{introduction-3}}

Antimicrobial resistance (AMR) is a global challenge in healthcare, livestock and agriculture, and the environment alike. The silent tsunami of AMR is already impacting our lives and the wave is constantly growing \textsuperscript{{[}1,2{]}}. One crucial action point in the fight against AMR is the appropriate use of antimicrobials. The choice and use of antimicrobials has to be integrated into a well-informed decision making process and supported by antimicrobial and diagnostic stewardship programmes \textsuperscript{{[}3,4{]}}. Next to essential local, national, and international guidelines on appropriate antimicrobial use, the information on AMR rates and antimicrobial use through reliable data analysis and reporting is vital. While data on national and international levels are typically easy to access through official reports, local data insights are often lacking, difficult to establish, and its generation requires highly trained professionals. Unfortunately, working with local AMR data is often furthermore complicated by very heterogeneous data structures and information systems within and between different settings \textsuperscript{{[}5,6{]}}. Yet, decision makers in the clinical context need to be able to access these important data in an easy and rapid manner. Without a dedicated team of epidemiologically trained professionals, providing these insights could be challenging and error-prone. Incorrect data or data analyses could even lead to biased/erroneous empirical antimicrobial treatment policies.

To overcome these hurdles, we previously developed new approaches to AMR data analysis and reporting to empower any expert on any level working with or relying on AMR data \textsuperscript{{[}7,8{]}}. We aimed at reliable, reproducible, and transparent AMR data analysis. The underlying concepts are based on open-source software, making them free to use and adaptable to any setting-specific needs. To specify, we developed a software package for the statistical language R to simplify and standardise AMR data analysis based on international guidelines \textsuperscript{{[}7{]}}. In addition, we demonstrated the application of this software package to create interactive analysis tools for rapid and user-friendly AMR data analysis and reporting \textsuperscript{{[}8{]}}.

However, while the use of our approach in research has been demonstrated \textsuperscript{{[}9--12{]}}, the impact on workflows for AMR data analysis and reporting in clinical settings is pending. AMR data analysis and reporting are typically performed at clinical microbiology departments in hospitals, in microbiological laboratories, or as part of multidisciplinary antimicrobial stewardship activities. AMR data analysis and reporting require highly skilled professionals. In addition, thorough and in-depth analyses can be time consuming and sufficient resources need to be allocated for consistent and repeated reporting. This is further complicated by the lack of available software tools that fulfil all requirements such as incorporation of (inter-) national guidelines or reliable reference data.

In this study, we aimed at demonstrating and studying the usability of our developed approach and its impact on clinicians' workflows in an institutional healthcare setting. The approach should enable better AMR data analysis and reporting in less time.

\hypertarget{methods-1}{%
\section{Methods}\label{methods-1}}

The study was initiated at the University Medical Center Groningen (UMCG), a 1339-bed tertiary care hospital in the Northern Netherlands and performed across the UMCG and Certe (a regional laboratory) in the Northern Netherlands. It was designed as a comparison study to evaluate the efficiency, effectiveness, and usability of a new AMR data analysis and reporting approach \textsuperscript{{[}7,8{]}} against traditional reporting.

\hypertarget{study-setup}{%
\subsection{Study setup}\label{study-setup}}

The setup of the study is visualised in Figure 1 and is explained in the following sections.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-01} 

}

\caption{Study setup; the same AMR data was used along all steps and rounds.}\label{fig:fig6-1}
\end{figure}

The study was based on a task document listing general AMR data analysis and reporting tasks (Table 1). This list served as the basis to compare effectiveness (solvability of each task for every user) and efficiency (time spent solving each task) of both approaches. Tasks were grouped into five related groups and analyses were performed per group (further referred to as five tasks). A maximum amount of time per task (group) was defined for each task. The list of tasks including correct results is available in Appendix A1.

Table 1. AMR data analysis and reporting tasks.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-t01} 

}

\end{figure}

\hypertarget{amr-data}{%
\subsection{AMR data}\label{amr-data}}

Anonymised microbiological data were obtained from the Department of Medical Microbiology and Infection Prevention at the UMCG. The data consisted of 23,416 records from 18,508 unique blood culture tests that were taken between January 1, 2019 and December 31, 2019 which were retrieved from the local laboratory information system (LIS). Available variables were: test date, sample identification number, sample specimen, anonymised patient identification number, microbial identification code (if culture positive), antimicrobial susceptibility test results (S, I, R - susceptible, susceptible at increased exposure, resistant) for 52 antimicrobials. The exemplified data structure is presented in Table 2.

Table 2. Raw data example.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-t02} 

}

\end{figure}

\hypertarget{amr-data-analysis-and-reporting}{%
\subsection{AMR data analysis and reporting}\label{amr-data-analysis-and-reporting}}

We used our previously developed approach \textsuperscript{{[}7,8{]}} to create a customised browser-based AMR data analysis and reporting application. This application was used in this study and applied to the AMR data analysis and reporting tasks listed in the task document (Table 1). The development of the application followed an agile approach using scrum methodologies \textsuperscript{{[}14{]}}. Agile development was used to effectively and iteratively work in a team of two developers, a clinical microbiologist, and an infection preventionist. The application was designed as an interactive web-browser based dashboard (Figure 2). The prepared dataset was already loaded into the system and interaction with the application was possible through any web-browser.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-02} 

}

\caption{Interactive dashboard for AMR data analysis used in this study.}\label{fig:fig6-2}
\end{figure}

\hypertarget{study-participants}{%
\subsection{Study participants}\label{study-participants}}

Participants in this study were recruited from the departments of Medical Microbiology, Critical Care Medicine, and Paediatrics, to reflect heterogeneous backgrounds of healthcare professionals working with AMR data. Members of the development team did not take part in the study.

\hypertarget{study-execution-and-data}{%
\subsection{Study execution and data}\label{study-execution-and-data}}

First, study participants were asked to fill in an online questionnaire capturing their personal backgrounds, demographics, software experience, and experience in AMR data analysis and reporting. Next, participants were provided the task document together with the AMR data (csv- or xlsx-format). The participants were asked to perform a comprehensive AMR data report following the task document using their software of choice (round 1). Task results and information on time spent per task were self-monitored and returned by the participant using a structured report form. Lastly, participants repeated the AMR data analysis and reporting process with the same task document but using the new AMR data analysis and reporting application (round 2). Task results and information on time spent per task were again self-monitored and returned by the participant using the same structured report form as in the first round. This last step was evaluated using a second online questionnaire. The study execution process is illustrated in Figure 1.

\hypertarget{evaluation-and-study-data-analysis}{%
\subsection{Evaluation and study data analysis}\label{evaluation-and-study-data-analysis}}

The utility of the new AMR data analysis and reporting application was evaluated according to ISO 9241-11:2018 \textsuperscript{{[}15{]}}. This international standard comprises several specific metrics to quantify the usability of a tool with regard to reaching its defined goals (Figure 3). In this study the goal was a comprehensive AMR data report and comprised several tasks as outlined in the task document. The equipment was the focus of this study (traditional AMR data analysis and reporting approach vs.~newly developed AMR data analysis and reporting approach).

\begin{figure}

{\centering \includegraphics[width=0.5\linewidth]{images/06-03} 

}

\caption{Usability framework based on ISO 9241-11.}\label{fig:fig6-3}
\end{figure}

The three ISO standard usability measures (in grey) were defined as follows in this study: Effectiveness was determined by degree of task completion coded using three categories: 1) completed; 2) not completed (task not possible to complete); 3) not completed (task completion would take too long, e.g., \textgreater{} 20 minutes). In addition, effectiveness was assessed by the variance in the task results stratified by study round. Deviation from the correct results was measured in absolute percent from the correct result. To account for potential differences in the results due to rounding, all numeric results were transformed to integers. Efficiency was determined by timing each individual task. Time on task started when the user started performing the task, all data was loaded, and the chosen analysis software was up and running. Time on task ended when the task reached one of the endpoints, as described above. In the analysis, the mean time for each task and the mean total time for the complete report across users was calculated. Statistically significant difference was tested using paired Student's t-test. All analyses were performed in R \textsuperscript{{[}16{]}}. Outcomes of tests were considered statistically significant for \emph{p} \textless{} 0.05. Accuracy of the reported results per task and round were studied by calculating the deviation of the reported result in absolute percent from the correct result. Satisfaction was measured using the System Usability Scale (SUS), a 10-item Likert scale with levels from 1 (strongly disagree) to 5 (strongly agree, see Appendix A3) \textsuperscript{{[}17{]}}. The SUS yields a single number from 0 to 100 representing a composite measure of the overall usability of the system being studied (SUS questions and score calculation in the Appendix A2).

\hypertarget{results-1}{%
\section{Results}\label{results-1}}

\hypertarget{study-participants-1}{%
\subsection{Study participants}\label{study-participants-1}}

In total 10 participants were recruited for this study. Most participants were clinical microbiologists (in training) (70\%). The median age of the participant group was 40.5 years with a median working experience in the field of 8.0 years. The relevance of AMR data as part of the participants' job was rated very high (median of 5.0; scale 1-5). AMR data analysis was part of the participants' job for 60\% of all participants. Participants reported to be very experienced in interpreting AMR data structures (median 5.0, scale 1-5). Participants were less experienced in epidemiological data analysis (median 3.0, scale 1-5). All participant characteristics are summarised in Table 3.

Table 3. Study participant characteristics.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-t03} 

}

\end{figure}

The participants reported a diverse background in software experience for data analysis, with most experience reported for Microsoft Excel (Figure 4).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-04} 

}

\caption{Data analysis software experience reported by study participants.}\label{fig:fig6-4}
\end{figure}

\hypertarget{effectiveness-and-accuracy}{%
\subsection{Effectiveness and accuracy}\label{effectiveness-and-accuracy}}

Not all participants were able to complete the tasks within the given time frame. Average task completion between the first round (traditional AMR data analysis and reporting) and the second round (new AMR data analysis and reporting) changed from 56\% (SD: 23\%) to 96\% (SD: 6\%) (\emph{p} \textless{} 0.05). Task completion per question and round is displayed in Figure 5. Variation in responses for each given task showed significant differences between the first and second round.

Figure 6 shows the deviation in absolute percent from the correct results from the correct result per round and task. The proportion of correct answers in the available results increased from 38\% in the first round to 98\% in the second round (\emph{p} \textless{} 0.001). A sub-analysis of species-specific results for task 3 round 1 is available in the appendix (A3).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-05} 

}

\caption{Task completion in percent by task number and round.}\label{fig:fig6-5}
\end{figure}

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-06} 

}

\caption{Deviation from the correct result by task and round in absolute percent from correct result. Only completed tasks (n) are shown.}\label{fig:fig6-6}
\end{figure}

\hypertarget{efficiency}{%
\subsection{Efficiency}\label{efficiency}}

Overall, the mean time spent per round was significantly reduced from 93.7 (SD: 21.6) minutes to 22.4 (SD: 13.7) minutes (\emph{p} \textless{} 0.001). Significant time reduction could be observed for tasks 2-5 (Figure 7). Analyses were further stratified to compare efficiency between participants that reported AMR data analysis as part of their job versus not part of the job. No significant time difference for completing all tasks could be found between the groups. However, in both groups the overall time for all tasks significantly decreased between the first and second round: on average by 70.7 minutes (\emph{p} \textless{} 0.001) in the group reporting AMR data analysis as part of their job and by 72.1 minutes (\emph{p} = 0.01) in the group not reporting AMR data analysis as part of their job.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-07} 

}

\caption{Mean time spent per task in minutes in each round (yellow = first round, red = second round). Statistical significance was tested using two-sided paired t-tests. All results were included irrespective of correctness of the results.}\label{fig:fig6-7}
\end{figure}

\hypertarget{satisfaction}{%
\subsection{Satisfaction}\label{satisfaction}}

Participants rated the usability of the new AMR reporting tool using the system usability scale (SUS) which takes values from 0 to 100 (Appendix A2). This resulted in a median of 83.8 on the SUS.

\hypertarget{discussion-2}{%
\section{Discussion}\label{discussion-2}}

This study demonstrates the effectiveness, efficiency, and accuracy of using open-source software tools to improve AMR data analysis and reporting. We applied our previously developed approach to AMR data analysis and reporting \textsuperscript{{[}7,8{]}} in a clinical scenario and tested these tools with study participants (users) working in the field of AMR. Comparing traditional reporting tools with our newly developed reporting tools in a two-step process, we demonstrated the usability and validity of our approach. Based on a five item AMR data analysis and reporting task list and the provided AMR data, study participants reported significantly less time spent on creating an AMR data report (on average 93.7 minutes vs.~22.4 minutes; \emph{p} \textless{} 0.01). Task completion increased significantly from 56\% to 96\%, which indicates that with traditional reporting approaches common questions around AMR are hard to answer in a limited time. The accuracy of the results greatly improved using the new AMR reporting approach, implicating that erroneous answers are more common when users rely on general non-AMR-specific traditional software solutions. The usability of our AMR reporting approach was rated with a median of 83.8 on the SUS. The SUS is widely used in usability assessments of software solutions. A systematic analysis of more than 1000 reported SUS scores for web-based applications across different fields has found a mean SUS score of 68.1 \textsuperscript{{[}18{]}}. The results thus demonstrate a good usability of our approach.

The task list used in this study reflects standard AMR reporting tasks. More sophisticated tasks, such as the detection of multi-drug resistance according to (inter-)national guidelines were not included. However, these analyses are vital in any setting but restrained since the required guidelines are not included in traditional reporting and analysis tools (e.g., Microsoft Excel, SPSS, etc.). Notably, the underlying software used in this study \textsuperscript{{[}7{]}} does provide methods to easily incorporate (inter)national guidelines such as the definitions for (multi-)drug resistance and country-specific (multi-)drug resistant organisms. The increase in task completion rate and accuracy of the results demonstrated that our tools empower specialists in the AMR field to generate reliable and valid AMR data reports. This is important as it enables detailed insights into the state of AMR on any level. These insights are often lacking. Our approach could fill this gap by democratising the ability for reliable and valid AMR data analysis and reporting.

This need is exemplified in the worrisome heterogeneity of the reporting results using traditional AMR reporting tools in the first round. Only 37.9\% of the results in the first round were correct. Together with a task completion rate of 56\%, this demonstrates that traditional tools are not suitable for AMR reporting. The inability of working in reproducible and transparent workflows further aggravates reporting with these traditional tools. All participants in the study should be able to produce standard AMR reports and 90\% indicated that they worked with AMR data before. Sixty percent reported AMR data analysis to be part of their job, but no efficiency difference between groups were found. Our results show that AMR data analysis and reporting is challenging and can be highly error-prone. But an approach such as the one we developed can lead to correct results in a short time while being reproducible and transparent.

We chose an agile workflow which enabled us to integrate clinical feedback throughout the development process in this study. We can highly recommend this efficient approach for projects that need to bridge clinical requirements, statistical approaches, and software development. Our approach was inspired by others not in the AMR field that describe the use of reproducible open-source workflows in ecology \textsuperscript{{[}19{]}}. We found that open-source software enables the transferability of methodological approaches across research fields. This transfer is a great example of the strength in the scientific community when working interdisciplinarily and sharing reliable and reproducible workflow.

This study is subject to limitations. Only ten participants were recruited for this study. Although low participant numbers are frequently observed in usability studies and reports show that only five participants suffice to study the usability of a new system, a larger sample size would be desirable \textsuperscript{{[}20--24{]}}. In addition, other methods (e.g., `think aloud' method) beyond the single use of the SUS for the evaluation of our approach would further improve insights in the usability but were not possible in the study setting \textsuperscript{{[}25{]}}. Although the introduction of new AMR data and reporting tools made use of an already available approach, implementation still requires staff experienced in R. Reporting requirements also differ per setting and tailor-made solutions incorporating different requirements are needed.

The present study shows that answering common AMR-related questions is tremendously burdened for professionals working with data. However, answers to such questions are the requirement to enable hospital-wide monitoring of AMR levels. The monitoring, be it on the institutional, regional, or (inter-) national level, can lead to alteration of treatment policies. It is thus of utmost importance that reliable results of AMR data analyses are ensured to avoid imprecise and erroneous results that could potentially be harmful to patients. We show that traditional reporting tools and applications that are not equipped for conducting microbiology epidemiological analyses seem unfit for this task - even for the most basic AMR data analyses. To fill this gap, we have developed new tools for AMR data analysis and reporting. In this study, we demonstrated that these tools can be used for better AMR data analysis and reporting in less time.

\hypertarget{acknowledgements-2}{%
\section*{Acknowledgements}\label{acknowledgements-2}}
\addcontentsline{toc}{section}{Acknowledgements}

We thank all study participants for their participation in this study and highly value their time spent on these tasks next to their clinical and other professional duties.

\hypertarget{funding}{%
\section*{Funding}\label{funding}}
\addcontentsline{toc}{section}{Funding}

This study was partly supported by the INTERREG V A (202085) funded project EurHealth-1Health (\url{http://www.eurhealth1health.eu}), part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport, the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the Ministry for National and European Affairs and Regional Development of Lower Saxony.

In addition, this study was part of a project funded by the European Union's Horizon 2020 research and innovation programme under the Marie Sklodowska-Curie grant agreement 713660 (MSCA-COFUND-2015-DP ``Pronkjewail'').

\hypertarget{conflict-of-interest}{%
\section*{Conflict of interest}\label{conflict-of-interest}}
\addcontentsline{toc}{section}{Conflict of interest}

The authors report no conflict of interests.

\hypertarget{references-5}{%
\section*{References}\label{references-5}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  O'Neill J. Review on antimicrobial resistance: tackling a crisis for the health and wealth of nations. London: Wellcome Trust; 2014.
\item
  OECD. Stemming the Superbug Tide. OECD Health Policy Studies 2018:224. \url{https://doi.org/10.1787/9789264307599-en}.
\item
  Dyar OJ, Huttner B, Schouten J, Pulcini C, ESGAP (ESCMID Study Group for Antimicrobial stewardshiP). What is antimicrobial stewardship? Clin Microbiol Infect 2017;23:793--8. \url{https://doi.org/10.1016/j.cmi.2017.08.026}.
\item
  Morjaria S, Chapin KC. who to test, when, and for what: why diagnostic stewardship in infectious diseases matters. J Mol Diagn 2020;22:1109--13. \url{https://doi.org/10.1016/j.jmoldx.2020.06.012}.
\item
  Tacconelli E, Sifakis F, Harbarth S, Schrijver R, van Mourik M, Voss A, \emph{et al.} Surveillance for control of antimicrobial resistance. Lancet Infect Dis 2018;18:e99--106. \url{https://doi.org/10.1016/S1473-3099(17)30485-1}.
\item
  Limmathurotsakul D, Dunachie S, Fukuda K, Feasey NA, Okeke IN, Holmes AH, \emph{et al.} Improving the estimation of the global burden of antimicrobial resistant infections. Lancet Infect Dis 2019;19:e392--8. \url{https://doi.org/10.1016/S1473-3099(19)30276-2}.
\item
  Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR - An R package for working with antimicrobial resistance data. Cold Spring Harbor Laboratory 2019:810622. \url{https://doi.org/10.1101/810622}.
\item
  Luz CF, Berends MS, Dik J-WH, Lokate M, Pulcini C, Glasner C, \emph{et al.} Rapid analysis of diagnostic and antimicrobial patterns in R (RadaR): interactive open-source software app for infection management and antimicrobial stewardship. J Med Internet Res 2019;21:e12843. \url{https://doi.org/10.2196/12843}.
\item
  Le Guern R, Titécat M, Loïez C, Duployez C, Wallet F, Dessein R. Comparison of time-to-positivity between two blood culture systems: a detailed analysis down to the genus-level. Eur J Clin Microbiol Infect Dis 2021. \url{https://doi.org/10.1007/s10096-021-04175-9}.
\item
  Kim S, Yoo SJ, Chang J. importance of susceptibility rate of ``the first'' isolate: evidence of real-world data. Medicina 2020;56. \url{https://doi.org/10.3390/medicina56100507}.
\item
  Tenea GN, Jarrin-V P, Yepez L. microbiota of wild fruits from the amazon region of ecuador: linking diversity and functional potential of lactic acid bacteria with their origin. In: Mikkola HJ, editor. Ecosystem and Biodiversity of Amazonia, Rijeka: IntechOpen; 2021. \url{https://doi.org/10.5772/intechopen.94179}.
\item
  Dutey-Magni PF, Gill MJ, McNulty D, Sohal G, Hayward A, Shallcross L, \emph{et al.} Feasibility study of hospital antimicrobial stewardship analytics using electronic health records. JAC Antimicrob Resist 2021;3. \url{https://doi.org/10.1093/jacamr/dlab018}.
\item
  Clinical and Laboratory Standards Institute. M39-A4, Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 4th ed.~Pittsburgh: Clinical and Laboratory Standards Institute; 2014.
\item
  Schwaber K, Beedle M. Agile software development with Scrum. London: Pearson; 2001.
\item
  International Organization for Standardization. Ergonomics of human-system interaction --- part 11: usability: definitions and concepts (ISO 9241-11:2018 ); Geneva: International Organization for Standardization; 2018.
\item
  R Core Team. R: A language and environment for statistical computing. Vienna: R Foundation for Statistical Computing; 2019.
\item
  Brooke J. SUS: A retrospective. Journal of Usability Studies 2013;8:29--40.
\item
  Bangor A, Kortum PT, Miller JT. An empirical evaluation of the system usability scale. International Journal of Human--Computer Interaction 2008;24:574--94. \url{https://doi.org/10.1080/10447310802205776}.
\item
  Lowndes JSS, Best BD, Scarborough C, Afflerbach JC, Frazier MR, O'Hara CC, \emph{et al.} Our path to better science in less time using open data science tools. Nat Ecol Evol 2017;1:160. \url{https://doi.org/10.1038/s41559-017-0160}.
\item
  Ledieu T, Bouzillé G, Thiessard F, Berquet K, Van Hille P, Renault E, \emph{et al.} Timeline representation of clinical data: usability and added value for pharmacovigilance. BMC Med Inform Decis Mak 2018;18:86. \url{https://doi.org/10.1186/s12911-018-0667-x}.
\item
  Rubin J, Chisnell D. Handbook of usability testing: how to plan, design, and conduct effective tests. Hoboken: Wiley; 2008.
\item
  Albert W, Tullis T. Measuring the user experience: collecting, analyzing, and presenting usability metrics. Burlington: Morgan Kaufmann; 2013.
\item
  Iordatii M, Venot A, Duclos C. Design and evaluation of a software for the objective and easy-to-read presentation of new drug properties to physicians. BMC Med Inform Decis Mak 2015;15:42. \url{https://doi.org/10.1186/s12911-015-0158-2}.
\item
  Bastien JMC. Usability testing: a review of some methodological and technical aspects of the method. Int J Med Inform 2010;79:e18--23. \url{https://doi.org/10.1016/j.ijmedinf.2008.12.004}.
\item
  Jaspers MWM. A comparison of usability methods for testing interactive health technologies: methodological aspects and empirical evidence. Int J Med Inform 2009;78:340--53. \url{https://doi.org/10.1016/j.ijmedinf.2008.10.002}.
\end{enumerate}

\hypertarget{appendix}{%
\section*{Appendix}\label{appendix}}
\addcontentsline{toc}{section}{Appendix}

\hypertarget{appendix-a1-task-lists-including-correct-results}{%
\subsection*{Appendix A1: Task lists including correct results}\label{appendix-a1-task-lists-including-correct-results}}
\addcontentsline{toc}{subsection}{Appendix A1: Task lists including correct results}

Table A1. AMR data analysis and reporting tasks with correct results

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-ta01} 

}

\end{figure}
\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-ta01b} 

}

\end{figure}
\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-ta01c} 

}

\end{figure}

\hypertarget{appendix-a2-system-usability-scale-sus}{%
\subsection*{Appendix A2: System Usability Scale (SUS)}\label{appendix-a2-system-usability-scale-sus}}
\addcontentsline{toc}{subsection}{Appendix A2: System Usability Scale (SUS)}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  I think that I would like to use this system frequently.
\item
  I found the system unnecessarily complex.
\item
  I thought the system was easy to use
\item
  I think that I would need the support of a technical person to be able to use this system.
\item
  I found the various functions in this system were well integrated.
\item
  I thought there was too much inconsistency in this system.
\item
  I would imagine that most people would learn to use this system very quickly.
\item
  I found the system very cumbersome to use.
\item
  I felt very confident using the system.
\item
  I needed to learn a lot of things before I could get going with this system.
\end{enumerate}

(Each item with levels: 1 = strongly disagrees to 5 = strongly agrees)

Scores for individual items are not meaningful on their own. To calculate the SUS score, the score contributions from each item must be summed. Each item's score contribution ranges from 0 to 4. For items 1, 3, 5, 7, and 9 the score contribution is the scale position minus 1. For items 2, 4, 6, 8, and 10, the contribution is 5 minus the scale position. The sum of the scores is multiplied by 2.5 to obtain the SUS.

\hypertarget{appendix-a3-task-3-sub-analysis}{%
\subsection*{Appendix A3: Task 3 sub-analysis}\label{appendix-a3-task-3-sub-analysis}}
\addcontentsline{toc}{subsection}{Appendix A3: Task 3 sub-analysis}

Task 3 asked participants to identify the ten most frequent species in the provided data set, while correcting for multiple occurrences of a species within a patient. Figure A3 illustrates the deviation from the correct result in the first round (traditional AMR reporting) per species. For this analysis also incomplete results were included (i.e., task not completed but some results provided).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/06-a03} 

}

\caption{Results from task 3 in round 1. Deviation in absolute percent from the correct result per identified species. Also, incomplete data from participants was used in this analysis (i.e., task not completed but some results given). The correct number per species is given in addition to the number provided answers.}\label{fig:fig6-a1}
\end{figure}

\hypertarget{ch07-cons}{%
\chapter{Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019}\label{ch07-cons}}

In preparation

(as of date of PhD defence: 25 August 2021)

Berends MS \textsuperscript{1,2}, Luz CF \textsuperscript{2}, Ott A \textsuperscript{1}, Andriesse GI \textsuperscript{1}, Becker K \textsuperscript{3,4}, Glasner C \textsuperscript{2‡}, Friedrich AW \textsuperscript{2‡}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, the Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, the Netherlands
\item
  Institute of Medical Microbiology, University Hospital Münster, Münster, Germany
\item
  Friedrich Loeffler-Institute of Medical Microbiology, University Medicine Greifswald, Greifswald, Germany
\end{enumerate}

\textsuperscript{‡} These authors contributed equally

\hypertarget{abstract-5}{%
\section*{Abstract}\label{abstract-5}}
\addcontentsline{toc}{section}{Abstract}

For years, coagulase-negative staphylococci (CoNS) were not considered a cause of bloodstream infections (BSIs) and were often regarded as contamination. However, the association of CoNS with nosocomial infections is increasingly recognised in research and clinical practice. At present, the CoNS group consists of 45 different species. Their identification has mainly been driven by the introduction of matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry. Yet, treatment guidelines consider CoNS as a whole group and rarely differentiate between species, despite increasing antibiotic resistance (ABR) in CoNS. Therefore, this retrospective study provides an in-depth analysis of CoNS isolates and their ABR profiles found in blood culture isolates between 2013 and 2019 in a novel full-region approach including the entire region of the Northern Netherlands. In total, 10,796 patients were included that were hospitalised in one of the 15 hospitals in the region leading to a sample of 14,992 first CoNS isolates for (ABR) data analysis. CoNS accounted for 27.6\% of all available 71,632 blood culture isolates. EUCAST Expert rules were applied to correct for errors in antibiotic test results. A total of 27 different species were found. Major differences were observed in the occurrence and ABR profiles of the different species. The top five species covered 97.1\% of all included isolates: \emph{S. epidermidis} (48.4\%), \emph{S. hominis} (33.6\%), \emph{S. capitis} (9.3\%), \emph{S. haemolyticus} (4.1\%), and \emph{S. warneri} (1.7\%). Regarding ABR, \emph{S. epidermidis} and \emph{S. haemolyticus} showed 50-80\% resistance to teicoplanin and macrolides while resistance to these agents remained lower than 10\% in most other CoNS species. Yet, such differences are neglected in national guideline development causing a focus on `ABR-safe' agents such as glycopeptides. Nonetheless, other agents could be considered viable options for some species where ABR never surpassed 10\%. In conclusion, a multi-year, full-region approach to extensively assess the trends in both the occurrence and AMR of CoNS species was carried out which could be used for evaluating treatment policies and understanding more about these important but still too often neglected pathogens.

\hypertarget{introduction-4}{%
\section{Introduction}\label{introduction-4}}

Sepsis is a syndrome of physiologic, pathologic, and biochemical abnormalities induced by bloodstream infections (BSIs). It is the most frequent cause of death in hospitalised patients and has been recognised by the WHO as a global health priority \textsuperscript{{[}1,2{]}}. For years, coagulase-negative staphylococci (CoNS) were not considered a cause of BSIs and were often regarded as contamination \textsuperscript{{[}3{]}}. Yet, it has been shown that CoNS can cause BSIs and a high mortality rate \textsuperscript{{[}4,5{]}}, especially in immunocompromised patients and newborns \textsuperscript{{[}6,7{]}}. Moreover, CoNS have become increasingly associated with nosocomial infections \textsuperscript{{[}8{]}}. This is attributed to (i) an increase of multimorbid and immunocompromised patients that are more prone to infections, (ii) the increased use of inserted foreign body material in modern medicine, and (iii) the property of CoNS to adapt molecularly to the hospital environment by diverging into new strains \textsuperscript{{[}8,9{]}}. Specifically, \emph{S. epidermidis} and \emph{S. haemolyticus} are associated with sepsis caused by foreign-body-related infections (FBRIs), such as central line-associated BSIs and prosthetic joint infections \textsuperscript{{[}10{]}}.

At present, the CoNS group consists of 45 different species \textsuperscript{{[}11{]}}. This group is highly heterogeneous in its prevalence in humans and, more importantly, its antibiotic resistance (ABR) patterns. Zooming in on CoNS at the species level is therefore useful to evaluate treatment options for CoNS causing BSI. The clinical interpretation and relevance of BSIs caused by CoNS are dependent on the determination at the species level, since not all species in the CoNS group are pathogenic and associated with sepsis or (other) nosocomial infections \textsuperscript{{[}8,12{]}}. While the microbiological diagnosis of BSIs has for decades been based on blood samples cultivated in automated blood-culture systems, molecular and mass spectrometry (MS) approaches enable more reliable microbiological diagnosis \textsuperscript{{[}13,14{]}}. Since 2012, matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) MS has become a standard for the identification of bacterial species and has, together with sequencing approaches, led to a rapid discovery of new species compared to formerly used techniques \textsuperscript{{[}15,16{]}}. Prior to the use of MALDI-TOF MS, identification of CoNS was primarily performed with biochemical and physiological tests, which yielded variable results, particularly in less prevalent species \textsuperscript{{[}16{]}}. Examples include \emph{S. warneri}, \emph{S. auricularis}, \emph{S. capitis}, and other CoNS species that primarily colonise the skin of animals or are found on food products \textsuperscript{{[}17{]}}. Due to less specific traditional test techniques, previously reported prevalences and ABR patterns of specific species in the CoNS group may have been unreliable or under-evaluated. Consequently, identification using MALDI-TOF MS has become crucial to analyse species-specific ABR.

ABR is a global healthcare problem and of great concern in the antibiotic therapy of BSIs. This also applies to the CoNS group where multi-drug resistance is common in species circulating in hospitals \textsuperscript{{[}18{]}}. The rise of beta-lactam resistance in CoNS species has led to vancomycin as a first-line therapy against CoNS-mediated BSI in many countries, even though information about the pharmacokinetics and pharmacodynamics (PK/PD) of vancomycin against CoNS is limited \textsuperscript{{[}5,19--21{]}}. To assess the constant change of ABR in CoNS, geo-spatial and temporal analyses of ABR are required.

In the Netherlands, country-wide ABR analyses are used to develop antibiotic treatment guidelines by the Dutch Working Party on Antibiotic Policy (Stichting Werkgroep Antibiotica Beleid, SWAB) \textsuperscript{{[}21,22{]}}. Their recommendations are based on NethMap, an annually released national report about ABR and antibiotic consumption by the Dutch National Institute for Public Health and the Environment (Rijksinstituut voor Volksgezondheid en Milieu, RIVM) \textsuperscript{{[}21{]}}. However, this national report does not specify nor address ABR on a patient, hospital, or regional level.

Therefore, to inform clinical decision-makers this cross-sectional retrospective study provides an in-depth ABR analysis of all CoNS isolates found in blood cultures from 2013 until 2019 in the Northern Netherlands that were determined by MALDI-TOF MS. We aim to evaluate the differences in the occurrence of CoNS species and their ABR patterns and to assess their clinical microbiological relevance using a full-region approach.

\hypertarget{materials-methods}{%
\section{Materials \& methods}\label{materials-methods}}

\hypertarget{study-setting-and-patient-cohort}{%
\subsection{Study setting and patient cohort}\label{study-setting-and-patient-cohort}}

This study was performed within the Northern Netherlands (Figure 1), a geographic region with 1.7 million inhabitants \textsuperscript{{[}23{]}}. Its three provinces are similar in population density: Drenthe (492,167 inhabitants, 184/km2), Friesland (647,672 inhabitants, 183/km2) and Groningen (583,990 inhabitants, 243/km2) \textsuperscript{{[}23{]}}. The study population consisted of 10,786 patients hospitalised with suspected BSI in 15 participating hospitals (14 secondary care, one tertiary care) located within this region between 1 January 2013 and 31 December 2019. All hospitals included at least one intensive care unit (ICU). There was no age restriction on including patients.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-01} 

}

\caption{Locations of the fifteen hospitals in the three provinces in the North of the Netherlands. Between 2013 and June 2018, the region comprised fourteen hospitals; in July 2018, two hospitals merged into one new hospital, leaving a total of thirteen currently active hospitals.}\label{fig:fig7-1}
\end{figure}

\hypertarget{microbiological-and-demographic-data}{%
\subsection{Microbiological and demographic data}\label{microbiological-and-demographic-data}}

All blood cultures were routinely drawn and analysed at one of the three medical microbiological laboratories in the region (Izore, Friesland; Certe, Groningen and Drenthe; University Medical Center Groningen). After routine processing, isolates were included in the study if the species was characterised as a member of the CoNS group and antibiotic test results were available. In the study period, CoNS species were the most prevalent microorganisms isolated from blood and accounted for 27.6\% of all available 71,632 blood culture isolates. The following variables were available for all isolates: date, name of laboratory, name of the hospital, age, gender, and ID of the patient and type of ward (ICU, clinical, outward). Genotypic data was not available for this study, as genotyping was not part of routine analysis.

\hypertarget{species-determination-and-antibiotic-susceptibility-testing-ast}{%
\subsection{Species determination and antibiotic susceptibility testing (AST)}\label{species-determination-and-antibiotic-susceptibility-testing-ast}}

Routine processing in the laboratories included the incubation of blood cultures allowing the colourimetric detection of CO2 produced by growing microorganisms. Determination of the taxonomic species level was done using MALDI-TOF MS. Two laboratories cultivated blood samples using the BacT/ALERT system (bioMérieux, France) and identified bacterial strains using the VITEK MS system (bioMérieux, France). One laboratory cultivated blood samples using the BACTEC (Becton Dickinson, UK) and identified bacterial strains using the Microflex System (Bruker Corporation, USA). Since the databases of these proprietary systems are not publicly available, a qualitative assessment could not be attained, nor was this available in public literature.

AST was performed using the VITEK 2 Advanced Expert System after isolates were incubated on blood agar plates containing 5\% sheep blood (BA+5\%SB). Two laboratories used the VITEK 2 P-586 cartridges and one laboratory used the VITEK 2 P-657 cartridge which are both developed specifically for Gram-positive bacteria such as staphylococci. All results were authorised and validated by at least two laboratory technicians and one clinical microbiologist. Since different VITEK 2 cartridges were used, not all isolates were tested for all antibiotics analysed in this study. Supplementary Material 2 contains a full list of all included isolates and their respective AST results.

\hypertarget{selection-of-bacterial-isolates}{%
\subsection{Selection of bacterial isolates}\label{selection-of-bacterial-isolates}}

First isolates were determined and selected using the AMR package for R to exclude duplicate findings following the M39-A4 guideline by the Clinical Laboratory Standards Institute (CLSI) \textsuperscript{{[}24,25{]}}. This guideline defines first isolates based on the species level per patient episode, regardless of body site and other phenotypical characteristics. The episode length for this study was defined as 365 days, resulting in the inclusion of a unique species once a year per patient.

In this study, several additions were made in extension to the CLSI guideline. As the CLSI guideline only considers the genus/species per episode, we investigated the added value to include changes in the ABR profile per genus/species and episode. For this purpose, we weighted the ABR profile of six preselected antibiotics, which were specifically chosen based on clinical relevance for Gram-positive bacteria, such as CoNS: erythromycin, oxacillin, rifampicin, teicoplanin, tetracycline, and vancomycin. Any change in these antibiotics from susceptible to resistant or vice-versa within the same species in the same patient within one episode was considered a `first weighted isolate'. ABR analysis results per species were included if at least 30 first isolates were available following the current CLSI guideline \textsuperscript{{[}24{]}}.

\hypertarget{eucast-rules-and-antibiotic-resistance-analysis}{%
\subsection{EUCAST rules and antibiotic resistance analysis}\label{eucast-rules-and-antibiotic-resistance-analysis}}

European Committee on Antimicrobial Susceptibility Testing (EUCAST) rules were applied to the AST results including EUCAST Expert Rules (v3.1, 2016), EUCAST Clinical Breakpoint Interpretations (v10.0, 2020), and EUCAST rules for Intrinsic Resistance and Unusual Phenotypes \textsuperscript{{[}26,27{]}}. All applied changes can be found in Supplementary Table 1. Resistance was defined as the number of isolates with an antibiotic interpretation of R (resistant) divided by the total number of susceptible (S or I) isolates, following the latest EUCAST guideline \textsuperscript{{[}27{]}}.

\hypertarget{statistical-analysis}{%
\subsection{Statistical analysis}\label{statistical-analysis}}

All statistical analyses were done using R v4.0.3, RStudio v1.4, and the AMR package v1.6.0 \textsuperscript{{[}25,28{]}}. To test for linear trends, linear regression analyses were performed. Contingency tables were tested with Fisher's exact test when the size was 2x2 and Chi-squared tests otherwise. For likelihood ratio tests exact binomial tests were used. Outcomes of statistical tests were considered significant when \emph{p} \textless{} 0.05.

\hypertarget{ethical-considerations}{%
\subsection{Ethical considerations}\label{ethical-considerations}}

Ethical approval and informed consent were not required according to the medical ethical committee of the University Medical Center Groningen (METc M21.277097). All data were anonymised at the associated laboratories before analysis.

\hypertarget{results-2}{%
\section{Results}\label{results-2}}

\hypertarget{patients-and-included-isolates}{%
\subsection{Patients and included isolates}\label{patients-and-included-isolates}}

A total of 10,796 patients were included in this seven-year study. The median age was 67 (IQR: 52-78) and 46.7\% (n = 5,040) of the patients was female. A total of 19,803 CoNS isolates were included, of which 14,992 isolates were used for ABR analysis based on the ``first weighted isolates'' algorithm. A selection of first isolates using solely the CLSI guideline \textsuperscript{{[}24{]}} would have yielded 12,971 isolates (-13.5\%, \emph{p} \textless{} 0.001). On ICUs, 25.7\% of the first weighted isolates was found in males compared to 17.0\% in females (\emph{p} \textless{} 0.001). The number of ICU patients with CoNS compared to non-ICU patient with CoNS showed a significant difference between secondary care (17.5\%, n = 1,403) and tertiary care (24.4\%, n = 670, \emph{p} \textless{} 0.001). Yet, no significant difference was observed in the number of CoNS isolates found in ICU patients between secondary care (21.0\%, n = 2,191) and tertiary care (22.8\%, n = 1,034).

Table 1. Numbers and characteristics per gender of included patients of the included CoNS isolates.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-t01} 

}

\end{figure}

At total of 27 different species of the CoNS group were found within the isolate collection (Table 2). The top five species covered 97.1\% (n = 14,560) of all first weighted isolates: \emph{S. epidermidis} (n = 7,260, 48.4\%), \emph{S. hominis} (n = 5,033, 33.6\%), \emph{S. capitis} (n = 1,395, 9.3\%), \emph{S. haemolyticus} (n = 612, 4.1\%), and \emph{S. warneri} (n = 260, 1.7\%).

The remaining 432 isolates (2,9\%) consisted of: \emph{S. lugdunensis} (n = 91, 0.6\%), \emph{S. saprophyticus} (n = 45, 0.3\%), \emph{S. pettenkoferi} (n = 44, 0.3\%), \emph{S. cohnii} (n = 43, 0.3\%), \emph{S. caprae} (n = 40, 0.2\%), and 17 other species (n = 169, 1.1\%).

Table 2. Overview of the total number of isolated CoNS species (not only first isolates) found between 2013 and 2019 in the Northern Netherlands.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-t02} 

}

\end{figure}

\hypertarget{occurrence-of-cons-species}{%
\subsection{Occurrence of CoNS species}\label{occurrence-of-cons-species}}

The occurrence of CoNS species was stratified by type of care, type of hospital ward, geographic province, gender, and age (Figure 2). Age was grouped into five groups: 0-11, 12-24, 12-24, 25-54, 55-74, and 75 or more years. When stratifying by species level and the different types of care, the proportion of \emph{S. epidermidis} among all CoNS isolates was 62.5\% in tertiary care (n = 2,834) versus 42.3\% in secondary care (n = 4,426; \emph{p} = 0.049). Overall, \emph{S. hominis} was less occurrent in tertiary care (20.3\%, n = 919) than in secondary care (39.4\%, n = 4,114, \emph{p} = 0.013), while the occurrence of other CoNS species was comparable between secondary and tertiary care. Yet, major differences in relative occurrence were observed between ICU and non-ICU status in secondary care. On secondary care ICUs, \emph{S. epidermidis} accounted for 55.9\% of all first weighted CoNS isolates found while on non-ICU wards this was 39.1\% (\emph{p} \textless{} 0.001). In contrast, \emph{S. hominis} accounted for 25.7\% on secondary care ICUs while on non-ICU wards this was 43.3\% (\emph{p} \textless{} 0.001). Notably, \emph{S. hominis} was found 105 times (7.53\%) in children under the age of one.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-02} 

}

\caption{The number of first weighted isolates of the top five CoNS species found in the study stratified by (A) type of care, (B) type of hospital ward, (C) province of the Netherlands, (D), gender, and (E) age group.}\label{fig:fig7-2}
\end{figure}

Although all three provinces in the study region are similar in population density and gender distribution \textsuperscript{{[}23{]}}, major differences were observed in the occurrence of CoNS species between those provinces in secondary care. The occurrence of \emph{S. epidermidis} among CoNS species in secondary care hospitals in Friesland was 38.7\% in contrast to 43.7\% and 45.9\% in Drenthe and Groningen respectively (\emph{p} \textless{} 0.001). \emph{S. hominis} was significantly more often found in secondary care hospitals in Friesland (45.9\%) than in Drenthe (33.3\%) and Groningen (36.0\%) (\emph{p} \textless{} 0.001). Drenthe and Groningen did not differ significantly in the occurrence of CoNS species in secondary care.

Overall, there was no significant change in species distribution over the years. Stratified by gender, a linear increase of \emph{S. hominis} over time (\emph{p} = 0.001) and a decrease of \emph{S. epidermidis} (\emph{p} = 0.005) was found in males. In females, the occurrence of \emph{S. hominis} also increased over time (\emph{p} = 0.008), but no decrease of \emph{S. epidermidis} or any other species was observed. In age groups, no significant trends in occurrence were observed.

\hypertarget{definition-of-cons-persistence}{%
\subsection{Definition of CoNS persistence}\label{definition-of-cons-persistence}}

In this retrospective study, it was impossible to differentiate between contaminated blood cultures and BSI-associated blood cultures, as clinical information was not available. Yet, to assess probable cases of BSIs caused by CoNS, we defined `CoNS persistence' as a surrogate. CoNS persistence was defined by at least three positive blood cultures drawn on three different days within 60 days containing the same CoNS species within the same patient. In total, we identified 294 cases of CoNS persistence (Table 3). Aside from \emph{S. massiliensis} that caused CoNS persistence in only one patient, the relatively most common causal agent of CoNS persistence was \emph{S. haemolyticus} (5.8\%, n = 32, \emph{p} \textless{} 0.001), followed by \emph{S. epidermidis} (3.7\%, n = 212, \emph{p} \textless{} 0.001), and \emph{S. lugdunensis} (3.4\%, n = 3, \emph{p} = 0.46).

Table 3. The number of patients with and without CoNS persistence per species.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-t03} 

}

\end{figure}

\hypertarget{antibiotic-resistance-analysis}{%
\subsection{Antibiotic resistance analysis}\label{antibiotic-resistance-analysis}}

Clinically relevant antibiotics and their respective ABR profiles were analysed and compared for the top five CoNS species. Figure 3 shows time trends regarding the ABR profiles to ten different clinically relevant antibiotics, while Table 4 contains resistance percentages of all applicable combinations of species and antibiotic agents. In the following subsections, more detail on occurrence and trends is provided per antibiotic class based on Figure 3 and Table 4. Comprehensive ABR analyses per species of all available variables can be found in Supplementary Table 3.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-03} 

}

\caption{Antibiotic resistance of the five most occurrent CoNS (n = 14,560) over time between 2013 and 2019. Lines and points are missing where there were less than 30 isolates available for analysis.}\label{fig:fig7-3}
\end{figure}

Table 4. Antibiotic resistance in all first weighted CoNS isolates in blood between 2013 and 2019 where at least 30 isolates were available for ABR analysis. Resistance of 100\% denotes intrinsic resistance, as defined by EUCAST. Between parentheses are the number of resistant first weighted isolates and the total number of first weighted isolates for that bug-drug combination. The antibiotic names are followed by the official EARS-Net code (European Antimicrobial Resistance Surveillance Network) and ATC code (Anatomical Therapeutic Chemical).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-t04} 

}

\end{figure}
\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-t04b} 

}

\end{figure}

\hypertarget{glycopeptides}{%
\subsubsection{Glycopeptides}\label{glycopeptides}}

Vancomycin resistance was found in six \emph{S. epidermidis} isolates (0.1\%) and in one \emph{S. hominis} isolate (0.0\%). Half of all \emph{S. epidermidis} isolates showed resistance to teicoplanin (50.5\%, n = 2,752), which increased over the seven study years (min-max: 44.8\%-54.5\%, \emph{p} = 0.001). An increase in teicoplanin resistance was observed in \emph{S. haemolyticus} (min-max: 10.9\%-44.0\%, \emph{p} \textless{} 0.001). Teicoplanin resistance remained low in \emph{S. capitis} (1.4\%, n = 17), \emph{S. hominis} (5.1\%, n = 202), and \emph{S. warneri} (9.6\%, n = 22).

\hypertarget{macrolides}{%
\subsubsection{Macrolides}\label{macrolides}}

Erythromycin resistance was highest in \emph{S. haemolyticus} (77.6\%, n = 437), followed by in \emph{S. epidermidis} (51.5\%, n = 3,471), \emph{S. hominis} (45.7\%, n = 2,086), \emph{S. warneri} (17.5\%, n = 40), and \emph{S. capitis} (11.0\%, n = 136). Resistance to azithromycin and clarithromycin was equal to erythromycin resistance, due to EUCAST expert rules. However, resistance to clindamycin remained lower than resistance to erythromycin in all species: 45.6\% (n = 253) in \emph{S. haemolyticus} and 43.4\% (n = 2,910) in \emph{S. epidermidis}, 29.6\% (n = 1,347) in \emph{S. hominis}, 4.4\% (n = 10) in \emph{S. warneri} ,and 10.8\% (n = 132) in \emph{S. capitis}.

\hypertarget{fluoroquinolones}{%
\subsubsection{Fluoroquinolones}\label{fluoroquinolones}}

The highest ciprofloxacin resistance was found in \emph{S. haemolyticus} (66.4\%; n = 374) and \emph{S. epidermidis} (51.5\%; n = 3,468). Resistance to moxifloxacin was 26.4\% (n = 24) in \emph{S. haemolyticus} and less than 10\% in all other species.

\hypertarget{beta-lactamspenicillins}{%
\subsubsection{Beta-lactams/penicillins}\label{beta-lactamspenicillins}}

Oxacillin resistance was as high as 61.9\% (n = 4,135) in \emph{S. epidermidis}, which was thus the proportion of MRSE (methicillin-resistant \emph{S. epidermidis}) among all \emph{S. epidermidis} isolates in this study. Oxacillin resistance in \emph{S. haemolyticus} was even higher (72.1\%, n = 403) but considerably lower in all other CoNS species (13.4\%-38.6\%). Almost all \emph{S. epidermidis}, \emph{S. haemolyticus}, and \emph{S. hominis} were resistant to amoxicillin (95.4\%, 93.6\%, and 92.8\% respectively), while all other species showed amoxicillin resistance ranging between 64.8\% and 73.5\%. Resistance to amoxicillin/clavulanic acid was 72.9\% (n = 3,026) in \emph{S. epidermidis}. \emph{S. haemolyticus} showed a strong linear increase in amoxicillin/clavulanic acid resistance (\emph{p} \textless{} 0.001) since 2013 with 87\% resistance in 2019 (n = 61).

\hypertarget{other-antibiotics}{%
\subsubsection{Other antibiotics}\label{other-antibiotics}}

Resistance remained low to rifampicin in \emph{S. haemolyticus} (5.0\%; n = 28) and \emph{S. epidermidis} (4.5\%; n = 300) and remained less than 0.6\% in all other species. Linezolid resistance was 0.4\% (n = 5) in \emph{S. capitis}, 0.4\% (n = 17) in \emph{S. hominis}, 0.2\% (n = 5) in \emph{S. haemolyticus}, 0.1\% (n = 5) in \emph{S. epidermidis}, and absent in \emph{S. warneri}. Mupirocin resistance was 14.8\% in \emph{S. epidermidis} (n = 987, of note: 166 additional isolates tested as ``I'') and between 1.7\% and 6.5\% in other species.

\hypertarget{other-relevant-species}{%
\subsubsection{Other relevant species}\label{other-relevant-species}}

Resistance in \emph{S. lugdunensis} (n = 82, sixth most occurrent species) remained generally low: 11.9\% (n = 5) to amoxicillin/clavulanic acid, 7.3\% (n = 6) to oxacillin, 4.8\% (n = 4) to ciprofloxacin, 15.4\% (n = 10) to tetracycline, 3.7\% (n = 3) to teicoplanin, and no resistance was observed to rifampicin, linezolid, and vancomycin.

\emph{S. saprophyticus} (n = 45, seventh-most occurrent species) showed no resistance to ciprofloxacin, teicoplanin, rifampicin, and vancomycin. Resistance to erythromycin was 15.4\% (n = 6), to linezolid 7.9\% (n = 3), and to oxacillin 16.2\% (n = 6).

\emph{S. pettenkoferi} (n = 44, eighth-most occurrent species) showed no resistance to gentamicin, tobramycin, linezolid, teicoplanin, or vancomycin but resistance to oxacillin was 40.4\% (n = 14). Resistance to ciprofloxacin (8.1\%, n = 3) and trimethoprim/sulfamethoxazole (2.7\%, n = 1) remained low.

\hypertarget{effect-of-patient-age-groups-on-antibiotic-resistance-in-cons}{%
\subsubsection{Effect of patient age groups on antibiotic resistance in CoNS}\label{effect-of-patient-age-groups-on-antibiotic-resistance-in-cons}}

Thirty bug-drug combinations were analysed of which 13 showed a significant linear trend associated with age groups (Figure 4). In \emph{S. epidermidis}, resistance to beta-lactam antibiotics was found to be lower in older patients (amoxicillin/clavulanic acid: \emph{p} = 0.002; cefuroxime: \emph{p} = 0.014). This was also observed in all aminoglycosides (e.g., gentamicin: \emph{p} = 0.017; tobramycin: \emph{p} = 0.009), except for kanamycin where higher age was associated with increasing resistance (\emph{p} = 0.011). \emph{S. epidermidis} was also less resistant to carbapenems in older patients (imipenem: \emph{p} = 0.046; meropenem: \emph{p} = 0.047). In \emph{S. hominis}, similar trends were observed, although the effect of resistance to kanamycin was stronger (\emph{p} = 0.006). \emph{S. capitis} showed significantly more resistance to tetracycline (\emph{p} = 0.022) in older patients.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/07-04} 

}

\caption{Age group comparison of ABR per antibiotic. Only bug-drug combinations are shown where at least 30 isolates were available for each age group and where results for all age groups were available.}\label{fig:fig7-4}
\end{figure}

\hypertarget{discussion-3}{%
\section{Discussion}\label{discussion-3}}

The present study provides a comprehensive analysis of species in the CoNS group and their associated ABR patterns in a full-region approach using solely MALDI-TOF MS for discriminating CoNS species. We selected and analysed a total of 14,992 first weighted CoNS isolates from 10,786 patients over seven years and identified significant differences in the trends of occurrence of the different CoNS species as well as in their ABR patterns.

Before MALDI-TOF MS, CoNS were often reported without the species name as formerly used techniques were not able to reliably discriminate species \textsuperscript{{[}16{]}}. The ratio of all CoNS species presented in the current study (Table 2) shows that five species accounted for 97.1\% of all 27 found CoNS species with \emph{S. epidermidis} accounting for the largest subgroup (48.4\%, n = 7,260). This distribution of species largely confirms results by previous reports \textsuperscript{{[}9,29{]}}.

For most CoNS species, pathogenicity has not been studied widely due to the lack of data. For this reason, we defined CoNS persistence as at least three positive blood cultures drawn on three different days within 60 days, containing the same CoNS species. This definition was applied for two reasons. Firstly, it rules out contamination since the chance of finding the same contaminating species three times on three different days is expected to be low. Secondly, it prevents underestimating the possible pathogenicity of CoNS species since three sequential findings indicate CoNS persistence. In total, 294 different cases of CoNS persistence were identified (Table 3) among the 10,786 included patients. \emph{S. haemolyticus} was found to be proportionally more associated with CoNS persistence (5.8\%) than \emph{S. epidermidis} (3.7\%) and \emph{S. hominis} (0.9\%), although the latter two were eight to ten times more prevalent than \emph{S. haemolyticus}. \emph{S. epidermidis} has widely been recognised as a pathogen and an important cause of BSIs \textsuperscript{{[}5,30{]}}. It was probably found more often than \emph{S. haemolyticus} due to its stronger association with skin colonisation \textsuperscript{{[}8{]}} although we could not confirm this finding. It has been reported that \emph{S. haemolyticus} is an emerging threat and one of the most frequent aetiological factors of staphylococcal infections \textsuperscript{{[}9,31{]}}. Adding to this worrisome trend is the great concern of ABR in \emph{S. haemolyticus} which was reported with 75\% of analysed \emph{S. haemolyticus} isolates to be multi-resistant \textsuperscript{{[}32{]}}. We confirmed this in the present study in which the ABR analysis showed that 72.1\% of \emph{S. haemolyticus} isolates were resistant to oxacillin and 77.6\% resistant to macrolides.

ABR analysis also showed substantial differences between CoNS species (Figure 3, Figure 4, Table 4). This observation could be supported by a recent study that showed strong heterogeneity in the resistance genes for CoNS species \textsuperscript{{[}33{]}}. For example, the \emph{blaZ} and \emph{aac-aphD} genes that can lead to penicillin and aminoglycoside resistance, respectively, were found to be up to four times more common in \emph{S. haemolyticus} than in other CoNS species \textsuperscript{{[}33{]}}. The level of resistance to oxacillin and consequent amount of methicillin-resistant \emph{S. epidermidis} (MRSE) identified in the present study (61.9\%) could also be supported by the mentioned study, that reported high prevalence of \emph{blaZ} in \emph{S. epidermidis} (64.2\%). Although differences in occurrence and ABR within CoNS species are known, they are often neglected, both in studies and in clinical practice. As an example, the Dutch national report on ABR and antibiotic consumption, NethMap, combines all CoNS species into one category making it impossible to distinguish between species. Nonetheless, Dutch treatment guidelines are based on NethMap \textsuperscript{{[}34{]}}. As an example, in 2019 NethMap reported for isolates found on ICUs 0\% linezolid resistance in CoNS, 8\% rifampicin resistance, and more than 20\% resistance in all other antibiotic classes in 2019. These results could be confirmed in the present study on the group level but not on the species level. The lack of acknowledging ABR differences within species might cause the development of treatment guidelines -- and the subsequent future treatment of BSI caused by CoNS -- to focus on `ABR-safe' agents for treating CoNS, such as vancomycin or linezolid. Still, agents such as tetracycline, co-trimoxazole, and erythromycin could be considered viable options for some species where, according to our results, ABR never surpassed 10\%. Furthermore, as age showed to have a significant effect on ABR (Figure 4), treatment guidelines could also be improved by incorporating age-specific recommendations. We could not find the correlation between ABR in CoNS species and age in previous literature.

In the present study, some CoNS species are noteworthy to be highlighted. For instance, \emph{S. pettenkoferi} was found only two to three times per year between 2013 and 2017 while this increased to 13 and 22 times per year in 2018 and 2019, respectively. Although recently named, multiple case studies showed that \emph{S. pettenkoferi} was found to be the causative agent of septic shock, bacteraemia, and wound infections and has also shown resistance to linezolid \textsuperscript{{[}35--37{]}}. Opposingly, no linezolid resistance was found in the present study. Cases of BSI caused by \emph{S. pettenkoferi} could incorrectly be assigned to \emph{S. capitis} that greatly resembles \emph{S. pettenkoferi} \textsuperscript{{[}38{]}}. The emerging neonatal pathogen \emph{S. capitis} is another noteworthy species causing sepsis and manifesting as a multidrug-resistant microorganism \textsuperscript{{[}39{]}}. In this study, 7.53\% of all first weighted \emph{S. capitis} isolates was found in one-year old children. Clinically relevant ABR (e.g., to chloramphenicol or vancomycin) was not found in these children in this study. This implies that the internationally emerging \emph{S. capitis} NRCS-A clone \textsuperscript{{[}39{]}} has not been found in the Northern Netherlands between 2013 and 2019.

Our study has limitations, mostly due to its sole source of routine diagnostic data. Firstly, it was not known which isolates were causal to BSI. This hinders the assessment of contamination as well as the determination of clinical importance. Secondly, the VITEK 2 systems between laboratories used different cartridges with different antibiotics which could lead to an incorporation bias towards some laboratories or hospitals. Additionally, the MALDI-TOF MS systems of all laboratories keep their taxonomic reference data, which is proprietary, and the recency could not be assessed. Thirdly, no genotyping was available for any of the included isolates since genotyping was not considered common practice for routine diagnostic workflows at the time of the study. For this reason, no assessment could be made about a hospital-associated cluster of strains. Lastly, vancomycin resistance might have been underdiagnosed in this study since Vitek2 AST is not optimal for testing glycopeptide resistance \textsuperscript{{[}40{]}}.

For the first time, a multi-year, full-region approach to comprehensively assess both the occurrence and ABR patterns of CoNS species based on MALDI-TOF MS results was carried out. Although CoNS often lack aggressive virulence properties, evaluating the occurrence and ABR patterns remains highly relevant \textsuperscript{{[}9{]}}. Stratification by region and demography unveiled a large heterogeneity in ABR between species, settings, and age groups which could be used for (re-)evaluating treatment policies and understanding more about these important but still too often neglected pathogens.

\hypertarget{supplementary-tables}{%
\section*{Supplementary tables}\label{supplementary-tables}}
\addcontentsline{toc}{section}{Supplementary tables}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Supplementary Table 1 (file ``supp\_tbl1.xlsx''): Extensive output of EUCAST changes to the original data set.
\item
  Supplementary Table 2 (file ``supp\_tbl2.xlsx''): List of species and all available AST test results. This file contains a SHA2 hash (256-bit) of the patient IDs, to be able to reproduce some part of the Results section on the patient level. The hash contains irretrievable information, rendering the data set strictly anonymous.
\item
  Supplementary Table 3 (file ``supp\_tbl3.xlsx''): ABR analysis per species for all available variables.
\end{enumerate}

\hypertarget{references-6}{%
\section*{References}\label{references-6}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Reinhart K, Daniels R, Kissoon N, Machado FR, Schachter RD, Finfer S. Recognizing Sepsis as a Global Health Priority --- A WHO Resolution. N Engl J Med 2017;377:414--7. \url{doi:10.1056/NEJMp1707170}.
\item
  Singer M, Deutschman CS, Seymour CW, Shankar-Hari M, Annane D, Bauer M, \emph{et al.} The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3). JAMA 2016;315:801--10. \url{doi:10.1001/jama.2016.0287}.
\item
  Cheung GYC, Otto M. Understanding the significance of \emph{S. epidermidis} bacteremia in babies and children. Curr Opin Infect Dis 2010;23:208--16. \url{doi:10.1097/QCO.0b013e328337fecb}.
\item
  Stearns-Kurosawa DJ, Osuchowski MF, Valentine C, Kurosawa S, Remick DG. The pathogenesis of sepsis. Annu Rev Pathol 2011;6:19--48. \url{doi:10.1146/annurev-pathol-011110-130327}.
\item
  Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. \emph{S. epidermidis} as a cause of bacteremia. Future Microbiol 2015;10:1859--79. \url{doi:10.2217/fmb.15.98}.
\item
  Rogers KL, Fey PD, Rupp ME. Coagulase-Negative Staphylococcal Infections. Infect Dis Clin North Am 2009;23:73--98. \url{doi:10.1016/j.idc.2008.10.001}.
\item
  Dong Y, Speer CP. The role of \emph{S. epidermidis} in neonatal sepsis: Guarding angel or pathogenic devil? Int J Med Microbiol 2014;304:513--20. \url{doi:10.1016/j.ijmm.2014.04.013}.
\item
  Heilmann C, Ziebuhr W, Becker K. Are coagulase-negative staphylococci virulent? Clin Microbiol Infect 2019;25:1071--80. \url{doi:10.1016/j.cmi.2018.11.012}.
\item
  Becker K, Heilmann C, Peters G. Coagulase-Negative staphylococci. Clin Microbiol Rev 2014;27:870--926. \url{doi:10.1128/CMR.00109-13}.
\item
  Sievert DM, Ricks P, Edwards JR, Schneider A, Patel J, Srinivasan A, \emph{et al.} Antimicrobial-Resistant Pathogens Associated with Healthcare-Associated Infections Summary of Data Reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2009--2010. Infect Control Hosp Epidemiol 2013;34:1--14. \url{doi:10.1086/668770}.
\item
  Becker K, Both A, Weißelberg S, Heilmann C, Rohde H. Emergence of coagulase-negative staphylococci. Expert Rev Anti Infect Ther 2020;18:349--66. \url{doi:10.1080/14787210.2020.1730813}.
\item
  Rosenstein R, Götz F. What distinguishes highly pathogenic staphylococci from medium- and non-pathogenic? Curr Top Microbiol Immunol 2013;358:33--89. \url{doi:10.1007/82_2012_286}.
\item
  Idelevich EA, Reischl U, Becker K. New Microbiological Techniques in the Diagnosis of Bloodstream Infections. Dtsch Arztebl Int 2018;115:822--32. \url{doi:10.3238/arztebl.2018.0822}.
\item
  Eubank TA, Long SW, Perez KK. Role of Rapid Diagnostics in Diagnosis and Management of Patients With Sepsis. J Infect Dis 2020;222:S103--9. \url{doi:10.1093/infdis/jiaa263}.
\item
  Pantůček R, Sedláček I, Indráková A, Vrbovská V, Mašlaňová I, Kovařovic V, \emph{et al.} \emph{S. edaphicus} sp. nov., Isolated in Antarctica, Harbors the mecC Gene and Genomic Islands with a Suspected Role in Adaptation to Extreme Environments. Appl Environ Microbiol 2018;84:e01746-17. \url{doi:10.1128/AEM.01746-17}.
\item
  Kassim A, Pflüger V, Premji Z, Daubenberger C, Revathi G. Comparison of biomarker based Matrix Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS) and conventional methods in the identification of clinically relevant bacteria and yeast. BMC Microbiol 2017;17:128. \url{doi:10.1186/s12866-017-1037-z}.
\item
  Spanu T, Sanguinetti M, Ciccaglione D, D'Inzeo T, Romano L, Leone F, \emph{et al.} Use of the VITEK 2 system for rapid identification of clinical isolates of Staphylococci from bloodstream infections. J Clin Microbiol 2003;41:4259--63. \url{doi:10.1128/JCM.41.9.4259-4263.2003}.
\item
  Lane MJ, Roy AF, Kearney MT, Pucheu-Haston CM. Characterization, distribution, antimicrobial resistance and resistance risk factors in staphylococci isolated from cats from 2001 to 2014. Vet Med Sci 2018;4:315--25. \url{doi:10.1002/vms3.122}.
\item
  Wisplinghoff H, Rosato AE, Enright MC, Noto M, Craig W, Archer GL. Related clones containing SCCmec type IV predominate among clinically significant \emph{S. epidermidis} isolates. Antimicrob Agents Chemother 2003;47:3574--9. \url{doi:10.1128/aac.47.11.3574-3579.2003}.
\item
  Ramos-Martín V, Johnson A, Livermore J, McEntee L, Goodwin J, Whalley S, \emph{et al.} Pharmacodynamics of vancomycin for CoNS infection: experimental basis for optimal use of vancomycin in neonates. J Antimicrob Chemother 2016;71:992--1002. \url{doi:10.1093/jac/dkv451}.
\item
  Greeff S, Mouton J. NethMap 2018: Consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in the Netherlands / MARAN 2018: Monitoring of Antimicrobial Resistance and Antibiotic Usage in Animals in the Netherlands in 2017. Bilthoven: Rijksinstituut voor Volksgezondheid en Milieu RIVM Stichting Werkgroep Antibiotica Beleid SWAB; 2018.
\item
  Stichting Werkgroep Antibiotica Beleid (SWAB), 2010. Antibacterial therapy of adult patients with sepsis. Retrieved from \url{https://www.swab.nl/english}.
\item
  Statistics Netherlands {[}Centraal Bureau voor de Statistiek; CBS{]}. Regionale kerncijfers Nederland. 2018.
\item
  Clinical and Laboratory Standards Institute (CLSI). M39-A4 Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 2014.
\item
  Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR - An R Package for Working with Antimicrobial Resistance Data. J Stat Softw 2021:Accepted for publication. \url{doi:10.1101/810622}.
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Intrinsic Resistance and Exceptional Phenotypes. Version 3.1. 2016.
\item
  EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Breakpoint tables for interpretation of MICs and zone diameters. Version 10.0. 2020.
\item
  R Core Team. R: A language and environment for statistical computing. 2021.
\item
  Otto M. \emph{S. epidermidis}: a major player in bacterial sepsis? Future Microbiol 2017;12:1031--3. \url{doi:10.2217/fmb-2017-0143}.
\item
  Ahlstrand E, Hellmark B, Svensson K, Söderquist B. Long-term molecular epidemiology of \emph{S. epidermidis} blood culture isolates from patients with hematological malignancies. PLoS One 2014;9:e99045. \url{doi:10.1371/journal.pone.0099045}.
\item
  Czekaj T, Ciszewski M, Szewczyk EM. \emph{S. haemolyticus} -- an emerging threat in the twilight of the antibiotics age. Microbiology 2015;161:2061--8. \url{doi:10.1099/mic.0.000178}.
\item
  Barros EM, Ceotto H, Bastos MCF, Dos Santos KRN, Giambiagi-Demarval M. \emph{S. haemolyticus} as an important hospital pathogen and carrier of methicillin resistance genes. J Clin Microbiol 2012;50:166--8. \url{doi:10.1128/JCM.05563-11}.
\item
  Pedroso SHSP, Sandes SHC, Filho RAT, Nunes AC, Serufo JC, Farias LM, \emph{et al.} Coagulase-Negative Staphylococci Isolated from Human Bloodstream Infections Showed Multidrug Resistance Profile. Microb Drug Resist 2018;24:635--47. \url{doi:10.1089/mdr.2017.0309}.
\item
  RIVM/SWAB. NethMap 2019 2019:87.
\item
  Vecchia M, Lombardi A, Zuccaro V, Mariani B, Morea A, Asperges E, \emph{et al.} A clinical case of sepsis due to staphylococcus pettenkoferi. Infez Med 2018;26:369--73.
\item
  Mihaila L, Defrance G, Levesque E, Ichai P, Garnier F, Derouin V, \emph{et al.} A dual outbreak of bloodstream infections with linezolid-resistant \emph{S. epidermidis} and \emph{S. pettenkoferi} in a liver Intensive Care Unit. Int J Antimicrob Agents 2012;40:472--4. \url{doi:10.1016/j.ijantimicag.2012.06.014}.
\item
  Hashi AA, Delport JA, Elsayed S, Silverman MS. \emph{S. pettenkoferi} Bacteremia: A Case Report and Review of the Literature. Can J Infect Dis Med Microbiol 2015;26:319--22. \url{doi:10.1155/2015/748154}.
\item
  Trülzsch K, Grabein B, Schumann P, Mellmann A, Antonenka U, Heesemann J, \emph{et al.} \emph{S. pettenkoferi} sp. nov., a novel coagulase-negative staphylococcal species isolated from human clinical specimens. Int J Syst Evol Microbiol 2007;57:1543--8. \url{doi:10.1099/ijs.0.64381-0}.
\item
  Carter GP, Ussher JE, Da Silva AG, Baines SL, Heffernan H, Riley T V., \emph{et al.} Genomic Analysis of Multiresistant \emph{S. capitis} Associated with Neonatal Sepsis. Antimicrob Agents Chemother 2018;62. \url{doi:10.1128/AAC.00898-18}.
\item
  Rybak MJ, Vidaillac C, Sader HS, Rhomberg PR, Salimnia H, Briski LE, \emph{et al.} Evaluation of vancomycin susceptibility testing for methicillin-resistant \emph{S. aureus}: comparison of Etest and three automated testing methods. J Clin Microbiol 2013;51:2077--81. \url{doi:10.1128/JCM.00448-13}.
\end{enumerate}

\hypertarget{ch08-defining-mdr}{%
\chapter{Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines}\label{ch08-defining-mdr}}

Published in Microorganisms, 2018 Jan 26;6(1):11

Köck R \textsuperscript{1,2,3}, Siemer P \textsuperscript{4}, Esser J \textsuperscript{5}, Kampmeier S \textsuperscript{2}, Berends MS \textsuperscript{6,7}, Glasner C \textsuperscript{7}, Arends JP \textsuperscript{7}, Becker K \textsuperscript{3}, Friedrich AW \textsuperscript{7}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Institute of Hospital Hygiene Oldenburg, Oldenburg, Germany
\item
  Institute of Hygiene, University Hospital Münster, Münster, Germany
\item
  Institute of Medical Microbiology, University Hospital Münster, Münster, Germany
\item
  European Medical School Oldenburg-Groningen, Oldenburg, Germany
\item
  Laborarztpraxis Osnabrück, Georgsmarienhütte, Germany
\item
  Certe Medical Diagnostics \& Advice Foundation, Groningen, the Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology, Groningen, the Netherlands
\end{enumerate}

\hypertarget{abstract-6}{%
\section*{Abstract}\label{abstract-6}}
\addcontentsline{toc}{section}{Abstract}

Preventing the spread of multidrug-resistant Gram-negative bacteria (MDRGNB) is a public health priority. However, the definition of MDRGNB applied for planning infection prevention measures such as barrier precautions differs depending on national guidelines. This is particularly relevant in the Dutch-German border region, where patients are transferred between healthcare facilities located in the two different countries, because clinicians and infection control personnel must understand antibiograms indicating MDRGNB from both sides of the border and using both national guidelines. This retrospective study aimed to compare antibiograms of Gram-negative bacteria and classify them using the Dutch and German national standards for MDRGNB definition. A total of 31,787 antibiograms from six Dutch and four German hospitals were classified. Overall, 73.7\% were no MDRGNB according to both guidelines. According to the Dutch and German guideline, 7772/31,787 (24.5\%) and 4586/31,787 (12.9\%) were MDRGNB, respectively (\emph{p} \textless{} 0.0001). Major divergent classifications were observed for extended-spectrum β-lactamase (ESBL) producing \emph{Enterobacteriaceae}, non-carbapenemase-producing carbapenem-resistant \emph{Enterobacteriaceae}, \emph{Pseudomonas aeruginosa} and \emph{Stenotrophomonas maltophilia}. The observed differences show that medical staff must carefully check previous diagnostic findings when patients are transferred across the Dutch-German border, as it cannot be assumed that MDRGNB requiring special hygiene precautions are marked in the transferred antibiograms in accordance with both national guidelines.

\hypertarget{introduction-5}{%
\section{Introduction}\label{introduction-5}}

Antimicrobial multidrug-resistant Gram-negative bacteria (MDRGNB) globally challenge clinicians and infection control personnel due to limited treatment options and the need to implement barrier precautions for preventing MDRGNB transmission \textsuperscript{{[}1{]}}. Comparing this challenge is particularly interesting in neighbouring regions characterised by highly developed but structurally different healthcare systems. An example for such a region is the Dutch-German border area, which is inhabited by 12 million people and comprises \textgreater100 hospitals.

In the Netherlands and Germany, surveillance systems currently indicate that 7.0\% and 11.8\% of all \emph{Escherichia coli} and 10.8\% and 14.3\% of all \emph{Klebsiella pneumoniae} isolated from blood cultures are non-susceptible to third-generation-cephalosporins indicative for production of extended-spectrum β-lactamases (ESBL) \textsuperscript{{[}2{]}}. Moreover, carbapenemase-producing \emph{Enterobacteriaceae} (CPE) occur in both countries, although the overall meropenem or imipenem resistance rates of \emph{Enterobacteriaceae} (e.g., \emph{Klebsiella spp.}) are still \textless1\% \textsuperscript{{[}2{]}}. Thirdly, carbapenem resistance in \emph{Acinetobacter baumannii}, which is often due to carbapenem-hydrolysing oxacillinase (OXA) production, affects 1.9\% and 5.4\% of all invasive isolates in The Netherlands and Germany respectively \textsuperscript{{[}2{]}}. A fourth clinically relevant species is \emph{Pseudomonas aeruginosa.} For this bacterium, 11\% and 18\% of all isolates from bloodstream infections were non-susceptible to ceftazidime and meropenem in Germany, respectively. In contrast, resistance rates were 3.5\% and 6.1\% in The Netherlands \textsuperscript{{[}2{]}}.

Nosocomial transmission is a major reason why the incidence of MDRGNB increases. Hence, infection control guidelines describing measures to prevent MDRGNB dissemination are implemented in many countries including The Netherlands and Germany. However, it should be noted that, according to data from the European Centre for Disease Prevention and Control (ECDC), Germany is currently considered as a country, where CPE are regionally endemic indicating inter-institutional spread, while their occurrence is more limited in The Netherlands. The same is observed for carbapenem-resistant \emph{A. baumannii} \textsuperscript{{[}3{]}}. This highlights the need to critically evaluate and compare infection control guidelines, as well as different risks for MDRGNB spread in these two countries.

In this context, one aspect is the definition of MDRGNB. Although definitions for multidrug resistance in epidemiological studies are available \textsuperscript{{[}4{]}}, and although theoretically CPE or ESBL-producing \emph{Enterobacteriaceae} are clearly defined by harbouring respective resistance encoding genes, the questions concerning what MDRGNB are in clinical routine and for which MDRGNB special barrier precautions should be implemented, are not universally defined. Moreover, it is important to differentiate between MDRGNB definitions established for therapeutic decisions and those created for epidemiological purposes and infection prevention \textsuperscript{{[}4{]}}. Recently, Müller \emph{et al.} have pointed out differences between the Dutch and German guidelines regarding the advice they give to laboratories and infection control personnel, which Gram-negative bacteria and antimicrobial resistance patterns should be considered as MDRGNB \textsuperscript{{[}5{]}}. As patient movement across the Dutch-German border is not infrequent, such divergent definitions could result in reduced patient safety, because isolates requiring isolation in the hospital abroad are not flagged as being multidrug-resistant on the microbiological reports.

Hence, in this article, we collected antibiograms of Gram-negative bacteria from Dutch and German hospitals located in the border region and applied both national MDRGNB definitions for infection prevention measures on this dataset. The results of this comparison shall clarify the differences between the two countries and estimate the impact of these differences for daily infection control practice.

\hypertarget{methods-2}{%
\section{Methods}\label{methods-2}}

We retrospectively extracted antibiograms of Gram-negative bacteria from laboratory information systems. Data about phenotypic and genotypic ESBL and carbapenemase tests performed for the respective bacterial isolates were also extracted, if available. All isolates included originate from patients treated in six Dutch and four German hospitals. All hospitals are located in the Dutch-German border region including the Northern part (Ems Dollart region) and the central part (EUREGIO). Five of six Dutch hospitals provided datasets from 1 January 2015 to 31 December 2016, because only a small number of samples was tested in these facilities; the sixth Dutch hospital and the German hospitals provided data for 2016 only. Antimicrobial susceptibility testing was done using guidelines of the European Committee on Antimicrobial Susceptibility Testing (EUCAST) guidelines and clinical breakpoints in all laboratories.

Anonymisation of patient-related and hospital-related data was done before analysis. We initially included all Gram-negative bacterial species and then restricted the dataset to \emph{Enterobacteriaceae}, P. aeruginosa, \emph{A. baumannii} complex, and \emph{Stenotrophomonas maltophilia}, as these are the species for which recommendations regarding MDRGNB definitions and special hygiene precautions are included in Dutch and German infection control guidelines \textsuperscript{{[}6,7{]}}. We included all isolates; duplicate isolates from the same patient were not removed. Classification of MDRGNB was done according the German national guideline (MDRGNB classified according to this guideline are henceforth designated ``Multiresistente Gramnegative Stäbchen'', MRGN, with the subtypes 3MRGN and 4MRGN) summarised in Table 1 \textsuperscript{{[}6{]}} and according to the Dutch national guideline (MDRGNB according to this guideline are henceforth designated ``Bijzonder Resistente Micro-Organismen'', BRMO) shown in Table 2 \textsuperscript{{[}7{]}}, for all isolates including complete phenotypic susceptibility test data for the antibiotics mentioned. Incomplete antibiograms were deleted from the dataset.

Table 1. Classification according to German guideline into 3MRGN and 4MRGN.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/08-t01} 

}

\end{figure}

Table 2. Classification according to Dutch guideline into BRMO.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/08-t02} 

}

\end{figure}

Statistical analysis was done by Epi Info (version 7.0, CDC Atlanta, Atlanta, GA, USA) using Chi-Square or (where appropriate) Fisher's exact test; \emph{p} \textless{} 0.05 was considered significant. The final dataset does not allow for conclusions about the epidemiology or the prevalence of MDRGNB, as it contains both isolates obtained from screening asymptomatic patients and clinical specimens. Moreover, the diagnostic procedures and indications for screening and clinical diagnostics were not harmonised in the participating laboratories and hospitals.

\hypertarget{results-3}{%
\section{Results}\label{results-3}}

\hypertarget{number-of-antibiograms-and-patients}{%
\subsection{Number of Antibiograms and Patients}\label{number-of-antibiograms-and-patients}}

Initially, 35,619 antibiograms were included of which 12,616 were from Dutch and 23,003 from German hospitals. The 12,616 isolates were from five Dutch secondary-care hospitals (n = 4,377; from 2015 to 2016) and one Dutch university-hospital (n = 8,239, 2016), and the 23,003 isolates were from three German secondary-care hospitals (n = 6,914, 2016) and one German university-hospital (n = 16,089, 2016). Overall, 80.9\% of all isolates were \emph{Enterobacteriaceae} and 19.1\% non-fermenting Gram-negative bacteria.

When analysing the data, two major limitations occurred: (i) For \emph{Enterobacteriaceae}, the Dutch classification system could not be applied for 3,832 isolates, because they were not tested for the presence of ESBLs or test results were unclear (n = 3,720 isolates from the German hospitals and n = 112 from Dutch hospitals). This is because testing for the presence of ESBL is not required by the German MRGN classification system (and is often not performed in German laboratories except for \emph{E. coli} and \emph{Klebsiella spp.}, where this test is routinely implemented in automated systems used for antimicrobial susceptibility testing). These isolates were therefore excluded from further analysis, which reduced the total number of isolates analysed to 31,787. (ii) Overall, we lacked data for the results of carbapenemase PCRs for non-fermenting bacteria. As carbapenemase PCRs are not required for classification in the German system, these results were not available for 4,651 \emph{P. aeruginosa} isolates from German hospitals. Since no VIM-carbapenemase was reported for the 1,205 \emph{P. aeruginosa} isolates from Dutch hospitals, we coped with this problem by assuming that the German \emph{P. aeruginosa} isolates were also VIM-negative and classified these isolates accordingly when applying the Dutch guideline. In contrast, for \emph{A. baumannii}, we considered all carbapenem-resistant isolates as carbapenemase producers when applying the Dutch guidelines. For \emph{Enterobacteriaceae}, test results were available, because German laboratories test the isolates in line with quality management measures.

\hypertarget{results-of-mrgn-and-brmo-classification}{%
\subsection{Results of MRGN and BRMO Classification}\label{results-of-mrgn-and-brmo-classification}}

According to the Dutch classification system, 7,772/31,787 (24.5\%) isolates were BRMO. Applying the German classification system on the same antibiograms resulted in the identification of 4,586/31l,787 (12.9\%) MRGN (\emph{p} \textless{} 0.0001). Table 3 shows where the two classification systems had the most divergent results on genus or species level.

Table 3. Differences in using Dutch and German multidrug resistance classification systems.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/08-t03} 

}

\end{figure}

The distribution of 3MRGN and 4MRGN among the 4,586 MRGN isolates is shown in Figure 1. Among all 152 carbapenem-resistant \emph{Enterobacteriaceae} isolates, carbapenemases were detected in 42 isolates (27.6\%) with OXA-48-like genes being predominant. The remaining 110 isolates were negative for carbapenemases (n = 87, 79.1\%) or not tested (n = 23, 20.9\%) and were meropenem-non-susceptible Morganella, Proteus, Providencia, and Serratia (n = 45), as well as \emph{Klebsiella spp.} (n = 31), Enterobacter (n = 24), \emph{E. coli} (n = 7), and Citrobacter (n = 3).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/08-01} 

}

\caption{Species distribution among isolates classified as 3MRGN and 4MRGN according to the German guideline.}\label{fig:fig8-1}
\end{figure}

Of all 6,882 isolates classified as BRMO-\emph{Enterobacteriaceae}, 34 harboured carbapenemase-encoding genes (0.5\%), 4,953 were ESBL-producers (80.0\%), and 3,037 (44.1\%) isolates were simultaneously resistant to fluoroquinolones and aminoglycosides. A total of 788 \emph{P. aeruginosa} isolates were classified as BRMO, because they had a resistance pattern in accordance with Table 2. Among the remaining 5,058 \emph{P. aeruginosa} isolates (3,961 from German and 1,107 from Dutch laboratories) not classified as BRMO, 1,107 (21.9\%) were carbapenem-resistant (981 and 126 from German and Dutch laboratories, respectively). A total of 72 BRMO-A. baumannii isolates were classified as such, because they were carbapenem-resistant (n = 70), quinolone/aminoglycoside-resistant (n = 2) or both (n = 59).

However, of all isolates 23,433 (73.7\%) were neither classified as MRGN, nor as BRMO. Among 3,780 and 806 isolates classified as 3MRGN and 4MRGN according to the German guideline, 3,271 (86.5\%) and 733 (90.9\%) were also classified as BRMO. In contrast, of the 7,772 isolates classified as BRMO, 3,768 were not classified MRGN (48.5\%). An agreement matrix between the Dutch and German guidelines for MDRGNB classification is shown in Table 4.

Table 4. Correlation matrix between the Dutch BRMO-classification and the German MRGN-classification system to define multidrug-resistant Gram-negative bacteria (MDRGNB) for 31,787 isolates of different bacterial species.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/08-t04} 

}

\end{figure}

\hypertarget{discussion-4}{%
\section{Discussion}\label{discussion-4}}

When patients are transferred between hospitals, information regarding MDRGNB colonisation or infection must also be transferred to ensure continuous implementation of infection control measures. This is usually supported by indicating on antibiograms, which are included in the records of a transferred patient, whether the respective bacteria are multidrug-resistant according to the national guideline. For cross-border healthcare, this implies that clinicians or infection control staff can interpret antibiograms according to guidelines from both countries or understand foreign `MDRGNB languages'. The aim of this study was to describe different classifications used in The Netherlands and Germany in order to estimate the risk, which might be caused when patients infected or colonised with MDRGNB are transferred across the border without recognizing the respective bacteria as multidrug-resistant.

When planning the data analysis, a first hurdle occurred when the authors tried to actually understand the respective classification guidelines in detail. We learned that parts of the practical applicability of the guidelines (from both sides of the border) are rather locally defined. For example, in the Dutch guideline, it is not explicitly mentioned for \emph{Enterobacteriaceae}, which fluoroquinolones (e.g., ciprofloxacin, levofloxacin, norfloxacin, moxifloxacin) and aminoglycosides (e.g., gentamicin, tobramycin, amikacin) should be considered for the classification of which bacterial species and how to categorise, if one quinolone is resistant and the other susceptible. In the German guideline, some exceptional rules, such as ignoring imipenem non-susceptibility in \emph{Serratia} or \emph{Proteus} for the classification (due to unreliable test results) are not mentioned and can only be concluded from other guidance papers or publications of German reference laboratories. This might cause problems if microbiological laboratories are working across the border and might be perceived as a lack of transparency. This issue could be improved when national policy makers published more detailed standard operating procedures for laboratories where the problems occurring in daily routine are more accurately described.

Overall, the Dutch guideline makes it more laborious for a microbiological laboratory to actually classify an isolate as MDRGNB (tests for phenotypic ESBL-production and VIM-carbapenemase encoding genes). This might reflect structural differences in the organisation of microbiological diagnostics between the two countries, as more laborious confirmation testing requires using more financial resources.

When comparing the Dutch and the German classification systems for MDRGNB (Table 4), we found very divergent results. The bulk of isolates, which were classified differently, were \emph{E. coli} and \emph{Klebsiella} isolates characterised by ESBL-production, but being susceptible to fluoroquinolones. In German hospitals, no other than basic hygiene measures are taken for patients colonised or infected with these strains. This can be criticised, because spread of ESBL-producers might increase carbapenem use. Moreover, ESBLs are usually encoded on plasmids, which can be transferred independently from the bacterial clone even to other bacterial species. However, recent investigations have shown, that clonal spread of ESBL-\emph{E. coli} in healthcare settings is rarely observed \textsuperscript{{[}8,9{]}}. A second reason for divergent classifications was that the Dutch guideline uses combined fluoroquinolone and aminoglycoside resistance as a criterion for multidrug resistance. Aminoglycosides are not considered in the German guideline, maybe due to their limited and decreasing use in German hospitals compared with the Netherlands (\textless0.5 vs.~3.7 daily defined doses (DDD)/100 patient-days) \textsuperscript{{[}10,11{]}}. Thirdly, major differences were also found for \emph{P. aeruginosa}. Many of the very broadly resistant \emph{P. aeruginosa} isolates, for which colistin, tobramycin, or new β-lactams (such as ceftolozane/tazobactam) were the only remaining treatment options, were not classified as MDRGNB by the Dutch guideline, because VIM-carbapenemases were not detected. In this context, we clearly overestimated the disagreement between the Dutch and German guideline (Table 4), because we considered all 1,107 carbapenem-resistant \emph{P. aeruginosa} isolates (of 5,058 not classified as BRMO) as VIM-negative. This is not probable as it is well known that in Germany up to 24\% of carbapenem-resistant \emph{P. aeruginosa} isolates harbour carbapenemases among which VIM is predominant \textsuperscript{{[}12{]}}. This points towards a major limitation of this study. Since the analysis was not prospectively planned, we had to cope with missing data. Of course, excluding 3,832 antibiograms (which is \textgreater10\% of the antibiograms collected in the participating hospitals) due to a lack of information about phenotypic ESBL-test results for non-\emph{E. coli}/non-\emph{Klebsiella} isolates might have caused significant impact on the results. However, the total numbers \emph{Enterobacter}, \emph{Citrobacter}, or \emph{Hafnia} isolates included from both sides of the border was comparable. Overall, the results of this study demonstrate that in contrast to other multidrug-resistant bacteria such as methicillin-resistant \emph{Staphylococcus aureus} or vancomycin-resistant enterococci, those resistance pheno- or genotypes that define Gram-negative bacteria as MDRGNB markedly differ between the Netherlands and Germany. For cross-border care, the easiest solution would be to harmonise the classification rules of both countries. As long as this is not done, the full antibiogram data of Gram-negative bacteria should be transferred together with the patient in order to enable classification by local infection control staff.

\hypertarget{acknowledgements-3}{%
\section*{Acknowledgements}\label{acknowledgements-3}}
\addcontentsline{toc}{section}{Acknowledgements}

This study was supported by the INTERREG V A (202085) funded project EurHealth-1Health, part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport (VWS), the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the German Federal State of Lower Saxony.

\hypertarget{conflicts-of-interest}{%
\section*{Conflicts of interest}\label{conflicts-of-interest}}
\addcontentsline{toc}{section}{Conflicts of interest}

The authors declare no conflict of interest. The founding sponsors had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.

\hypertarget{references-7}{%
\section*{References}\label{references-7}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Magiorakos, A.P.; Burns, K.; Rodríguez Baño, J.; Borg, M.; Daikos, G.; Dumpis, U.; Lucet, J.C.; Moro, M.L.; Tacconelli, E.; Simonsen, G.S.; \emph{et al.} Infection prevention and control measures and tools for the prevention of entry of carbapenem-resistant \emph{Enterobacteriaceae} into healthcare settings: Guidance from the European Centre for Disease Prevention and Control. Antimicrob. Resist. Infect. Control 2017, 6, 113, \url{doi:10.1186/s13756-017-0259-z}.
\item
  European Centre for Disease Prevention and Control. EARS-Net Resistance Data for 2016. Available online: \url{http://atlas.ecdc.europa.eu/public/index.aspx} (accessed on 15 December 2017).
\item
  Albiger, B.; Glasner, C.; Struelens, M.J.; Grundmann, H.; Monnet, D.L.; European Survey of Carbapenemase-Producing \emph{Enterobacteriaceae} (EuSCAPE) Working Group. Carbapenemase-producing \emph{Enterobacteriaceae} in Europe: Assessment by national experts from 38 countries, May 2015. Euro Surveill. 2015, 20, 30062, \url{doi:10.2807/1560-7917.ES.2015.20.45.30062}.
\item
  Magiorakos, A.-P.; Srinivasan, A.; Carey, R.B.; Carmeli, Y.; Falagas, M.E.; Giske, C.G.; Harbarth, S.; Hindler, J.F.; Kahlmeter, G.; Olsson-Liljequist, B.; \emph{et al.} Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: An international expert proposal for interim standard definitions for acquired resistance. Clin. Microbiol. Infect. 2012, 18, 268--281, \url{doi:10.1111/j.1469-0691.2011.03570.x}.
\item
  Müller, J.; Voss, A.; Köck, R.; Sinha, B.; Rossen, J.W.; Kaase, M.; Mielke, M.; Daniels-Haardt, I.; Jurke, A.; Hendrix, R.; \emph{et al.} Cross-border comparison of the Dutch and German guidelines on multidrug-resistant Gram-negative microorganisms. Antimicrob. Resist. Infect. Control 2015, 4, 7, \url{doi:10.1186/s13756-015-0047-6}.
\item
  Kommission für Krankenhaushygiene und Infektionsprävention (KRINKO). Hygienemaßnahmen bei Infektionen oder Besiedlung mit multiresistenten gramnegativen Stäbchen. Bundesgesundheitsbl 2012, 55, 1311--1354.
\item
  Werkgroep Infectiepreventie. BRMO (Bijzonder Resistente Micro-Organismen)---Ziekenhuizen. Available online: \url{http://www.rivm.nl/dsresource?objectid=b6b99580-44e2-4b9c-8183-52871e61764f\&type=org\&disposition=inline} (accessed on 17 December 2017).
\item
  Mellmann, A.; Bletz, S.; Böking, T.; Kipp, F.; Becker, K.; Schultes, A.; Prior, K.; Harmsen, D. Real-Time Genome Sequencing of Resistant Bacteria Provides Precision Infection Control in an Institutional Setting. J. Clin. Microbiol. 2016, 54, 2874--2881, \url{doi:10.1128/JCM.00790-16}.
\item
  Kluytmans-van den Bergh, M.F.Q.; van Mens, S.P.; Haverkate, M.R.; Bootsma, M.C.J.; Kluytmans, J.A.J.W.; Bonten, M.J.M.; SoM Study Group. Quantifying Hospital-Acquired Carriage of Extended-Spectrum Beta-Lactamase-Producing \emph{Enterobacteriaceae} Among Patients in Dutch Hospitals. Infect. Control Hosp. Epidemiol. 2018, 39, 32--39, \url{doi:10.1017/ice.2017.241}.
\item
  SWAB. NethMap 2017 - Consumption of Antimicrobial Agents and Antimicrobial Resistance among Medically Important Bacteria in The Netherlands in 2016. Available online: \url{http://www.swab.nl} (accessed on 17 December 2017).
\item
  Paul-Ehrlich Gesellschaft (PEG). Bundesamt für Verbraucherschutz und Lebensmittelsicherheit, GERMAP 2015---Antibiotika-Resistenz und --Verbrauch. Available online: \url{http://www.p-e-g.org/econtext/germap} (accessed on 17 December 2017).
\item
  Kaase, M. Bericht des Nationalen Referenzzentrums (NRZ) für gramnegative Krankenhauserreger. Epidemiol. Bull. 2016, 2, 11--14.
\end{enumerate}

\hypertarget{ch09-changing-epidemiology}{%
\chapter{\texorpdfstring{Changing Epidemiology of Methicillin-Resistant \emph{Staphylococcus aureus} in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow Policy}{Changing Epidemiology of Methicillin-Resistant Staphylococcus aureus in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow Policy}}\label{ch09-changing-epidemiology}}

Published in Eurosurveillance. 2019 Apr 11; 24(15): 180024

Jurke A \textsuperscript{1}, Daniels-Haardt I \textsuperscript{2}, Silvis W \textsuperscript{3}, Berends MS \textsuperscript{4,5}, Glasner C \textsuperscript{5}, Becker K \textsuperscript{6}, Köck R \textsuperscript{6,7,8}, Friedrich AW \textsuperscript{5}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  North Rhine-Westphalian Centre for Health, Section Infectious Disease Epidemiology, Bochum, Germany
\item
  North Rhine-Westphalian Centre for Health, Department Health Promotion, Health Protection, Bochum, Germany
\item
  Laboratory for Medical Microbiology and Public Health (LabMicTA), Hengelo, Netherlands
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, Netherlands
\item
  University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, Netherlands
\item
  University Hospital Münster, University of Münster, Institute of Medical Microbiology, Münster, Germany
\item
  University Hospital Münster, University of Münster, Institute for Hygiene, Münster, Germany
\item
  Institute of Hygiene, DRK Kliniken Berlin, Berlin, Germany
\end{enumerate}

\hypertarget{abstract-7}{%
\section*{Abstract}\label{abstract-7}}
\addcontentsline{toc}{section}{Abstract}

Methicillin-resistant \emph{Staphylococcus aureus} (MRSA) is a major cause of healthcare-associated infections. We describe MRSA colonisation/infection and bacteraemia rate trends in Dutch--German border region hospitals (NL--DE-BRH) in 2012--16. All 42 NL--DE BRH (8 NL-BRH, 34 DE-BRH) within the cross-border network EurSafety Health-net provided surveillance data (on average ca 620,000 annual hospital admissions, of these 68.0\% in Germany). Guidelines defining risk for MRSA colonisation/infection were reviewed. MRSA-related parameters and healthcare utilisation indicators were derived. Medians over the study period were compared between NL- and DE-BRH. Measures for MRSA cases were similar in both countries, however defining patients at risk for MRSA differed. The rate of nasopharyngeal MRSA screening swabs was 14 times higher in DE-BRH than in NL-BRH (42.3 vs 3.0/100 inpatients; \emph{p} \textless{} 0.0001). The MRSA incidence was over seven times higher in DE-BRH than in NL-BRH (1.04 vs 0.14/100 inpatients; \emph{p} \textless{} 0.0001). The nosocomial MRSA incidence-density was higher in DE-BRH than in NL-BRH (0.09 vs 0.03/1,000 patient days; \emph{p} = 0.0002) and decreased significantly in DE-BRH (\emph{p} = 0.0184) during the study. The rate of MRSA isolates from blood per 100,000 patient days was almost six times higher in DE-BRH than in NL-BRH (1.55 vs 0.26; \emph{p} = 0.0041). The patients had longer hospital stays in DE-BRH than in NL-BRH (6.8 vs 4.9; \emph{p} \textless{} 0.0001). DE-BRH catchment area inhabitants appeared to be more frequently hospitalised than their Dutch counterparts. Ongoing IPC efforts allowed MRSA reduction in DE-BRH. Besides IPC, other local factors, including healthcare systems, could influence MRSA epidemiology.

\hypertarget{introduction-6}{%
\section{Introduction}\label{introduction-6}}

Cross-border patient mobility is a priority in the European Union (EU), because the most accessible or appropriate care for citizens living in border regions may be available abroad. When, in 2013, the directive 2011/24/EU came into force, patients' right to access healthcare in other Member States including reimbursement and medical follow-up in their respective home countries was entitled in an EU law for the first time. With this, cross-border cooperation in infection prevention and control (IPC) using comprehensive strategies is important \textsuperscript{{[}1{]}}.

Antimicrobial resistant (AMR) pathogens are a serious threat to public health in Europe, leading to increased healthcare costs, treatment failure and deaths. For invasive bacterial infections, prompt treatment with effective antimicrobial agents is essential and is one of the most effective interventions to reduce the risk of fatal outcomes \textsuperscript{{[}2{]}}. Currently, the epidemiological situation is cause for concern especially with regard to AMR Gram-negative pathogens, e.g., characterised by carbapenem resistance (CR) \textsuperscript{{[}3{]}}. However, the Gram-positive methicillin-resistant \emph{Staphylococcus aureus} (MRSA) is still one of the most important causes of healthcare-associated infections due to AMR pathogens \textsuperscript{{[}3{]}}.

In 2017 in a consensus report of the European Centre for Disease Prevention and Control (ECDC), the European Food Safety Authority (EFSA) and the European Medicines Agency (EMA), the proportion of MRSA in invasive \emph{S. aureus} infections was proposed as an indicator for surveillance of AMR pathogens in humans \textsuperscript{{[}4{]}}. Although in 2016 the proportion of MRSA in invasive \emph{S. aureus} infections in Europe reached its lowest level (13.7\%) since the ECDC first presented population-weighted data for the EU in 2009, large inter-country variations (1.2 to 50.5\%) remain in Europe \textsuperscript{{[}3{]}}. For example, in the most populated German federal state, North Rhine-Westphalia (NRW), the incidence of MRSA bacteraemia per inhabitants was 32-fold higher compared with the Dutch neighbouring region with similar population size in 2009--10 \textsuperscript{{[}5{]}}.

The occurrence of MRSA still necessitates continuous surveillance and preparedness to optimise IPC to further decrease MRSA rates \textsuperscript{{[}6-9{]}}. Since 1999, MRSA screening of various sites including at least nares, pharynx and wounds (if present) and additionally perineum or groin (in case of known previous carriage) before or at admission to hospitals is recommended in Germany, if patients have defined risk factors \textsuperscript{{[}10{]}}. For MRSA carriers IPC measures including isolation in single rooms, barrier precautions and decolonisation therapies are also recommended \textsuperscript{{[}10,11{]}}. Within the EU-funded community initiative INTERREG IIIA in 2006, all hospitals in the German Münsterland region, located directly at the Dutch--German border, started to establish a network to control MRSA -- the EUREGIO MRSA net. They agreed to monitor the implementation of the IPC measures, harmonise local standards, exchange hospital utilisation data and MRSA data, perform molecular typing of MRSA isolates and establish regional benchmarks \textsuperscript{{[}12{]}}. This `search-and-follow' strategy was inspired from the `search-and-destroy' policy implemented in Dutch hospitals since the 1980s. It aimed to improve application of the German national MRSA recommendations, the regional cooperation between hospitals, other healthcare facilities and public health authorities, as well as to create a more robust MRSA surveillance system \textsuperscript{{[}9,12-14{]}}. Further to this strategy, screening for MRSA carriage among risk patients at hospital admission increased between 2009 and 2011 in these regional German hospitals and the nosocomial MRSA incidence density significantly decreased \textsuperscript{{[}15{]}}.

The cross-border IPC network cooperation, i.e.~the Dutch−German web-based communication portal for handling MRSA problems for healthcare workers, patients and the public was continued from 2009 to 2015 within the INTERREG IVA funded project EurSafety Health-net. This enabled hospitals and nursing homes to acquire Euregional Quality and Transparency certificates. Moreover, since 2016, the collaboration was further prolonged within the INTERREG VA funded project EurHealth-1Health inter alia. Within this, the Dutch signaling meeting of the Hospital-acquired Infection and Antimicrobial Resistance Monitoring Group (SO-ZI/AMR) occurs in the German study region.

Here, we analysed 2012 to 2016 MRSA surveillance data from Dutch and German border region hospitals (NL-BRH and DE-BRH) in the network in order to describe temporal and spatial trends of MRSA rates and find differences between these groups of hospitals. We also used the data to calculate the MRSA rates per inpatient and per patient days in both groups of hospitals and the MRSA rates per inhabitants in the patient catchment areas of NL-BRH and DE-BRH respectively in order to compare the two groups in relation to these parameters.

\hypertarget{methods-3}{%
\section{Methods}\label{methods-3}}

\hypertarget{setting}{%
\subsection{Setting}\label{setting}}

Within the EurSafety Health-net project (\url{http://www.eursafety.eu/}) the German part of the project region geographically comprised six districts in the Münsterland region (codes DEA33--35, DEA37, DEA38 and DE94B, level 3, according to the Nomenclature of Territorial Units for Statistics, NUTS \textsuperscript{{[}16{]}}) and was inhabited by ca 1.73 million people \textsuperscript{{[}17{]}}. The Dutch part comprised eight districts in the provinces of Groningen, Drenthe and in the region Twente-Achterhoek (codes NL111--113, NL131--133, NL213 and NL225) inhabited by ca 2.10 million people (Figure 1) \textsuperscript{{[}17{]}}. Initially, there were 42 hospitals located in the Dutch--German region (reduced in 2015 to 41 due to a structural merging of two DE-BRH) treating ca 620,000 admitted patients (68.0\% in the German part of the study region) with ca 3,900,000 patient days per year. All 34 (since 2015, 33) regional DE-BRH (9.5\% of hospitals in NRW in 2016) and all eight regional NL-BRH (8.8\% of hospitals in the Netherlands in 2016) took part in the project. Among the DE-BRH, 29 were acute care hospitals, one was a university hospital, one was a rehabilitation clinic and three hospitals were specialised in psychiatry, while the NL-BRH comprised one university- and seven acute care hospitals.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-01} 

}

\caption{Location of the study region in the Netherlands and Germany, 2012-2016. The dark grey area represents the study region, including the Dutch regions East Groningen (NL111), Delfzijl and surroundings (NL112), rest of Groningen (NL113), North Drenthe (NL 131), South East Drenthe (NL132), South West Drenthe (NL133), Twente (NL213), Achterhoek (NL225), and the German regions Grafschaft Bentheim region (DE94B) and the Münsterland-region with the urban district Münster (DEA33) and the rural districts Borken (DEA34), Coesfeld (DEA35), Steinfurt (DEA37) and Warendorf (DEA38).}\label{fig:fig9-1}
\end{figure}

\hypertarget{guidelines-for-patients-at-risk-for-mrsa-and-infection-prevention-and-control-measures}{%
\subsection{Guidelines for patients at risk for MRSA and infection prevention and control measures}\label{guidelines-for-patients-at-risk-for-mrsa-and-infection-prevention-and-control-measures}}

Both NL-BRH and DE-BRH implemented MRSA-related IPC measures according to their national guidelines and recommendations, issued by the Dutch Working Group on Infection Prevention (WIP) and the German Commission for Hospital Hygiene and Infection Prevention (KRINKO) at the Robert Koch-Institute, respectively \textsuperscript{{[}10,18{]}}. Of note, the definitions of whom to screen at admission differed for NL-BRH and DE-BRH based on the national guidelines and recommendations (Table 1), as well as screening sites (DE-BRH: at least nose, pharynx, throat and wounds, if present, additionally perineum and groin swab, when indicated; NL-BRH: nasal-, throat- and perineum or rectum swab plus additional cultures depending on clinical signs) \textsuperscript{{[}10,19{]}}. In all hospitals, positive screenings or any other detection of MRSA was followed by single room isolation, contact precautions and decolonisation, if applicable. Pre-emptive isolation of patients with MRSA risk factors was performed according to local guidelines (in DE-BRH only for patients with previous MRSA carriage, for NL-BRH see Table 1.

The levels of isolation for inpatients with risk categories were the following:

\begin{itemize}
\tightlist
\item
  RMRSA: MRSA positive- or (RH) high-risk category patients in high-risk departments of the hospital (e.g., intensive care unit, haematology): single room isolation with contact- and airborne precautions.
\item
  RH: High-risk category patients who are not in high-risk departments and who have an MRSA screening result available within 24 hours of admission: single room with contact precautions.
\item
  RL: Low-risk category: no isolation, awaiting new MRSA screening test results.
\end{itemize}

In both countries adherence to the MRSA-IPC guidelines- and recommendations was periodically checked by the responsible local public health authorities (Germany) and national health inspectorate (Netherlands). The implementation of other IPC measures in the participating hospitals, such as standards for the prevention of catheter-related bloodstream infections, was not planned or assessed within the project.

Table 1. Risk factors for MRSA carriage at admission according to Dutch and German MRSA guidelines, 2012--2016.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-t01} 

}

\end{figure}

\hypertarget{data-collection}{%
\subsection{Data collection}\label{data-collection}}

An MRSA case was defined as an inpatient who was colonised or infected with MRSA at admission or for nosocomial MRSA cases, after admission. A blood culture positive for MRSA, from a single inpatient and from a single hospital stay was qualified as MRSAB case. If an MRSA case, or MRSAB case, had several stays in a year, each hospital stay was counted as an MRSA case, or MRSAB case, in the surveillance.

On both sides of the border, the collected surveillance data of inpatients (i.e.~excluding outpatients) included the number of nasopharyngeal swabs performed for MRSA screening before or at admission, the numbers of MRSA cases (one isolate per patient per hospital stay) − in DE-BRH and in several NL-BRH MRSA cases were additionally classified as imported or nosocomial (i.e.~nosocomial, if the case was detected ≥ 3 days after hospital admission unless the patient was a known MRSA carrier), the number of cases and the number of patient days. Additionally, in DE-BRH and in several NL-BRH the patient days of MRSA cases (i.e.~the number of days, which an MRSA-positive patient spent in hospital) were also recorded. Moreover, the number of inpatients with a blood culture positive for MRSA (MRSAB, one isolate per patient case) and the number of \emph{S. aureus} in blood cultures (one isolate per patient case) were assessed. The MRSA-surveillance data as described above were collected in all DE-BRH using a protocol adapted from the national German Nosocomial Infections Surveillance System (MRSA-KISS \textsuperscript{{[}20{]}}); see Supplement Table S1). For cross-border analysis, the laboratories serving for all NL-BRH provided retrospectively collected data for the period 2012 to 2016, according to the same protocol.

\hypertarget{ethical-statement}{%
\subsection{Ethical statement}\label{ethical-statement}}

Ethical approval was asked from ethical committee at the University Medical Center Groningen (UMCG), and approval was not necessary for this study.

\hypertarget{data-analysis}{%
\subsection{Data analysis}\label{data-analysis}}

We analysed the surveillance data of five years (2012--16) and calculated the following parameters: (i) screening rate (nasopharyngeal swabs for MRSA/100 inpatients), (ii) MRSA incidence (MRSA cases/100 inpatients), (iii) percentage of MRSA isolates per all \emph{S. aureus} isolates detected in blood cultures, (iv) incidence density of MRSA isolates detected from blood cultures (MRSAB cases/100,000 patient days), (v) nosocomial MRSA incidence density (nosocomial MRSA-cases/1,000 patient days), (vi) length of stay in hospital (number of patient days/inpatients, (vii) length of stay in the hospital of MRSA cases (number of patient days of MRSA cases/MRSA cases). We calculated the mean annual numbers of inpatients per 100 inhabitants and of patient days per 100 inhabitants of the patient catchment area of NL-BRH and DE-BRH. Furthermore, we calculated the mean annual number of nasopharyngeal swabs performed for MRSA screening before or at admission to hospital per 100 inhabitants in the patient catchment area of the regional hospitals (DE-BR and NL-BR) as well as of inpatient MRSA cases per 1,000 inhabitants and the MRSAB/1,000,000 inhabitants using our surveillance data of inpatients (i.e., excluding outpatients). The number of inhabitants were assessed from the official statistical database \textsuperscript{{[}17{]}}.

Time trends of MRSA parameters were analysed by Friedman tests. The percentage of nosocomial MRSA cases on all MRSA cases was assessed by Cochran--Armitage test of linear trend. The cross-border regional comparisons were analysed using Wilcoxon rank sum test. All statistical analyses were done using SAS 9.4 software (SAS Institute Inc., Cary, United States); \emph{p} \textless{} 0.05 was considered significant. Results of significance tests were discarded if the software displayed an alert due to more than 10\% of missing values in the respective dataset. The map was made using RegioGraph10 (GFK Geomarketing GmbH, Bruchsal, Germany).

\hypertarget{results-4}{%
\section{Results}\label{results-4}}

\hypertarget{trend-and-cross-border-comparison-of-mrsa-rates}{%
\subsection{Trend and cross-border comparison of MRSA rates}\label{trend-and-cross-border-comparison-of-mrsa-rates}}

The total numbers of MRSA cases (detected in DE-BRH and NL-BRH are shown in Table 2. In both DE-BRH and NL-BRH the median nasopharyngeal MRSA screening rate increased significantly between 2012 and 2016 (Table 3). Overall, the median screening rate was 14 times higher in DE-BRH than in NL-BRH (\emph{p} \textless{} 0.0001, Table 4).

Table 2. Numbers of methicillin-resistant \emph{Staphylococcus aureus} cases documented in all study hospitals in the German region of Münsterland and the Dutch regions of Twente-Achterhoek, Drenthe and Groningen, 2012--2016 (n = 42 hospitals) \textsuperscript{a.}

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-t02} 

}

\end{figure}

Table 3. Annual medians of methicillin-resistant \emph{Staphylococcus aureus} parameters in all study hospitals in the German region Münsterland and the Dutch regions of Twente-Achterhoek, Drenthe and Groningen, 2012--2016 (n = 42 hospitals) \textsuperscript{a.}

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-t03} 

}

\end{figure}

Table 4. Methicillin-resistant \emph{Staphylococcus aureus} parameters in all study hospitals in the German region of Münsterland and the Dutch regions of Twente-Achterhoek, Drenthe and Groningen, 2012--2016 (n = 42 hospitals) \textsuperscript{a.}

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-t04} 

}

\end{figure}

The median MRSA incidence remained stable over time at both sides of the border (Table 3), but was more than seven times higher in DE-BRH than in NL-BRH (\emph{p} \textless{} 0.0001) (Table 4). The median percentage of MRSA in \emph{S. aureus} blood culture isolates decreased from 12.5\% in 2012 to 5.0\% in 2016 in DE-BRH (\emph{p} = 0.0959), while it remained stable in NL-BRH (\emph{p} = 0.1679) (Table 3), but was more than 34 times higher in DE-BRH (\emph{p} = 0.0001) (Table 4). The median of MRSAB per 100,000 patient days remained stable over time in DE-BRH (\emph{p} = 0.4272) and NL-BRH (\emph{p} = 0.0620) (Table 3) and was six-fold greater in DE-BRH than in NL BRH (\emph{p} = 0.0041) (Table 4). The percentages of nosocomial cases on all MRSA cases (Table 2) decreased significantly in DE-BRH (\emph{p} \textless{} 0.0001), but did not change in NL-BRH (\emph{p} \textless{} 0.6474). Over the study period the median nosocomial MRSA incidence-density decreased significantly in DE-BRH (\emph{p} = 0.0184) (Table 3), but did not change in NL-BRH (\emph{p} = 0.3532) and was approximately three times higher in DE-BRH than in NL-BRH (\emph{p} = 0.0002) (Table 4).

\hypertarget{cross-border-comparison-of-healthcare-utilisation}{%
\subsection{Cross-border comparison of healthcare utilisation}\label{cross-border-comparison-of-healthcare-utilisation}}

We compared the available data on healthcare utilisation in DE-BRH and NL-BRH. The median length of stay (LOS) in the hospital was 6.8 days in DE-BRH compared with 4.9 days in NL-BRH (\emph{p} \textless{} 0.0001) (Table 4); LOS of MRSA patients was similar in DE-BRH vs NL-BRH (11.1 days vs 11.7 days; \emph{p} = 0.8774) (Table 4). The hospitalisation rate was 24.3 inpatients/100 inhabitants annually in the patient catchment area of DE-BRH, almost thrice the rate in the NL-BRH's catchment area (9.27/100). To put this difference in healthcare utilisation into context, we calculated the mean annual number of nasopharyngeal MRSA screening swabs before or at admission to hospital per 100 inhabitants in the German border region (DE-BR) vs the Dutch border region (NL-BR) (12.2 vs 0.36). Additionally, we compared the MRSA surveillance data of inpatients (i.e., excluding outpatients) in the patient catchment area of DE-BRH and NL-BRH. The calculated numbers of inpatient MRSA cases per 1,000 inhabitants in DE-BR and NL-BR were 2.52 vs.~0.14. Furthermore, the calculated MRSAB/1,000,000 inhabitants in DE-BR and NL-BR was 38.4 vs 4.09 (Table 5).

Table 5. Calculated parameters in the patient catchment area of all study hospitals in the German region of Münsterland and Dutch regions of Twente-Achterhoek, Drenthe and Groningen, 2012--2016 (n = 42 hospitals) \textsuperscript{a.}

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/09-t05} 

}

\end{figure}

\hypertarget{discussion-5}{%
\section{Discussion}\label{discussion-5}}

As patients in the EU have the right to healthcare across the borders of Member States (EU directive 2011/24/EU), it is of interest to compare the quality of care, safety standards and risks of nosocomial infection by AMR pathogens between EU countries. In this respect, the cross-border systematic and continuous MRSA surveillance is one of the cornerstones to ensure equal quality of healthcare \textsuperscript{{[}21{]}}.

Our study revealed significant differences between Dutch and German hospitals (Table 4). The median MRSA-incidence in DE-BRH was more than seven times higher compared with NL-BRH. We also found that the median MRSA percentage of \emph{S. aureus} detected in blood cultures was more than 34 times higher in DE-BRH than in NL-BRH (Table 4). The incidence density of MRSAB was six times higher in DE-BRH (Table 4) and there were nine times more MRSAB per 1,000,000 inhabitants for the patient catchment area of DE-BRH compared with NL-BRH (Table 5).

According to the ECDC, differences in the occurrence of AMR pathogens between European countries are most likely caused by differences in healthcare utilisation, antimicrobial use and IPC practices \textsuperscript{{[}3{]}}.

Concerning healthcare utilisation in our context, we found that inhabitants in the German part of the study region were almost three times as often hospitalised (Table 5) and had a significantly longer LOS than patients on the Dutch part (Table 4). This may be due to socioeconomic factors or a different organisation of ambulatory healthcare.

While antimicrobial consumption was not the focus of the current study, NRW has been reported as the region in Germany with the highest antimicrobial consumption in outpatients (19.2 daily defined doses (DDD/1,000 inhabitants) \textsuperscript{{[}22{]}}. In this respect, the MRSA incidence in DE-BRH was slightly above the incidences in German hospitals participating in the nationwide surveillance system MRSA-KISS \textsuperscript{{[}20{]}}. The antimicrobial consumption level in NRW seems to be also considerably higher than in the Netherlands (10.39 DDD/1,000 inhabitants) \textsuperscript{{[}23{]}}, not only in terms of total antibiotics consumed, but also for the oral use of second-generation cephalosporins. Promoting rational regional antibiotic use is therefore one of the major goals in the INTERREG VA project EurHealth-1Health (\url{http://www.eurhealth-1health.eu/}).

For MRSA IPC, the recommendations in Germany and the corresponding guidelines in the Netherlands were comparable regarding the measures performed for MRSA carriers \textsuperscript{{[}10,18{]}}. However, there were differences between the two countries in identifying people at risk of MRSA infection/colonisation \textsuperscript{{[}10,18{]}}. In this study, we found that the DE-BRH performed 14 times more nasopharyngeal screening swabs for MRSA than their Dutch counterparts.

The higher screening rates on the German side of the border may be ascribed to the fact that in German IPC recommendations, previous hospitalisation in Germany is a risk factor for MRSA carriage. This constitutes a main difference in defined risk factors between Dutch- and German MRSA IPC guidelines, whereby Dutch guidelines mostly consider screening for patients previously hospitalised outside the Netherlands (Tables 1 and and3)3) \textsuperscript{{[}14,24{]}}. In this respect, we observed that although the densities of nosocomial MRSA cases were lower in NL-BRH than in DE-BRH (Table 3), the proportion of nosocomial MRSA cases among all MRSA detected was slightly higher in the Dutch hospitals (Table 2). The reason for this remains unclear, but it might be speculated that a larger proportion of MRSA carriers in the Netherlands had no risk factors for MRSA and were hence not screened at admission.

Another explanation for screening rate differences between the two countries may be distinct underlying epidemiological situations regarding MRSA. For example, the MRSA prevalence is higher in the population in Germany than that in patients at hospital admission in the Netherlands (0.7\% vs.~0.13\%) \textsuperscript{{[}25,26{]}}. Moreover, in the German part of the study region, a possible additional MRSA burden due to the exceptionally frequent occurrence of livestock-associated MRSA might have an effect \textsuperscript{{[}27,28{]}}.

The screening and IPC measures in the DE-BRH appeared to be nevertheless appropriate. In 2006, in the project region excluding Groningen and Drenthe (Figure), investigations evaluating the numbers of patients with MRSA risk factors at admission to German hospitals demonstrated that ca 35.6\% of patients had a risk factor requiring screening \textsuperscript{{[}29{]}}. A corresponding level of screening was implemented by DE-BRH during the study period 2009--11 \textsuperscript{{[}15{]}}. This level remained high in the 2012--16 period (Table 3), indicating a very good implementation of the screening standards.

About 1\% of all patients admitted in DE-BRH carried MRSA, which corresponds well to results of investigations evaluating the prevalence of MRSA carriage in the regional general, non-hospitalised population in 2012 \textsuperscript{{[}25{]}}. In terms of difference with the Netherlands, this has for consequence that it is more expensive to provide isolation capacities for ca 1.0\% of inpatients with MRSA in DE-BRH vs 0.15\% in NL-BRH. Moreover, the higher MRSA incidence in DE-BRH could lead to a higher probability for nosocomial MRSA cases as they are not completely avoidable \textsuperscript{{[}30-32{]}}.

From 2012 to 2016 however, the nosocomial MRSA incidence density in DE-BRH decreased significantly, a trend already observed from 2009 to 2011 \textsuperscript{{[}15{]}}. Moreover, the nosocomial MRSA incidence density (Table 3) appeared to be below the densities reported for hospitals participating in the nationwide surveillance system MRSA-KISS (median nosocomial MRSA cases per 1,000 patient days in DE-BRH/MRSA KISS, 2012--16: 0.11/0.14, 0.09/0.12, 0.09/0.10, 0.08/0.09, 0.07/0.08) \textsuperscript{{[}15,20{]}}. This may indicate the successful implementation of concerted IPC standards in DE-BRH in the EurSafety Health-net network \textsuperscript{{[}15{]}}.

We also observed for that the difference of the incidence of MRSA bacteraemia per inhabitants between the German and Dutch border region (38.4 vs 4.09 per 1,000,000) was apparently smaller than calculated in a previous study, which used 2009 Dutch and 2010 German data respectively to derive the difference between NRW and the Netherlands (57.6 vs 1.8 per 1,000,000) \textsuperscript{{[}5{]}}. In addition, according to the population-based German mandatory notification system for invasive MRSA infections (SurvStat) from 2012 to 2016, 40.7 MRSA isolates were detected in blood or cerebrospinal fluid per 1,000,000 inhabitants in the German project region \textsuperscript{{[}33{]}}, which is lower compared with data from the federal state of NRW (70.3 per 1,000,000 inhabitants) as well as from Germany (47.9 per 1,000,000 inhabitants) \textsuperscript{{[}34{]}}.

Comparing our results with those of other German laboratories participating in a voluntary, national surveillance system (ARS) \textsuperscript{{[}35{]}}, revealed that, for each year of the period 2012--16 the median percentage of MRSA in \emph{S. aureus} from blood cultures was lower in DE-BRH than in other laboratories in western Germany (DE-BRH/ARS-region west (NRW), 2012--16: 12.5\%/19.0\%, 14.3\%/15.0\%, 10.5\%/13.5\%, 9.8\%/13.3\%, 5.0\%/12.0\%) (Table 3), as well as below the middle lower range of the EU/European Economic Association (EEA) population-weighted mean between 18.8\% in 2012 and 13.7\% in 2016 \textsuperscript{{[}3,34,36{]}}.

In contrast, the mean MRSA percentage of \emph{S. aureus} detected in blood culture during 2012--16 was higher (1.5\% vs 1.3\%) in NL-BRH compared with Dutch national data of Infectious Disease Surveillance Information System for Antibiotic Resistance, (ISIS-AR) covering data of 52\% of diagnostic laboratories \textsuperscript{{[}37{]}}.

As typical for all passive surveillance systems, bias due to differences in reporting behaviour cannot be excluded and is a limitation of this study. However, as MRSA surveillance in DE-BRH started in 2007, a stabilised compliance in reporting can be assumed for the period from 2012--16. The higher number of MRSA cases per inhabitants on the German side compared with the Netherlands is biased if there is more than one episode of MRSA detection per year for one individual patient among the number of cases. Also, the inclusion of three psychiatric hospitals and one rehabilitation clinic, which have usually longer average lengths of stay, may have prolonged hospital stay in the DE-BRH. However, the data are in accordance with German-wide assessment systems. The clinical relevance of MRSA isolates detected in blood cultures is undisputable, but variations in blood culture diagnostics (e.g., frequency, performance) may result in bias when comparing MRSA percentages of \emph{S. aureus} blood culture isolates between different countries \textsuperscript{{[}38{]}}. A limitation of the study design is that the implementation of IPC standards, which are not directly targeted to control MRSA, such as bundles to prevent central-line-associated bloodstream infections (CLABSI), was not assessed and compared in the participating hospitals. Hence, changes of the incidence of MRSA bacteraemia could also be attributable to improvements in CLABSI prevention or other IPC standards.

This study on MRSA covering all hospitals across part of a European border as well as hospitals of all three care-categories demonstrated that routine MRSA surveillance may be helpful to monitor trends of MRSA parameters, to compare the MRSA rates and to indicate needs for further improvement to reach low MRSA rates EU-wide. Our results supplement the European and national surveillance systems. Ongoing efforts in MRSA prevention are recommended, including all healthcare sectors, especially with focus on One Health \textsuperscript{{[}39-42{]}}. Moreover, cross-border surveillance should be extended to other multidrug-resistant organisms, such as CR \emph{Enterobacteriaceae} in the future.

\hypertarget{acknowledgements-4}{%
\section*{Acknowledgements}\label{acknowledgements-4}}
\addcontentsline{toc}{section}{Acknowledgements}

We acknowledge all the active participants of the EurSafety Health-net and EurHealth-1Health projects: The infection control nurses and the physicians responsible for infection control of the 42 participating hospitals, as well as the staff of the regional laboratories participating in the project. We thank the project representatives appointed by the public health offices in the EUREGIO, especially Ms.~Scherwinski and Ms.~Winkler (both Borken), Dr.~Toepper (Coesfeld), Dr.~Bierbaum and Dr.~Lürwer (both Münster), Dr.~Schmeer and Ms.~Suhr (both Steinfurt), Dr.~König and Ms.~Clemens (Warendorf). Furthermore, we thank Ms.~Schmidt, Ms.~Lunemann, Ms.~Jessen and Ms.~Ganser (NRW Centre for Health) and Dr.~Gunnar Andriesse from Certe in Groningen for their support.

\hypertarget{funding-1}{%
\section*{Funding}\label{funding-1}}
\addcontentsline{toc}{section}{Funding}

The EurSafety Health-Net project was financially supported by external funding within the INTERREG IVA program `Germany-Netherlands' of the EU (EurSafety Health-net: INTERREG IVA III-1-01=073), by the German states of NRW and Lower Saxony and by the Dutch provinces Overijssel, Gelderland and Limburg. The EurHealth-1Health project is implemented within the framework of the INTERREG VA `Germany-Netherlands' program (grant number EU/INTERREG VA-202085) and is co-financed by the European Union, the Dutch Ministry of Health, Welfare and Sport (VWS), the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State North Rhine-Westphalia and by the German Federal State Lower Saxony.

\hypertarget{references-8}{%
\section*{References}\label{references-8}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Poljak M, Akova M, Friedrich AW, Rodríguez-Baño J, Sanguinetti M, Tacconelli E, \emph{et al.} ESCMID-an international Europe-based society committed to fostering cross-border collaboration and education to improve patient care. Clin Microbiol Infect. 2018;24(1):1-2. 10.1016/j.cmi.2017.05.024
\item
  Cassini A, Högberg LD, Plachouras D, Quattrocchi A, Hoxha A, Simonsen GS, \emph{et al.} Attributable deaths and disability-adjusted life-years caused by infections with antibiotic-resistant bacteria in the EU and the European Economic Area in 2015: a population-level modelling analysis. Lancet Infect Dis. 2019;19(1):56-66. 10.1016/S1473-3099(18)30605-4
\item
  European Centre for Disease Prevention and Control (ECDC). Antimicrobial resistance surveillance in Europe 2016. Annual Report of the European Antimicrobial Resistance Surveillance Network (EARS-Net). Stockholm: ECDC; 2017. Available from: \url{https://ecdc.europa.eu/sites/portal/files/documents/AMR-surveillance-Europe-2016.pdf}
\item
  European Centre for Disease Prevention and Control (ECDC) ECDC, EFSA and EMA Joint Scientific Opinion on a list of outcome indicators as regards surveillance of antimicrobial resistance and antimicrobial consumption in humans and food-producing animals. EFSA J. 2017;15(10):5017.
\item
  van Cleef BAGL, Kluytmans JAJW, van Benthem BHB, Haenen A, Monen J, Daniels-Haardt I, \emph{et al.} Cross border comparison of MRSA bacteraemia between The Netherlands and North Rhine-Westphalia (Germany): a cross-sectional study. PLoS One. 2012;7(8):e42787. 10.1371/journal.pone.0042787
\item
  Köck R, Becker K, Cookson B, van Gemert-Pijnen JE, Harbarth S, Kluytmans J, \emph{et al.} Systematic literature analysis and review of targeted preventive measures to limit healthcare-associated infections by meticillin-resistant \emph{Staphylococcus aureus}. Euro Surveill. 2014;19(29):23-49. 10.2807/1560-7917.ES2014.19.29.20860
\item
  Sai N, Laurent C, Strale H, Denis O, Byl B. Efficacy of the decolonization of methicillin-resistant \emph{Staphylococcus aureus} carriers in clinical practice. Antimicrob Resist Infect Control. 2015;4(56):56. 10.1186/s13756-015-0096-x
\item
  Sassmannshausen R, Deurenberg RH, Köck R, Hendrix R, Jurke A, Rossen JWA, \emph{et al.} MRSA Prevalence and Associated Risk Factors among Health-Care Workers in Non-outbreak Situations in the Dutch-German EUREGIO. Front Microbiol. 2016;7(1273):1273. 10.3389/fmicb.2016.01273
\item
  Dik JW, Sinha B, Friedrich AW, Lo-Ten-Foe JR, Hendrix R, Köck R, \emph{et al.} Cross-border comparison of antibiotic prescriptions among children and adolescents between the North of the Netherlands and the North-West of Germany. Antimicrob Resist Infect Control. 2016;5(1):14. 10.1186/s13756-016-0113-8
\item
  Commission for Hospital Hygiene and Infection Prevention at Robert Koch-Institute Empfehlungen zur Prävention und Kontrolle von Methicillin-resistenten \emph{Staphylococcus aureus}-Stämmen (MRSA) in medizinischen und pflegerischen Einrichtungen. {[}Recommendations for prevention and control of methicillin-resistant \emph{Staphylococcus aureus} (MRSA) in medical and nursing facilities{]}. Bundesgesundheitsblatt Gesundheitsforschung Gesundheitsschutz. 2014;57(6):695-732. German. 10.1007/s00103-014-1980-x
\item
  Commission for Hospital Hygiene and Infection Prevention at Robert Koch-Institute Empfehlungen zur Prävention und Kontrolle von Methicillinresistenten \emph{Staphylococcus aureus}-Stämmen (MRSA) in medizinischen und pflegerischen Einrichtungen. {[}Recommendations for the prevention and control of methicillin-resistant \emph{Staphylococcus aureus} isolates (MRSA) in hospitals and other healthcare facilities{]}. GMS Krankenhhyg Interdiszip. 2009;4(1):Doc01.
\item
  Friedrich AW, Daniels-Haardt I, Köck R, Verhoeven F, Mellmann A, Harmsen D, \emph{et al.} EUREGIO MRSA-net Twente/Münsterland--a Dutch-German cross-border network for the prevention and control of infections caused by methicillin-resistant \emph{Staphylococcus aureus}. Euro Surveill. 2008;13(35):18965. 10.2807/ese.13.35.18965-en
\item
  Ciccolini M, Donker T, Köck R, Mielke M, Hendrix R, Jurke A, \emph{et al.} Infection prevention in a connected world: the case for a regional approach. Int J Med Microbiol. 2013;303(6-7):380-7. 10.1016/j.ijmm.2013.02.003
\item
  Souverein D, Houtman P, Euser SM, Herpers BL, Kluytmans J, Den Boer JW. Costs and Benefits Associated with the MRSA Search and Destroy Policy in a Hospital in the Region Kennemerland, The Netherlands. PLoS One. 2016;11(2):e0148175. 10.1371/journal.pone.0148175
\item
  Jurke A, Köck R, Becker K, Thole S, Hendrix R, Rossen J, \emph{et al.} Reduction of the nosocomial meticillin-resistant \emph{Staphylococcus aureus} incidence density by a region-wide search and follow-strategy in forty German hospitals of the EUREGIO, 2009 to 2011. Euro Surveill. 2013;18(36):20579. 10.2807/1560-7917.ES2013.18.36.20579
\item
  Eurostat. Nomenclature of territorial units for statistics (NUTS); National structures (EU). Eurostat; 2018. Available from: \url{https://ec.europa.eu/eurostat/web/nuts/national-structures-eu}
\item
  Eurostat. Population density by NUTS 3 region. Eurostat: 2018. Available from: \url{https://ec.europa.eu/eurostat/web/products-datasets/product?code=demo_r_d3dens}
\item
  Werkgroep Infectiepreventie (WIP). Meticilline-resistente \emph{Staphylococcus aureus} (MRSA) -- Ziekenhuizen {[} Meticilline-resistente \emph{Staphylococcus aureus} (MRSA) - hospitals{]}. Leiden; WIP; 2012. Dutch. Available from: \url{http://www.rivm.nl/dsresource?objectid=3f054354-ff4a-43ef-91f9-7c6f0417be95\&type=org\&disposition=inline}
\item
  Nederlandse Vereniging voor Medische Microbiologie (NVMM). NVvMM ---richtlijn {[}Dutch Society for Medical Microbiology---Guideline: Laboratory detection of highly resistant microorganisms (HRMO) version 2.0{]}. Leeuwarden; NVMM; 2012. {[}Accessed 03 Feb 2019{]}. Available from: \url{https://www.nvmm.nl/media/1051/2012_hrmo_mrsa_esbl.pdf}.
\item
  Surveillance NI. MRSA-KISS-Protocoll-2016; MRSA KISS Reference data 2016. {[}Accessed 03 Feb 2019{]}. Available from: \url{https://www.nrz-hygiene.de/surveillance/kiss/mrsa-kiss/archiv/} 2017.
\item
  European Commission. Communication from the Commission to the European Parliament and the Council. Action plan against the rising threats from Antimicrobial Resistance. Brüssel: Publications Office of the European Union; 2011. Available from: \url{https://www.eumonitor.eu/9353000/1/j4nvke1fm2yd1u0_j9vvik7m1c3gyxp/vkcwedxv7dzy/v=s7z/f=/com(2011)748_en.pdf}
\item
  PEG) P-E-GfC. GERMAP 2015 -- Bericht über den Antibiotikaverbrauch und die Verbreitung von Antibiotikaresistenzen in der Human- und Veterinärmedizin in Deutschland. Rheinbach: Antiinfectives Intelligence; 2016. Report No.: 978-3-9818383-0-5. Available from: \url{http://www.p-e-g.org/econtext/germap}
\item
  SWAB. NethMap 2017, Consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in The Netherlands in 2016. MARAN 2017, Monitoring of Antimicrobial Resistance and Antibiotic Usage in Animals in the Netherlands in 2016. Bergen; 2019. Available from: \url{https://www.rivm.nl/bibliotheek/rapporten/2017-0056.pdf}
\item
  van der Zee A, Hendriks WD, Roorda L, Ossewaarde JM, Buitenwerf J. Review of a major epidemic of methicillin-resistant \emph{Staphylococcus aureus}: the costs of screening and consequences of outbreak management. Am J Infect Control. 2013;41(3):204-9. 10.1016/j.ajic.2012.02.033
\item
  Köck R, Werner P, Friedrich AW, Fegeler C, Becker K, Prevalence of Multiresistant Microorganisms (PMM) Study Group. Prevalence of Multiresistant Microorganisms PMM Study Group Persistence of nasal colonization with human pathogenic bacteria and associated antimicrobial resistance in the German general population. New Microbes New Infect. 2015;9:24-34. 10.1016/j.nmni.2015.11.004
\item
  Weterings V, Veenemans J, van Rijen M, Kluytmans J. Prevalence of nasal carriage of methicillin-resistant \emph{Staphylococcus aureus} in patients at hospital admission in the Netherlands, 2010 - 2017: an observational study. Clin Microbiol Infect. 2019;S1198-743X(19)30114-4. 10.1016/j.cmi.2019.03.012
\item
  Köck R, Schaumburg F, Mellmann A, Köksal M, Jurke A, Becker K, \emph{et al.} Livestock-associated methicillin-resistant \emph{Staphylococcus aureus} (MRSA) as causes of human infection and colonization in Germany. PLoS One. 2013;8(2):e55040. 10.1371/journal.pone.0055040
\item
  Schaumburg F, Köck R, Mellmann A, Richter L, Hasenberg F, Kriegeskorte A, \emph{et al.} study group Population dynamics among methicillin-resistant \emph{Staphylococcus aureus} isolates in Germany during a 6-year period. J Clin Microbiol. 2012;50(10):3186-92. 10.1128/JCM.01174-12
\item
  Köck R, Brakensiek L, Mellmann A, Kipp F, Henderikx M, Harmsen D, \emph{et al.} Cross-border comparison of the admission prevalence and clonal structure of meticillin-resistant \emph{Staphylococcus aureus}. J Hosp Infect. 2009;71(4):320-6. 10.1016/j.jhin.2008.12.001
\item
  Haley RW. Surveillance by objective: a new priority-directed approach to the control of nosocomial infections. The National Foundation for Infectious Diseases lecture. Am J Infect Control. 1985;13(2):78-89. 10.1016/0196-6553(85)90085-9
\item
  Gastmeier P, Bräuer H, Forster D, Dietz E, Daschner F, Rüden H. A quality management project in 8 selected hospitals to reduce nosocomial infections: a prospective, controlled study. Infect Control Hosp Epidemiol. 2002;23(2):91-7. 10.1086/502013
\item
  Pronovost P, Needham D, Berenholtz S, Sinopoli D, Chu H, Cosgrove S, \emph{et al.} An intervention to decrease catheter-related bloodstream infections in the ICU. N Engl J Med. 2006;355(26):2725-32. 10.1056/NEJMoa061115
\item
  Robert-Koch-Institute. Surveillance Statistics (Germany). \href{mailto:SurvStat@RKI}{\nolinkurl{SurvStat@RKI}} 2.0. {[}Accessed 03 Feb 2019{]}. Available from: \url{https://survstat.rki.de}
\item
  Walter J, Haller S, Blank HP, Eckmanns T, Abu Sin M, Hermes J. Incidence of invasive meticillin-resistant \emph{Staphylococcus aureus} infections in Germany, 2010 to 2014. Euro Surveill. 2015;20(46):30067. 10.2807/1560-7917.ES.2015.20.46.30067
\item
  Robert-Koch-Institute. Antimicrobial Resistance Surveillance (ARS, Germany). {[}Accessed 03 Feb 2019{]}. Available from: \url{https://ars.rki.de}
\item
  Olearo F, Albrich WC, Vernaz N, Harbarth S, Kronenberg A, Swiss Centre For Antibiotic Resistance Anresis \emph{Staphylococcus aureus} and methicillin resistance in Switzerland: regional differences and trends from 2004 to 2014. Swiss Med Wkly. 2016;146:w14339. 10.4414/smw.2016.14339
\item
  NVMM. Infectieziekten Surveillance Informatie Systeem - Antibiotica Resistentie (ISIS-AR). {[}Accessed 03 Feb 2019{]}. Available from: \url{https://www.isis-web.nl}
\item
  Raupach-Rosin H, Duddeck A, Gehrlich M, Helmke C, Huebner J, Pletz MW, \emph{et al.} Deficits in knowledge, attitude, and practice towards blood culture sampling: results of a nationwide mixed-methods study among inpatient care physicians in Germany. Infection. 2017;45(4):433-41. 10.1007/s15010-017-0990-7
\item
  Köck R, Kreienbrock L, van Duijkeren E, Schwarz S. Antimicrobial resistance at the interface of human and veterinary medicine. Vet Microbiol. 2017;200:1-5. 10.1016/j.vetmic.2016.11.013
\item
  Maier GS, Thorey F, Kolbow K, Lazovic D, Lühmann M, Ohnsorge J, \emph{et al.} Livestock-assoziierter Methicillin-resistenter \emph{Staphylococcus aureus} -- Erhebung einer orthopädischen Fachklinik im Hochrisikogebiet Nord-West. {[}Livestock-Associated Methicillin-Resistant \emph{Staphylococcus aureus}: Epidemiological Data from an Orthopaedic Department in North-West Germany{]}. Z Orthop Unfall. 2017;155(3):304-9.
\item
  Nillius D, von Müller L, Wagenpfeil S, Klein R, Herrmann M. Methicillin-Resistant \emph{Staphylococcus aureus} in Saarland, Germany: The Long-Term Care Facility Study. PLoS One. 2016;11(4):e0153030. 10.1371/journal.pone.0153030
\item
  Paget J, Aangenend H, Kühn M, Hautvast J, van Oorschot D, Olde Loohuis A, \emph{et al.} MRSA Carriage in Community Outpatients: A Cross-Sectional Prevalence Study in a High-Density Livestock Farming Area along the Dutch-German Border. PLoS One. 2015;10(11):e0139589. 10.1371/journal.pone.0139589
\end{enumerate}

\hypertarget{ch10-multi-mdro-screening}{%
\chapter{A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures}\label{ch10-multi-mdro-screening}}

Accepted in Eurosurveillance (ahead of print)

(as of date of PhD defence: 25 August 2021)

Berends MS \textsuperscript{1,2}*, Glasner C \textsuperscript{2}*, Becker K \textsuperscript{3,4}, Esser J \textsuperscript{5}, Gieffers J \textsuperscript{6}, Jurke A \textsuperscript{7}, Kampinga G \textsuperscript{2}, Kampmeier S \textsuperscript{8}, Klont R \textsuperscript{9}, Köck R \textsuperscript{8,10}, Al Naemi N \textsuperscript{9}, Ott A \textsuperscript{1}, Ruijs G \textsuperscript{11}, Saris K \textsuperscript{12}, Tami A \textsuperscript{2}, Van Zeijl J \textsuperscript{13}, Von Müller L \textsuperscript{14}, Voss A \textsuperscript{12}, Waar K \textsuperscript{13}, Friedrich AW \textsuperscript{2}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Certe Medical Diagnostics and Advice Foundation, Groningen, The Netherlands
\item
  Department of Medical Microbiology and Infection Control, University of Groningen, University Medical Center Groningen, Groningen, The Netherlands
\item
  Institute of Medical Microbiology, University Hospital Münster, Münster, Germany
\item
  Friedrich Loeffler-Institute of Medical Microbiology, University Medicine Greifswald, Greifswald, Germany
\item
  Practice of Laboratory Medicine and University Osnabrück, Department of Dermatology, Environmental Medicine and Health Theory, Osnabrück, Germany
\item
  Institute for Microbiology, Hygiene and Laboratory Medicine, Klinikum Lippe, Detmold, Germany
\item
  North Rhine-Westphalian Centre for Health, Section Infectious Disease Epidemiology, Bochum, Germany
\item
  Institute of Hygiene, University Hospital Münster, Münster, Germany
\item
  Laboratory Microbiology Twente Achterhoek, Hengelo, The Netherlands
\item
  Institute of Hygiene, DRK Kliniken Berlin, Berlin, Germany
\item
  Laboratory for Medical Microbiology and Infectious Diseases, Isala, Zwolle, The Netherlands
\item
  Department of Medical Microbiology, Radboud University Medical Centre and Canisius-Wilhelmina Hospital, Nijmegen, The Netherlands
\item
  Izore, Centre for Infectious Diseases Friesland, Leeuwarden, The Netherlands
\item
  Institute for Laboratory Medicine, Microbiology and Hygiene, Christophorus-Kliniken GmbH, Coesfeld, Germany
\end{enumerate}

* These authors contributed equally

\hypertarget{abstract-8}{%
\section*{Abstract}\label{abstract-8}}
\addcontentsline{toc}{section}{Abstract}

Antimicrobial resistance poses a risk for healthcare, both in the community and hospitals. The spread of multi-drug resistant organisms (MDROs) occurs mostly on a local and regional level, following movement of patients, also across national borders. The aim of this observational study was to determine the prevalence of MDROs in a European cross-border region to understand differences and improve infection prevention based on real-time routine data and workflows. Between September 2017 and June 2018, 23 hospitals in the Dutch-German cross-border region (NL-BR and DE-BR) participated in the study. During eight consecutive weeks, patients were screened upon admission to intensive care units (ICUs) for nasal carriage of methicillin-resistant \emph{Staphylococcus aureus} (MRSA) and rectal carriage of vancomycin-resistant \emph{Enterococcus faecium}/\emph{E. faecalis} (VRE), third-generation cephalosporin-resistant \emph{Enterobacteriaceae} (3GCRE) and carbapenem-resistant \emph{Enterobacteriaceae} (CRE). All samples were processed in the associated laboratories. A total of 3,365 patients were screened (NL-BR: 1,202, DE-BR: 2,163). The median screening compliance was 60.4\% (NL-BR: 56.9\%, DE-BR: 62.9\%). The MDRO prevalence was higher in the DE-BR than in the NL-BR, namely 1.7\% vs 0.6\% for MRSA (\emph{p} = 0.006), 2.7\% vs 0.1\% for VRE (\emph{p} \textless{} 0.001) and 6.6\% vs 3.6\% for 3GCRE (\emph{p} \textless{} 0.001), whereas the prevalence for CRE was comparable, with 0.2\% in DE-BR ICUs vs 0.0\% in NL-BR ICUs. This first prospective multi-centre screening study in a European cross-border region, shows high heterogenicity in MDRO carriage prevalence on NL-BR and DE-BR ICUs. This indicates that the prevalence is influenced by the different healthcare structures.

\hypertarget{introduction-7}{%
\section{Introduction}\label{introduction-7}}

Antimicrobial resistance (AMR) is a growing public health threat worldwide. Like global pandemics, multi-drug resistant bacteria pose one of the largest health risks to humans both in the community and within healthcare facilities \textsuperscript{{[}1,2{]}}. Specifically, hospitals are exposed to this risk and are challenged at multiple levels, e.g., the individual patient, the healthcare team, the organization and the political and economic environment. In hospitals, patients colonised and/or infected with multi-drug resistant organisms (MDROs) lead to higher costs, have prolonged hospital stays, have higher risks for complications, and an increased morbidity and mortality \textsuperscript{{[}3,4{]}}. To decrease these risks, the World Health Organization (WHO) urgently advised to change the way antibiotics are prescribed, and in addition highlighted that behavioural changes, resulting from the implementation of infection prevention measures, are indispensable to successfully combat AMR \textsuperscript{{[}5,6{]}}. According to WHO analyses, one key pitfall is that international AMR surveillance is neither coordinated nor harmonised and that there are still information gaps, especially with respect to twelve MDROs, which have been categorised as urgently requiring new antibiotics and improved combat strategies \textsuperscript{{[}6,7{]}}. These MDROs include amongst others: methicillin-resistant \emph{Staphylococcus aureus} (MRSA), carbapenem-resistant \emph{Enterobacteriaceae} (CRE), extended spectrum beta-lactamase (ESBL)-producing \emph{Enterobacteriaceae} and vancomycin-resistant \emph{Enterococcus faecium} (VRE) \textsuperscript{{[}7{]}}.

The prevalence of such MDROs varies not only between countries, but also between different regions (henceforth called healthcare regions), within one country or comprising a cross-border region, such as the Dutch-German cross-border region \textsuperscript{{[}8,9{]}}. Hospital transfer of patients within or between healthcare regions (i.e., from a local or regional hospital to a university medical centre or vice versa) can be a substantial driver of AMR \textsuperscript{{[}9{]}}. Thus, prevalence estimates of MDROs at regional level may better reflect the actual reality and allow the implementation of interventions more effectively. This is of utmost importance especially since the European Union (EU) directive from 2011 allows patients to seek medical treatment in any EU country. As approx. 30\% of all EU citizens live in a cross-border region, this underlines the importance of a non-national-only, but a regional cross-border approach.

The Dutch-German cross-border region has been at the forefront in cooperating in the domain of AMR and infection prevention since 2005 with the support of European INTERREG programmes (www.deutschland-nederland.eu). Ever since, the projects developed within the INTERREG program have been denoted a `best practice' for studying the prevalence of MDRO in a European cross-border region (Interact, European Cooperation Day, 2013). Importantly, among all cross-border regions in Europe, the Dutch-German cross-border region exhibits the most frequent exchange of citizens, with 74\% of Germans and Dutch citizens living close to the border indicating to have visited the other country \textsuperscript{{[}10{]}}. On top of that, patient movements, exchange of patients between different healthcare institutes, across this particular border occur on a regular basis \textsuperscript{{[}9{]}}.

A recently published comparison of the national Dutch and German guidelines on Gram-negative MDROs urged the usage of consistent terminology and harmonised diagnostic procedures for the improvement of infection prevention, treatment and patient safety \textsuperscript{{[}11{]}}. Gathering and comparing regional data from both sides of the border was considered essential because of two reasons. Firstly, the EU treaty of Lisbon and directives in vigour will lead to an increasing number of patients seeking medical treatment in a neighbouring country. Secondly, particularly in cross-border regions between two high-income countries with cost-extensive, highly advanced and technological driven healthcare systems, the number of neonates, immuno-compromised and elderly patients that are seeking treatment will continue to increase \textsuperscript{{[}12{]}}.

With the advancements in healthcare, the demographic changes and increase in the number of multimorbidity, intensive care units (ICUs) have become the main hubs for patients in any hospital \textsuperscript{{[}13,14{]}}. ICUs represent a distinct hospital environment with high-frequent contact between specially trained hospital staff and critically ill patients requiring advanced technology and increased antibiotic prescription \textsuperscript{{[}15{]}}. Thus, ICUs are hotspots for the emergence and transmission of MDROs, frequently causing infections in these critically ill patients \textsuperscript{{[}16{]}}.

Therefore, the aim of this observational prospective multicentre screening study was to determine the prevalence of selected MDROs on admission to adult ICUs in the Dutch-German cross-border region based on real-time routine data and workflows and to correlate those with the existing healthcare structures.

\hypertarget{methods-4}{%
\section{Methods}\label{methods-4}}

\hypertarget{study-design}{%
\subsection{Study Design}\label{study-design}}

This observational prospective multicentre screening study was carried out between the 1st of September 2017 and the 18th of June 2018 in the Dutch-German cross-border region (NL-DE-BR) to determine the prevalence of MDROs on adult ICUs. All adult patients (≥18 years) were included in the study. The screening period for all hospitals lasted eight consecutive weeks (Supplementary Figure 1). A total of 23 hospitals, eight Dutch and 15 German, participated in this study. The 23 hospitals were served by ten laboratories, six on the Dutch (Dutch border region; NL-BR) and four on the German (German border region; DE-BR) side. Both regions have a similar geographical size, population density and type of hospital care (one university hospital, several secondary care hospitals). During the screening period, each participating hospital aimed at screening all patients at admission to their participating ICU for nasal carriage of MRSA and rectal carriage of VRE (both \emph{E. faecium} and \emph{E. faecalis}), 3GCRE and CRE. For the definition of 3GCRE, the European Centre for Disease Prevention and Control (ECDC) guideline was followed: all of cefotaxime, ceftazidime and ceftriaxone were considered. Moreover, although defined as \emph{Enterobacteriaceae}, the present study focussed solely on \emph{Escherichia coli} and \emph{Klebsiella spp}. An overview of all MDRO definitions used in this study is summarised in the Supplementary Material. All samples were processed at the associated routine diagnostic laboratory, which were all ISO certified at the time of the study, following local standard operating procedures which were adapted to the study protocol when necessary (Supplementary Material Table 1). Bacterial species were confirmed by matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry and antibiotic susceptibility was determined using VITEK 2 automated systems with EUCAST (European Committee on Antimicrobial Susceptibility Testing) clinical breakpoints \textsuperscript{{[}17{]}}. Moreover, data about the number of beds per hospital and ICU, hospital and ICU admissions and hospital and ICU patient days were provided by all participating hospitals for 2016.

\hypertarget{statistical-analysis-software}{%
\subsection{Statistical Analysis \& Software}\label{statistical-analysis-software}}

Data analysis was done in R using the software application RStudio and the R package AMR (R v4.0.2, RStudio v1.3.959 and AMR package v1.3.0), which are all free, open-source and publicly available \textsuperscript{{[}18{]}}.

Contingency tables were tested with Fisher's exact test when the size was 2x2 and Chi2 tests otherwise. To test for equality in prevalence between countries, the exact binomial test was used. Outcomes of statistical tests were considered significant when two-sided \emph{p} \textless{} 0.05.

\hypertarget{ethics}{%
\subsection{Ethics}\label{ethics}}

The medical ethical committee of the University Medical Center Groningen (UMCG, The Netherlands) was informed and patients or their relatives were approached to voluntarily participate in the study. Ethical approval and informed consent were not required (METc 2015.535). All data were collected in accordance with the European Parliament and Council decisions on the epidemiological surveillance and control of communicable disease in the European Community. The board of directors of all other participating hospitals agreed to conduct the study.

\hypertarget{results-5}{%
\section{Results}\label{results-5}}

\hypertarget{healthcare-structure-of-the-participating-hospitals}{%
\subsection{Healthcare structure of the participating hospitals}\label{healthcare-structure-of-the-participating-hospitals}}

Between the 1\textsuperscript{st} of September 2017 and the 18\textsuperscript{th} of June 2018, 23 hospitals in the NL-DE-BR participated in the study, eight in the NL-BR and 15 in the DE-BR. The total number of beds from all participating ICUs was 443 beds (NL-BR: 182 {[}41.1\%{]}, DE-BR: 261 {[}58.9\%{]}). The bed capacity of the ICUs in relation to the respective hospital bed capacity did not differ between hospitals within either country or between the two countries (NL-BR: 3.2\% {[}IQR: 3.0-3.7\%{]}, DE-BR: 3.6\% {[}IQR: 1.8-5.5\%{]}). The participating hospitals are characterised by the data shown in Table 1.

Table 1. Overview of the number of hospitals, laboratories, number of beds per hospital and ICU, hospital and ICU admissions, hospital and ICU patient days and average length of stay, Dutch-German cross-border region, 2016.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-t01} 

}

\end{figure}

\hypertarget{study-population-and-screening-samples-from-icus}{%
\subsection{Study population and screening samples from ICUs}\label{study-population-and-screening-samples-from-icus}}

A total of 3,365 patients were screened: 1,202 (35.7\%) on NL-BR and 2,163 (64.3\%) on DE-BR ICUs (Table 2). The screening period per hospital lasted eight consecutive weeks (56 days, IQR: 55-58 days, Supplementary Figure 1). In both, NL-BR and DE-BR, significantly more males than females were screened (\emph{p} \textless{} 0.001) and in NL-BR relatively less females were screened than in DE-BR (\emph{p} \textless{} 0.01). The median age of all screened patients was 68 years (IQR: 57-77), while patients in DE-BR were significantly older than patients in the NL-BR (\emph{p} \textless{} 0.001).

A total of 6,462 swabs were taken, 2,308 (35.7\%) in NL-BR and 4,154 (64.3\%) in DE-BR ICUs. Of those, 3,292 were taken from the nasopharynx and 3,170 were from the rectum. The overall screening compliance (screened for at least one MDRO group) was 60.4\% (3,365 out of 5,568). For ICUs in the NL-BR this was 56.9\% (1,202 out of 2,111) and for ICUs in the DE-BR this was 62.9\% (2,163 out of 3,457), \emph{p} \textless{} 0.001. The median screening compliance for all four MDRO groups (i.e., nasopharyngeal swab for MRSA, rectal swab for VRE, 3GCRE and CRE) on the other hand was in total 55.3\% (3,081 out of 5,568), and 52.1\% (1,100 out of 2,111) in NL-BR and 57.3\% (1,981 out of 3,457) in DE-BR ICUs (\emph{p} \textless{} 0.001). Most patients (91.5\% for NL-DE-BR ICUs) that were screened while present on the ICU were screened for all MDRO groups.

In total, 3,291 patients were screened for MRSA (1,174 {[}35.7\%{]} in NL-BR and 2,117 {[}64.3\%{]} in DE-BR ICUs), 3,145 for VRE (1,110 {[}35.3\%{]} in NL-BR and 2,035 {[}64.7\%{]} in DE-BR ICUs) and 3,152 for 3GCRE (1,126 {[}35.7\%{]} in NL-BR and 2,026 {[}64.3\%{]} in DE-BR ICUs). Of note, in some patients multiple MDROs were found from the same or different species, meaning that some patients are included in multiple MDRO groups.

Table 2. Overview of total number of patients present and screened, swabs and type of bacteria tested for in NL-BR and DE-BR, September 2017 -- June 2018.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-t02} 

}

\end{figure}

\hypertarget{prevalence-of-gram-positive-mdros-mrsa-and-vre}{%
\subsection{Prevalence of Gram-positive MDROs: MRSA and VRE}\label{prevalence-of-gram-positive-mdros-mrsa-and-vre}}

The overall prevalence for MRSA carriage at ICU admission was 1.3\% (43 out of 3,291), and for VRE carriage 1.8\% (56 out of 3,145). The prevalence was higher in DE-BR than in NL-BR ICUs, namely 1.7\% (36 of 2,117) vs 0.6\% (7 of 1,174) for MRSA (\emph{p} = 0.006) and 2.7\% (55 of 2,035) vs 0.1\% (1 of 1,110) for VRE (\emph{p} \textless{} 0.001), respectively (Figure 1). The prevalence ranged from 0\% to 1.5\% in NL-BR ICUs and from 0\% to 4.1\% in DE-BR ICUs for MRSA and from 0\% to 0.3\% in NL-BR ICUs and from 0\% to 4.8\% in DE-BR ICUs for VRE (Figure 1). An overview of all isolated MRSA and VRE isolates can be found in the Supplementary Table 2. Notably, all 56 cases of VRE were caused by \emph{E. faecium}.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-01} 

}

\caption{Prevalence of MRSA and VRE in NL-BR ICUs, in DE-BR ICUs and in both cross-border regions together (NL-DE-BR ICUs). Numbers above in squares represent the number of positive patients divided by the total number of patients screened for the respective pathogen with the calculated prevalence. Boxplots show the median prevalence in participating ICUs (thick line within each box), the first and third quartile (upper and lower border of the box, the difference is the IQR), and the whiskers with error bars represent 1.5 times the IQR denoting the normal range. The dots are outside this range. DE-BR: German cross-border region; ICU: intensive care unit; IQR: interquartile range; MRSA: methicillin-resistant *Staphylococcus aureus*; NL-BR: Dutch cross-border region; NL-DE-BR: total Dutch-German cross-border region; VRE: vancomycin-resistant enterococci.}\label{fig:fig10-1}
\end{figure}

\hypertarget{prevalence-of-gram-negative-mdro-3gcre-and-cre}{%
\subsection{Prevalence of Gram-negative MDRO: 3GCRE and CRE}\label{prevalence-of-gram-negative-mdro-3gcre-and-cre}}

The overall prevalence at ICU admission for 3GCRE carriage was 5.5\% (173 out of 3,152) and 0.1\% (4 out of 3,152) for CRE carriage. The prevalence for 3GCRE was significantly higher in DE-BR than in NL-BR ICUs, namely 6.6\% (133 out of 2,026) vs 3.6\% (40 out of 1,122), \emph{p} \textless{} 0.001, whereas the prevalence for CRE was comparable, with 0.0\% (0 out of 1,126) in NL-BR ICUs vs 0.2\% (4 out of 2,026) in DE-BR ICUs (Figure 2 and Table 2). Most of the isolated 3GCRE were \emph{E. coli} isolates, namely 166 (92.2\%). Twelve isolates were \emph{K. pneumoniae} (6.8\%), one \emph{K. variicola} (0.6\%) and one \emph{K. oxytoca} (0.6\%). The four CRE isolates were found in three different DE-BR ICUs, three were \emph{E. coli} and one was a \emph{K. pneumoniae} isolate. The prevalence for 3GCRE differed within both countries between hospitals, ranging from 0\% to 10.0\% in NL-BR ICUs and from 2.3\% to 15.2\% in DE-BR ICUs (Figure 2). Table 2 presents an overview of the prevalence of MRSA, VRE, 3GCRE and CRE. An overview of all isolated 3GCRE and CRE isolates can be found in the Supplementary Table 2.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-02} 

}

\caption{Prevalence of 3GCRE and CRE in NL-BR ICUs, in DE-BR ICUs and in both cross-border regions together (NL-DE-BR ICUs). Numbers above in squares represent the number of positive patients divided by the total number of patients screened for the respective pathogen with the calculated prevalence. 3GCR: third-generation cephalosporin-resistant *Enterobacteriaceae*, CRE: carbapenem-resistant *Enterobacteriaceae*; DE-BR: German cross-border region; NL-BR: Dutch cross-border region; NL-DE-BR: total Dutch-German cross-border region.}\label{fig:fig10-2}
\end{figure}

\hypertarget{prevalence-of-gram-negative-mdros-based-on-dutch-and-german-definitions}{%
\subsection{Prevalence of Gram-negative MDROs based on Dutch and German definitions}\label{prevalence-of-gram-negative-mdros-based-on-dutch-and-german-definitions}}

The national guidelines for The Netherlands and Germany differ greatly in the way Gram-negative MDROs are being defined (while definitions for MRSA and VRE are identical) \textsuperscript{{[}12,19{]}}. An overview of the specific Dutch and German definitions of MDROs is summarised in the Supplementary Material.

The German national infection prevention guideline classifies Gram-negative MDROs into 3MRGN and 4MRGN (German: `Multiresistente Gram-negative Stäbchen', multidrug-resistant Gram-negative rods) based on phenotypic susceptibility. When the German MRGN definition is being applied to all Gram-negative isolates, the overall prevalence for 3MRGN is 2.9\% (91 out of 3,152) and for 4MRGN 0.1\% (4 out of 3,152). The prevalence was significantly lower in NL-BR than in DE-BR ICUs for 3MRGN, namely 1.1\% (12 out of 1,126) vs 3.9\% (79 out of 2,026) {[}\emph{p} \textless{} 0.001{]}, whereas the prevalence for 4MRGN was comparable, namely 0\% (0 out of 1,126) vs 0.2\% (4 out of 2,026) {[}\emph{p} = 0.30{]} (Figure 3). The prevalence for 3MRGN differed within both countries between hospitals, ranging from 0\% to 5.0\% in NL-BR and from 1.2\% to 10.9\% in DE-BR ICUs. The four 4MRGN were three \emph{E. coli} isolates and one \emph{K. pneumoniae} isolate and originated from three different DE-BR ICUs. Of note, for the definition of 3MRGN, piperacillin results could not be included since only results for piperacillin-tazobactam were reported.

The Dutch national guideline defines exceptional resistant microorganisms as BRMO (`Bijzonder Resistente Microorganismen') using strict interpretation guidelines \textsuperscript{{[}20{]}}. When the Dutch BRMO definition is applied to all Gram-negative isolates, the overall BRMO prevalence is 5.6\% (176 out of 3,152). The prevalence was lower in NL-BR than in DE-BR ICUs, namely 3.9\% (44 out of 1,126) vs 6.5\% (132 out of 2,026) for BRMOs {[}\emph{p} = 0.002{]} (Figure 3). The prevalence for BRMO differed within both countries between hospitals, ranging from 0\% to 10.0\% in NL-BR and from 2.3\% to 15.2\% in DE-BR ICUs.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-03} 

}

\caption{Prevalence of 3MRGN, 4MRGN and BRMO in NL-BR ICUs, DE-BR ICUs and both cross-border regions together (NL-DE-BR ICUs). Numbers above in squares represent the number of positive patients divided by the total number of patients screened for the respective pathogen with the calculated prevalence. 3MRGN: Multiresistente Gram-negative Stäbchen mit Resistenz gegen 3 der 4 Antibiotikagruppen (multiresistant Gram-negative rods with resistance to 3 of the 4 antibiotic groups); 4MRGN: Multiresistente Gram-negative Stäbchen mit Resistenz gegen 4 der 4 Antibiotikagruppen (multidrug-resistant Gram-negative rods with resistance to 4 of the 4 antibiotic groups); BRMO: bijzonder-resistente microorganism (particularly resistant microorganisms); NL-BR: Dutch cross-border region; NL-DE-BR: total Dutch-German cross-border region.}\label{fig:fig10-3}
\end{figure}

\hypertarget{comparison-of-mdro-prevalence-between-nl-br-and-de-br-icus-in-university-and-non-university-hospitals}{%
\subsection{Comparison of MDRO prevalence between NL-BR and DE-BR ICUs in university and non-university hospitals}\label{comparison-of-mdro-prevalence-between-nl-br-and-de-br-icus-in-university-and-non-university-hospitals}}

For NL-BR ICUs, the prevalence of all MDRO groups was not significantly different between the non-university and the university hospital (Figure 4). This was different for the participating DE-BR ICUs where the prevalence of 3GCRE {[}\emph{p} \textless{} 0.001{]}, 3MRGN {[}\emph{p} = 0.005{]} and BRMO {[}\emph{p} \textless{} 0.001{]} were significantly higher in the non-university hospitals (Figure 4). Interestingly, the prevalence of almost all investigated MDROs was not significantly different between the two university hospitals, except for the prevalence of VRE, which was significantly higher in the German university ICU {[}\emph{p} \textless{} 0.001{]}. Comparing the prevalence of all investigated MDROs between NL-BR and DE-BR non-university hospital ICUs revealed a significant difference for VRE {[}\emph{p} \textless{} 0.001{]}, 3GCRE {[}\emph{p} \textless{} 0.001{]}, 3MRGN {[}\emph{p} \textless{} 0.001{]} and BRMO {[}\emph{p} \textless{} 0.001{]}, whereas the difference for MRSA {[}\emph{p} = 0.83{]} differed only slightly (Figure 4).

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/10-04} 

}

\caption{Comparison between prevalence of MRSA, VRE, 3GCRE, CRE, 3MRGN, 4MRGN and BRMO between non-university and university hospital ICUs in the NL-BR and DE-BR. 3MRGN: multiresistant Gram-negative rods with resistance to 3 of the 4 antibiotic groups (multiresistant Gram-negative rods with resistance to 3 of the 4 antibiotic groups); 4MRGN: multidrug-resistant Gram-negative rods with resistance to 4 of the 4 antibiotic groups (multidrug-resistant Gram-negative rods with resistance to 4 of the 4 antibiotic groups); 3GCR: third-generation cephalosporin-resistant *Enterobacteriaceae*; BRMO: bijzonder-resistente microorganisme (particularly resistant microorganisms); DE-BR: German cross-border region; ICU: intensive care unit; IQR: interquartile range; MRSA: methicillin-resistant Staphylococcus aureus; NL-BR: Dutch cross-border region; NL-DE-BR: total Dutch-German cross-border region; VRE: vancomycin-resistant enterococci, CRE: carbapenem-resistant *Enterobacteriaceae*.}\label{fig:fig10-4}
\end{figure}

\hypertarget{discussion-6}{%
\section{Discussion}\label{discussion-6}}

To the best of our knowledge this is the first prospective observational multicentre screening study focusing on ICU admission prevalence of the most common MDROs in a healthcare region that comprises a national border. This study has been performed within the Dutch-German cross-border network, which has a long-lasting experience in close cooperation in the domain of AMR and infection prevention and control \textsuperscript{{[}21,22{]}}. Interestingly, the Dutch and German healthcare systems differ in many aspects, creating a natural ``living lab'' situation to study AMR and other healthcare-related topics. One difference is the overall hospital activity, as shown in Table 1. In the NL-BR, 4.8 per 100 hospital admissions lead to an ICU admission. In contrast, in the DE-BR this is 7.7 per 100 hospital admissions. This difference can be explained by the higher ICU capacity in DE-BR hospitals, namely 261 of 5,388 (4.8\%) vs 182 of 7,514 (2.4\%) in NL-BR. Interestingly, the median hospital-wide length of stay (LOS) is shorter in the NL-BR than in the DE-BR (4.98 vs 6.10 days), whereas the ICU-specific LOS is longer in the NL-BR (4.06 vs 3.57 days). When comparing our data with the LOS by Eurostat from 2017, it can be observed that the hospital-wide LOS of the NL-BR is comparable to the national average (5.0 vs 4.5 days), whereas for Germany, the LOS of the DE-BR is much lower (6.1 vs 9.0) \textsuperscript{{[}19{]}}. Although no information was available for the present study with regard to staffing in hospitals and ICUs, it has been shown by others that the number of available staff on German ICUs is much less than on Dutch ICUs, while understaffing has been found to be inversely proportional to finding MDROs \textsuperscript{{[}15,23,24{]}}. Strikingly, a more recent study focussing on the Dutch-German cross-border region presented that healthcare workers of both sides of the border have a similar awareness and perception towards AMR and both struggle with the limitations to cope with the application of preventive measures \textsuperscript{{[}25{]}}.

The success of infection prevention and other actions to combat AMR within a hospital can be measured by the occurrence of MDROs. To this end, the ECDC reports overviews of MDRO proportions based on nationally aggregated data from blood cultures on a regular basis. On a more country-specific level, MDRO proportions are also reported by national health institutes (NHI); the Rijkinstituut voor Volgsgezondheid en Milieu (RIVM) in the Netherlands and the Robert-Koch Institut (RKI) in Germany \textsuperscript{{[}26,27{]}}. These MDRO proportions differ greatly from the here reported prevalence of MDRO carriage. MDRO proportions are the fraction of e.g.~MRSA isolates among \emph{S. aureus} isolates, whereas MDRO prevalence is the fraction of patients with e.g.~MRSA colonisation in a certain patient population. MDRO proportions are thus based on the microorganism and the respective resistance pattern, information that can be easily extracted from any laboratory information system, whereas MDRO prevalence is based on the patient or a certain population and requires mostly active screening. While both are of high importance and serve different purposes, only MDRO prevalence informs us about the carriage or infection rate in patients.

In the present study, the overall carriage prevalence for the different MDROs was higher in the DE-BR ICUs, although some differences were marginal. Specifically, prevalence of MRSA carriage was three times higher in the DE-BR (1.7\%) than in the NL-BR (0.6\%). These prevalences are consistent with a recent study about all nosocomial MRSA cases in this region from 2012 until 2016 \textsuperscript{{[}22{]}}. For 2018, reports on the country level published by the ECDC, show that the proportion of MRSA among \emph{S. aureus} isolates from blood cultures was 1.2\% in the Netherlands and 7.6\% in Germany (with regional variations as per the Dutch and German NHIs; e.g.~0.3\% in the Northern Netherlands and 14.5\% in Northern-West Germany in any blood culture) \textsuperscript{{[}26-28{]}}. Differences between proportions and prevalence are of course expected, and the higher MRSA proportions can, for example, be explained by an increased antibiotic use to foster the occurrence of MRSA. Nevertheless, the rather low prevalence of MRSA carriage on both sides of the border demonstrates that national efforts to control MRSA specifically in this cross-border region, that are continuously successful in the Netherlands since decades, have now led to a decrease on the German side of the border as well.

For VRE, the prevalence measured in this study was 0.1\% in the NL-BR and also remained low in the DE-BR (2.7\%), although almost 30 times higher than in the NL-BR. This difference is also reflected by different proportions of VRE among \emph{E. faecium} from blood: 1.1\% in the Netherlands vs 23.8\% in Germany in 2018 as reported by the ECDC and 0.6\% vs 7.6\% in any blood culture in 2018 as reported by the Dutch and German NHIs, respectively \textsuperscript{{[}26-28{]}}. The large difference in the German VRE proportion between the data from ECDC and the German NHI cannot be explained. Moreover, Germany has seen a rapid increase in the proportion of VRE among \emph{E. faecium}, from 1.4\% in 2001 to 14.5\% in 2013 and thus 23.8\% in 2018 \textsuperscript{{[}28{]}}. The cause of this is still unknown. Probably due to the stringent infection prevention and outbreak control in the Netherlands, the proportion of VRE from blood cultures among \emph{E. faecium} never exceeded 1.5\% in the Netherlands \textsuperscript{{[}28{]}}.

The difference in MDRO prevalence between NL-BR and DE-BR was also observed for Gram-negative MDROs. Since the Netherlands and Germany have different guidelines to classify Gram-negative bacteria as MDRO (BRMO vs 3MRGN/4MRGN) but both phenotypically test for 3rd generation cephalosporins, a comparison was made based on 3GCRE. The 3GCRE carriage prevalence in the DE-BR was almost twice as high (6.6\%) as in the NL-BR (3.6\%), but both were still lower than national averages. The ECDC reported proportions of 3GCRE among \emph{E. coli} and \emph{K. pneumoniae} from blood in 2018 as \emph{E. coli}: 12.2\% and \emph{K. pneumoniae}: 12.9\% for Germany and \emph{E. coli}: 7.3\% and \emph{K. pneumoniae}: 11.1\% for the Netherlands. The same year the NHIs reported a slightly lower prevalence with \emph{E. coli} at 10.7\% and \emph{K. pneumoniae} at 12.0\% in Germany and \emph{E. coli} at 6.6\% and \emph{K. pneumoniae}: at 10.1\% in the Netherlands \textsuperscript{{[}26-28{]}}. This highlights that there are important differences to be found when studying carriage in specified populations versus looking at the proportion of invasive isolates, but that the lower carriage of Gram-negative MDROs in the participating NL-DE-BR hospitals shows the importance of a regional compared to a national view. Notably, in the present study only four CRE isolates were identified, all from the DE-BR. Interestingly, when applying the country specific guidelines to the Gram-negative MDROs study isolates, the Dutch BRMO guideline yields more MDRO than the German 3MRGN/4MRGN guideline (overall BRMO: 5.6\% vs overall 3MRGN/4MRGN: 2.9\%/0.1\%). This difference is comparable to results from a previous study where those guidelines were compared between the countries \textsuperscript{{[}12{]}}. Since the Dutch guideline classifies all third-generation cephalosporin-resistant \emph{E. coli} and \emph{Klebsiella spp}. as BRMO, while the German guideline only classifies them as MRGN if they are additionally ciprofloxacin-resistant, a higher prevalence of BRMO than MRGN was expected.

As both university and non-university hospitals participated in the study, a comparison of MDRO carriage prevalence on ICUs based on the type of hospital could be realised. In the NL-BR no significant difference for all investigated MDROs between university and non- university hospitals was observed. In the DE-BR, on the other hand, significant differences were observed for 3GCRE, 3MRGN and BRMO between university and non-university hospitals, but not for MRSA, VRE, 4MRGN and CRE. Non-university hospitals presented a significantly higher MDRO prevalence for 3GCRE, 3MRGN and BRMO at ICU admission. Explaining this observed dissimilarity requires additional studies on e.g.~hospital activity, size, staff availability, hospital geography and inter-hospital distance. A recent report highlighted that a higher density of inpatient care, a higher number of hospitals, a longer length of stay and lower staffing ratios all might facilitate MDRO dissemination \textsuperscript{{[}29{]}}. Interestingly, when comparing the hospital types between the two border regions, the university hospitals have a very similar prevalence of all MDROs on ICUs. Our results show that ICUs in non-university hospitals in the DE-BR are being challenged more frequently with Gram-negative MDROs compared to MRSA and VRE. Especially, with respect to third-generation cephalosporin resistance, this problem seems very prominent. This contradicts the general consensus that MDROs are less prevalent in smaller hospitals. The reason for this difference and problem is unknown and requires further investigation. However, experts claim that, especially in smaller hospital settings, up to one third of all hospital-associated infections can be prevented by solely improving infection prevention \textsuperscript{{[}30{]}}. To investigate this, more information about the staff and patients admitted to ICUs would be required, e.g., number of staff and hours available for infection prevention, information on severity of disease, antibiotic exposure or length of hospital stay prior to ICU admission.

The limitations of this study exemplify the challenge to compare AMR prevalence rates within or between healthcare regions, especially when comprising a national border. Firstly, the median screening compliance was dissatisfying in both border regions, although significantly higher in the DE-BR (62.6\%) than in the NL-BR (56.9\%). Only two hospitals were equipped with sufficient staff, one each side of the border; their screening compliance was 99.3\% and 83.2\%, respectively. This underlines the need for more (research) guidance and/or more staffing, education and material, to implement better infection prevention and control. It also accentuates the inherently limited maximum compliance to be gained from routine wards and workflows, which is also an important point of consideration when using (inter)nationally published results. Secondly, collection of information about infection control staff, MDRO outbreaks, infections, antibiotic use and risk factors of patients was outside the scope of this study. Although this would have allowed for the analysis of origin and source of the identified MRDOs, this information was practically impossible to retrieve from the 23 different hospitals and 3,365 patients included in this study due to legislative and organizational constraints. Thirdly, the participating laboratories in this study were not homogeneous in their diagnostic test methodologies and since for most of the laboratory's molecular confirmation (e.g., of resistance encoding genes) was not part of their standard operating procedures, it was also not included in the study protocol. Fourthly, not all hospitals conducted the screening in the same eight consecutive weeks, as this was practically unfeasible. While this might have improved comparability, others found almost no seasonality in bacterial bloodstream infections and we therefore consider this issue to be of low impact \textsuperscript{{[}31{]}}.

This study highlights the importance of a regional and cross-border approach in any European cross-border region, to illustrate the difference of AMR prevalence between the regions and to highlight potential differences with country-wide reports. Moreover, the focus on routine workflows in both the hospital and laboratories make this study valuable since it offers an honest perspective on the reality. To be able to emphasise on this further, attaining a deeper level of detail is a vast prerequisite, for example by collecting information about staff on the wards and infection control staff, MDRO outbreaks, infections, antibiotic use, and risk factors of patients. Standard reporting based on the Nomenclature of Territorial Units for Statistics (NUTS) on a NUTS3 or at least NUTS2 level instead of NUTS1 or the national level would also improve the resolution of the AMR prevalence within a country or healthcare region and improve the understanding thereof. Interestingly, comparisons with national data on MDRO proportions as reported by the ECDC and the respective NHIs revealed rather low numbers of submitted isolates which highlights a bottleneck of using this data source. Moreover, only a limited number of hospitals, mostly large (university) hospitals especially in Germany, actively participate in national or international surveillance systems arguing for the inclusion of small and medium-sized hospitals when determining and analysing MDRO prevalences. Additionally, generalising guidelines and definitions between countries, preferably on the European level, will improve comparability between countries which is of great importance for cross-border regions. In conclusion, geographical and political borders do not seem to be ``respected'' by MDROs, although healthcare systems, geographic nature and guidelines are very different between countries. Proportions of MDROs of certain pathogens, as reported on the national and international level, do not reflect MDRO prevalence in the patient or general population. This should be taken into serious consideration when interpreting reports on the country or even continental level.

\hypertarget{supplementary-files}{%
\section*{Supplementary files}\label{supplementary-files}}
\addcontentsline{toc}{section}{Supplementary files}

\begin{itemize}
\tightlist
\item
  Supplementary Table 1. Overview of the used media for screening in all participating laboratories in this study.
\item
  Supplementary Table 2. Overview of all antibiotic results of all positive isolates found in this study (one isolate per row).
\item
  Supplementary Figure 1. Screening period per hospital. All hospitals screened between September 2017 and July 2018. Hospitals \#5 and \#17 were university hospitals and started almost immediately after the start of the study. Hospital \#7 could only start in May 2018 due to lack of available personnel.
\item
  Supplementary Material. Overview and summary of MDRO definitions based on different national and international guidelines mentioned and used in the manuscript ``A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures''.
\end{itemize}

\hypertarget{acknowledgements-5}{%
\section*{Acknowledgements}\label{acknowledgements-5}}
\addcontentsline{toc}{section}{Acknowledgements}

This study was supported by the INTERREG V A (202085) funded project EurHealth-1Health (\url{http://www.eurhealth1health.eu}), part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport, the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the Ministry for National and European Affairs and Regional Development of Lower Saxony. The authors would like to thank all hospitals, laboratory and ICU staff for participating in this study.

\hypertarget{conflict-of-interest-1}{%
\section*{Conflict of interest}\label{conflict-of-interest-1}}
\addcontentsline{toc}{section}{Conflict of interest}

The authors declare no conflict of interest. The founding sponsors had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.

\hypertarget{references-9}{%
\section*{References}\label{references-9}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  Roca I, Akóva M, Baquero F, Carlet J, Cavaleri M, Coenen S, \emph{et al.} The global threat of antimicrobial resistance: science for intervention. New Microbes New Infect. 2015 Jul;6:22--9.
\item
  de Kraker MEA, Davey PG, Grundmann H, on behalf of the BURDEN study group. Mortality and hospital stay associated with resistant \emph{Staphylococcus aureus} and \emph{Escherichia coli} bacteremia: estimating the burden of antibiotic resistance in europe. PLoS Med. 2011;8(10):e1001104.
\item
  Cosgrove SE. The relationship between antimicrobial resistance and patient outcomes: mortality, length of hospital stay, and health care costs. Clin Infect Dis. 2006 Jan:S82--9.
\item
  Jung E, Byun S, Lee H, Moon SY, Lee H. Vancomycin-resistant \emph{Enterococcus} colonization in the intensive care unit: clinical outcomes and attributable costs of hospitalization. Am J Infect Control. 2014 Oct;42(10):1062--6.
\item
  O'Neill J. Tackling drug-resistant infections globally: final report and recommendations. 2016 May 18:1--84.
\item
  World Health Organization (WHO). Antimicrobial resistance: Global report on surveillance. 2014 Jun:1--256.
\item
  World Health Organization (WHO). Global priority list of antibiotic-resistant bacteria to guide research, discovery, and development of new antibiotics. 2017 Feb:1--7.
\item
  European Centre for Disease Control and Prevention (ECDC). Point prevalence survey of healthcare-associated infections and antimicrobial use in European acute care hospitals. 2013 Nov:1--216.
\item
  Ciccolini M, Donker T, Köck R, Mielke M, Hendrix R, Jurke A, \emph{et al.} Infection prevention in a connected world: the case for a regional approach. Int J Med Microbiol. 2013 Aug;303(6-7):380--7.
\item
  European Commission. Eurobarometer Results Programme Germany-Netherlands. 2015 Dec 14:1--5.
\item
  Köck R, Siemer P, Esser J, Kampmeier S, Berends MS, Glasner C, \emph{et al.} Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines. Microorganisms. 2018 Mar;6(1):11--8.
\item
  Müller J, Voss A, Köck R, Sinha B, Rossen JW, Kaase M, \emph{et al.} Cross-border comparison of the Dutch and German guidelines on multidrug-resistant Gram-negative microorganisms. Antimicrob Resist In; 2014 Dec;4:7--7.
\item
  Multimorbidity: Technical Series on Safer Primary Care. Geneva: World Health Organization; 2016. Licence: CC BY-NC-SA 3.0 IGO.
\item
  Tahir H, López-Cortés LE, Kola A, Yahav D, Karch A, Xia H, \emph{et al.} Relevance of intra-hospital patient movements for the spread of healthcare-associated infections within hospitals - a mathematical modeling study. PLoS Comput Biol. Public Library of Science; 2021 Feb;17(2):e1008600.
\item
  Depasse B, Pauwels D, Somers Y, Vincent JL. A profile of European ICU nursing. Intensive Care Med. 1998 Sep;24(9):939--45.
\item
  Van Duijn PJ, Dautzenberg MJD, Oostdijk EAN. Recent trends in antibiotic resistance in European ICUs. Curr Opin Crit Care. 2011 Dec;17(6):658--65.
\item
  EUCAST guidelines for detection of resistance mechanisms and specific resistances of clinical and/or epidemiological importance. 2017 Jul:1--43.
\item
  Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR: Antimicrobial Resistance Analysis. 2020. Available from: \url{https://msberends.github.io/AMR}.
\item
  KRINKO: Ergänzung zur Empfehlung der KRINKO „Hygienemaß- nahmen bei Infektionen oder Besiedlung mit multiresistenten Gram- negativen Stäbchen`` (2012) im Zusammenhang mit der von EUCAST neu definierten Kategorie „I`` bei der Antibiotika-Resistenzbestimmung: Konsequenzen für die Definition von MRGN. Epid Bull 2019 Sep;9:82--83.
\item
  Werkgroep Infectiepreventie (WIP). Ziekenhuizen Bijzonder resistente micro-organismen (BRMO) (Concept). 2013 Apr 24;:1--22.
\item
  Jurke A, Köck R, Becker K, Thole S, Hendrix R, Rossen J, \emph{et al.} Reduction of the nosocomial meticillin-resistant \emph{Staphylococcus aureus} incidence density by a region-wide search and follow-strategy in forty German hospitals of the EUREGIO, 2009 to 2011. Euro Surveill. 2013 Sep;18(36):pii=20579.
\item
  Jurke A, Daniels-Haardt I, Silvis W, Berends MS, Glasner C, Becker K, \emph{et al.} Changing epidemiology of meticillin-resistant \emph{Staphylococcus aureus} in 42 hospitals in the Dutch-German border region, 2012 to 2016: results of the search-and-follow-policy. Euro Surveill. 2019 Apr;24(15).
\item
  Alp E, Leblebicioglu H, Doganay M, Voss A. Infection control practice in countries with limited resources. Ann Clin Microbiol Antimicrob; 2011 Oct 22;10(1):36.
\item
  Vuichard Gysin D, Cookson B, Saenz H, Dettenkofer M, Widmer AF. Variability in contact precautions to control the nosocomial spread of multi-drug resistant organisms in the endemic setting: a multinational cross-sectional survey. Antimicrob Resist In; 2018 Jul;7(81).
\item
  Keizer J, Braakman-Jansen LMA, Kampmeier S, Köck R, Naiemi Al N, Riet-Warning Te R, \emph{et al.} Cross-border comparison of antimicrobial resistance (AMR) and AMR prevention measures: the healthcare workers' perspective. Antimicrob Resist In; 2019;8(1):123--13.
\item
  Robert-Koch-Institute, Antimicrobial Resistance Surveillance (ARS), Germany {[}Internet{]}. Available from: \url{https://ars.rki.de}. {[}Accessed 01 Sep 2020{]}.
\item
  Infectieziekten Surveillance Informatie Systeem-Antibiotica Resistentie (ISIS-AR), Netherlands {[}Internet{]}. Available from: \url{https://www.rivm.nl/isis-ar}. {[}Accessed 01 Sep 2020{]}.
\item
  European Centre for Disease Prevention and Control (ECDC). Surveillance Atlas of Infectious Diseases {[}Internet{]}. Available from: \url{https://atlas.ecdc.europa.eu/public/index.aspx} {[}Accessed 01 Sep 2020{]}.
\item
  Köck R, Becker K, Idelevich EA, Jurke A, Glasner C, Hendrix R, \emph{et al.} Prevention and Control of Multidrug-Resistant Bacteria in The Netherlands and Germany - The Impact of Healthcare Structures. Int J Env Res Pub He; 2020 Apr;17(7):2337--10.
\item
  Harbarth S, Sax H, Gastmeier P. The preventable proportion of nosocomial infections: an overview of published reports. J Hosp Infect. 2003 Aug;54(4):258--66.
\item
  Eber MR, Shardell M, Schweizer ML, Laxminarayan R, Perencevich EN. Seasonal and temperature-associated increases in gram-negative bacterial bloodstream infections among hospitalized patients. PLoS ONE. Public Library of Science; 2011;6(9):e25298.
\end{enumerate}

\hypertarget{ch11-summary}{%
\chapter{Summary and Future Perspectives}\label{ch11-summary}}

\hypertarget{section-i}{%
\section*{Section I}\label{section-i}}
\addcontentsline{toc}{section}{Section I}

Where did microbial epidemiology start? How did it originate? And how does it add to the holistic approach of infection management? These questions are answered in this first section. It is subsequently outlined which important current limitations exist when applying microbial epidemiology in practice and how they could be overcome.

The general introduction of this thesis outlines in \textbf{chapter 1} that microbial epidemiology is a part of infectious disease epidemiology, which in turn is a part of clinical microbiology. Microbial epidemiology can be seen, among other things, as the scientific field for acquiring new insights about spreading microorganisms and their respective antimicrobial resistance (AMR) patterns. The advancements in information technology have brought us not only the possibilities to look beyond regional, national, and international borders to get an understanding of the spread of microorganisms and AMR, but even to observe, analyse and understand pandemics in real-time. Methods we develop and use today can be implemented on the other side of the world tomorrow. This is an important advantage in modern microbial epidemiology, which focus is increasingly becoming more data-driven.

To expedite this focus, data are the primary requirement. The data used as input for microbial epidemiological analyses are often obtained from laboratory information systems (LIS). These data consist of routine diagnostic results from laboratory tests. \textbf{Chapter 2} brings an opinionated view that diagnostics might lead to raw results, but not to a direct answer to the clinical question that a physician treating a patient might have. Providing physicians with answers requires the approach of a multidisciplinary, intertwined stewardship concept with a focus on diagnostics \textsuperscript{{[}1,2{]}}. This demands medical specialists in general and microbiologists, in particular, to closely interact for optimal quality of care and patient safety in successful infection management: diagnostic stewardship (DSP). The concept of stewardships, in general, has been widely used to facilitate communication and clinical decision-making, while it proved challenging to establish a clear definition of `stewardship' \textsuperscript{{[}3,4{]}}. Moreover, diagnostics in clinical microbiology laboratories are currently advancing fast with regards to improved workflows and new technologies, such as matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry \textsuperscript{{[}5,6{]}}. Yet, diagnostics in infection management is broader than this and covers many clinical areas where communication and interaction are fundamental to make the best use of knowledge and expertise, leading to all specialisms contributing to patient care. The right test at the right time for the right patient to answer the right questions and start the right treatment -- this is what DSP in clinical microbiology is about. Microbial epidemiology can be utilised for a small aspect of this diagnostic entirety, by recycling the test results and subsequently bringing enrichments to the answer-generating process that DSP embodies.

This is where \textbf{chapter 3} continues, by highlighting important current limitations when applying microbial epidemiology, in particular AMR data analysis. Specifically, AMR data analysis has to be conducted in a clinically and epidemiologically sensible way \textsuperscript{{[}7{]}}, but is challenging since it requires expertise in (clinical) epidemiology and (clinical) microbiology, and tools to handle the AMR data analysis itself. This is further hindered by the common lack of accessibility of data stored in LIS-es, as most LIS-es are not designed with a focus on epidemiology. As an example, every LIS keeps its own taxonomic data and laboratories are responsible for their regular update. Given that AMR guidelines are strongly based on the microbial taxonomy (some rules only apply to a specific genus, other rules apply to a specific family), this information must be correct and up to date \textsuperscript{{[}8--10{]}}. Unfortunately, from studying seven clinical microbiology laboratories in the Netherlands, it became apparent that all their LIS-es contained severely outdated taxonomic names. This can impact both routine result reporting and (future) epidemiological analyses. For these reasons, the \texttt{AMR} package for \texttt{R} was introduced in this chapter as a new epidemiological instrument for AMR data analysis that is free, independent, open-source, and publicly available. Developed with a team from twelve different public health organisations in seven different countries, it provides tools to simplify AMR data cleaning, transformation and analysis, as well as methods to easily incorporate (inter)national guidelines, and scientifically reliable reference data. As of May 2021, it has been downloaded at least 50,000 times from 162 different countries since its first release in 2018 \textsuperscript{{[}11{]}}. The results of a survey among users presented in this chapter showed that its use leads to more reproducibility of analysis results, more reliable outcomes of AMR data analyses, and new or improved insights in AMR for the users' institutions and regions. Users also stated that the \texttt{AMR} package was used to support clinical decision-making. The package solves the inconvenience of being dependent on (inter)national guidelines and reliable (reference) data, while also providing a comprehensive toolbox for the analysis itself. The \texttt{AMR} package for \texttt{R} can therefore empower any specialist in the field working with AMR data.

\hypertarget{section-ii}{%
\section*{Section II}\label{section-ii}}
\addcontentsline{toc}{section}{Section II}

Following the challenges outlined in the previous section, this section introduces the \texttt{AMR} package for \texttt{R} as a new instrument to cope with these challenges. From multiple viewpoints, the \texttt{AMR} package and its advantages are put into perspective: from a technical viewpoint, from an infection management viewpoint and from a clinical viewpoint. These combined provide a common ground for understanding the explications that the \texttt{AMR} package can yield in the field and how it can set a new empowered starting point for future applications of microbial epidemiology.

The technical functionalities of the \texttt{AMR} package for \texttt{R} have been described in \textbf{chapter 4}, where it is described how the \texttt{AMR} package has been developed to standardise clean and reproducible AMR data analyses using international standardised recommendations \textsuperscript{{[}9,12{]}}. To facilitate this, scientifically reliable reference data are incorporated regarding valid laboratory results (as opposed to e.g., non-existing MIC values), antimicrobial agents, and the complete biological taxonomy of microorganisms. Source data should be analysed in the most reliable way, especially when for example the outcome will be used to evaluate patient treatment options. This requires reproducible and field-specific, specialised data cleaning and transforming. The \texttt{AMR} package provides a standardised and automated way of cleaning, transforming, and enhancing common LIS data, independent of the underlying data source and data accuracy. For this reason, general algorithms were developed to clean AMR test results and to validate the names of microorganisms and antimicrobial agents. The equation for taxonomic name validation takes into account the human pathogenic prevalence of microorganisms and is context-aware about other taxonomic properties such as the kingdom, phylum, order and family. To exemplify, a data value ``E. coli'' will be translated to the bacterium \emph{Escherichia coli}, while informing the user that the parasite \emph{Entamoeba coli} is also eligible but has a lower likelihood. Using convenient functions, users can quickly retrieve consistent microbial properties, such as the taxonomic kingdom, phylum, class, order, family, genus, species, subspecies, previously accepted names and even the Gram stain. Aside from information about microorganisms, the package also includes reference data about antibiotics, which comprises common laboratory information system codes, official names, ATC (Anatomical Therapeutic Chemical) codes, ATC group names, defined daily doses (DDD) and more than 5,000 trade names of 456 antimicrobial agents. Using these reference data, users can translate raw data and retrieve properties about any microorganism or antimicrobial drug. Furthermore, the \texttt{AMR} package is capable of determining multi-drug resistant organisms (MDROs) based on national and international guidelines, interpreting raw minimum inhibitory concentrations (MICs) and can determine first isolates to be used for calculating AMR of both monotherapy and combination therapies. The \texttt{AMR} package itself was meant as a comprehensive instrument for data-technical staff working in the field of AMR, although its use is not limited to this group.

To exemplify this, \textbf{chapter 5} shows that the \texttt{AMR} package was used as a backbone in an interactive open-source software app for infection management and antimicrobial stewardship, called RadaR (rapid analysis of diagnostic and antimicrobial patterns in R). Infection management in the form of antimicrobial stewardship (AMS) programs has emerged as an effective solution to address this global health problem in hospitals \textsuperscript{{[}3{]}}. Connecting to \textbf{chapter 2}, stewardship interventions and activities focus on individual patients (personalised medicine and consulting) as well as patient groups or clinical syndromes (guidelines, protocols, information technology infrastructure, and clinical decision support systems) while prioritising improvement in quality of care and patient safety for any intervention \textsuperscript{{[}13,14{]}}. However, easy access to analyse patient groups (e.g., stratified by departments or wards, specific antimicrobials, or diagnostic procedures used) is difficult to implement in daily practice. It is even more challenging to rapidly analyse larger patient populations (e.g., spread over multiple specialities) even though this information might be available in the data. Therefore, the development of RadaR was intended to serve AMS teams with a user-friendly and time-saving data analysis resource, without the need for profound technical expertise. RadaR was developed for graphical exploratory (AMR) data analysis. Among others, it provides Kaplan-Meier curves about lengths of hospitals stays, time trends for the number of admissions, antimicrobial consumption, and an automated AMR data analysis for which the \texttt{AMR} package for \texttt{R} was used. RadaR was validated by 12 ESGAP members (European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship) from 9 different countries. It has the potential to be a highly useful tool for infection management and AMS teams in daily practice. Additionally, this chapter shows that the \texttt{AMR} package can be used as part of another software solution to empower integrated infection management.

Following this insight, \textbf{Chapter 6} demonstrates the effectiveness of the \texttt{AMR} package among users, by evaluating its usability and impact on clinicians' workflows in a typical hospital scenario. Although the use of the \texttt{AMR} package in research has been demonstrated in multiple studies from different countries already \textsuperscript{{[}15--18{]}}, the impact on workflows for AMR data analysis and reporting in clinical settings was still pending. AMR data analysis and reporting, unfortunately, require specifically skilled personnel. Moreover, thorough and in-depth analyses can be time-consuming and sufficient resources need to be allocated for consistent and repeated reporting. To determine the impact of these facts in a clinical setting, common questions about blood culture data were formulated that had to be answered by routine clinical personnel, including clinical microbiologists, paediatricians and intensivists. In total, ten clinicians participated in the study. Additionally, participants were asked to fill in an online questionnaire capturing their backgrounds, demographics, software experience, and experience in AMR data analysis and reporting. All participants had to answer the study questions twice: the first time with their software of choice (round 1) and the second time using a newly developed web application built around the \texttt{AMR} package for \texttt{R} (round 2). The development of this web application was utilised in a highly efficient and agile workflow. The answers to the list of questions served as the basis to compare the effectiveness (solvability of each task for every user) and efficiency (time spent solving each task) between the two rounds. Not all participants were able to complete the tasks within the given time frame. Average task completion between the first and second round increased from 56\% (SD: 23\%) to 96\% (SD: 6\%). The proportion of correct answers between the first and second round increased from 38\% to 98\%. The mean time spent per round was reduced from 94 minutes (SD: 22 minutes) to 22 minutes (SD: 14 minutes). This chapter demonstrates the increased effectiveness, efficiency, and accuracy of using the \texttt{AMR} package for \texttt{R} for AMR data analysis compared to traditional software applications such as Microsoft Excel and SPSS.

\hypertarget{section-iii}{%
\section*{Section III}\label{section-iii}}
\addcontentsline{toc}{section}{Section III}

Many clinical studies in the field of infectious diseases and microbiology rely on some form of (microbial) epidemiology. While the \texttt{AMR} package was presented in the previous section and its use in different settings was showcased, this section starts with an epidemiological research projects in the Northern Dutch region, and then extends to the Dutch-German cross-border region to better understand the occurrence and AMR patterns of pathogens on a (eu)regional level. Focusing on the regions on each side of a national border allows comparisons between two different nations on the micro level. And different nations ultimately mean different healthcare systems. What is left of `One Health'? What are the implications on comparison of having differences between countries in AMR test methodologies, MDRO interpretations and screening policies? This section provides answers to these questions.

\textbf{Chapter 7} zooms in on coagulase-negative staphylococci (CoNS), which are known to cause bloodstream infection (BSI) and a high mortality rate, although for years they had often been regarded as contamination \textsuperscript{{[}19--23{]}}. Moreover, CoNS have become increasingly associated with nosocomial infections \textsuperscript{{[}24{]}}. At present, the CoNS group consists of 45 different species, although determining the species level has only recently been made possible for routine diagnostic laboratories \textsuperscript{{[}25--27{]}}. Since 2012, MALDI-TOF mass spectrometry has become the standard for the identification of bacterial species such as CoNS. Before that, identification of CoNS was primarily done with biochemical and physiological tests, which yielded generally variable results, in particular in less prevalent species \textsuperscript{{[}27{]}}. AMR, and especially multi-drug resistance, is also an increasing problem in CoNS \textsuperscript{{[}28{]}}. Nonetheless, treatment guidelines and national surveillance programs (such as the Dutch NethMap) still gather CoNS as a whole group, lacking differentiation between species \textsuperscript{{[}29{]}}. Consequently, little is known about trends in occurrence and AMR in CoNS on the local and regional level. Therefore, this retrospective study shows an in-depth AMR analysis of 19,803 CoNS isolates found in all available 71,632 blood culture isolates between 2013 and 2019 in the Northern Netherlands that were determined by MALDI-TOF MS. This study followed a full-region approach by covering the whole Northern Netherlands. Through this analysis, we aimed to evaluate the differences in the occurrence of CoNS species and their AMR patterns and to assess their clinical microbiological relevance to this end. A total of 27 different species of the CoNS group were found. Major differences were observed in the occurrence of the different species: the top five species covered 97.1\% of all included isolates. These were: \emph{S. epidermidis} (48.4\%), \emph{S. hominis} (33.6\%), \emph{S. capitis} (9.3\%), \emph{S. haemolyticus} (4.1\%) and \emph{S. warneri} (1.7\%), meaning that the remaining 2.9\% of isolates consisted of 22 different CoNS species. The proportion of CoNS in intensive care units (ICUs) compared to other departments was also found to be significantly different between secondary care (17.5\% of isolates from ICU) and tertiary care (24.4\%\% of isolates from ICU). As it was unknown which patients had BSI, `CoNS persistence' was defined as a surrogate having at least three positive blood cultures drawn on three different days within 60 days, containing the same CoNS species, within the same patient. The relatively most common causal agent of CoNS persistence was \emph{S. haemolyticus} (5.8\% of all patients with \emph{S. haemolyticus}), followed by \emph{S. epidermidis} (3.7\%,) and \emph{S. lugdunensis} (3.4\%). AMR analysis has shown substantial differences between CoNS species and was presented thoroughly per antibiotic class in tables and text. For example, \emph{S. epidermidis} and \emph{S. haemolyticus} showed 50\% to 80\% resistance to teicoplanin, erythromycin, ciprofloxacin, and oxacillin, while resistance to these agents remained lower than 10\% in most other CoNS species. Yet, these differences are neglected on the national level such as in NethMap, which might cause the development of treatment guidelines to focus on `AMR-safe' agents for treating CoNS, such as vancomycin or linezolid. Nonetheless, agents such as tetracycline, co-trimoxazole, and erythromycin could be considered viable options for some species, where according to the study results, AMR never surpassed 10\%. In conclusion, a multi-year full-region approach to extensively assess the trends in both the occurrence and AMR of CoNS species was carried out, which could be used for evaluating treatment policies and understanding more about these important but still too often neglected pathogens. Furthermore, this study served as a practical research example of how the \texttt{AMR} package for \texttt{R} can be used to gain new AMR insights using epidemiologically sounds methods.

Following new insights by studying AMR test results in the Northern Netherlands, \textbf{chapter 8} provides a comparison of AMR test results and their national interpretations of MDROs in the Dutch-German cross-border region, especially concerning the practical impact on cross-border healthcare workers. Comparing AMR in general, not only MDROs, in this cross-border region is particularly interesting since both countries are characterised by highly developed but structurally different healthcare systems. AMR interpretations in patient records are transferred between healthcare facilities located in these two different countries, while the underlying definitions differ. This causes the need for clinicians and infection control personnel to understand AMR results from both sides of the border and to be able to comprehend both national MDRO interpretation guidelines. By comparing antibiograms of Gram-negative bacteria from both sides of the border, the degree of impact of these challenges was sought to determine. To this end, 35,619 antibiograms from six Dutch and four German hospitals were analysed between 2015 and 2016 of all species of \emph{Enterobacteriaceae}, and P. aeruginosa, the A. baumannii complex and Stenotrophomonas maltophilia. MDRO recommendations and special hygiene precautions exist in this region for all of these species. On the Dutch side of the border, isolate selection was carried out using the \texttt{AMR} package. From the Dutch hospitals, 12,616 antibiograms were selected using the \texttt{AMR} package for \texttt{R} applying the Dutch MDRO interpretation guideline. Of note, other national and international guidelines, such as the German MDRO interpretation guideline, are also included in the \texttt{AMR} package for \texttt{R}. From German hospitals, 23,003 antibiograms were selected using other methods. According to the Dutch guideline, 24.5\% of all isolates were an MDRO. According to the German guideline, 12.9\% of all isolates were an MDRO. However, of all isolates, 73.7\% were not classified as an MDRO according to either guideline. Among all carbapenem-resistant \emph{Enterobacteriaceae} isolates, carbapenemases were detected in 27.6\% with OXA-48-like genes being predominant. The remaining isolates were negative for carbapenemases (79.1\%) or not tested (20.9\%). When patients are transferred between hospitals, information regarding MDRO colonisation or infection must also be transferred to ensure continuous implementation of infection control measures. For cross-border healthcare, this implies that clinicians or infection control staff should be able to determine MDROs based on antibiograms according to guidelines from either of the two countries. For cross-border healthcare, the easiest solution would be to harmonise the classification rules of both countries. This would likewise solve the understandable confusion patients might experience if infection control measures are imposed in one country, but relieved after transfer to another country. As long as the harmonisation is not done, the full AMR data of Gram-negative bacteria should be transferred together with the patient to enable classification by local infection control staff.

Other AMR-related cross-border challenges and differences are illustrated in \textbf{chapter 9}, which comprises a comprehensive microbial epidemiological analysis of MRSA occurrence, policies, and healthcare effects in the Dutch-German border region. MRSA is still one of the major causes of healthcare-associated infections due to AMR pathogens \textsuperscript{{[}30{]}}. In this study, MRSA surveillance data of five years (2012-2016) from Dutch and German cross-border region hospitals were analysed to describe temporal and spatial trends of MRSA rates and find differences between these groups of hospitals. The research setting comprised 42 hospitals located in the Dutch-German cross-border region treating approximately 620,000 admitted patients (68.0\% in the German part of the study region) with 3.9 million patient days per year. All hospitals had implemented MRSA-related infection prevention control measures according to their national guidelines and recommendations, and the guideline differences between the two countries were compared. On both sides of the border, the median nasopharyngeal MRSA screening rate increased significantly between 2012 and 2016, although the median MRSA incidence remained stable over time at both sides of the border. Overall, the median screening rate was 14 times higher in the German border region (DE-BR) than in the Dutch border region (NL-BR). The median percentage of MRSA in \emph{S. aureus} blood culture isolates decreased from 12.5\% in 2012 to 5.0\% in 2016 in DE-BR, while it remained stable at 0\% to 1.9\% in NL-BR. Nonetheless, MRSA among \emph{S. aureus} isolates was 34 times higher in DE-BR. The in-hospital length of stay of MRSA patients was similar in both regions, while the general length of stay differed significantly. Furthermore, the number of nasopharyngeal MRSA screening swabs before or at admission to hospital per 100 inhabitants was 12.2 in DE-BR and 0.36 in NL-BR, also 34 times higher in DE-BR. The number of inpatient MRSA cases per 1,000 inhabitants was 2.52 in DE-BR and 0.14 in NL-BR. Thus, this study revealed significant differences between Dutch and German hospitals. The median MRSA incidence in DE-BR hospitals was more than seven times higher than in NL-BR hospitals. According to the European Centre of Disease Prevention and Control (ECDC), differences in the occurrence of AMR pathogens between European countries are most likely caused by differences in healthcare utilisation, antimicrobial use and infection prevention control practices \textsuperscript{{[}31{]}}. Concerning healthcare utilisation in our context, we found that inhabitants in the German part of the study region were almost three times as often hospitalised and had a significantly longer length of stay than patients on the Dutch part. This may be due to socioeconomic factors or a different organisation of ambulatory healthcare. This comprehensive study on MRSA covering hospitals across a European border demonstrated that routine MRSA surveillance may be helpful to monitor trends of MRSA parameters, to enable (inter)national comparisons.

The discussion of this study concluded with ``cross-border surveillance should be extended to other multidrug-resistant organisms'', which is where \textbf{chapter 10} continues. Given that not only MRSA but MDROs, in general, pose a risk for healthcare, both in the community and hospitals, the study aimed to determine the prevalence of multiple MDROs in this cross-border region to understand differences and improve infection prevention based on real-time routine data and workflows. To this end, 23 hospitals in the Dutch-German cross-border region (NL-BR and DE-BR) participated between 2017 and 2018 in this prospective study by screening all patients upon admission to intensive care units (ICUs). All hospitals (8 in NL-BR, 15 in DE-BR) enrolled for eight consecutive weeks and screened patients for nasal carriage of MRSA and rectal carriage of vancomycin-resistant \emph{Enterococcus faecium}/\emph{E. faecalis} (VRE), third-generation cephalosporin-resistant \emph{Enterobacteriaceae} (3GCRE) and carbapenem-resistant \emph{Enterobacteriaceae} (CRE). A total of 3,365 patients were screened: 35.7\% on NL-BR ICUs and 64.3\% on DE-BR ICUs. The median age of all screened patients was 68 years (IQR: 57-77), while patients in DE-BR were significantly older than patients in the NL-BR. A total of 6,462 swabs were processed. The overall screening compliance (screened for at least one MDRO group) was 60.4\%, in NL-BR 56.9\% and in DE-BR 62.9\%. All AMR data analyses were carried out and automated using the \texttt{AMR} package for \texttt{R}. The prevalence of MRSA was 1.7\% in DE-BR ICUs and 0.6\% in NL-BR ICUs. The prevalence of VRE was 2.7\% in DE-BR ICUs and 0.1\% in NL-BR ICUs. Notably, this prevalence ranged from 0\% to 4.1\% in DE-BR. All 56 cases of VRE were caused by E. faecium. The prevalence of 3GCRE was 6.6\% in DE-BR ICUs and 3.6\% in NL-BR ICUs, whereas the prevalence for CRE was practically non-existent on both sides of the border. The prevalence for Gram-negative MDROs differed within both countries between hospitals, ranging from 0\% to 5.0\% in NL-BR and from 1.2\% to 10.9\% in DE-BR ICUs. For NL-BR ICUs, the prevalence of all MDRO groups was not significantly different between the non-university and the university hospital. For the DE-BR ICUs however, the prevalence of Gram-negative MDROs was significantly higher in the non-university hospitals. In the NL-BR, 4.8 per 100 hospital admissions led to ICU admission. In contrast, in the DE-BR this was 7.7 per 100 hospital admissions. This difference can be explained by the higher ICU capacity in DE-BR hospitals (4.8\% of all hospital beds) compared to NL-BR hospitals (2.4\% of all hospital beds). The overall carriage prevalence for the different MDROs was higher in the DE-BR ICUs, although some differences were marginal. Specifically, the prevalence of MRSA carriage was three times higher in the DE-BR (1.7\%) than in the NL-BR (0.6\%). These prevalences are consistent with the study mentioned in \textbf{chapter 9}. The difference in MDRO prevalence between NL-BR and DE-BR was observed for all MDROs groups. Yet, the study findings were not all comparable with (inter)national averages. For example, the 3GCRE carriage prevalence in the DE-BR was almost twice as high (6.6\%) as in the NL-BR (3.6\%), but both were still lower than national averages. The ECDC reported 3GCRE proportions among blood culture isolates of \emph{E. coli} and \emph{K. pneumoniae} as 12.2\% to 12.9\% for Germany and 7.3\% to 11.1\% for the Netherlands. This highlights that there are important differences to be found when studying carriage in specified populations versus looking at the proportion of (probably) invasive isolates. Thus, this study highlights the importance of a regional and cross-border approach in any European cross-border region, to illustrate the difference in AMR prevalence between the regions and to highlight potential differences with country-wide reports. Attaining a deeper level of detail is required to be able to elaborate on this further, for example by collecting information about staff on the wards and infection control staff, MDRO outbreaks, infections, antibiotic use and risk factors of patients. In conclusion, geographical and political borders do not seem to be ``respected'' by MDROs, although healthcare systems, geographic nature and guidelines are very different between countries. Proportions of MDROs of certain pathogens, as reported on the national and international level, do not reflect MDRO prevalence in the patient or general population. This should be taken into serious consideration when interpreting reports on the country or even continental level.

\hypertarget{future-perspectives}{%
\section*{Future perspectives}\label{future-perspectives}}
\addcontentsline{toc}{section}{Future perspectives}

\begin{quote}
After hearing for several decades that computers will soon be able to assist with difficult diagnoses, the practising physician may well wonder why the revolution has not occurred. Scepticism at this point is understandable. Few, if any, programs currently have active roles as consultants to physicians. The story behind these unfulfilled expectations is instructive and, we believe, offers hope for the future.
\end{quote}

These words are from Schwartz \emph{et al.} and, unfortunately, not very recent. It was published 34 years ago in The New England Journal of Medicine in 1987 \textsuperscript{{[}36{]}}. Many might find it quite disappointing that this exact quote can still apply to current times. Yet, this is not due to a lack of technological advancements -- computational power and software capabilities have increased significantly over the last decades. And with them, the enablement of making optimal use of existing data to aid clinical decision-making and to support medicine as a whole. Hence, if it is not due to lack of technological advancements, what is then inhibiting the use of these advancements for clinical use? Others pointed out that the answer might be the gap in culture between the clinicians, biomedical scientists, and those skilled in computer programming \textsuperscript{{[}37,38{]}}. To this end, one might contemplate whether multi-disciplinarity was imbedded well enough into our integral medical field, since the differences are not only cultural. While both clinicians and biomedical scientists endure more than a decade of specialised training and education in a similar field, they often (1) do not speak each other's language, (2) lack a common value system, even regarding knowledge and ignorance, and (3) have different sources of passion and emotional intensity \textsuperscript{{[}37{]}}. Scientists have to focus on asking ``why?'' and ``how?'', whereas clinicians have to focus on acquiring practical answers to ``how?'' and ``what?''. From a clinician's perspective, asking ``why?'' distracts from the sense of mastery that comes from accumulating information and applying it in a clinical setting. Neither perspectives are wrong, they are just inherently different, and this results in a cultural gap. Unfortunately, this cultural gap hinders the translation of scientific discoveries into medical advances and may even hinder scientific progress \textsuperscript{{[}37{]}}.

While this gap may be existent, this thesis aims to narrow this gap for clinicians and scientists working in the fields of clinical microbiology and microbial epidemiology, by providing an instrument that can be beneficial and usable for clinicians and scientists alike. Ultimately, it could yield more collaboration, communication, and efficacy between scientists and clinicians. The \texttt{AMR} package for \texttt{R} has empowered the four studies mentioned in \textbf{SECTION III}, which were conducted in the Northern Netherlands as well as in the Dutch-German cross-border region. In these studies, the \texttt{AMR} package affected the selection of isolates, determination of MDROs, or the entire AMR data analysis. Combined with the user survey results in \textbf{chapter 3} (that also included the use of the \texttt{AMR} package by both clinicians and scientists), the proof of concept of an integrated design in \textbf{chapter 5}, and the positive effects on clinical staff working with AMR data in \textbf{chapter 6}, this indicates that this new instrument can be deployed and used in a multi-disciplinarily fashion. Many others have pointed out the challenges in AMR data analysis on (cumulative) antibiograms and, inter alia, the necessity for correcting duplicate isolates \textsuperscript{{[}7,18,39--45{]}}. Still, all these are theoretical and did not provide a pragmatic solution for those conducting microbial epidemiology. Hindler \emph{et al.} presented a practical example of a data set that might require a correction for duplicate isolates (Table 1) \textsuperscript{{[}7{]}}. The algorithm of choice could be isolate-based, patient-based, episode-based, or phenotype-based. This choice is dependent on the type of analysis and desired outcome. Table 2 illustrates the scope of the isolates that should be included based on a chosen algorithm and, more importantly, shows how the \texttt{AMR} package for \texttt{R} can be used to accomplish this in one simple command, underlining its approachability. Some of those functions to apply the respective algorithm using the \texttt{AMR} package for \texttt{R} have been used by others \textsuperscript{{[}15--18{]}}.

Table 1. Example AMR test results of four \emph{Staphylococcus aureus} isolates from a single patient.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/11-t01} 

}

\end{figure}

Table 2. Algorithms for including isolates and the accompanying function in the \texttt{AMR} package for \texttt{R} for use in the AMR data analysis. The `x' in the last column denotes any data set in a similar structure as Table 1.

\begin{figure}

{\centering \includegraphics[width=1\linewidth]{images/11-t02} 

}

\end{figure}

User feedback as presented in \textbf{chapter 3} implies that usage of the \texttt{AMR} package has led to higher reproducibility, higher reliability, new AMR insights and improved clinical decision-making. From \textbf{chapter 5} until \textbf{chapter 10}, it is shown that the \texttt{AMR} package can be a sensible and reliable tool for microbial isolate selection and conducting AMR data analysis. These examples indicate that the \texttt{AMR} package for \texttt{R} has the potential to become a centrepiece in AMR data analyses, which is further supported by its use in other scientific publications \textsuperscript{{[}15--18{]}}. One of its most important features -- enabling users to transform raw data into valuable new insights -- allows data sets from any clinical source to be used. For example, data sets from different regions could be analysed every year in the same manner by reusing an automated AMR script, comparing trends in the occurrence of MDROs. This uniformity is an important advantage for gaining new AMR insights on the local, regional or national level and should be exploited to the fullest.

From an international point of view, it could be viable to achieve a common workflow with AMR interpretation guideline suppliers such as the Clinical and Laboratory Standards Institute (CLSI) and the European Committee on Antimicrobial Susceptibility Testing (EUCAST) \textsuperscript{{[}8,10{]}}. These organisations provide clinical microbiology laboratories around the globe with static and manually formatted Microsoft Excel and Portable Document Format (PDF) files, requiring laboratory staff to manually apply guideline updated into their LIS. Since the \texttt{AMR} package for \texttt{R} contains machine-readable files of these (often yearly) guidelines, a collaborative workflow could lead to a more seamless implementation and update process in clinical laboratories worldwide, increasing reliability and reducing the workload on laboratory staff. These possible effects have yet to be studied.

Similarly, LIS manufacturers could benefit from the freely available comprehensive reference data about antimicrobial agents and the taxonomy of microorganisms that the \texttt{AMR} package provides. The data provided in the \texttt{AMR} package are automatically updated using services from the World Health Organization Collaborating Centre for Drug Statistics Methodology, PubChem, the Catalogue of Life, the List of Prokaryotic names with Standing in Nomenclature and SNOMED CT. LIS manufacturers could provide this same automated process or the data in the \texttt{AMR} package directly to their end-users (the clinical microbiology laboratories), to ensure a continuously up to date version of the reference data about antimicrobial agents and the taxonomy of microorganisms. This would mean that laboratories could be unburdened by losing the necessity of keeping their local data up to date. Maintaining these local data is of paramount importance, as all AMR interpretation guidelines are based on these data. It would strongly optimise the quality of the output of clinical routine laboratories. Aside from this optimisation, with less manual and tedious work to conduct microbial epidemiology for data-technical staff, using this presented new instrument hopefully also leads to faster availability of higher-quality research in the field of AMR, as well as a better patient outcome in clinical settings.

A more clinical example of the possibilities of the \texttt{AMR} package is to analyse microbiological data from urinary tract infections in comparison with blood culture data. Some patients suffer from a urinary tract infection but are admitted to the hospital with urosepsis sometime after. As these are the kind of clinical complications we should thrive to prevent, a full-region analysis of these data might shed light on the reasons why these clinical complications were not or could not be prevented. Fortunately, it is not difficult anymore to select patients who had a laboratory-confirmed urinary tract infection in primary care and a positive blood culture with the same pathogen in the weeks after. The \texttt{AMR} package can be used to do so, and to calculate suggestions for more specific and probably effective antibiotic treatment. This puzzle may not be easily solved, but it is at least now possible to get the data into the right format and have them generate the answers to back our hypotheses. Research initiatives to study this clinical example have recently commenced in both our Northern Dutch region and in the Dutch-German cross-border region.

Yet, the \texttt{AMR} package could be used for even more sophisticated outcomes by combining microbial epidemiology with computational intelligence, and this is where the real potential lies. For example, empirical sepsis therapy could become more personalised or, as others call it, become precision medicine by performing in-depth analyses of blood cultures isolates \textsuperscript{{[}46{]}}. Blood cultures are namely the most reliable diagnostic measure for analysing microbes and their AMR, even if they are drawn from e.g.~arterial catheters \textsuperscript{{[}47--49{]}}. Combining AMR test results from blood culture isolates with patient demographics and hospital-specific traits might enable a comprehensive and multi-angle view on the patient's disease. To specify, by stratifying patient demographics (such as age, gender, comorbidities, history of antibiotic consumption) and comparing them with hospital-specific traits (such as geographic location, common microbial findings, infection control measures, allowed number of patients per room), AMR data analysis could show major differences between all these patient stratifications. The subsequent results could be used to calculate the likelihood of finding similar pathogens and AMR in similar cases, leading to predictive modelling for upcoming septic patients. For example, a septic 60-year-old male patient with a long antibiotic consumption history due to chronic obstructive pulmonary disease (COPD) might require different empiric antiseptic treatment than a septic 60-year-old male patient without COPD and no antibiotic consumption history. In other words, this modelling could lead to personalised empiric treatment guidelines, increasing the chance of therapeutic success. For a study to investigate this, the \texttt{AMR} package for \texttt{R} could be used to identify eligible patients, compare the antibiotic consumption histories, and calculate the AMR rates for pre-defined groups. The \texttt{AMR} package can also calculate the empiric chance of success of different monotherapies and combination therapies using different algorithms. The output of the models will most probably be different between regions and could perhaps even differ per hospital, although the model itself could be universally implementable. Using predictive modelling for treating patients opens a new way of how we make the best use of our data; data we already have and have had for many years. Better yet, new data are generated each day and their quality is constantly improving, due to technical laboratory advancements. Although this specific example of predicting therapeutic success would have been impossible to study twenty years ago, it is highly feasible now. Others have already shown similar approaches recently to predict sepsis using neutrophil-to-lymphocyte ratios, neutrophil dysregulation, or high-resolution vital signs time series \textsuperscript{{[}50--52{]}}. Yet, these modern approaches predict occurrence of sepsis and do not predict the likelihoods of the most effective empiric treatment if patients are already septic. Microbial epidemiology could pose an effective perspective to this end when collaborating with specialities such as acute care medicine and pharmacy, which links back to \textbf{chapter 2} about DSP: the right prediction at the right time for the right patient to (answer the right questions and) start the right predicted treatment. Still, improving empiric antiseptic treatment may feel like extensively training the goalkeeper. This may be necessary, but we should also realise that when the ball enters the penalty area, a lot has gone wrong already.

One might deduce that microbial epidemiology is not yet utilised to the fullest within clinical microbiology and this has a clear explanation. Advancements in information technology have progressed fast over the last decades, even more so over the last years. These advancements have led to improved LIS systems, enhanced software to apply complicated statistics and advanced mathematics, and even to this thesis. Thus, these advancements are quite novel, which means that they can bring new input to existing scientific fields such as clinical microbiology. Training and education are key in accelerating the required knowledge to apply these new advancements. This in turn will lead to the effect that e.g., clinical microbiologists and researchers in the field of clinical microbiology are urged to collaboratively think, develop and learn to work with these advancements. Yet, the cultural gap between clinicians and scientists as outlined earlier might inhibit progress to this end. Still, only collaborations and multi-disciplinary approaches will make sure that we can utilise the advancements in information technology up to their full potential, so patients will benefit most from our future scientific developments. For this reason, we should all strive to narrow and bridge the cultural gap. With regard to the practical labour concerning (predictive) modelling, it should perhaps become more common for research groups within our field, and probably many other research fields, to include (more) modellers and other data-technical staff.

\hypertarget{references-10}{%
\section*{References}\label{references-10}}
\addcontentsline{toc}{section}{References}

\begin{enumerate}
\def\labelenumi{\arabic{enumi}.}
\tightlist
\item
  World Health Organization. Diagnostic stewardship: A guide to implementation in antimicrobial resistance surveillance sites. 2016.
\item
  Morgan DJ, Malani P, Diekema DJ. Diagnostic Stewardship-Leveraging the Laboratory to Improve Antimicrobial Use. JAMA 2017;171:157--64. \url{doi:10.1001/jama.2017.8531}.
\item
  Dyar OJ, Huttner B, Schouten J, Pulcini C, ESGAP (ESCMID Study Group for Antimicrobial stewardshiP). What is antimicrobial stewardship? Clin Microbiol Infect 2017;23:793--8. \url{doi:10.1016/j.cmi.2017.08.026}.
\item
  Mendelson M, Balasegaram M, Jinks T, Pulcini C, Sharland M. Antibiotic resistance has a language problem. Nature 2017;545:23--5. \url{doi:10.1038/545023a}.
\item
  Greub G, Moran-Gilad J, Rossen J, Egli A, ESCMID Study Group for Genomic and Molecular Diagnostics (ESGMD). ESCMID postgraduate education course: applications of MALDI-TOF mass spectrometry in clinical microbiology. Microbes Infect 2017;19:433--42. \url{doi:10.1016/j.micinf.2017.06.004}.
\item
  Didelot X, Bowden R, Wilson DJ, Peto TEA, Crook DW. Transforming clinical microbiology with bacterial genome sequencing. Nat Rev Genet 2012;13:601--12. \url{doi:10.1038/nrg3226}.
\item
  Hindler JF, Stelling J. Analysis and Presentation of Cumulative Antibiograms: A New Consensus Guideline from the Clinical and Laboratory Standards Institute. Clin Infect Dis 2007;44:867--73. \url{doi:10.1086/511864}.
\item
  European Committee on Antimicrobial Susceptibility Testing (EUCAST) 2021. \url{https://eucast.org} (accessed March 21, 2021).
\item
  Clinical and Laboratory Standards Institute (CLSI). M39-A4 Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 2014.
\item
  Clinical and Laboratory Standards Institute (CLSI) 2021. \url{https://clsi.org} (accessed March 21, 2021).
\item
  RStudio PBC. CRAN package download logs 2021. \url{http://cran-logs.rstudio.com} (accessed March 31, 2021).
\item
  Leclercq R, Cantón R, Brown DFJ, Giske CG, Heisig P, MacGowan AP, \emph{et al.} EUCAST expert rules in antimicrobial susceptibility testing. Clin Microbiol Infect 2013;19:141--60. \url{doi:10.1111/j.1469-0691.2011.03703.x}.
\item
  Barlam TF, Cosgrove SE, Abbo LM, MacDougall C, Schuetz AN, Septimus EJ, \emph{et al.} Implementing an Antibiotic Stewardship Program: Guidelines by the Infectious Diseases Society of America and the Society for Healthcare Epidemiology of America. Clin Infect Dis 2016;62:e51-77. \url{doi:10.1093/cid/ciw118}.
\item
  Davey P, Marwick CA, Scott CL, Charani E, McNeil K, Brown E, \emph{et al.} Interventions to improve antibiotic prescribing practices for hospital inpatients. Cochrane Database Syst Rev 2017;2:CD003543. \url{doi:10.1002/14651858.CD003543.pub4}.
\item
  Le Guern R, Titécat M, Loïez C, Duployez C, Wallet F, Dessein R. Comparison of time-to-positivity between two blood culture systems: a detailed analysis down to the genus-level. Eur J Clin Microbiol Infect Dis 2021. \url{doi:10.1007/s10096-021-04175-9}.
\item
  Dutey-Magni PF, Gill MJ, McNulty D, Sohal G, Hayward A, Shallcross L, \emph{et al.} Feasibility study of hospital antimicrobial stewardship analytics using electronic health records. JAC-Antimicrobial Resist 2021;3. \url{doi:10.1093/jacamr/dlab018}.
\item
  N. Tenea G, Jarrin-V P, Yepez L. Microbiota of Wild Fruits from the Amazon Region of Ecuador: Linking Diversity and Functional Potential of Lactic Acid Bacteria with Their Origin. Ecosyst. Biodivers. Amaz., IntechOpen; 2021. \url{doi:10.5772/intechopen.94179}.
\item
  Kim S, Yoo SJ, Chang J. Importance of Susceptibility Rate of `the First' Isolate: Evidence of Real-World Data. Medicina (B Aires) 2020;56:507. \url{doi:10.3390/medicina56100507}.
\item
  Rogers KL, Fey PD, Rupp ME. Coagulase-Negative Staphylococcal Infections. Infect Dis Clin North Am 2009;23:73--98. \url{doi:10.1016/j.idc.2008.10.001}.
\item
  Dong Y, Speer CP. The role of \emph{Staphylococcus epidermidis} in neonatal sepsis: Guarding angel or pathogenic devil? Int J Med Microbiol 2014;304:513--20. \url{doi:10.1016/j.ijmm.2014.04.013}.
\item
  Cheung GYC, Otto M. Understanding the significance of \emph{Staphylococcus epidermidis} bacteremia in babies and children. Curr Opin Infect Dis 2010;23:208--16. \url{doi:10.1097/QCO.0b013e328337fecb}.
\item
  Stearns-Kurosawa DJ, Osuchowski MF, Valentine C, Kurosawa S, Remick DG. The pathogenesis of sepsis. Annu Rev Pathol 2011;6:19--48. \url{doi:10.1146/annurev-pathol-011110-130327}.
\item
  Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. \emph{Staphylococcus epidermidis} as a cause of bacteremia. Future Microbiol 2015;10:1859--79. \url{doi:10.2217/fmb.15.98}.
\item
  Heilmann C, Ziebuhr W, Becker K. Are coagulase-negative staphylococci virulent? Clin Microbiol Infect 2018;0. \url{doi:10.1016/j.cmi.2018.11.012}.
\item
  Becker K, Both A, Weißelberg S, Heilmann C, Rohde H. Emergence of coagulase-negative staphylococci. Expert Rev Anti Infect Ther 2020;18:349--66. \url{doi:10.1080/14787210.2020.1730813}.
\item
  Pantůček R, Sedláček I, Indráková A, Vrbovská V, Mašlaňová I, Kovařovic V, \emph{et al.} \emph{Staphylococcus edaphicus} sp. nov., Isolated in Antarctica, Harbors the mecC Gene and Genomic Islands with a Suspected Role in Adaptation to Extreme Environments. Appl Environ Microbiol 2018;84:e01746-17. \url{doi:10.1128/AEM.01746-17}.
\item
  Kassim A, Pflüger V, Premji Z, Daubenberger C, Revathi G. Comparison of biomarker based Matrix Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS) and conventional methods in the identification of clinically relevant bacteria and yeast. BMC Microbiol 2017;17:128. \url{doi:10.1186/s12866-017-1037-z}.
\item
  Lane MJ, Roy AF, Kearney MT, Pucheu-Haston CM. Characterization, distribution, antimicrobial resistance and resistance risk factors in staphylococci isolated from cats from 2001 to 2014. Vet Med Sci 2018;4:315--25. \url{doi:10.1002/vms3.122}.
\item
  Greeff S, Mouton J. NethMap 2018: Consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in the Netherlands / MARAN 2018: Monitoring of Antimicrobial Resistance and Antibiotic Usage in Animals in the Netherlands in 2017. Bilthoven: Rijksinstituut voor Volksgezondheid en Milieu RIVM Stichting Werkgroep Antibiotica Beleid SWAB; 2018.
\item
  European Centre for Disease Prevention and Control. European Antimicrobial Resistance Surveillance Network (EARS-Net) 2021. \url{https://www.ecdc.europa.eu/en/about-us/partnerships-and-networks/disease-and-laboratory-networks/ears-net} (accessed March 25, 2021).
\item
  European Centre for Disease Prevention and Control. Antimicrobial resistance surveillance in Europe 2016. Annual Report of the European Antimicrobial Resistance Surveillance Network (EARS-Net). Stockholm: 2017.
\item
  van Cleef BAGL, Kluytmans JAJW, van Benthem BHB, Haenen A, Monen J, Daniels-Haardt I, \emph{et al.} Cross border comparison of MRSA bacteraemia between The Netherlands and North Rhine-Westphalia (Germany): a cross-sectional study. PLoS One 2012;7:e42787. \url{doi:10.1371/journal.pone.0042787}.
\item
  Teicher CL, Ronat J-B, Fakhri RM, Basel M, Labar AS, Herard P, \emph{et al.} Antimicrobial drug-resistant bacteria isolated from Syrian war-injured patients, August 2011-March 2013. Emerg Infect Dis 2014;20:1949--51. \url{doi:10.3201/eid2011.140835}.
\item
  Balkhair A, Al-Farsi YM, Al-Muharrmi Z, Al-Rashdi R, Al-Jabri M, Neilson F, \emph{et al.} Epidemiology of multi-drug resistant organisms in a teaching hospital in oman: a one-year hospital-based study. ScientificWorldJournal 2014;2014:157102. \url{doi:10.1155/2014/157102}.
\item
  Szmigielski S, Prévost G, Monteil H, Colin DA, Jeljaszewicz J. Leukocidal toxins of staphylococci. Zentralbl Bakteriol 1999;289:185--201. \url{doi:10.1016/s0934-8840(99)80105-4}.
\item
  Schwartz WB, Patil RS, Szolovits P. Artificial intelligence in medicine. Where do we stand? N Engl J Med 1987;316:685--8. \url{doi:10.1056/NEJM198703123161109}.
\item
  Restifo LL, Phelan GR. The cultural divide: exploring communication barriers between scientists and clinicians. Dis Model Mech 2011;4:423--6. \url{doi:10.1242/dmm.008177}.
\item
  Bargiela D, Verkerk MM. The clinician-programmer: designing the future of medical practice. BMJ 2013;347. \url{doi:10.1136/bmj.f4563}.
\item
  Bax R, Bywater R, Cornaglia G, Goossens H, Hunter P, Isham V, \emph{et al.} Surveillance of antimicrobial resistance--what, how and whither? Clin Microbiol Infect 2001;7:316--25. \url{doi:10.1046/j.1198-743x.2001.00239.x}.
\item
  den Heijer CDJ, Penders J, Donker GA, Bruggeman CA, Stobberingh EE. The importance of gender-stratified antibiotic resistance surveillance of unselected uropathogens: a Dutch Nationwide Extramural Surveillance study. PLoS One 2013;8:e60497. \url{doi:10.1371/journal.pone.0060497}.
\item
  Chopra I, Roberts M. Tetracycline antibiotics: mode of action, applications, molecular biology, and epidemiology of bacterial resistance. Microbiol Mol Biol Rev 2001;65:232-60 ; second page, table of contents. \url{doi:10.1128/MMBR.65.2.232-260.2001}.
\item
  Patel D, Nimbalkar A, Sethi A, Kungwani A, Nimbalkar S. Blood culture isolates in neonatal sepsis and their sensitivity in Anand District of India. Indian J Pediatr 2014;81:785--90. \url{doi:10.1007/s12098-013-1314-2}.
\item
  Critchley IA, Karlowsky JA. Optimal use of antibiotic resistance surveillance systems. Clin Microbiol Infect 2004;10:502--11. \url{doi:10.1111/j.1469-0691.2004.00911.x}.
\item
  Bosso JA, Mauldin PD, Steed LL. Consequences of combining cystic fibrosis- and non-cystic fibrosis-derived Pseudomonas aeruginosa antibiotic susceptibility results in hospital antibiograms. Ann Pharmacother 2006;40:1946--9. \url{doi:10.1345/aph.1H377}.
\item
  Horvat RT, Klutman NE, Lacy MK, Grauer D, Wilson M. Effect of duplicate isolates of methicillin-susceptible and methicillin-resistant \emph{Staphylococcus aureus} on antibiogram data. J Clin Microbiol 2003;41:4611--6. \url{doi:10.1128/jcm.41.10.4611-4616.2003}.
\item
  Atreya MR, Wong HR. Precision medicine in pediatric sepsis. Curr Opin Pediatr 2019;31:322--7. \url{doi:10.1097/MOP.0000000000000753}.
\item
  Ombelet S, Barbé B, Affolabi D, Ronat J-B, Lompo P, Lunguya O, \emph{et al.} Best Practices of Blood Cultures in Low- and Middle-Income Countries. Front Med 2019;6:131. \url{doi:10.3389/fmed.2019.00131}.
\item
  Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. \emph{Staphylococcus epidermidis} as a cause of bacteremia. Future Microbiol 2015;10:1859--79. \url{doi:10.2217/fmb.15.98}.
\item
  Berger I, Gil Margolis M, Nahum E, Dagan O, Levy I, Kaplan E, \emph{et al.} Blood Cultures Drawn From Arterial Catheters Are Reliable for the Detection of Bloodstream Infection in Critically Ill Children. Pediatr Crit Care Med 2018;19:e213--8. \url{doi:10.1097/PCC.0000000000001462}.
\item
  Huang Z, Fu Z, Huang W, Huang K. Prognostic value of neutrophil-to-lymphocyte ratio in sepsis: A meta-analysis. Am J Emerg Med 2020;38:641--7. \url{doi:10.1016/j.ajem.2019.10.023}.
\item
  Shen X-F, Cao K, Jiang J-P, Guan W-X, Du J-F. Neutrophil dysregulation during sepsis: an overview and update. J Cell Mol Med 2017;21:1687--97. \url{doi:10.1111/jcmm.13112}.
\item
  Nemati S, Holder A, Razmi F, Stanley MD, Clifford GD, Buchman TG. An Interpretable Machine Learning Model for Accurate Prediction of Sepsis in the ICU. Crit Care Med 2018;46:547--53. \url{doi:10.1097/CCM.0000000000002936}.
\end{enumerate}

\hypertarget{gearfetting-yn-frysk}{%
\chapter*{Gearfetting yn Frysk}\label{gearfetting-yn-frysk}}
\addcontentsline{toc}{chapter}{Gearfetting yn Frysk}

\textbf{Seksje I}

Wêr is mikrobiale epidemiology begûn? Hoe is it ûntstien? En hoe kin it bydrage oan de holistyske oanpak fan ynfeksjebehear? Dy fragen wurde beäntwurde yn dizze earste seksje. Dêrnei wurdt sketst hokker wichtige hjoeddeistige beheiningen besteane yn `e tapassing fan mikrobiale epidemiology yn `e praktyk en hoe't dy oerwûn wurde kinne.

Yn `e algemiene ynlieding wurdt yn \textbf{haadstik 1} sketst dat mikrobiale epidemiology diel útmakket fan ynfeksjesykte-epidemiology, dy't op syn beurt ûnderdiel is fan klinyske mikrobiology. Under oaren kin mikrobiale epidemyology sjoen wurde as it wittenskiplike fjild foar it krijen fan nije ynsjoggen oer de ferdieling fan mikro-organismen en harren ûnderskate patroanen yn antymikrobiale resistinsje (AMR). Foarútgong yn ynformaasjetechnology hat ús net allinich de mooglikheden brocht om oer regionale, nasjonale en ynternasjonale grinzen hinne te sjen om ynsicht te krijen yn `e fersprieding fan mikro-organismen en AMR, mar sels ek om pandemys yn real-time wier te nimmen, te analysearjen en te begripen. Metoaden dy't wy hjoed ûntwikkelje en brûke, kinne moarn oan `e oare kant fan `e wrâld tapast wurde. Dat is in wichtich foardiel yn moderne mikrobiale epidemiology, dêr't de klam hieltyd mear op data komt te lizzen.

De data dy't brûkt wyrde as ynput foar mikrobiale epidemiologyske analyzes, wurde faaks ferkrigen út laboratoariumynformaasjesystemen (LIS). Dizze data binne routine-diagnostyske resultaten fan laboratoariumtests. Yn \textbf{haadstik 2} is de eigensinnige opfetting oanfierd dat diagnostyk liedt ta rûge resultaten, mar net needsaaklikerwiis ta in direkt antwurd op de klinyske fraach dy't in behanneljend arts fan in pasjint hawwe kin. Antwurden oan dokters fereaskje in oanpak fan in multydissiplinêr, ferweve ``stewardship''-konsept mei in fokus op diagnostyk. Dat fereasket fan medyske spesjalisten yn `t algemien (en artsen-mikrobiolooch yn it bysûnder) in nauwe ynteraksje mei kollega's, sadat dat soarget foar optimale kwaliteit fan soarch en feiligens fan pasjinten; dat is it saneamde Diagnostic Stewardship Program (DSP). De term ``stewardship'' (rintmasterskip) wurdt breed brûkt om kommunikaasje en klinyske beslútfoarming te fasilitearjen, mar it fêststellen fan in dúdlike definysje fan ``stewardship'' hat in útdaging bewiisd. Boppedat giet de diagnostyk yn medysk-mikrobiologyske laboratoaria op it stuit fluch foarút mei betrekking ta ferbettere workflows en nije technologyen, lykas matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) massaspektrometry. Dochs is diagnostyk yn ynfeksjebehear breder as dit en omfettet it in protte klinyske gebieten wêr't kommunikaasje en ynteraksje fûneminteel binne om it bêste gebrûk te meitsjen fan kennis en saakkundigens, sadat alle spesjalismen in bydrage leverje kinne oan pasjintesoarch. De juste test op de juste tiid foar de juste pasjint om de juste fragen te beäntwurdzjen en mei de juste behanneling te begjinnen − dêr giet DSP yn medyske mikrobiology oer. Mikrobiale epidemiology kin brûkt wurde foar in lyts aspekt fan dat diagnostyske gehiel, troch de testresultaten te resirkulearjen om dêrnei ferriking oan te bringen yn it antwurd- generearjende proses dat DSP is.

\textbf{Haadstik 3} giet fierder mei it beljochtsjen fan wichtige hjoeddeistige beheiningen by it tapassen fan mikrobiale epidemiology, benammen AMR-analyze. AMR-analyze moat útfierd wurde op in klinysk en epidemiologysk sinfolle manier, mar it is útdaagjend om't it ekspertize fereasket yn sawol (klinyske) epidemiology as (medyske) mikrobiology, en derneist de juste instruminten om de AMR-analyzes út te fieren. Dit wurdt nochris fierder yngewikkeld troch it ûntbrekken fan de tagonklikens fan LIS-data, om't de measte LIS-en net ûntwurpen binne mei epidemiologyske analyzes yn `e holle. Elk LIS ûnderhâldt bygelyks syn eigen taksonomyske gegevens en laboratoaria binne sels ferantwurdlik foar it regelmjittich bywurkjen dêrfan. Sûnt AMR-rjochtlinen bot basearre binne op de mikrobiale taksonomy (guon regels jilde allinnich foar in spesifyk skaai, oare regels jilde foar in spesifike famylje), moat dizze ynformaasje akkuraat wêze en by de tiid. Spitigernôch is troch ûndersyk ûnder sân medysk-mikrobiologyske laboratoaria yn Nederlân bliken dien dat al harren LIS-en tige ferâldere taksonomyske nammen befetsje. Dit kin slimme gefolgen hawwe foar sawol de routinerapportaazje fan resultaten as foar (takomstige) epidemyologyske analyzes. Om dy redenen waard yn dit haadstik it AMR-pakket foar R yntrodusearre as in nij epidemiologysk ynstrumint foar AMR-analyze dat fergees, ûnôfhinklik, open-source en iepenbier beskikber is. It waard ûntwikkele troch in team fan tolve ferskillende iepenbiere soarchorganisaasjes út sân ferskillende lannen en biedt help om it opskjinjen, transfomearjen en analysearjen fan AMR-gegevens te ferienfâldigjen, en biedt tagelyk metoaden om maklik (ynter)nasjonale rjochtlinen en wittenskiplik betroubere referinsjegegevens ta te passen. Tsjin maaie 2021 wie it mear as 50.000 kear ynladen troch brûkers út 162 ferskillende lannen sûnt de earste release yn 2018. De resultaten fan in enkête ûnder brûkers presintearre yn dit haadstik, litte sjen dat it gebrûk liedt ta mear reprodusearberens fan analyzeresultaten, betrouberdere resultaten fan AMR-analyzes, en sawol nije as ferbettere ynsjoggen yn AMR foar de ynstellings en regio's fan `e brûkers. Brûkers stelden ek dat it AMR-pakket brûkt wie om klinyske beslútfoarming te stypjen. It pakket lost it ûngemak op fan it ôfhinklik wêzen fan (ynter)nasjonale rjochtlinen en betroubere (referinsje)gegevens, wylst it ek in wiidweidige `toolbox' biedt foar de analyze sels. It AMR-pakket foar R kin dêrom in help wêze foar elke spesjalist yn it fjild dy't mei AMR-gegevens wurket.

\textbf{Seksje II}

Nei de útdagings dy't sketst binne yn `e foarige seksje, wurdt yn dizze seksje it AMR-pakket foar R beskreaun as in nij ynstrumint om dy útdagingen oan te pakken. Fanút ferskate perspektiven wurdt it AMR-pakket en syn foardielen beljochte: fanút in technysk perspektyf, fanút it perspektyf fan ynfeksjebehear en fanút in klinysk perspektyf. Dy kombinaasje jout in mienskiplike basis foar it begripen fan de oplossingen dy't it AMR-pakket biede kin en hoe't it in nij begjinpunt foarmje kin foar takomstige tapassingen fan mikrobiale epidemiology.

De technyske skaaimerken fan it AMR-pakket foar R wurde beskreaun yn \textbf{haadstik 4}, dêr't yn beskreaun wurdt hoe't it AMR-pakket ûntwurpen is om reprodusearbere AMR-analyzes te standerdisearjen oan `e hân fan ynternasjonale standert oanrikkemedaasjes. Om dat mooglik te meitsjen, wurde wittenskiplik betroubere referinsjegegevens brûkt foar de falidaasje fan laboratoariumresultaten, antymikrobiale middels en de folsleine biologyske taksonomy fan mikro-organismen. Boarnegegevens moatte analysearre wurde yn de meast betroubere wei, foaral wannear't it resultaat, bygelyks, brûkt wurde sil om de behannelopsjes foar in psjint te evaluearjen. Dit freget reprodusearbere en spesjalisearre ferwurking fan gegevens. It AMR-pakket biedt in standerdisearre en automatisearre manier om mienskiplike LIS-data op te skjinjen, te transformearjen en te ferbetterjen, ûnôfhinklik fan de ûnderlizzende databoarne en de krektens fan `e data. Foar dit doel, binne algemien tapasbere algoritmen ûntwikkele, om AMR-testresultaten opskinje te kinnen en nammen fan mikro-organismen en antymikrobiale middels falidearje te kinnen. De formule foar de falidaasje fan taksonomyske nammen hâldt rekken mei it foarkommen fan siikmeitsjende mikro-organismen en is kontekstbewust oangeande oare taksonomyske skaaimerken sa as it keninkryk, fylum, oarder en famylje. Bygelyks wurdt de wearde ``E. coli'' oersetten nei de baktearje Escherichia coli, wylst de brûker ek ynformeare wurdt dat de parasyt Entamoeba coli ek in mooglikheid is, mar in legere kâns hat. Mei help fan handige funksjes kinne brûkers fluch konsistinte mikrobiale eigenskippen weromfine, lykas it taksonomyske keninkryk, famylje, skaai, soarte, ferâldere taksonomyske nammen en sels de Gram-kleur. Neist ynformaasje oer mikro-organismen, befettet it pakket ek referinsjegegevens oangeande antibiotika, wêrûnder in protte foarkommende LIS-koades, offisjele nammen, ATC-koades (Anatomical Therapeutic Chemical), definearre deistiche doses (defined daily doses, DDD), en mear as 5.000 hannelsnammen fan 456 antymikrobiale middels. Mei dizze referinsjegegevens kinne brûkers rauwe gegevens oersette en eigenskippen weromfine oer elk mikro-organisme of antibiotikum. Boppedat is it AMR-pakket yn steat om multiresistinte organismen (multidrug-resistant organisms, MDRO's) te identifisearjen basearre op nasjonale en ynternasjonale rjochtlinen, minimum inhibitory concentrations (MIC's) te ynterpretearjen, en kin it de earste isolaten bepale dy't brûkt wurde moatte foar it berekkenjen fan AMR foar sawol monoterapy as kombinaasje-terapyen. It AMR-pakket is bedoeld om in breed helpmiddel te wêzen foar data-technysk personiel dat wurket yn it gebiet fan AMR, hoewol't it gebrûk net beheind is ta dy groep.

As yllustraasje hjirfan wurdt yn \textbf{haadstik 5} sjen litten dat it AMR-pakket likernôch brûkt wurde kin as in soarte fan rêchbonke yn in ynteraktive open-source software-applikaasje foar ynfeksjemangement en antimicrobial stewardship, neamd RadaR (rapid analysis of diagnostic and antimicrobial patterns in R). Ynfeksjemangement yn `e foarm fan Antimicrobial Stewardship Programma's (ASP), hat him ûntpopt as in effektive oplossing om it globale sûnensprobleem fan antibioatikaresistinsje yn sikehuzen oan te pakken. Dit is yn oerienstimming mei \textbf{haadstik 2}; stewarship-yntervinsjes en -aktiviteiten rjochtsje harren op yndividuele pasjinten (persoanlike genêskunde en konsultatie), mar likegoed op pasjintgroepen of klinyske syndromen, dêr't elke yntervinsje liede moat ta ferbettering fan de kwaliteit fan `e soarch en de feiligens fan de pasjint. It is lykwols dreech om pasjintgroepen yn `e deistige praktyk te analysearjen (bgl. stratifisearre nei ôfdieling, spesifike antymikrobiale middels, of brûkte diagnostyske prosedueres). It is sels noch lestiger om fluch grutte pasjintpopulaasjes te analysearjen (bgl. ferspraat oer meardere spesjaliteiten), ek al is dizze ynformaasje beskiber yn `e data. Dêrom wie de ûntwikkeling fan RadaR bedoeld om ASP-teams te foarsjen fan in brûkersfreonlik en tiidsbesparjend ynstrumint, sûnder dat de djippe technyske ekspertize nedich is. RadaR biedt ûnder oaren Kaplan-Meier -curves oer de lisduur yn sikehuzen, tiidtrends foar it oantal opnames, antibiotikagebrûk, en in automatisearre AMR-data-analyze dêr't it AMR-pakket foar R foar brûkt is. RadaR is falidearre troch 12 ESGAP-leden (European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship) út 9 ferskillende lannen. It hat it potinsjeel in tige brûkber ynstrumint te wêzen yn `e deistige praktyk fan sawol ynfeksjebehear as ASP-teams. Dêrnjonken waard yn dit haadstik dúdlik dat it AMR-pakket brûkt wurde kin as ûnderdiel fan in oare software-oplossing om yntegrearre ynfeksjebehear mooglik te meitsjen.

Dêrút folgjend yllustrearret \textbf{haadstik 6} de effektiviteit fan it AMR-pakket ûnder brûkers, troch it beoardieljen fan de brûkberens en it effekt op de wurkstream fan dokters yn in typysk klinysk senario. Hoewol't it AMR-pakket yn wittenskiplik ûndersyk al yn ferskate stúdzjes út ferskate lannen brûkt is, wie der noch gjin analyze fan `e ynfloed op AMR-analyze en -rapportaazje yn in klinyske omjouwing. De analyze en rapportaazje fan AMR-data fereaskje spitigernôch spesjaal oplaat personiel. Derneist kinne AMR-data-analyzes tiidsrôvjend wêze. Om de impact hjirfan yn in klinyske omjouwing te beoardieljen, waarden algemiene ûndersyksfragen oer bloedkultuerdata gearstald dy't troch klinysk routinepersoniel beäntwurde wurde moasten, wêrûnder artsen-mikrobiolooch, bernedokters en yntinsivisten. Yn totaal diene tsien klinici mei oan `e stúdzje. Boppedat waard dielnimmers frege in online fragelist yn te foljen oer har eftergrûn, demografy (lykas leeftyd en geslacht), en eardere ûnderfining mei software en AMR-data-analyze en -rapportaazje. Alle dielnimmers moasten de fragen twa kear beäntwurdzje: de earste kear mei de software fan harren eigen kar (earste ronde) en de twadde kear mei in nij ûntwikkele webapplikaasje, boud om it AMR-pakket foar R hinne (twadde ronde). In effisjinte agile workflow waard brûkt foar de ûntwikkeling fan dizze webapplikaasje. De antwurden op de ûndersyksfragen tsjinnen as basis om de effektiviteit (antwurden op elke taak foar elke brûker) en de effisjinsje (tiid bestege oan it oplossen fan elke taak) te fergelykjen tusken de twa rondes. Net alle dielnimmers koene taken binnen it foarskreaune tiidsbestek foltôgje. De gemiddelde foltôging per taak tusken de earste en twadde ronde naam ta fan 56\% nei 96\% en it persintaazje goede antwurden wie tanommen fan 38\% nei 98\%. De gemiddelde tiid per ronde waard fermindere mei mear as in oere. Dit haadstik toant dêrmei de ferhege effektiviteit, effisjinsje en krektens fan it brûken fan it AMR-pakket foar R foar AMR-analyze yn ferliking mei tradisjonele software lykas Microsoft Excel en SPSS.

\textbf{Seksje III}

In protte klinyske stúdzjes op it gebiet fan ynfeksjesykten en medyske mikrobiology binne ôfhinklik fan ien of oare foarm fan (mikrobiale) epidemiology. Wylst yn `e foarige seksje it AMR-pakket yntrodusearre waard en it gebrûk yn ferskate senario's ûndersocht waard, begjint dizze seksje mei in epidemiologysk ûndersyksprojekt yn `e Noard-Nederlânske regio, en wreidet dizze seksje dêrnei út nei de Nederlânsk-Dútske grinsregio om de distribúsje, it foarkommen en de AMR fan ferskate sykteferwekkende mikro-organismen op in (je)regionaal nivo better begripe te kinnen. Troch yn te zoomen op de regio's oan beide kanten fan in lânsgrins, kinne op mikronivo ferlikings makke wurde tusken twa ferskillende naasjes. En ferskillende naasjes betsjut úteinlik ferskillende soarchstruktueren. Wat bliuwt oer fan `One Health`? Wat binne de konsekwinsjes fan it hawwen fan ferskillen tusken lannen yn testtechniken, antibiotika-ynterpretaasjes en mikrobiologysk screeningsbelied? Dizze seksje jout antwurden op dy fragen.

\textbf{Haadstik 7} rjochtet him op koägulaze-negative stafylokokken (KNS), wêrfan't bekend is dat se bloedstreamynfeksjes (BSY) en hege mortaliteit feroarsaakje kinne, hoewol't se jierrenlang mar as `gewoan' kontaminaasje beskôge waarden. Boppedat wurde KNS-en hieltyd faker assosjeare mei nosokomiale ynfeksjes. Op it stuit bestiet de KNS-groep út 45 ferskillende soarten (`species'), hoewol it bepalen fan it soartnivo pas koartlyn mooglik makke is foar routine-diagnostyske laboratoaria. Sûnt 2012 is nammentlik MALDI-TOF massaspektrometry de standert foar de identifikaasje fan bakteriële soarten lykas KNS. Hjirfoar waard de ydentifikaasje benammen dien mei biogemyske en fysiologyske testmetoaden, dy't fariearjende resultaten opleveren, yn it bysûnder by minder foarkommende soarten. AMR, en yn it bysûnder multyresistinsje, is in tanimmend probleem yn KNS-en. Dochs wurde KNS-en yn behannelrjochtlinen en nasjonale tafersjochprogramma's (lykas it Nederlânske NethMap), noch hieltyd as ien groep sjoen, sûnder differinsjaasje tusken soarten. Om dizze reden is net folle bekend oer trends yn it foarkommen fan, en AMR yn, KNS-en op lokaal en regionaal nivo. Dêrom toant dizze retrospektive stúdzje in detaillearre AMR-analyze fan hast 20 tûzen KNS-isolaten dy't fûn wiene yn alle beskikbere 70 tûzen bloedkultuerisolaten tusken 2013 en 2019 yn Noard-Nederlân. Mei dizze analyze hawwe wy stribbe om ferskillen yn it foarkommen fan KNS-soarten en harren AMR-patroanen te evaluearjen en om harren klinyske mikrobiologyske relevânsje te beoardieljen. Yn totaal waarden 27 ferskillende soarten fan `e KNS-groep fûn. Grutte ferskillen waarden sjoen yn it foarkommen fan `e ferskillende soarten: de top fiif bestie út 97\% fan alle isolaten (S. epidermidis, S. hominis, S. capitis, S. haemolyticus en S. warneri). It oanpart fan KNS-en op `e intensive care (IC) neffens oare ôfdielings wie signifikant ferskillend tusken perifeare sikehuzen en it universitêr sikehûs. Om't net bekend wie hokker pasjinten in BSY hienen, waard ``KNS-persistinsje'' definieare as in surrogaat wêrfoar teminsten trije positive bloedkulturen nommen wurde moasten op trije ferskillende dagen, binnen 60 dagen, wêr't deselde KNS yn fûn wie, by deselde pasjint. De relatyf meast foarkommende oarsaaklike ferwekker fan KNS-persistinsje wie S. haemolyticus, folge troch S. epidermidis en S. lugdunensis. AMR-analyze hat wichtige ferskillen iepenbiere tusken de KNS- soarten. Bygelyks eksposearren S. epidermidis en S. haemolyticus 50\% oant 80\% resistinsje tsjin de measte antibiotika, wylst de resistinsje tsjin dizze middels by `e measte oare KNS-en leger as 10\% bleau. En dochs op nasjonaal nivo, lykas yn NethMap, wurde dizze ferskillen ferwaarleazge, wat liede kin ta de ûntwikkeling fan behannelrjochtlinen dy't har rjochtsje op feilige en fertroude middels foar de behanneling fan KNS, lykas vancomycine of linezolid. Middels lykas tetracycline, cotrimoxazol, en erythromycine soenen as alternative opsjes beskôge wurde kinne foar guon soarten, wêr't de AMR, neffens ús ûndersyksresultaten, nea boppe de 10\% útkaam is. Ta beslút kin steld wurde dat in mearjierrige regio-oerstiigjende oanpak tapast is om de ûntwikkelingen yn sawol it foarkommen as de antibioatikaresistinsje fan LNS-soarten wiidweidich te beskôgjen, om sadwaande it behannelbelied te evaluearjen en mear te begripen oer dizze wichtige, mar noch net faak genôch serieus nommen sykteferwekkers. Dêrneist tsjinne dizze stúdzje as in praktysk foarbyld fan hoe't it AMR-pakket foar R brûkt wurde kin yn stúdzjes om nije ynsjoggen te krijen oer antibiotikaresistinsje mei epidemiologysk ûnderboude metoaden.

Nei oanlieding fan de nije befinings troch it bestudearjen fan AMR-testresultaten yn Noard-Nederlân, jout \textbf{haadstik 8} in ferliking fan nasjonale ynterpretaasjes fan MDRO's yn de Nederlânsk-Dútske grinsregio, benammen oangeande de praktyske gefolgen foar personiel yn de sûnenssoarch dy't ticht by de grins wurkje. It fergelykjen fan AMR yn it algemien, net allinne MDRO's, yn dizze grinsregio is tige ynteressant om't beide lannen karakterisearre wurde troch heech ûntwikkele, mar dochs struktureel oars ynrjochte soarchsystemen. Antibioatika-ynterpretaasjes fan pasjinten wurde oerdroegen tusken soarchynstellings yn dizze twa lannen, wylst de ûnderlizzende definysjes ferskille. Dêrtroch moatte dokters en ynfeksjeprevinsje-meiwurkers de antibioatikaresultaten fan beide kanten fan `e grins begripe kinne en yn steat wêze beide nasjonale MDRO-rjochtlinen tapasse te kinnen. Troch antibiogrammen fan Gram-negative baktearjes fan beide kanten fan `e grins mei-inoar te fergelykjen, waard besocht de omfang fan ynfloed fan dizze útdagingen te bepalen. Dêrta waarden tusken 2015 en 2016 35.619 antibiogrammen út seis Nederlânske en fjouwer Dútske sikehûzen analysearre foar alle soarten Enterobacteriaceae, en P. aeruginosa, it A. baumannii-kompleks en Stenotrophomonas maltophilia. Foar al dizze soarten besteane yn dizze regio MDRO-oanbefellings en spesjale ynfeksjeprevinsjemaatrigels. Út de Nederlânske sikehûzen waarden 12.616 antibiogrammen selekteare mei it AMR-pakket foar R, wêrmei ek de Nederlânske MDRO-rjochtline tapast wurde koe. Wichtich is dat oare nasjonale en ynternasjonale rjochtlinen, lykas de Dútske MDRO-rjochtline, ek opnommen binne yn it AMR-pakket foar R. Út Dútske sikehûzen waarden 23.003 antibiogrammen selekteare. Neffens de Nederlânske rjochtline wie 24,5\% fan alle isolaten in MDRO. Neffens de Dútske rjochtline wie 12,9\% fan alle isolaten in MDRO. Lykwols waard 73,7\% fan alle isolaten net klassifisearre as in MDRO neffens ien fan `e beide rjochtlinen. By it oerdragen fan pasjinten tusken sikehûzen, moat ek ynformaasje oer MDRO-kolonisaasje of -ynfeksje oerdroegen wurde om de trochgeande útfiering fan ynfeksjeprevinsjemaatrigels te garandearjen. Foar regio-oerstiigjende sûnenssoarch betsjut dit dat klinici en ynfeksjeprevinsjemeiwurkers yn steat wêze moatte om MDRO's te bepalen basearre op antibiogrammen neffens de rjochtlinen fan ien fan beide lannen. Foar regio-oerstiigjende sûnenssoarch soe dêrom de ienfâldichste oplossing wêze om de rjochtlinen fan beide lannen te harmonisearjen. Dat soe ek de begryplike betizing oplosse kinne dy't pasjinten ûnderfine kinne as ynfeksjeprevinsjemaatrigels oplein wurde yn it iene lân, mar opheft wurde nei oerdracht nei it oare lân. Oant de harmonisaasje berikt is, soenen de folsleine AMR-gegevens tegearre mei de pasjint oerdroegen wurde moatte om't klassifikaasje foar lokale ynfeksjeprevinsje-meiwurkers mooglik te meitsjen.

Oare AMR-relatearre grinsoerstiigjende útdagings en ferskillen wurde yllustrearre yn \textbf{haadstik 9}, dat in wiidweidige mikrobiale epidemiologyske analyze omfettet fan it foarkommen fan MRSA, it belied en de ynfloed op sûnenssoarch yn `e Nederlânsk-Dútske grinsregio. MRSA is noch altyd ien fan `e liedende oarsaken fan sikehûs-relatearre ynfeksjes troch resistinte baktearjes. Yn dizze stúdzje waarden MRSA-tafersjochgegevens fan fiif jier (2012-2016) út Nederlânske en Dútske sikehuzen yn `e grinsregio analysearre om regio-spesifike trends oer tiid fan MRSA beskriuwe te kinnen en om ferskillen tusken sikehûsgroepen fêst te stellen. De stúdzje omfette 42 sikehûzen yn `e Nederlânsk-Dútske grinsregio mei sawat 620.000 opnommen pasjinten (68,0\% yn it Dútske diel fan 'e ûndersyksregio) mei hast fjouwer miljoen pasjintdagen per jier. Alle sikehuzen hiene MRSA-relatearre previnsjemaatrigels ymplementeare neffens harren nasjonale rjochtlinen en oanbefellings, en ferskillen yn rjochtlinen tusken de twa lannen waarden fergelike. Oan beide kanten fan `e grins naam it MRSA-screeningspersintaazje tusken 2012 en 2016 bot ta, hoewol de MRSA-ynsidinsje oer de tiid stabyl bleau oan beide kanten fan `e grins. Yn totaal wie it screeningspersintaazje yn `e Dútske grinsregio 14 kear heger as yn 'e Nederlânske grinsregio. It persintaazje MRSA yn bloedkultuerisolaten mei S. aureus sakke fan 13\% yn 2012 nei 5\% yn 2016 yn `e Dútske grinsregio, wylst it stabyl bleau yn `e Nederlânske grinsregio (0\% oant 2\%). Dochs wie MRSA ûnder S. aureus-isolaten 34 kear heger yn `e Dútske grinsregio. De listiid yn it sikehûs by MRSA-pasjinten wie yn beide regio's lyksoartich, wylst de algemiene listiid flink fariearre. Fierder wie it oantal MRSA-útstriken foar of by sikehûsopname per 100 ynwenners 12,2 yn `e Dútske grinsregio en 0,36 yn `e Nederlânske grinsregio; 34 kear heger yn `e Dútske grinsregio. It oantal yntramurale MRSA-gefallen per 1.000 ynwenners wie 2,52 yn `e Dútske grinsregio en 0,14 yn `e Nederlânske grinsregio. Dizze stúdzje toande dus signifikante ferskillen oan tusken Nederlânske en Dútske sikehûzen. De MRSA-ynsidinsje yn Dútske sikehûzen wie mear as sân kear heger as yn Nederlânske sikehûzen. Neffens it European Centre of Disease Prevention and Control (ECDC) wurde ferskillen yn it foarkommen fan resistente sykteferwekkers tusken Jeropeeske lannen wierskynlik feroarsake troch ferskillen yn soarchgebrûk, antimykrobieel gebrûk en ynfeksjeprevinsjemaatrigels. Wat it soarchgebrûk yn ús kontekst oanbelanget, fûnen wy dat ynwenners yn it Dútske diel fan `e stúdzje hast trije kear sa faak yn it sikehûs opnommen wiene en in tige langere listiid hiene as pasjinten yn it Nederlânske diel. Dit kin wêze troch sosjaal-ekonomyske faktoaren of in oare ynrjochting fan ambulante sûnenssoarch. Dizze wiidweidige stúdzje oer MRSA yn sikehûzen rûn in Jeropeeske grins hat sjen litten dat trochgeand MRSA-tafersjoch nuttich wêze kin om trends fan MRSA te folgjen, om (ynter)nasjonale fergelikingen ta te stean.

De diskusje fan dizze stúdzje waard ôfsletten mei (oersetten) ``grinsoerstiigjend tafersjoch soe útwreide wurde moatte nei oare multyresistinte mikro-organismen'', wat krekt is wêr't \textbf{haadstik 10} op trochgiet. Sûnt net allinne MRSA's mar MDRO's yn it algemien in risiko posearje foar de sûnenssoarch, sawol yn `e mienskip as yn de sikehuzen, hie dizze stúdzje ta doel om it foarkommen fan meardere MDRO's yn dizze grinsregio fêst te stellen om sadwaande verskillen better begripe te kinnen, en om ynfeksjeprevinsje te ferbetterjen, baseare op real-time routinegegevens. Foar dat doel namen 23 sikehûzen yn `e Nederlânsk-Dútske grinsregio tusken 2017 en 2018 diel oan dizze prospective stúdzje troch alle pasjinten op tagong ta de intensive care (IC) te ûndersykjen. Alle sikehûzen (8 yn `e Nederlânske grinsregio, 15 yn `e Dútske grinsregio) dienen elk mei foar acht opienfolgjende wiken en ûndersochten yn dy perioade pasjinten foar dragerskip fan MRSA, vancomycine-resistinte Enterococcus faecium/E. faecalis (VRE), tredde-generaasje cefalosporine-resistinte Enterobacteriaceae (3GCRE) en carbapenem-resistinte Enterobacteriaceae (CRE). Yn totaal waarden 3.365 pasjinten ûndersocht: 35,7\% op Nederlânske IC's en 64,3\% op Dútske IC's. De mediane leeftyd fan alle screenede pasjinten wie 68 jier (IQR: 57-77), wêrby pasjinten yn `e Dútske grinsregio signifikant âlder wiene as pasjinten yn `e Nederlânske grinsregio. De algemiene screening compliance (screened foar teminsten ien MDRO-groep) wie 60\%. Alle AMR-data-analyzes waarden útfierd en automatisearre mei help fan it AMR-pakket foar R. It foarkommen fan MRSA wie 1,7\% op Dútske IC's en 0,6\% op Nederlânske IC's. It foarkommen fan VRE wie 2,7\% op Dútske IC's en 0,1\% op Nederlânske IC`s. Opfallend wie dat dit foarkommen yn it Dútske grinsgebiet útienrûn fan 0\% oant 4,1\%. Alle 56 gefallen fan VRE waarden feroarsake troch E. faecium. It foarkommen fan 3GCRE wie 6,6\% op Dútske IC's en 3,6\% op Nederlânske IC's, wylst it foarkommen fan CRE oan beide kanten fan de grins tichtby nul bleau. It foarkommen fan Gram-negative MDRO's ferskilden tusken sikehuzen yn beide lannen, fariearjend fan 0\% oant 5,0\% yn `e Nederlânske grinsregio en fan 1,2\% oant 10,9\% yn `e Dútske grinsregio. Foar de ynbegrepen Nederlandse IC's wie it foarkommen fan alle MDRO-groepen net gâns oars tusken perifeare sikehuzen en it universitêr sikehûs. Op de Dútske IC's wie it foarkommen fan Gram-negative MDRO's lykwols heger yn perifeare sikehuzen. Yn `e Nederlânske grinsregio liede 4,8 per 100 sikehûsopnames ta in IC-opname. Yn `e Dútske grinsregio wie dit oars 7,7 per 100 sikehûsopnames. Dit ferskil kin ferklearre wurde troch de hegere IC-kapasiteit yn Dútske sikehûzen (4,8\% fan alle sikehûsbêden) yn ferliking mei Nederlânske sikehûzen (2,4\% fan alle sikehûsbêden). It algehiele foarkommen fan ferskillende MDRO's wie heger op Dútske IC's, hoewol't guon ferskillen tige lyts wiene. Benammen it foarkommen fan MRSA wie trije kear heger yn `e Dútske grinsregio as yn `e Nederlânske grinsregio, wat konsistint is mei de ûndersyksresultaten yn \textbf{haadstik 9}. Dochs wiene de ûndersyksresultaten net konsistint mei (ynter)nasjonale gemiddelden. Bygelyks, it foarkommen fan 3GCRE wie hast twa kear sa heech yn `e Dútske grinsregio as yn `e Nederlânske grinsregio, mar beide wiene noch hieltyd leger as it nasjonale gemiddelden; de ECDC rapporteare 6\% hegere 3GCRE-persintaazjes ûnder E. coli en K. pneumoniae út bloedkulturen foar Dútslân en Nederlân. Dit lit sjen dat der wichtige ferskillen binne tusken it bestudearjen fan dragerskip yn bepaalde populaasjes en it bestudearjen fan it oanpart fan (wierskynlik) invasive isolaten. Dizze stúdzje beklammet dêrom it belang fan in regionale en grinsoerstiigjende oanpak yn in Jeropeeske grinsregio, om it ferskil yn foarkommen fan AMR tusken de regio's te yllustrearjen en om potinsjele ferskillen mei nasjonale rapporten te beljochtsjen. Om dat fierder út te wurkjen is in djipper nivo fan detail nedich, bygelyks troch ynformaasje te sammeljen oer (ynfeksjeprevinsje) personiel, MDRO-útbraken, ynfeksjes, antibiotikagebrûk en risikofaktoaren fan pasjinten. Yn konklúzje lykje geografyske en politike grinzen troch MDRO's net ``respektearre'' te wurden, hoewol sûnenssoarchsystemen, geografyske lokaasje en rjochtlinen ferskille fan lân nei lân. De persintaazjes MDRO's fan guon sykteferwekkers, lykas nasjonaal en ynternasjonaal rapportearre, reflektearje net it foarkommen fan MDRO's yn in pasjint en/of yn `e algemiene befolking. Dat moat yn alle earnst beskôge wurde by it ynterpretearjen fan rapporten op nasjonaal of sels kontinintaal nivo.

\textbf{Konklúzje}

Fanút ferskate perspektiven wurdt it AMR-pakket en syn foardielen beljochte: fanút in technysk perspektyf, fanút it perspektyf fan ynfeksjebehear en fanút in klinysk perspektyf. Dy kombinaasje jout in mienskiplike basis foar it begripen fan de oplossingen dy't it AMR-pakket biede kin en hoe't it in nij begjinpunt foarmje kin foar takomstige tapassingen fan mikrobiale epidemiology, sawol yn klinyske omjouwings as yn wittenskiplik ûndersyk. Dit proefskrift giet yn op dizze perspektiven troch it gebrûk fan dit nije ynstrumint te yllustrearjen yn epidemiologyske stúdzjes yn `e Nederlânsk-Dútske grinsregio om de distribúsje, it foarkommen en de AMR fan ferskate sykteferwekkende mikro-organismen op in (je)regionaal nivo better begripe te kinnen. Ta beslút toant dit proefskrift de tafoege wearde fan in konsekwint data-analytysk ynstrumint om AMR-data foar te meitsjen en te analysearjen yn in regio-oerstiigjende oanpak, om nije ynsjoggen te krijen yn AMR-trends.

\hypertarget{samenvatting-in-het-nederlands}{%
\chapter*{Samenvatting in het Nederlands}\label{samenvatting-in-het-nederlands}}
\addcontentsline{toc}{chapter}{Samenvatting in het Nederlands}

\textbf{Sectie I}

Waar is de microbiële epidemiologie begonnen? Hoe is het ontstaan? En hoe draagt het bij tot de holistische benadering van infectiemanagement? Deze vragen worden in deze eerste sectie beantwoord. Vervolgens wordt geschetst welke belangrijke huidige beperkingen er bestaan bij de toepassing van microbiële epidemiologie in de praktijk en hoe deze zouden kunnen worden ondervangen.

In de algemene inleiding in \textbf{hoofdstuk 1} van dit proefschrift wordt geschetst dat microbiële epidemiologie een onderdeel is van de epidemiologie van infectieziekten, die op haar beurt weer een onderdeel is van de medische microbiologie. Microbiële epidemiologie kan onder andere worden gezien als het wetenschappelijke veld voor het verwerven van nieuwe inzichten over de verspreiding van micro-organismen en hun respectievelijke antimicrobiële resistentie (AMR). De vooruitgang in de informatietechnologie heeft ons niet alleen de mogelijkheden gebracht om over regionale, nationale en internationale grenzen heen te kijken om inzicht te krijgen in de verspreiding van micro-organismen en AMR, maar zelfs om pandemieën real-time te analyseren en te begrijpen. Methoden die we vandaag ontwikkelen en gebruiken, kunnen bij wijze van spreken morgen aan de andere kant van de wereld worden toegepast. Dit is een belangrijk voordeel van moderne microbiële epidemiologie, waarin de focus steeds meer op data komt te liggen.

De data die als input voor microbieel epidemiologische analyses worden gebruikt, worden vaak verkregen uit laboratoriuminformatiesystemen (LIS). Deze data bestaan uit routine-diagnostische resultaten van laboratoriumtests. In \textbf{hoofdstuk 2} wordt de mening naar voren gebracht dat diagnostiek wél kan leiden tot ruwe resultaten, maar níet noodzakelijkerwijs leidt tot een direct antwoord op de klinische vraag die een behandelend arts van een patiënt kan hebben. Om artsen van antwoorden te voorzien is de aanpak van een multidisciplinair, verweven ``stewardship''-concept nodig met een focus op diagnostiek. Dit vraagt van medisch specialisten in het algemeen (en artsen-microbioloog in het bijzonder) een nauwe interactie voor optimale kwaliteit van zorg en patiëntveiligheid dat leidt tot succesvol infectiemanagement: diagnostisch stewardship (DSP). Het begrip ``stewardship'' wordt veel gebruikt om communicatie en klinische besluitvorming te vergemakkelijken, maar het is een uitdaging gebleken om een duidelijke definitie van ``stewardship'' vast te stellen. Bovendien boekt de diagnostiek in medisch microbiologische laboratoria momenteel snelle vooruitgang met betrekking tot verbeterde workflows en nieuwe technologieën, zoals matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) massaspectrometrie. Diagnostiek bij infectiemanagement is echter breder dan dit en bestrijkt veel klinische gebieden waar communicatie en onderlinge interactie van fundamenteel belang zijn om optimaal gebruik te kunnen maken van kennis en expertise, zodat alle specialismen een bijdrage kunnen leveren aan patiëntenzorg. De juiste test op het juiste moment voor de juiste patiënt om de juiste vragen te beantwoorden en de juiste behandeling te starten -- dat is waar het bij DSP in de medische microbiologie om draait. Microbiële epidemiologie kan ingezet worden voor een klein aspect van het diagnostische geheel, door testresultaten te recyclen en vervolgens verrijkingen aan te brengen in het antwoord-genererende proces dat DSP belichaamt.

\textbf{Hoofdstuk 3} gaat verder met het belichten van belangrijke huidige beperkingen bij de toepassing van microbiële epidemiologie, in het bijzonder bij de analyse van AMR-data. De analyse van AMR-data moet worden verricht op een klinisch en epidemiologisch zinvolle manier, hoewel dit een uitdaging is door de vereiste expertise in (klinische) epidemiologie en (medische) microbiologie, en goede instrumenten om de AMR-data-analyse uit te voeren. Dit wordt nog eens verder bemoeilijkt door het gebrek aan de toegankelijkheid van LIS-data, aangezien de meeste LIS-en niet zijn ontworpen om epidemiologische analyses te doen. Elk LIS houdt bijvoorbeeld zijn eigen taxonomische gegevens bij en de laboratoria zijn verantwoordelijk voor de regelmatige bijwerking ervan. Aangezien AMR-richtlijnen sterk gebaseerd zijn op de microbiële taxonomie (sommige regels gelden bijv. alleen voor een specifieke genus, andere regels gelden voor een specifieke familie), moet deze informatie correct en up-to-date zijn. Helaas is uit onderzoek onder zeven medisch microbiologische laboratoria in Nederland gebleken dat al hun LIS-en sterk verouderde taxonomische namen bevatten. Dit kan gevolgen hebben voor zowel de routinematige rapportage van resultaten als voor (toekomstige) epidemiologische analyses. Om deze redenen is in dit hoofdstuk het AMR-pakket voor R (een programmeertaal voor statistische berekeningen) geïntroduceerd als een nieuw epidemiologisch instrument voor de analyse van AMR-data. Het AMR-pakket is gratis, onafhankelijk, open-source, en openbaar beschikbaar. Het is ontwikkeld met een team van twaalf verschillende gezondheidsorganisaties in zeven verschillende landen en biedt hulpmiddelen om het opschonen, transformeren en analyseren van AMR-data te vereenvoudigen, mar ook om gemakkelijk (inter)nationale richtlijnen te kunnen toepassen en wetenschappelijk betrouwbare referentiedata te kunnen gebruiken. In mei 2021 was het meer dan 50.000 keer gedownload door 162 verschillende landen sinds de eerste release in 2018. Uit de resultaten van een enquête onder gebruikers die in dit hoofdstuk worden gepresenteerd, blijkt dat het gebruik ervan leidt tot meer reproduceerbaarheid van analyseresultaten, betrouwbaardere uitkomsten van AMR-data-analyses, en nieuwe of verbeterde inzichten in AMR voor de instellingen en regio's van de gebruikers. Gebruikers gaven ook aan dat het AMR-pakket gebruikt is om klinische besluitvorming te ondersteunen. Het pakket lost het ongemak op van het afhankelijk zijn van (inter)nationale richtlijnen en betrouwbare (referentie)data, terwijl het ook een uitgebreide `toolbox' biedt voor de analyse zelf. Het AMR-pakket voor R kan daarom elke specialist in ons veld die met AMR-gegevens werkt in staat stellen zijn werk makkelijker te doen.

\textbf{Sectie II}

Na de uitdagingen die in de vorige sectie zijn geschetst, wordt in deze sectie het AMR-pakket voor R geïntroduceerd als een nieuw instrument om deze uitdagingen aan te gaan. Vanuit verschillende invalshoeken worden het AMR-pakket en zijn voordelen in perspectief geplaatst: vanuit een technisch perspectief, vanuit het perspectief van infectiemanagement en vanuit een klinisch perspectief. Deze combinatie biedt een gemeenschappelijke basis voor het begrijpen van de oplossingen die het AMR-pakket kan bieden en hoe het een nieuw startpunt kan vormen voor toekomstige toepassingen van microbiële epidemiologie.

De technische functionaliteiten van het AMR-pakket voor R zijn beschreven in \textbf{hoofdstuk 4}, waarin wordt beschreven hoe het AMR-pakket is ontwikkeld om reproduceerbare AMR-data-analyses te standaardiseren aan de hand van internationale gestandaardiseerde aanbevelingen. Om dit mogelijk te maken zijn wetenschappelijk betrouwbare referentiedata gebruikt met betrekking tot de validatie van laboratoriumresultaten, antimicrobiële middelen en de volledige biologische taxonomie van micro-organismen. Brondata moeten op de meest betrouwbare manier worden geanalyseerd, vooral wanneer het resultaat bijvoorbeeld gebruikt gaat worden om de behandelingsopties voor een patiënt te evalueren. Dit vereist een reproduceerbare en gespecialiseerde verwerking van data. Het AMR-pakket biedt een gestandaardiseerde en geautomatiseerde manier om gemeenschappelijke LIS-data op te schonen, te transformeren en te verbeteren, onafhankelijk van de onderliggende databron en de nauwkeurigheid van de data. Hiervoor zijn algemeen toepasbare algoritmen ontwikkeld, teneinde AMR-testresultaten te kunnen opschonen en namen van micro-organismen en antimicrobiële middelen te kunnen valideren. De formule voor de validatie van taxonomische namen houdt rekening met het vóórkomen van ziekteverwekkende micro-organismen en is contextbewust wat betreft andere taxonomische eigenschappen zoals het koninkrijk, het fylum, de orde en de familie. Ter illustratie: een waarde ``E. coli'' wordt vertaald naar de bacterie Escherichia coli, terwijl de gebruiker ook wordt geïnformeerd dat de parasiet Entamoeba coli in aanmerking komt, maar een lagere waarschijnlijkheid heeft. Met behulp van behendige functies kunnen gebruikers snel consistente microbiële eigenschappen opvragen, zoals het taxonomische koninkrijk, de familie, het geslacht, de soort, verouderde taxonomische namen en zelfs de Gram-kleur. Naast informatie over micro-organismen bevat het pakket ook referentiedata over antibiotica, waaronder veelvoorkomende LIS-codes, officiële namen, ATC-codes (Anatomical Therapeutic Chemical), gedefinieerde dagelijkse doses (defined daily doses, DDD) en meer dan 5.000 handelsnamen van 456 antimicrobiële middelen. Met behulp van deze referentiedata kunnen gebruikers ruwe data vertalen en eigenschappen ophalen over elk micro-organisme of antibioticum. Bovendien is het AMR-pakket in staat om multiresistente organismen (multidrug-resistant organisms, MDRO's) te identificeren op basis van nationale en internationale richtlijnen, minimum inhibitory concentrations (MIC's) te interpreteren en kan het de eerste isolaten bepalen die gebruikt zouden moeten worden voor het berekenen van AMR voor zowel monotherapie als combinatietherapieën. Het AMR-pakket is bedoeld als een uitgebreid instrument voor data-technisch personeel dat werkzaam is op het gebied van AMR, hoewel het gebruik ervan niet beperkt is tot deze groep.

Om dit te illustreren, toont \textbf{hoofdstuk 5} aan dat het AMR-pakket gebruikt kan worden als ruggengraat in een interactieve open-source software app voor infectiemanagement en antimicrobial stewardship, genaamd RadaR (rapid analysis of diagnostic and antimicrobial patterns in R). Infectiemanagement in de vorm van Antimicrobial Stewardship Programma's (ASP) heeft zich ontpopt als een effectieve oplossing om het mondiale gezondheidsprobleem van antibioticaresistentie in ziekenhuizen aan te pakken. Dit sluit aan bij \textbf{hoofdstuk 2}; stewardship-interventies en -activiteiten richten zich zowel op individuele patiënten (gepersonaliseerde geneeskunde en consultatie) als op patiëntengroepen of klinische syndromen, waarbij bij elke interventie moet leiden tot verbetering van de kwaliteit van de zorg en de veiligheid van de patiënt. Het is echter moeilijk om in de dagelijkse praktijk patiëntengroepen te analyseren (bijv. gestratificeerd naar afdeling, specifieke antimicrobiële middelen, of gebruikte diagnostische procedures). Het is zelfs nog moeilijker om snel grote patiëntpopulaties te analyseren (bijv. verspreid over meerdere specialismen), ook al is deze informatie beschikbaar in de data. Daarom is de ontwikkeling van RadaR bedoeld om ASP-teams te voorzien van een gebruiksvriendelijk en tijdbesparend hulpmiddel voor data-analyse, zonder dat dit diepgaande technische expertise vereist. RadaR biedt onder andere Kaplan-Meier-curves over de ligduur in ziekenhuizen, tijdstrends voor het aantal opnames, antibioticaconsumptie, en een geautomatiseerde AMR-data-analyse waarvoor het AMR-pakket voor R gebruikt is. RadaR werd geëvalueerd door 12 ESGAP-leden (European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship) uit 9 verschillende landen. Het heeft de potentie om een zeer nuttig middel te zijn voor infectiemanagement en ASP-teams in de dagelijkse praktijk. Bovendien toont dit hoofdstuk aan, dat het AMR-pakket gebruikt kan worden als onderdeel van een andere softwareoplossing om geïntegreerd infectiemanagement mogelijk te maken.

Hieruit volgend, illustreert \textbf{hoofdstuk 6} de effectiviteit van het AMR-pakket onder gebruikers, door het evalueren van de bruikbaarheid en de impact op het werk van artsen in een typisch klinisch scenario. Hoewel het AMR-pakket in research al in meerdere studies uit verschillende landen gebruikt is, was er nog geen analyse naar de impact op workflows voor AMR-analyse en -rapportage in een klinische omgeving. De analyse en rapportage van AMR-data vereisen helaas specifiek opgeleid personeel. Bovendien kunnen AMR-data-analyses tijdrovend zijn. Om de impact hiervan in een klinische setting te bepalen, werden algemene vragen over bloedkweekdata opgesteld die door klinisch routinepersoneel moesten worden beantwoord, waaronder artsen-microbioloog, kinderartsen en intensivisten. In totaal namen tien clinici deel aan het onderzoek. Bovendien werd de deelnemers gevraagd een online vragenlijst in te vullen over hun achtergrond, demografische gegevens (zoals leeftijd en geslacht), software-ervaring en eerdere ervaring met AMR-data-analyse en -rapportage. Alle deelnemers moesten de onderzoeksvragen tweemaal beantwoorden: de eerste keer met de software van hun keuze (ronde 1) en de tweede keer met behulp van een nieuw ontwikkelde webapplicatie gebouwd rond het AMR-pakket voor R (ronde 2). Voor de ontwikkeling van deze webapplicatie werd gebruik gemaakt van een zeer efficiënte agile workflow. De antwoorden op de onderzoeksvragen dienden als basis om de effectiviteit (antwoorden op elke taak voor elke gebruiker) en efficiëntie (tijd besteed aan het oplossen van elke taak) tussen de twee rondes te vergelijken. Niet alle deelnemers waren in staat de taken binnen het gestelde tijdsbestek af te ronden. De gemiddelde taakvoltooiing tussen de eerste en tweede ronde steeg van 56\% naar 96\% en het percentage correcte antwoorden steeg van 38\% naar 98\%. De gemiddelde bestede tijd per ronde werd met meer dan een uur verminderd. Dit hoofdstuk demonstreert de verhoogde effectiviteit, efficiëntie en nauwkeurigheid van het gebruik van het AMR-pakket voor R voor AMR-data-analyse in vergelijking met traditionele software zoals Microsoft Excel en SPSS.

\textbf{Sectie III}

Veel klinische studies op het gebied van infectieziekten en microbiologie berusten op een of andere vorm van (microbiële) epidemiologie. Terwijl in de vorige sectie het AMR-pakket is gepresenteerd en het gebruik ervan in verschillende scenario's is gedemonstreerd, begint deze sectie met een epidemiologisch researchproject in de Noord-Nederlandse regio, en breidt de sectie zich vervolgens uit tot de Nederlands-Duitse grensregio om het vóórkomen van ziekteverwekkers en diens AMR-patronen op een (eu)regionaal niveau beter te begrijpen. Door in te zoomen op de regio's aan weerszijden van een landsgrens kunnen op microniveau vergelijkingen worden gemaakt tussen twee verschillende naties. En verschillende naties betekenen uiteindelijk verschillende gezondheidszorgsystemen. Wat blijft er over van `One Health'? Wat zijn de gevolgen van de verschillen tussen landen wat betreft AMR-testmethoden, MDRO-interpretaties en screeningsbeleid? Deze sectie geeft antwoorden op deze vragen.

\textbf{Hoofdstuk 7} zoomt in op coagulase-negatieve stafylokokken (CNS), waarvan bekend is dat ze bloedbaaninfecties (BBI) en een hoog sterftecijfer veroorzaken, hoewel ze jarenlang vaak als `slechts' besmettelijk werden beschouwd. Bovendien worden CNS-en steeds vaker in verband gebracht met nosocomiale infecties. Momenteel bestaat de CNS-groep uit 45 verschillende species (soorten), hoewel het bepalen van het speciesniveau pas onlangs mogelijk gemaakt is voor routinediagnostische laboratoria. Sinds 2012 is namelijk MALDI-TOF-massaspectrometrie de standaard geworden voor de identificatie van bacteriële species zoals CNS. Hiervoor gebeurde de identificatie van CNS-en hoofdzakelijk met biochemische en fysiologische tests, die doorgaans variërende resultaten opleverden, in het bijzonder bij minder prevalente species. AMR, en met name multiresistentie, is ook een toenemend probleem bij CNS-en. Niettemin worden CNS-en in behandelrichtlijnen en nationale surveillanceprogramma's (zoals het Nederlandse NethMap) nog steeds als één groep beschouwd, zonder differentiatie tussen de species. Om deze reden is er weinig bekend over trends in het vóórkomen van, en AMR in, CNS-en op lokaal en regionaal niveau. Daarom toont deze retrospectieve studie een gedetailleerde AMR-analyse van bijna 20 duizend CNS-isolaten die gevonden waren in alle beschikbare 70 duizend bloedkweekisolaten tussen 2013 en 2019 in Noord-Nederland. Met deze analyse hebben we beoogd om de verschillen in het vóórkomen van CNS-species en hun AMR-patronen te evalueren en om hun klinisch microbiologische relevantie te beoordelen. In totaal werden 27 verschillende species van de CNS-groep gevonden. Er werden grote verschillen waargenomen in het vóórkomen van de verschillende species: de top vijf omvatte 97\% van alle geïncludeerde isolaten (S. epidermidis, S. hominis, S. capitis, S. haemolyticus en S. warneri). Het aandeel van CNS-en op de intensive care (IC) in vergelijking met andere afdelingen bleek ook significant te verschillen tussen tweedelijns zorg en derdelijns zorg. Omdat onbekend was welke patiënten een BBI hadden, werd `CNS-persistentie' gedefinieerd als een surrogaat waarvoor ten minste drie positieve bloedkweken afgenomen moesten zijn op drie verschillende dagen binnen 60 dagen, met dezelfde CNS, bij dezelfde patiënt. De relatief meest voorkomende veroorzaker van CNS-persistentie was S. haemolyticus, gevolgd door S. epidermidis en S. lugdunensis. AMR-analyse heeft aanzienlijke verschillen tussen CNS-species aangetoond. Zo vertoonden S. epidermidis en S. haemolyticus 50\% tot 80\% resistentie tegen de meeste antibiotica, terwijl de resistentie tegen deze middelen bij de meeste andere CNS-en lager dan 10\% bleef. Toch worden deze verschillen op nationaal niveau, zoals in NethMap, verwaarloosd, wat ertoe zou kunnen leiden dat bij de ontwikkeling van behandelrichtlijnen de nadruk wordt gelegd op veilige en vertrouwde middelen voor de behandeling van CNS, zoals vancomycine of linezolid. Niettemin kunnen middelen zoals tetracycline, cotrimoxazol en erythromycine als alternatieve opties worden beschouwd voor sommige species, waar volgens de studieresultaten de AMR nooit boven de 10\% is uitgekomen. Concluderend kan worden gesteld dat een meerjarige regio-totale benadering gebruikt is om de trends in zowel het vóórkomen als de AMR van CNS-species uitgebreid te beoordelen, wat kan worden gebruikt om het behandelingsbeleid te evalueren en meer te begrijpen over deze belangrijke maar nog steeds vaak niet serieus genomen pathogenen. Bovendien diende deze studie als een praktisch voorbeeld van hoe het AMR-pakket voor R kan worden gebruikt om nieuwe AMR-inzichten te verkrijgen met behulp van epidemiologisch onderbouwde methoden.

Als vervolg op de nieuwe inzichten door het bestuderen van AMR-testresultaten in Noord-Nederland, geeft \textbf{hoofdstuk 8} een vergelijking van nationale interpretaties van MDRO's in de Nederlands-Duitse grensregio, vooral wat betreft de praktische gevolgen voor grenspersoneel in de gezondheidszorg. Het vergelijken van AMR in het algemeen, niet alleen MDRO's, in deze grensoverschrijdende regio is bijzonder interessant omdat beide landen worden gekenmerkt door hoog ontwikkelde, maar desondanks structureel verschillende gezondheidszorgsystemen. AMR-interpretaties in patiëntendossiers worden overgedragen tussen zorginstellingen in deze twee verschillende landen, terwijl de onderliggende definities verschillen. Hierdoor moeten clinici en deskundigen infectiepreventie de AMR-resultaten van beide kanten van de grens begrijpen en in staat zijn om beide nationale MDRO-richtlijnen toe te kunnen passen. Door antibiogrammen van Gram-negatieve bacteriën uit beide kanten van de grens met elkaar te vergelijken, werd getracht de mate van impact van deze uitdagingen te bepalen. Hiertoe werden tussen 2015 en 2016 35.619 antibiogrammen van alle soorten Enterobacteriaceae, en P. aeruginosa, het A. baumannii-complex en Stenotrophomonas maltophilia uit zes Nederlandse en vier Duitse ziekenhuizen geanalyseerd. Voor al deze soorten bestaan in deze regio MDRO-aanbevelingen en speciale infectiepreventiemaatregelen. Uit de Nederlandse ziekenhuizen werden 12.616 antibiogrammen geselecteerd met behulp van het AMR-pakket voor R waarmee ook de Nederlandse MDRO-richtlijn toegepast kon worden. Van belang is dat andere nationale en internationale richtlijnen, zoals de Duitse MDRO-richtlijn, ook zijn opgenomen in het AMR-pakket voor R. Uit Duitse ziekenhuizen werden 23.003 antibiogrammen geselecteerd. Volgens de Nederlandse richtlijn was 25\% van alle isolaten een MDRO. Volgens de Duitse richtlijn was 13\% van alle isolaten een MDRO. Echter, van alle isolaten werd 74\% niet geclassificeerd als een MDRO volgens een van beide richtlijnen. Wanneer patiënten tussen ziekenhuizen worden overgebracht, moet ook informatie over MDRO-kolonisatie of -infectie worden overgedragen om de continue uitvoering van infectiepreventiemaatregelen te waarborgen. Voor grensoverschrijdende gezondheidszorg houdt dit in dat clinici of deskundigen infectiepreventie in staat moeten zijn MDRO's te bepalen op basis van antibiogrammen volgens de richtlijnen van een van beide landen. Voor grensoverschrijdende gezondheidszorg zou de eenvoudigste oplossing zijn de richtlijnen van beide landen te harmoniseren. Dit zou ook een oplossing bieden voor de begrijpelijke verwarring die patiënten zouden kunnen ondervinden wanneer infectiepreventiemaatregelen in het ene land worden opgelegd, maar na overplaatsing naar een ander land weer worden opgeheven. Zolang de harmonisatie niet is gerealiseerd, moeten de volledige AMR-gegevens samen met de patiënt worden overgedragen om classificatie voor lokale deskundigen infectiepreventie mogelijk te maken.

Andere AMR-gerelateerde grensoverschrijdende uitdagingen en verschillen worden geïllustreerd in \textbf{hoofdstuk 9}, dat een uitgebreide microbiële epidemiologische analyse omvat van het vóórkomen van MRSA, en het beleid en de gevolgen voor de gezondheidszorg in het Nederlands-Duitse grensgebied. MRSA is nog steeds een van de belangrijkste oorzaken van gezondheidszorg-geassocieerde infecties als gevolg van resistente ziekteverwekkers. In deze studie werden MRSA-surveillancegegevens van vijf jaar (2012-2016) van Nederlandse en Duitse grensoverschrijdende regioziekenhuizen geanalyseerd om regio-specifieke trends in de tijd van MRSA te beschrijven en om verschillen tussen ziekenhuizengroepen vast te stellen. De studie omvatte 42 ziekenhuizen in de Nederlands-Duitse grensregio met ongeveer 620.000 opgenomen patiënten (68,0\% in het Duitse deel van de onderzoeksregio) met bijna vier miljoen patiëntdagen per jaar. Alle ziekenhuizen hadden MRSA-gerelateerde infectiepreventiemaatregelen geïmplementeerd volgens hun nationale richtlijnen en aanbevelingen, en de verschillen in richtlijnen tussen de twee landen werden vergeleken. Aan beide zijden van de grens nam het MRSA-screeningspercentage tussen 2012 en 2016 aanzienlijk toe, hoewel de MRSA-incidentie in de loop van de tijd aan beide zijden van de grens stabiel bleef. In totaal was het screeningspercentage in de Duitse grensregio 14 keer hoger dan in de Nederlandse grensregio. Het percentage MRSA's in bloedkweekisolaten met S. aureus daalde van 13\% in 2012 tot 5\% in 2016 in de Duitse grensregio, terwijl het stabiel bleef in de Nederlandse grensregio (0\% tot 2\%). Niettemin was het ruwe aantal MRSA's onder S. aureus-isolaten 34 keer hoger in de Duitse grensregio. De ligduur in het ziekenhuis bij MRSA-patiënten was in beide regio's vergelijkbaar, terwijl de algemene ligduur aanzienlijk verschilde. Verder bedroeg het aantal MRSA-uitstrijken voor of bij opname in het ziekenhuis per 100 inwoners 12,2 in de Duitse grensregio en 0,36 in de Nederlandse grensregio; 34 keer zo hoog in de Duitse grensregio. Het aantal intramurale MRSA-gevallen per 1.000 inwoners bedroeg 2,52 in de Duitse grensregio en 0,14 in de Nederlandse grensregio. Dit onderzoek liet dus significante verschillen zien tussen Nederlandse en Duitse ziekenhuizen. De MRSA-incidentie in Duitse ziekenhuizen was meer dan zeven keer hoger dan in Nederlandse ziekenhuizen. Volgens het European Centre of Disease Prevention and Control (ECDC) worden verschillen in het vóórkomen van resistente ziekteverwekkers tussen Europese landen hoogstwaarschijnlijk veroorzaakt door verschillen in zorggebruik, antimicrobieel gebruik en infectiepreventiemaatregelen. Wat het zorggebruik in onze context betreft, stelden wij vast dat inwoners in het Duitse deel van het studiegebied bijna drie keer zo vaak in het ziekenhuis werden opgenomen en een aanzienlijk langere ligduur hadden dan patiënten in het Nederlandse deel. Dit kan te wijten zijn aan sociaaleconomische factoren of een andere inrichting van ambulante gezondheidszorg. Deze uitgebreide studie over MRSA in ziekenhuizen rond een Europese grens heeft aangetoond dat routinematige MRSA-surveillance nuttig kan zijn om trends van MRSA te volgen, om zodoende (inter)nationale vergelijkingen mogelijk te maken.

De discussie van deze studie werd afgesloten met (vertaald) ``grensoverschrijdende surveillance moet worden uitgebreid naar andere multiresistente micro-organismen'', wat precies is waar \textbf{hoofdstuk 10} op voortborduurt. Aangezien niet alleen MRSA's maar MDRO's in het algemeen een risico vormen voor de gezondheidszorg, zowel in de gemeenschap als in ziekenhuizen, had deze studie tot doel de prevalentie van meerdere MDRO's in deze grensoverschrijdende regio vast te stellen om verschillen te begrijpen en infectiepreventie te verbeteren op basis van real-time routinegegevens. Hiertoe namen 23 ziekenhuizen in de Nederlands-Duitse grensregio tussen 2017 en 2018 deel aan deze prospectieve studie door alle patiënten bij opname op de IC te screenen. Alle ziekenhuizen (8 in Nederland, 15 in Duitsland) screenden patiënten gedurende acht opeenvolgende weken op dragerschap van MRSA, vancomycineresistente Enterococcus faecium/E. faecalis (VRE), derde-generatie cefalosporine-resistente Enterobacteriaceae (3GCRE) en carbapenem-resistente Enterobacteriaceae (CRE). In totaal werden 3.365 patiënten gescreend: 36\% op Nederlandse IC's en 64\% op Duitse IC's. De mediane leeftijd van alle gescreende patiënten was 68 jaar (IQR: 57-77), waarbij patiënten in de Duitse grensregio significant ouder waren dan patiënten in de Nederlandse grensregio. De algemene screening compliance (gescreend op ten minste één MDRO-groep) was 60\%. Alle AMR-data-analyses werden uitgevoerd en geautomatiseerd met behulp van het AMR-pakket voor R. De prevalentie van MRSA was 1,7\% op Duitse IC's en 0,6\% op Nederlandse IC's. De prevalentie van VRE was 2,7\% op Duitse IC's en 0,1\% op Nederlandse IC's. Opmerkelijk is dat deze prevalentie varieerde van 0\% tot 4,1\% in het Duitse grensgebied. Alle 56 gevallen van VRE werden veroorzaakt door E. faecium. De prevalentie van 3GCRE was 6,6\% op Duitse IC's en 3,6\% op Nederlandse IC's, terwijl de prevalentie voor CRE aan beide zijden van de grens nagenoeg nihil was. De prevalentie voor Gram-negatieve MDRO's verschilde tussen ziekenhuizen in beide landen, variërend van 0\% tot 5,0\% in de Nederlandse grensregio en van 1,2\% tot 10,9\% in de Duitse grensregio. Voor de geïncludeerde Nederlandse IC's was de prevalentie van alle MDRO-groepen niet significant verschillend tussen tweedelijns en derdelijns ziekenhuizen. Voor de Duitse IC's was de prevalentie van Gram-negatieve MDRO's echter significant hoger in de tweedelijns ziekenhuizen. In de Nederlandse grensregio leidde 4,8 per 100 ziekenhuisopnamen tot een IC-opname. In de Duitse grensregio was dit daarentegen 7,7 per 100 ziekenhuisopnames. Dit verschil kan worden verklaard door de hogere IC-capaciteit in Duitse ziekenhuizen (4,8\% van alle ziekenhuisbedden) in vergelijking met Nederlandse ziekenhuizen (2,4\% van alle ziekenhuisbedden). De algemene prevalentie van de verschillende MDRO's was hoger op de Duitse IC's, hoewel sommige verschillen marginaal waren. Met name de prevalentie van MRSA was drie keer hoger in de Duitse grensregio dan in de Nederlandse grensregio, wat consistent is met de onderzoeksresultaten in \textbf{hoofdstuk 9}. Toch waren de onderzoeksresultaten niet consistent met (inter)nationale gemiddelden. Zo was de 3GCRE-prevalentie bijna twee keer zo hoog in de Duitse grensregio als in de Nederlandse grensregio, maar beide waren nog steeds lager dan de nationale gemiddelden; de ECDC meldde 6\% hogere 3GCRE-percentages onder E. coli en K. pneumoniae uit bloedkweken voor Duitsland en Nederland. Hieruit blijkt dat er belangrijke verschillen zijn tussen het bestuderen van dragerschap in bepaalde populaties en het bestuderen van het aandeel van (waarschijnlijk) invasieve isolaten. Deze studie benadrukt daarmee het belang van een regionale en grensoverschrijdende aanpak in een Europese grensregio, om het verschil in AMR-prevalentie tussen de regio's te illustreren en om potentiële verschillen met nationale rapporten te belichten. Om dit verder te kunnen uitwerken is een dieper detailniveau nodig, bijvoorbeeld door informatie te verzamelen over (infectiepreventie)personeel, MDRO-uitbraken, infecties, antibioticagebruik en risicofactoren van patiënten. Concluderend lijken geografische en politieke grenzen door MDRO's niet te worden ``gerespecteerd'', hoewel de gezondheidszorgsystemen, de geografische aard en de richtlijnen van land tot land sterk verschillen. De percentages MDRO's van bepaalde ziekteverwekkers, zoals gerapporteerd op nationaal en internationaal niveau, weerspiegelen niet de prevalentie van MDRO's in de patiënt of in de algemene bevolking. Dit moet ernstig in overweging worden genomen bij de interpretatie van rapporten op nationaal of zelfs continentaal niveau.

\textbf{Conclusie}

Vanuit verschillende invalshoeken worden het AMR-pakket en zijn voordelen in perspectief geplaatst: vanuit een technisch perspectief, vanuit het perspectief van infectiemanagement en vanuit een klinisch perspectief. Deze combinatie biedt een gemeenschappelijke basis voor het begrijpen van de oplossingen die het AMR-pakket kan bieden en hoe het een nieuw startpunt kan vormen voor toekomstige toepassingen van microbiële epidemiologie, zowel in klinische settings als in wetenschappelijk onderzoek. Dit proefschrift gaat vervolgens in op deze verschillende invalshoeken door het gebruik van dit nieuwe instrument te illustreren in epidemiologische studies in de Nederlands-Duitse grensregio om het vóórkomen en de AMR-trends van micro-organismen op (eu)regionaal niveau beter te begrijpen. Concluderend toont dit proefschrift de toegevoegde waarde aan van een consistent data-analytisch instrument om AMR-data voor te bereiden en te analyseren in een regio-overstijgende benadering, om nieuwe inzichten te verkrijgen in AMR-trends.

\hypertarget{zusammenfassung-auf-deutsch}{%
\chapter*{Zusammenfassung auf Deutsch}\label{zusammenfassung-auf-deutsch}}
\addcontentsline{toc}{chapter}{Zusammenfassung auf Deutsch}

\emph{Ein wichtiger Teil dieser Dissertation (insbesondere Abschnitt III) wurde durch die sehr gute und vor allem herzliche Zusammenarbeit mit deutschen Kollegen:innen ermöglicht. Diese Zusammenfassung ist eine freundliche Geste an meine deutschen Kollegen:innen.}

\textbf{Abschnitt I}

Wo liegen die Anfänge der mikrobiellen Epidemiologie? Wie ist sie entstanden? Und wie trägt sie zu einem umfassenden Ansatz in der Behandlung von Patient:innen mit Infektionen bei? Diese Fragen werden in diesem ersten Abschnitt beantwortet. Anschließend werden die wichtigsten derzeitigen Hindernisse bei der Anwendung der mikrobiellen Epidemiologie in der Praxis beschrieben und wie diese überwunden werden könnten.

Die allgemeine Einleitung dieser Dissertation skizziert in \textbf{Kapitel 1}, dass die mikrobielle Epidemiologie ein Teil der Infektionsepidemiologie ist, die wiederum ein Teil der klinischen Mikrobiologie ist. Die mikrobielle Epidemiologie kann unter anderem als das wissenschaftliche Feld zur Gewinnung neuer Erkenntnisse über sich ausbreitende Mikroorganismen und deren jeweilige Muster der antimikrobiellen Resistenz (AMR) gesehen werden. Die Fortschritte in der Informationstechnologie haben uns nicht nur die Möglichkeiten gebracht, über regionale, nationale und internationale Grenzen hinweg zu schauen, um ein Verständnis für die Ausbreitung von Mikroorganismen und AMR zu bekommen, sondern sogar Pandemien in Echtzeit zu beobachten, zu analysieren und zu verstehen. Methoden, die wir heute entwickeln und anwenden, können morgen auf der anderen Seite der Welt eingesetzt werden. Dies ist ein wichtiger Vorteil in der modernen mikrobiellen Epidemiologie, deren Fokus zunehmend datengetriebener wird.

Um diesen Fokus voranzutreiben, sind Daten die wichtigste Voraussetzung. Die Daten, die als Input für mikrobielle epidemiologische Analysen verwendet werden, werden häufig aus Laborinformationssystemen (LIS) gewonnen. Diese Daten bestehen aus Routine-diagnoseergebnissen von Labortests. In \textbf{Kapitel 2} wird die Ansicht erörtert, dass die Diagnostik zwar zu Rohdaten führt, aber nicht zu einer direkten Antwort auf die klinische Frage, die ein Arzt, der einen Patient:innen behandelt, haben könnte. Um Ärzten Antworten zu geben, ist der Ansatz eines multidisziplinären, ineinandergreifenden ‚Stewardship'-Konzepts mit Schwerpunkt auf der Diagnostik erforderlich. Dies erfordert ein enges Zusammenspiel von Fachärzten im Allgemeinen und Mikrobiologen im Besonderen für eine optimale Versorgungsqualität und Patientensicherheit um erfolgreiches Infektions-management ausführen zu können: Diagnostic Stewardship Programme (DSP). Das Konzept des Stewardships wurde im Allgemeinen weithin verwendet, um die Kommunikation und die klinische Entscheidungsfindung zu erleichtern, wobei es sich als schwierig erwies, eine klare Definition des Begriffs ``Stewardship'' festzulegen. Darüber hinaus macht die Diagnostik in klinisch-mikrobiologischen Laboratorien derzeit rasante Fortschritte im Hinblick auf verbesserte Arbeitsabläufe und neue Technologien, wie z. B. die matrixunterstützte Laser-Desorptions/Ionisations-Time-of-Flight (MALDI-TOF) Massenspektrometrie. Die Diagnostik im Infektionsmanagement ist jedoch breiter angelegt und umfasst viele klinische Bereiche, in denen Kommunikation und Interaktion von grundlegender Bedeutung sind, um das Wissen und die Expertise optimal zu nutzen, was dazu führt, dass alle Fachrichtungen einen Beitrag zur Patientenversorgung leisten. Der richtige Test zur richtigen Zeit für den richtigen Patient:innen, um die richtigen Fragen zu beantworten und die richtige Behandlung einzuleiten - darum geht es beim DSP in der klinischen Mikrobiologie. Die mikrobielle Epidemiologie kann für einen kleinen Aspekt dieser diagnostischen Gesamtheit genutzt werden, indem die Testergebnisse wiederverwertet werden und anschließend Anreicherungen in den Prozess zur Generierung von Antworten einbringen, den DSP verkörpert.

Hier setzt \textbf{Kapitel 3} an, indem es wichtige aktuelle Einschränkungen bei der Anwendung der mikrobiellen Epidemiologie, insbesondere der AMR-Datenanalyse, hervorhebt. Insbesondere muss die AMR-Datenanalyse auf eine klinisch und epidemiologisch sinnvolle Weise durchgeführt werden, was jedoch eine Herausforderung darstellt, da man dafür Fachwissen in (klinischer) Epidemiologie und (klinischer) Mikrobiologie sowie Werkzeuge zur Handhabung der AMR-Datenanalyse benötigt. Dies wird zusätzlich durch die häufig fehlende Zugänglichkeit der in LIS-es gespeicherten Daten erschwert, da die meisten LIS-es nicht mit einem Fokus auf Epidemiologie konzipiert sind. So führt beispielsweise jedes LIS seine eigenen taxonomischen Daten und die Labore sind für deren regelmäßige Aktualisierung verantwortlich. Da die AMR-Richtlinien stark auf der mikrobiellen Taxonomie basieren (einige Regeln gelten nur für eine bestimmte Gattung, andere Regeln gelten für eine bestimmte Familie), müssen diese Informationen korrekt und aktuell sein. Leider wurde bei der Untersuchung von sieben klinischen Mikrobiologie-Laboren in den Niederlanden festgestellt, dass alle ihre LIS-es stark veraltete taxonomische Namen enthielten. Dies kann sowohl die routinemäßige Ergebnismeldung als auch (zukünftige) epidemiologische Analysen beeinträchtigen. Aus diesen Gründen wurde in diesem Kapitel das AMR-Paket für R, eine Programmiersprache für statistische Berechnungen, als neues epidemiologisches Instrument zur AMR-Datenanalyse vorgestellt. Das AMR-Paket ist kostenlos, unabhängig, Open Source und öffentlich zugänglich. Es wurde mit einem Team aus zwölf verschiedenen Organisationen des öffentlichen Gesundheitswesens in sieben verschiedenen Nationen entwickelt und bietet Werkzeuge zur Vereinfachung der AMR-Datenbereinigung, -transformation und -analyse sowie Methoden zur einfachen Einbindung (inter)nationaler Richtlinien und wissenschaftlich zuverlässiger Referenzdaten. Mit Stand Mai 2021 wurde das AMR-Paket seit seiner ersten Veröffentlichung im Jahr 2018 mindestens 50.000-mal aus 162 verschiedenen Nationen heruntergeladen. Die Ergebnisse einer Umfrage unter den Nutzern, die in diesem Kapitel vorgestellt werden, zeigten, dass seine Verwendung zu einer besseren Reproduzierbarkeit von Analyseergebnissen, verlässlicheren Ergebnissen von AMR-Datenanalysen und neuen oder verbesserten Erkenntnissen zu AMR für die Institutionen und Regionen der Nutzer führte. Die Anwender gaben auch an, dass das AMR-Paket zur Unterstützung der klinischen Entscheidungsfindung eingesetzt wurde. Das Paket löst die Unannehmlichkeit, von (inter)nationalen Richtlinien und zuverlässigen (Referenz-)Daten abhängig zu sein, und bietet gleichzeitig eine umfassende Toolbox für die Analyse selbst. Das AMR-Paket für R kann daher jeden Spezialisten auf dem Gebiet, der mit AMR-Daten arbeitet, unterstützen.

\textbf{Abschnitt II}

Nach den im vorherigen Abschnitt beschriebenen Herausforderungen wird in diesem Abschnitt das AMR-Paket für R als neues Instrument zur Bewältigung dieser Herausforderungen vorgestellt. Das AMR-Paket und seine Vorteile werden aus verschiedenen Blickwinkeln betrachtet: aus technischer Sicht, aus Sicht des Infektionsmanagements und aus klinischer Sicht. Diese Kombination bietet eine gemeinsame Grundlage für das Verständnis der Erklärungen, die das AMR-Paket im Feld liefern kann und wie es einen neuen Ausgangspunkt für zukünftige Anwendungen der mikrobiellen Epidemiologie setzen kann.

Die technischen Funktionalitäten des AMR-Pakets für R wurden in \textbf{Kapitel 4} beschrieben. Dort wird beschrieben, wie das AMR-Paket entwickelt wurde, um saubere und reproduzierbare AMR-Datenanalysen unter Verwendung internationaler Richtlinien zu standardisieren. Um dies zu ermöglichen, werden wissenschaftlich verlässliche Laborreferenzdaten, antimikrobieller Wirkstoffe und der vollständigen biologischen Taxonomie der Mikroorganismen einbezogen. Die Quelldaten sollten so zuverlässig wie möglich analysiert werden, vor allem, wenn die Ergebnisse z. B. zur Bewertung von Behandlungsoptionen für Patient:innen herangezogen werden sollen. Dies erfordert eine reproduzierbare und feldspezifische, spezialisierte Datenbereinigung und -transformation. Das AMR-Paket bietet eine standardisierte und automatisierte Möglichkeit, allgemeine LIS-Daten zu bereinigen, zu transformieren und zu verbessern, unabhängig von der zugrunde liegenden Datenquelle und der Datengenauigkeit. Aus diesem Grund wurden allgemeine Algorithmen zur Bereinigung von AMR-Daten und zur Validierung der Namen von Mikroorganismen und antimikrobiellen Wirkstoffen entwickelt. Die Gleichung zur taxonomischen Namensvalidierung berücksichtigt die humanpathogene Prävalenz von Mikroorganismen und ist kontextbewusst über andere taxonomische Eigenschaften wie das Königreich, Phylum, Ordnung und Familie. So wird z. B. ein Datenwert ``E. coli'' in das Bakterium Escherichia coli übersetzt, während der Benutzer darüber informiert wird, dass der Parasit Entamoeba coli ebenfalls in Frage kommt, aber eine geringere Wahrscheinlichkeit hat. Mit Hilfe einfacher Funktionen können Benutzer schnell konsistente mikrobielle Eigenschaften abrufen, wie z. B. das taxonomische Königreich, die Familie, Gattung, Art, früher akzeptierte Namen und sogar die Gram-Färbung. Neben den Informationen über Mikroorganismen enthält das Paket auch Referenzdaten über Antibiotika, die gängige Laborinformationssystem-Codes, offizielle Wirkstoffnamen, ATC-Codes (Anatomical Therapeutic Chemical), definierte Tagesdosen (DDD) und mehr als 5.000 Handelsnamen von 456 antimikrobiellen Wirkstoffen umfassen. Mit Hilfe dieser Referenzdaten können Anwender Rohdaten übersetzen und Eigenschaften über jeden Mikroorganismus oder antimikrobiellen Wirkstoff abrufen. Darüber hinaus ist das AMR-Paket in der Lage, multiresistente Organismen (multi-drug resistant organisms, MDROs) auf der Grundlage nationaler und internationaler Richtlinien zu bestimmen, minimale Hemmkonzentrationen (MHKs) zu interpretieren und erste Isolate zu bestimmen, die für die Berechnung der AMR sowohl von Monotherapien als auch von Kombinationstherapien verwendet werden können. Das AMR-Paket selbst war als umfassendes Instrument für datentechnisches Personal gedacht, das auf dem Gebiet der AMR arbeitet, obwohl seine Verwendung nicht auf diese Gruppe beschränkt ist.

Um dies zu veranschaulichen, zeigt \textbf{Kapitel 5}, dass das AMR-Paket als Rückgrat in einer interaktiven Open-Source-Software-App für Infektionsmanagement und antimikrobiellem Stewardship, genannt RadaR (Rapid Analysis of Diagnostic and Antimicrobial patterns in R; schnelle Analyse von diagnostischen und antimikrobiellen Mustern in R), verwendet wurde. Infektionsmanagement in Form von Antimicrobial Stewardship (AMS)-Programmen hat sich als effektive Lösung herausgestellt, um dieses globale Gesundheitsproblem in Krankenhäusern anzugehen. Anknüpfend an \textbf{Kapitel 2} konzentrieren sich Stewardship-Interventionen und -Aktivitäten sowohl auf einzelne Patient:innen (personalisierte Medizin und Beratung) als auch auf Patientengruppen oder klinische Syndrome (Richtlinien, Protokolle, informations-technische Infrastruktur und klinische Entscheidungsunterstützungssysteme), wobei die Verbesserung der Versorgungsqualität und der Patientensicherheit bei jeder Intervention im Vordergrund steht. Der einfache Zugang zur Analyse von Patientengruppen (z. B. stratifiziert nach Abteilungen oder Stationen, spezifischen antimikrobiellen Mitteln oder verwendeten diagnostischen Verfahren) ist jedoch in der täglichen Praxis schwer umzusetzen. Noch schwieriger ist es, größere Patientenpopulationen (z. B. über mehrere Fachrichtungen verteilt) schnell zu analysieren, auch wenn diese Informationen in den Daten vorhanden sein könnten. Daher war die Entwicklung von RadaR darauf ausgerichtet, AMS-Teams eine benutzerfreundliche und zeitsparende Datenanalyseressource zur Verfügung zu stellen, ohne dass tiefgreifende technische Fachkenntnisse erforderlich sind. RadaR wurde für die grafische explorative (AMR) Datenanalyse entwickelt. Es bietet unter anderem Kaplan-Meier-Kurven über die Verweildauer im Krankenhaus, Zeittrends für die Anzahl der Aufnahmen, den Verbrauch von antimikrobiellen Mitteln und eine automatisierte AMR-Datenanalyse, für die das AMR-Paket für R verwendet wurde. RadaR wurde von 12 ESGAP-Mitgliedern (European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship) aus 9 verschiedenen Nationen evaluiert. Es hat das Potenzial, ein sehr nützliches Werkzeug für Infektionsmanagement und AMS-Teams in der täglichen Praxis zu sein. Zusätzlich zeigt dieses Kapitel, dass das AMR-Paket als Teil einer anderen Softwarelösung verwendet werden kann, um ein integriertes Infektionsmanagement zu ermöglichen.

Nach dieser Erkenntnis wird in \textbf{Kapitel 6} die Effektivität des AMR-Pakets bei den Anwendern demonstriert, indem die Benutzerfreundlichkeit und die Auswirkungen auf die Arbeitsabläufe der Kliniker:innen in einem typischen Krankenhausszenario bewertet werden. Obwohl die Verwendung des AMR-Pakets in der Forschung bereits in mehreren Studien aus verschiedenen Nationen nachgewiesen wurde, stand die Auswirkung auf die Arbeitsabläufe für die AMR-Datenanalyse und -berichterstattung in klinischen Umgebungen noch aus. AMR-Datenanalyse und -reporting erfordern speziell geschultes Personal. Darüber hinaus können gründliche und tiefgehende Analysen zeitaufwendig sein und es müssen ausreichend Ressourcen für eine konsistente und wiederholte Berichterstattung bereitgestellt werden. Um die Auswirkungen dieser Tatsachen in einem klinischen Umfeld zu ermitteln, wurden allgemeine Fragen zu Blutkulturdaten formuliert, die von klinischem Personal, einschließlich klinischen Mikrobiologen:innen, Pädiater:innen und Intensivmedizinern:innen, beantwortet werden mussten. Insgesamt nahmen zehn Kliniker:innen an der Studie teil. Zusätzlich wurden die Teilnehmer:innen gebeten, einen Online-Fragebogen auszufüllen, in dem ihr Hintergrund, ihre demografischen Daten, ihre Software-Erfahrung und ihre Erfahrung mit der Analyse und Berichterstattung von AMR-Daten erfasst wurden. Alle Teilnehmer:innen mussten die Studienfragen zweimal beantworten: das erste Mal mit der Software ihrer Wahl (Runde 1) und das zweite Mal mit einer neu entwickelten Webanwendung, die auf dem AMR-Paket für R aufbaut (Runde 2). Die Entwicklung dieser Webanwendung wurde in einem hocheffizienten und agilen Workflow ausgeführt. Die Antworten auf den Fragenkatalog dienten als Grundlage, um die Effektivität (Lösbarkeit jeder Aufgabe für jeden Benutzer) und Effizienz (Zeitaufwand für die Lösung jeder Aufgabe) zwischen den beiden Runden zu vergleichen. Nicht alle Teilnehmer waren in der Lage, die Aufgaben innerhalb des vorgegebenen Zeitrahmens zu lösen. Die durchschnittliche Aufgabenerfüllung zwischen der ersten und zweiten Runde stieg von 56\% auf 96\% und der Anteil der richtigen Antworten stieg von 38\% auf 98\%. Der mittlere Zeitaufwand pro Runde wurde mit mehr als einer Stunde reduziert. Dieses Kapitel zeigt die erhöhte Effektivität, Effizienz und Genauigkeit der Verwendung des AMR-Pakets für R zur AMR-Datenanalyse im Vergleich zu herkömmlichen Softwareanwendungen wie Microsoft Excel und SPSS.

\textbf{Abschnitt III}

Viele klinische Studien auf dem Gebiet der Infektionskrankheiten und der Mikrobiologie stützen sich auf eine Form der (mikrobiellen) Epidemiologie. Während das AMR-Paket im vorherigen Abschnitt vorgestellt und seine Verwendung in verschiedenen Umgebungen gezeigt wurde, beginnt dieser Abschnitt mit einem epidemiologischen Forschungsprojekt in der nordniederländischen Region und dehnt sich dann auf die niederländisch-deutsche Grenzregion aus, um das Auftreten und die AMR-Muster von Krankheitserregern auf (eu)regionaler Ebene zu verstehen. Die Fokussierung auf die Regionen auf beiden Seiten einer nationalen Grenze ermöglicht Vergleiche zwischen zwei verschiedenen Nationen auf der Mikroebene. Und unterschiedliche Nationen bedeuten letztlich unterschiedliche Gesundheitssysteme. Was bleibt von „One Health`` übrig? Welche Auswirkungen hat es auf den Vergleich, wenn sich die Nationen in Bezug auf AMR-Testmethoden, MDRO-Interpretationen und Screening-Richtlinien unterscheiden? Dieser Abschnitt gibt Antworten auf diese Fragen.

\textbf{Kapitel 7} befasst sich mit Koagulase-negativen Staphylokokken (KNS), die bekanntermaßen Blutbahninfektionen (BSI) und eine hohe Sterblichkeitsrate verursachen, obwohl sie jahrelang oft als Kontamination angesehen wurden. Außerdem werden KNS zunehmend mit nosokomialen Infektionen in Verbindung gebracht. Derzeit besteht die Gruppe der KNS aus 45 verschiedenen Spezies, wobei die Bestimmung der Spezies-Ebene erst seit kurzem für diagnostische Routinelabore möglich ist. Seit 2012 ist die MALDI-TOF-Massenspektrometrie zum Standard für die Identifizierung von Bakterienarten wie KNS geworden. Davor erfolgte die Identifizierung von KNS vor allem mit biochemischen und physiologischen Tests, die insbesondere bei weniger verbreiteten Spezies generell variable Ergebnisse lieferten. AMR, und insbesondere Multiresistenz, ist auch bei KNS ein zunehmendes Problem. Dennoch erfassen Behandlungsrichtlinien und nationale Überwachungsprogramme (wie z. B. das niederländische NethMap) KNS immer noch als eine Gesamtgruppe, wobei eine Differenzierung zwischen den Arten fehlt. Folglich ist wenig über Trends im Auftreten und AMR bei KNS auf lokaler und regionaler Ebene bekannt. Daher zeigt diese retrospektive Studie eine eingehende AMR-Analyse von fast 20 Tausend KNS-Isolaten, die in allen verfügbaren 70 Tausend Blutkulturisolaten zwischen 2013 und 2019 in den nördlichen Niederlanden gefunden wurden. Diese Studie folgte einem flächendeckenden Ansatz, indem sie die gesamten nördlichen Niederlande abdeckte. Ziel dieser Analyse war es, die Unterschiede im Vorkommen von KNS-Spezies und deren AMR-Muster zu bewerten und die klinisch-mikrobiologische Relevanz zu beurteilen. Insgesamt wurden 27 verschiedene Spezies der KNS-Gruppe gefunden. Es wurden große Unterschiede im Vorkommen der verschiedenen Spezies beobachtet: Die fünf wichtigsten Spezies deckten 97\% aller eingeschlossenen Isolate ab (S. epidermidis, S. hominis, S. capitis, S. haemolyticus und S. warneri). Der Anteil von KNS auf Intensivstationen (ICUs) im Vergleich zu anderen Abteilungen unterschied sich ebenfalls signifikant zwischen der Sekundärversorgung und der Tertiärversorgung. Da nicht bekannt war, bei welchen Patient:innen BSI auftraten, wurde ``KNS-Persistenz'' als Surrogat für mindestens drei positive Blutkulturen definiert, die an drei verschiedenen Tagen innerhalb von 60 Tagen gezogen wurden und dieselbe KNS-Spezies enthielten, und zwar bei demselben Patient:innen. Der relativ häufigste Erreger von KNS-Persistenz war S. haemolyticus, gefolgt von S. epidermidis und S. lugdunensis. Die AMR-Analyse zeigte erhebliche Unterschiede zwischen den KNS-Spezies. Zum Beispiel zeigten S. epidermidis und S. haemolyticus 50\% bis 80\% Resistenz gegen die meisten Antibiotika, während die Resistenz gegen diese Mittel bei den meisten anderen KNS-Spezies unter 10\% lag. Dennoch werden diese Unterschiede auf nationaler Ebene wie in NethMap vernachlässigt, was dazu führen könnte, dass sich die Entwicklung von Behandlungsrichtlinien auf „AMR-sichere`` Wirkstoffe zur Behandlung von KNS konzentriert, wie z. B. Vancomycin oder Linezolid. Nichtsdestotrotz könnten Wirkstoffe wie Tetracyclin, Co-Trimoxazol und Erythromycin als brauchbare Optionen für einige Spezies angesehen werden, bei denen die AMR laut den Studienergebnissen nie mehr als 10\% betrug. Zusammenfassend lässt sich sagen, dass ein mehrjähriger, flächendeckender Ansatz zur umfassenden Bewertung der Trends sowohl des Auftretens als auch der AMR von KNS-Spezies durchgeführt wurde, der zur Bewertung von Behandlungsstrategien und zum besseren Verständnis dieser wichtigen, aber immer noch zu oft vernachlässigten Krankheitserreger genutzt werden konnte. Darüber hinaus diente diese Studie als praktisches Forschungsbeispiel dafür, wie das AMR-Paket für R genutzt werden kann, um mit epidemiologisch fundierten Methoden neue Erkenntnisse über AMR zu gewinnen.

Nach neuen Erkenntnissen durch die Untersuchung von AMR-Testergebnissen in den nördlichen Niederlanden bietet \textbf{Kapitel 8} einen Vergleich von ihren nationalen Interpretationen von MDROs in der deutsch-niederländischen Grenzregion, insbesondere hinsichtlich der praktischen Auswirkungen auf das grenzüberschreitende Gesundheitspersonal. Der Vergleich von AMR im Allgemeinen, nicht nur MDROs, in dieser grenzüberschreitenden Region ist besonders interessant, da beide Nationen durch hoch entwickelte, aber strukturell unterschiedliche Gesundheitssysteme gekennzeichnet sind. AMR-Interpretationen in Patientenakten werden zwischen Gesundheitseinrichtungen in diesen beiden unterschiedlichen Nationen übertragen, während die zugrunde liegenden MDRO Definitionen unterschiedlich sind. Daraus ergibt sich die Notwendigkeit für Kliniker:innen und Hygienespezialist:innen, AMR-Ergebnisse von beiden Seiten der Grenze zu verstehen und in der Lage zu sein, beide nationalen MDRO-Interpretationsrichtlinien nachzuvollziehen. Durch den Vergleich von Antibiogrammen Gram-negativer Bakterien von beiden Seiten der Grenze wurde versucht, den Grad der Auswirkungen dieser Herausforderungen zu bestimmen. Zu diesem Zweck wurden 35 Tausend Antibiogramme aus sechs niederländischen und vier deutschen Krankenhäusern zwischen 2015 und 2016 von allen Arten von Enterobacteriaceae sowie P. aeruginosa, dem A. baumannii-Komplex und Stenotrophomonas maltophilia analysiert. Für alle diese Spezies gibt es in dieser Region MDRO-Empfehlungen und spezielle Hygienemaßnahmen. Aus den niederländischen Krankenhäusern wurden Antibiogramme mit Hilfe des AMR-Pakets für R unter Anwendung der niederländischen MDRO-Interpretationsleitlinie ausgewählt. Es sei darauf hingewiesen, dass auch andere nationale und internationale Richtlinien, wie die deutsche MDRO-Interpretationsrichtlinie, im AMR-Paket für R enthalten sind. Nach der niederländischen Leitlinie waren 25\% aller Isolate ein MDRO. Nach der deutschen Leitlinie waren 13\% aller Isolate ein MDRO. Von allen Isolaten wurden jedoch 74\% nach keiner der beiden Richtlinien als MDRO eingestuft. Wenn Patient:innen zwischen Krankenhäusern verlegt werden, müssen auch Informationen über MDRO-Besiedlung oder -Infektion übertragen werden, um eine kontinuierliche Umsetzung von Infektionskontrollmaßnahmen zu gewährleisten. Für die grenzüberschreitende Gesundheitsversorgung bedeutet dies, dass Kliniker:innen oder Hygienespezialist:innen in der Lage sein sollten, MDROs anhand von Antibiogrammen gemäß den Richtlinien beider Nationen zu bestimmen. Für die grenzüberschreitende Gesundheitsversorgung wäre die einfachste Lösung, die Definitionen der beiden Nationen zu harmonisieren. Dies könnte auch die verständliche Verwirrung lösen, die bei Patient:innen auftreten kann, wenn in einem Land Maßnahmen zur Infektionsprävention auferlegt werden, diese aber nach der Verlegung in ein anderes Land wieder aufgehoben werden. Solange die Harmonisierung nicht erfolgt ist, sollten die vollständigen AMR-Daten von Gram-negativen Bakterien zusammen mit dem Patient:innen übertragen werden, um eine Klassifizierung durch das lokale Infektionskontrollpersonal zu ermöglichen.

Weitere AMR-bedingte grenzüberschreitende Herausforderungen und Unterschiede werden in \textbf{Kapitel 9} veranschaulicht, das eine umfassende mikrobielle epidemiologische Analyse des MRSA-Vorkommens, der Maßnahmen und der Auswirkungen auf das Gesundheitswesen in der deutsch-niederländischen Grenzregion umfasst. MRSA ist immer noch eine der Hauptursachen für therapieassoziierte Infektionen aufgrund von AMR-Erregern. In dieser Studie wurden MRSA-Surveillance-Daten von fünf Jahren (2012-2016) aus Krankenhäusern der niederländischen und deutschen Grenzregion analysiert, um zeitliche und räumliche Trends der MRSA-Raten zu beschreiben und Unterschiede zwischen diesen Gruppen von Krankenhäusern zu finden. Das Forschungssetting umfasste 42 Krankenhäuser in der deutsch-niederländischen Grenzregion, die etwa 620.000 aufgenommene Patient:innen (68\% im deutschen Teil der Studienregion) mit fast vier Millionen Patiententagen pro Jahr behandelten. Alle Krankenhäuser hatten MRSA-bezogene Maßnahmen zur Infektionsprävention entsprechend ihren nationalen Richtlinien und Empfehlungen implementiert, und die Richtlinienunterschiede zwischen den beiden Nationen wurden verglichen. Auf beiden Seiten der Grenze stieg die MRSA-Screening-Rate zwischen 2012 und 2016 signifikant an, obwohl die MRSA-Inzidenz auf beiden Seiten der Grenze im Zeitverlauf stabil blieb. Insgesamt war die Screening-Rate in der deutschen Grenzregion 14-mal höher als in der niederländischen Grenzregion. Der Teil von MRSA in S. aureus-Blutkulturisolaten sank von 13\% im Jahr 2012 auf 5\% im Jahr 2016 in der deutschen Grenzregion, aber blieb stabil in der niederländischen Grenzregion (0\% bis 2\%). Dennoch war die Anzahl von MRSA unter den S. aureus-Isolaten in der deutschen Grenzregion 34-mal höher. Die Länge des Krankenhausaufenthalts von MRSA-Patient:innen war in beiden Regionen ähnlich, während sich die allgemeine Länge signifikant unterschied. Außerdem war die Anzahl der MRSA-Screening-Abstriche vor oder bei der Aufnahme ins Krankenhaus 12,2 pro 100 Einwohner in der deutschen Grenzregion und 0,36 in der niederländischen Grenzregion, ebenfalls 34-mal höher in DE-BR. Die Anzahl der stationären MRSA-Fälle pro 1.000 Einwohner lag in der deutschen Grenzregion bei 2,52 und in der niederländischen Grenzregion bei 0,14. Somit zeigte diese Studie signifikante Unterschiede zwischen niederländischen und deutschen Krankenhäusern. Die MRSA-Inzidenz in deutschen Krankenhäusern war mehr als siebenmal höher als in niederländischen Krankenhäusern. Nach Angaben des Europäischen Zentrums für die Prävention und die Kontrolle von Krankheiten (ECDC) werden Unterschiede im Auftreten von AMR-Erregern zwischen den europäischen Nationen höchstwahrscheinlich durch Unterschiede in der Nutzung des Gesundheitswesens, der Verwendung von antimikrobiellen Mitteln und den Praktiken zur Infektionsprävention verursacht. In Bezug auf die Inanspruchnahme des Gesundheitswesens in unserem Kontext stellten wir fest, dass die Bewohner im deutschen Teil der Studienregion fast dreimal so häufig ins Krankenhaus eingeliefert wurden und eine signifikant längere Länge des Krankenhausaufenthalts hatten als die Patient:innen im niederländischen Teil. Dies könnte auf sozioökonomische Faktoren oder eine unterschiedliche Organisation der ambulanten Gesundheitsversorgung zurückzuführen sein. Diese umfassende Studie zu MRSA, die Krankenhäuser über eine europäische Grenze hinweg abdeckt, hat gezeigt, dass eine routinemäßige MRSA-Surveillance hilfreich sein kann, um Trends von MRSA-Parametern zu überwachen, um (inter)nationale Vergleiche zu ermöglichen.

Die Diskussion dieser Studie schloss mit (übersetzt) „die grenzüberschreitende Überwachung sollte auf andere multiresistente Organismen ausgeweitet werden``, womit \textbf{Kapitel 10} fortgesetzt wird. Da nicht nur MRSA, sondern MDROs im Allgemeinen ein Risiko für die Gesundheitsversorgung darstellen, sowohl in der Allgemeinbevölkerung als auch in Krankenhäusern, zielte die Studie darauf ab, die Prävalenz mehrerer MDROs in dieser grenzüberschreitenden Region zu bestimmen, um Unterschiede zu verstehen und die Infektionsprävention auf der Grundlage von Echtzeit-Routinedaten und Arbeitsabläufen zu verbessern. Zu diesem Zweck nahmen 23 Krankenhäuser in der deutsch-niederländischen Grenzregion zwischen 2017 und 2018 an dieser prospektiven Studie teil, indem sie alle Patient:innen bei der Aufnahme auf Intensivstationen (ICUs) screenten. Alle Krankenhäuser (8 in den Niederlanden, 15 in Deutschland) nahmen für acht aufeinanderfolgende Wochen an der Studie teil und untersuchten die Patient:innen auf die Kolonisierung von MRSA, Vancomycin-resistenten Enterococcus faecium/E. faecalis (VRE), Cephalosporin-resistenten Enterobacteriaceae der dritten Generation (3GCRE) und Carbapenem-resistenten Enterobacteriaceae (CRE). Insgesamt wurden 3.365 Patient:innen gescreent: 36\% auf niederländischen Intensivstationen und 64\% auf deutschen Intensivstationen. Das mediane Alter aller gescreenten Patient:innen betrug 68 Jahre (IQR: 57-77), wobei die Patient:innen in der deutschen Grenzregion signifikant älter waren als die Patient:innen in der niederländischen Grenzregion. Die allgemeine Screening-Compliance (auf mindestens eine MDRO-Gruppe gescreent) lag bei 60\%. Alle AMR-Datenanalysen wurden mit dem AMR-Paket für R durchgeführt und automatisiert. Die Prävalenz von MRSA betrug 1,7\% in deutschen Intensivstationen und 0,6\% in niederländischen Intensivstationen. Die Prävalenz von VRE betrug 2,7\% in deutschen Intensivstationen und 0,1\% in niederländischen Intensivstationen. Bemerkenswert ist, dass diese Prävalenz in der deutschen Grenzregion von 0\% bis 4,1\% reichte. Alle 56 Fälle von VRE wurden durch E. faecium verursacht. Die Prävalenz von 3GCRE betrug 6,6\% in deutschen und 3,6\% in niederländischen Intensivstationen, während die Prävalenz für CRE auf beiden Seiten der Grenze praktisch nicht präsent war. Die Prävalenz für gramnegative MDROs unterschied sich innerhalb beider Nationen zwischen den Krankenhäusern und reichte von 0\% bis 5,0\% in der niederländischen Grenzregion und von 1,2\% bis 10,9\% in der deutschen Grenzregion. Für die eingeschlossenen niederländischen Intensivstationen war die Prävalenz aller MDRO-Gruppen nicht signifikant unterschiedlich zwischen der nicht-universitären und der universitären Klinik. Für die deutschen Intensivstationen war jedoch die Prävalenz von gramnegativen MDROs in den nicht-universitären Krankenhäusern signifikant höher. In der niederländischen Grenzregion führten 4,8 von 100 Krankenhauseinweisungen zur Aufnahme auf der Intensivstation. Im Gegensatz dazu waren es in der deutschen Grenzregion 7,7 pro 100 Krankenhauseinweisungen. Dieser Unterschied lässt sich durch die höhere ICU-Kapazität in deutschen Krankenhäusern (4,8\% aller Krankenhausbetten) im Vergleich zu niederländischen Krankenhäusern (2,4\% aller Krankenhausbetten) erklären. Die Gesamtprävalenz für die verschiedenen MDROs war auf den deutschen Intensivstationen höher, obwohl einige Unterschiede marginal waren. Insbesondere war die Prävalenz von MRSA-Kolonisierung in der deutschen Grenzregion dreimal so hoch wie in der niederländischen Grenzregion, was mit den in \textbf{Kapitel 9} erwähnten Studienergebnissen übereinstimmt. Dennoch waren die Studienergebnisse nicht übereinstimmend mit (inter)nationalen Durchschnittswerten. Zum Beispiel war die Prävalenz der 3GCRE-Kolonisierung in der deutschen Grenzregion fast doppelt so hoch wie in der niederländischen Grenzregion, aber beide waren immer noch niedriger als der nationale Durchschnitt. Das ECDC meldete für Deutschland und die Niederlande 6\% höhere 3GCRE-Anteile unter den Blutkulturisolaten von E. coli und K. pneumoniae. Dies unterstreicht, dass es wichtige Unterschiede gibt, wenn man die Kolonisierung in bestimmten Populationen untersucht und nicht den Anteil der (wahrscheinlich) invasiven Isolate betrachtet. Somit unterstreicht diese Studie die Bedeutung eines regionalen und grenzüberschreitenden Ansatzes in jeder europäischen grenzüberschreitenden Region, um die Unterschiede in der AMR-Prävalenz zwischen den Regionen zu verdeutlichen und mögliche Unterschiede zu landesweiten Berichten aufzuzeigen. Um dies weiter ausarbeiten zu können, ist ein tieferer Detaillierungsgrad erforderlich, z. B. durch die Erfassung von Informationen über das Personal auf den Stationen und das Personal der Infektionskontrolle, MDRO-Ausbrüche, Infektionen, Antibiotikaeinsatz und Risikofaktoren der Patient:innen. Zusammenfassend lässt sich sagen, dass geografische und politische Grenzen von MDROs anscheinend nicht „respektiert`` werden, obwohl die Gesundheitssysteme, die geografische Beschaffenheit und die Richtlinien in den einzelnen Nationen sehr unterschiedlich sind. Die Anteile von MDROs bestimmter Erreger, wie sie auf nationaler und internationaler Ebene berichtet werden, spiegeln nicht die MDRO-Prävalenz in der Patient:innen - oder Allgemeinbevölkerung wider. Dies sollte bei der Interpretation von Berichten auf Nationen - oder sogar Kontinentalebene ernsthaft in Betracht gezogen werden.

\textbf{Fazit}

Das AMR-Paket und seine Vorteile werden aus verschiedenen Blickwinkeln betrachtet: aus technischer Sicht, aus der Sicht des Infektionsmanagements und aus klinischer Sicht. Diese Kombination bietet eine gemeinsame Grundlage für das Verständnis, was das AMR-Paket in der Praxis bewirken kann und wie es einen neuen, befähigten Ausgangspunkt für zukünftige Anwendungen der mikrobiellen Epidemiologie, sowohl in klinischen als auch in Forschungsumgebungen, setzen kann. Die vorliegende Dissertation vertieft diese vielfältigen Gesichtspunkte anschließend, indem sie den Einsatz dieses neuen Instruments in epidemiologischen Forschungsprojekten in der deutsch-niederländischen Grenzregion illustriert, um das Vorkommen und die AMR-Muster von Mikroorganismen auf (eu)regionaler Ebene besser zu verstehen. Zusammenfassend zeigt diese Dissertation den Mehrwert eines konsistenten datenanalytischen Instruments zur Aufbereitung und Analyse von AMR-Daten in einem flächendeckenden Ansatz, der auch im klinischen Umfeld eingesetzt werden kann, um neue Erkenntnisse über AMR-Muster zu erhalten.

  \bibliography{packages.bib}

\end{document}
