<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>7 Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019 | A New Instrument for Microbial Epidemiology</title>
  <meta name="description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="7 Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019 | A New Instrument for Microbial Epidemiology" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology" />
  <meta property="og:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />
  <meta property="og:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="github-repo" content="msberends/a_new_instrument_for_microbial_epidemiology" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="7 Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019 | A New Instrument for Microbial Epidemiology" />
  
  <meta name="twitter:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="twitter:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />

<meta name="author" content="Matthijs S. Berends" />


<meta name="date" content="2021-08-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ch06-radar2.html"/>
<link rel="next" href="ch08-defining-mdr.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://doi.org/10.33612/diss.177417131" target="blank"><img src="images/cover.jpg" style="width:90%; margin-left:5%;"></a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preamble</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#short-summary-250-words"><i class="fa fa-check"></i>Short summary (250 words)</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="colophon.html"><a href="colophon.html"><i class="fa fa-check"></i>Colophon</a></li>
<li class="chapter" data-level="" data-path="contents.html"><a href="contents.html"><i class="fa fa-check"></i>Contents</a></li>
<li class="chapter" data-level="1" data-path="ch01-introduction.html"><a href="ch01-introduction.html"><i class="fa fa-check"></i><b>1</b> General Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#microbial-epidemiology"><i class="fa fa-check"></i><b>1.1</b> Microbial epidemiology</a></li>
<li class="chapter" data-level="1.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#antimicrobial-resistance-in-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Antimicrobial resistance in microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#laboratory-diagnostics"><i class="fa fa-check"></i><b>1.2.1</b> Laboratory diagnostics</a></li>
<li class="chapter" data-level="1.2.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#interpretation-of-raw-results"><i class="fa fa-check"></i><b>1.2.2</b> Interpretation of raw results</a></li>
<li class="chapter" data-level="1.2.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#multi-drug-resistant-organisms"><i class="fa fa-check"></i><b>1.2.3</b> Multi-drug resistant organisms</a></li>
<li class="chapter" data-level="1.2.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#surveillance-programs"><i class="fa fa-check"></i><b>1.2.4</b> Surveillance programs</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#data-analysis-using-r"><i class="fa fa-check"></i><b>1.3</b> Data analysis using R</a></li>
<li class="chapter" data-level="1.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#setting-for-this-thesis"><i class="fa fa-check"></i><b>1.4</b> Setting for this thesis</a></li>
<li class="chapter" data-level="1.5" data-path="ch01-introduction.html"><a href="ch01-introduction.html#aim-of-this-thesis-and-introduction-to-its-chapters"><i class="fa fa-check"></i><b>1.5</b> Aim of this thesis and introduction to its chapters</a></li>
<li class="chapter" data-level="" data-path="ch01-introduction.html"><a href="ch01-introduction.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html"><i class="fa fa-check"></i><b>2</b> Diagnostic Stewardship: Sense or Nonsense?!</a>
<ul>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-1"><i class="fa fa-check"></i><b>2.1.1</b> Case 1</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-2"><i class="fa fa-check"></i><b>2.1.2</b> Case 2</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#the-general-concept"><i class="fa fa-check"></i><b>2.2</b> The general concept</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#diagnostics"><i class="fa fa-check"></i><b>2.2.1</b> ‘Diagnostics’</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-in-the-microbiological-laboratory"><i class="fa fa-check"></i><b>2.2.2</b> DSP in the microbiological laboratory</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-as-process-optimisation"><i class="fa fa-check"></i><b>2.2.3</b> DSP as process optimisation</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#multidisciplinary-aspects-of-dsp-and-infection-management"><i class="fa fa-check"></i><b>2.2.4</b> Multidisciplinary aspects of DSP and infection management</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#conclusion"><i class="fa fa-check"></i><b>2.3</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#financing"><i class="fa fa-check"></i>Financing</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html"><i class="fa fa-check"></i><b>3</b> Introducing a New, Free, and Independent Method for Standardised, Reproducible and Reliable Analyses of Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#background"><i class="fa fa-check"></i><b>3.1</b> Background</a></li>
<li class="chapter" data-level="3.2" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#standardising-amr-data-analysis"><i class="fa fa-check"></i><b>3.2</b> Standardising AMR data analysis</a></li>
<li class="chapter" data-level="3.3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#comparison-with-existing-software-methods"><i class="fa fa-check"></i><b>3.3</b> Comparison with existing software methods</a></li>
<li class="chapter" data-level="3.4" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#user-feedback"><i class="fa fa-check"></i><b>3.4</b> User feedback</a></li>
<li class="chapter" data-level="3.5" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#conclusion-1"><i class="fa fa-check"></i><b>3.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html"><i class="fa fa-check"></i><b>4</b> <code>AMR</code> - An <code>R</code> Package for Working with Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.2</b> Antimicrobial resistance data</a></li>
<li class="chapter" data-level="4.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data-transformation"><i class="fa fa-check"></i><b>4.3</b> Antimicrobial resistance data transformation</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-taxonomically-valid-microorganism-names"><i class="fa fa-check"></i><b>4.3.1</b> Working with taxonomically valid microorganism names</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-names-or-codes"><i class="fa fa-check"></i><b>4.3.2</b> Working with antimicrobial names or codes</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-susceptibility-test-results"><i class="fa fa-check"></i><b>4.3.3</b> Working with antimicrobial susceptibility test results</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#interpretative-rules-by-eucast"><i class="fa fa-check"></i><b>4.3.4</b> Interpretative rules by EUCAST</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-defined-daily-doses-ddd"><i class="fa fa-check"></i><b>4.3.5</b> Working with defined daily doses (DDD)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#enhancing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.4</b> Enhancing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-first-isolates"><i class="fa fa-check"></i><b>4.4.1</b> Determining first isolates</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-multi-drug-resistant-organisms-mdro"><i class="fa fa-check"></i><b>4.4.2</b> Determining multi-drug resistant organisms (MDRO)</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#analysing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.5</b> Analysing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#calculation-of-antimicrobial-resistance"><i class="fa fa-check"></i><b>4.5.1</b> Calculation of antimicrobial resistance</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#design-decisions"><i class="fa fa-check"></i><b>4.6</b> Design decisions</a></li>
<li class="chapter" data-level="4.7" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#reproducible-example"><i class="fa fa-check"></i><b>4.7</b> Reproducible example</a></li>
<li class="chapter" data-level="4.8" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#discussion"><i class="fa fa-check"></i><b>4.8</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#computational-details"><i class="fa fa-check"></i>Computational Details</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#references-3"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#appendix-a-included-data-sets"><i class="fa fa-check"></i>Appendix A: Included Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch05-radar.html"><a href="ch05-radar.html"><i class="fa fa-check"></i><b>5</b> Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship</a>
<ul>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="ch05-radar.html"><a href="ch05-radar.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch05-radar.html"><a href="ch05-radar.html#background-1"><i class="fa fa-check"></i><b>5.1.1</b> Background</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch05-radar.html"><a href="ch05-radar.html#objectives"><i class="fa fa-check"></i><b>5.1.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch05-radar.html"><a href="ch05-radar.html#methods"><i class="fa fa-check"></i><b>5.2</b> Methods</a></li>
<li class="chapter" data-level="5.3" data-path="ch05-radar.html"><a href="ch05-radar.html#results"><i class="fa fa-check"></i><b>5.3</b> Results</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch05-radar.html"><a href="ch05-radar.html#overview"><i class="fa fa-check"></i><b>5.3.1</b> Overview</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch05-radar.html"><a href="ch05-radar.html#application-design"><i class="fa fa-check"></i><b>5.3.2</b> Application Design</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch05-radar.html"><a href="ch05-radar.html#development-process"><i class="fa fa-check"></i><b>5.3.3</b> Development Process</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch05-radar.html"><a href="ch05-radar.html#workflow"><i class="fa fa-check"></i><b>5.3.4</b> Workflow</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch05-radar.html"><a href="ch05-radar.html#customisation"><i class="fa fa-check"></i><b>5.3.5</b> Customisation</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch05-radar.html"><a href="ch05-radar.html#discussion-1"><i class="fa fa-check"></i><b>5.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch05-radar.html"><a href="ch05-radar.html#principal-findings"><i class="fa fa-check"></i><b>5.4.1</b> Principal Findings</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch05-radar.html"><a href="ch05-radar.html#conclusions"><i class="fa fa-check"></i><b>5.4.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#conflicts-of-interests"><i class="fa fa-check"></i>Conflicts of interests</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#references-4"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch06-radar2.html"><a href="ch06-radar2.html"><i class="fa fa-check"></i><b>6</b> Better Antimicrobial Resistance Data Analysis and Reporting in Less Time</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="6.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#methods-1"><i class="fa fa-check"></i><b>6.2</b> Methods</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-setup"><i class="fa fa-check"></i><b>6.2.1</b> Study setup</a></li>
<li class="chapter" data-level="6.2.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data"><i class="fa fa-check"></i><b>6.2.2</b> AMR data</a></li>
<li class="chapter" data-level="6.2.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data-analysis-and-reporting"><i class="fa fa-check"></i><b>6.2.3</b> AMR data analysis and reporting</a></li>
<li class="chapter" data-level="6.2.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants"><i class="fa fa-check"></i><b>6.2.4</b> Study participants</a></li>
<li class="chapter" data-level="6.2.5" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-execution-and-data"><i class="fa fa-check"></i><b>6.2.5</b> Study execution and data</a></li>
<li class="chapter" data-level="6.2.6" data-path="ch06-radar2.html"><a href="ch06-radar2.html#evaluation-and-study-data-analysis"><i class="fa fa-check"></i><b>6.2.6</b> Evaluation and study data analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#results-1"><i class="fa fa-check"></i><b>6.3</b> Results</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants-1"><i class="fa fa-check"></i><b>6.3.1</b> Study participants</a></li>
<li class="chapter" data-level="6.3.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#effectiveness-and-accuracy"><i class="fa fa-check"></i><b>6.3.2</b> Effectiveness and accuracy</a></li>
<li class="chapter" data-level="6.3.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#efficiency"><i class="fa fa-check"></i><b>6.3.3</b> Efficiency</a></li>
<li class="chapter" data-level="6.3.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#satisfaction"><i class="fa fa-check"></i><b>6.3.4</b> Satisfaction</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#discussion-2"><i class="fa fa-check"></i><b>6.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#acknowledgements-2"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#funding"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#conflict-of-interest"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#references-5"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix"><i class="fa fa-check"></i>Appendix</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a1-task-lists-including-correct-results"><i class="fa fa-check"></i>Appendix A1: Task lists including correct results</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a2-system-usability-scale-sus"><i class="fa fa-check"></i>Appendix A2: System Usability Scale (SUS)</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a3-task-3-sub-analysis"><i class="fa fa-check"></i>Appendix A3: Task 3 sub-analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ch07-cons.html"><a href="ch07-cons.html"><i class="fa fa-check"></i><b>7</b> Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019</a>
<ul>
<li class="chapter" data-level="7.1" data-path="ch07-cons.html"><a href="ch07-cons.html#abstract-5"><i class="fa fa-check"></i><b>7.1</b> Abstract</a></li>
<li class="chapter" data-level="7.2" data-path="ch07-cons.html"><a href="ch07-cons.html#introduction-4"><i class="fa fa-check"></i><b>7.2</b> Introduction</a></li>
<li class="chapter" data-level="7.3" data-path="ch07-cons.html"><a href="ch07-cons.html#materials-methods"><i class="fa fa-check"></i><b>7.3</b> Materials &amp; methods</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="ch07-cons.html"><a href="ch07-cons.html#study-setting-and-patient-cohort"><i class="fa fa-check"></i><b>7.3.1</b> Study setting and patient cohort</a></li>
<li class="chapter" data-level="7.3.2" data-path="ch07-cons.html"><a href="ch07-cons.html#microbiological-and-demographic-data"><i class="fa fa-check"></i><b>7.3.2</b> Microbiological and demographic data</a></li>
<li class="chapter" data-level="7.3.3" data-path="ch07-cons.html"><a href="ch07-cons.html#species-determination-and-antibiotic-susceptibility-testing-ast"><i class="fa fa-check"></i><b>7.3.3</b> Species determination and antibiotic susceptibility testing (AST)</a></li>
<li class="chapter" data-level="7.3.4" data-path="ch07-cons.html"><a href="ch07-cons.html#selection-of-bacterial-isolates"><i class="fa fa-check"></i><b>7.3.4</b> Selection of bacterial isolates</a></li>
<li class="chapter" data-level="7.3.5" data-path="ch07-cons.html"><a href="ch07-cons.html#eucast-rules-and-antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.3.5</b> EUCAST rules and antibiotic resistance analysis</a></li>
<li class="chapter" data-level="7.3.6" data-path="ch07-cons.html"><a href="ch07-cons.html#statistical-analysis"><i class="fa fa-check"></i><b>7.3.6</b> Statistical analysis</a></li>
<li class="chapter" data-level="7.3.7" data-path="ch07-cons.html"><a href="ch07-cons.html#ethical-considerations"><i class="fa fa-check"></i><b>7.3.7</b> Ethical considerations</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="ch07-cons.html"><a href="ch07-cons.html#results-2"><i class="fa fa-check"></i><b>7.4</b> Results</a>
<ul>
<li class="chapter" data-level="7.4.1" data-path="ch07-cons.html"><a href="ch07-cons.html#patients-and-included-isolates"><i class="fa fa-check"></i><b>7.4.1</b> Patients and included isolates</a></li>
<li class="chapter" data-level="7.4.2" data-path="ch07-cons.html"><a href="ch07-cons.html#occurrence-of-cons-species"><i class="fa fa-check"></i><b>7.4.2</b> Occurrence of CoNS species</a></li>
<li class="chapter" data-level="7.4.3" data-path="ch07-cons.html"><a href="ch07-cons.html#definition-of-cons-persistence"><i class="fa fa-check"></i><b>7.4.3</b> Definition of CoNS persistence</a></li>
<li class="chapter" data-level="7.4.4" data-path="ch07-cons.html"><a href="ch07-cons.html#antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.4.4</b> Antibiotic resistance analysis</a></li>
</ul></li>
<li class="chapter" data-level="7.5" data-path="ch07-cons.html"><a href="ch07-cons.html#discussion-3"><i class="fa fa-check"></i><b>7.5</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#supplementary-tables"><i class="fa fa-check"></i>Supplementary tables</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#references-6"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html"><i class="fa fa-check"></i><b>8</b> Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines</a>
<ul>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#abstract-6"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="8.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#introduction-5"><i class="fa fa-check"></i><b>8.1</b> Introduction</a></li>
<li class="chapter" data-level="8.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#methods-2"><i class="fa fa-check"></i><b>8.2</b> Methods</a></li>
<li class="chapter" data-level="8.3" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-3"><i class="fa fa-check"></i><b>8.3</b> Results</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#number-of-antibiograms-and-patients"><i class="fa fa-check"></i><b>8.3.1</b> Number of Antibiograms and Patients</a></li>
<li class="chapter" data-level="8.3.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-of-mrgn-and-brmo-classification"><i class="fa fa-check"></i><b>8.3.2</b> Results of MRGN and BRMO Classification</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#discussion-4"><i class="fa fa-check"></i><b>8.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#acknowledgements-3"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#conflicts-of-interest"><i class="fa fa-check"></i>Conflicts of interest</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#references-7"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html"><i class="fa fa-check"></i><b>9</b> Changing Epidemiology of Methicillin-Resistant <em>Staphylococcus aureus</em> in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow Policy</a>
<ul>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#abstract-7"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="9.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#introduction-6"><i class="fa fa-check"></i><b>9.1</b> Introduction</a></li>
<li class="chapter" data-level="9.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#methods-3"><i class="fa fa-check"></i><b>9.2</b> Methods</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#setting"><i class="fa fa-check"></i><b>9.2.1</b> Setting</a></li>
<li class="chapter" data-level="9.2.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#guidelines-for-patients-at-risk-for-mrsa-and-infection-prevention-and-control-measures"><i class="fa fa-check"></i><b>9.2.2</b> Guidelines for patients at risk for MRSA and infection prevention and control measures</a></li>
<li class="chapter" data-level="9.2.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-collection"><i class="fa fa-check"></i><b>9.2.3</b> Data collection</a></li>
<li class="chapter" data-level="9.2.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#ethical-statement"><i class="fa fa-check"></i><b>9.2.4</b> Ethical statement</a></li>
<li class="chapter" data-level="9.2.5" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-analysis"><i class="fa fa-check"></i><b>9.2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#results-4"><i class="fa fa-check"></i><b>9.3</b> Results</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#trend-and-cross-border-comparison-of-mrsa-rates"><i class="fa fa-check"></i><b>9.3.1</b> Trend and cross-border comparison of MRSA rates</a></li>
<li class="chapter" data-level="9.3.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#cross-border-comparison-of-healthcare-utilisation"><i class="fa fa-check"></i><b>9.3.2</b> Cross-border comparison of healthcare utilisation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#discussion-5"><i class="fa fa-check"></i><b>9.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#acknowledgements-4"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#funding-1"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#references-8"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html"><i class="fa fa-check"></i><b>10</b> A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures</a>
<ul>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#abstract-8"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="10.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#introduction-7"><i class="fa fa-check"></i><b>10.1</b> Introduction</a></li>
<li class="chapter" data-level="10.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#methods-4"><i class="fa fa-check"></i><b>10.2</b> Methods</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-design"><i class="fa fa-check"></i><b>10.2.1</b> Study Design</a></li>
<li class="chapter" data-level="10.2.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#statistical-analysis-software"><i class="fa fa-check"></i><b>10.2.2</b> Statistical Analysis &amp; Software</a></li>
<li class="chapter" data-level="10.2.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#ethics"><i class="fa fa-check"></i><b>10.2.3</b> Ethics</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#results-5"><i class="fa fa-check"></i><b>10.3</b> Results</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#healthcare-structure-of-the-participating-hospitals"><i class="fa fa-check"></i><b>10.3.1</b> Healthcare structure of the participating hospitals</a></li>
<li class="chapter" data-level="10.3.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-population-and-screening-samples-from-icus"><i class="fa fa-check"></i><b>10.3.2</b> Study population and screening samples from ICUs</a></li>
<li class="chapter" data-level="10.3.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-positive-mdros-mrsa-and-vre"><i class="fa fa-check"></i><b>10.3.3</b> Prevalence of Gram-positive MDROs: MRSA and VRE</a></li>
<li class="chapter" data-level="10.3.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdro-3gcre-and-cre"><i class="fa fa-check"></i><b>10.3.4</b> Prevalence of Gram-negative MDRO: 3GCRE and CRE</a></li>
<li class="chapter" data-level="10.3.5" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdros-based-on-dutch-and-german-definitions"><i class="fa fa-check"></i><b>10.3.5</b> Prevalence of Gram-negative MDROs based on Dutch and German definitions</a></li>
<li class="chapter" data-level="10.3.6" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#comparison-of-mdro-prevalence-between-nl-br-and-de-br-icus-in-university-and-non-university-hospitals"><i class="fa fa-check"></i><b>10.3.6</b> Comparison of MDRO prevalence between NL-BR and DE-BR ICUs in university and non-university hospitals</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#discussion-6"><i class="fa fa-check"></i><b>10.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#supplementary-files"><i class="fa fa-check"></i>Supplementary files</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#acknowledgements-5"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#conflict-of-interest-1"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#references-9"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="ch11-summary.html"><a href="ch11-summary.html"><i class="fa fa-check"></i><b>11</b> Summary and Future Perspectives</a>
<ul>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-i"><i class="fa fa-check"></i>Section I</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-ii"><i class="fa fa-check"></i>Section II</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-iii"><i class="fa fa-check"></i>Section III</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#future-perspectives"><i class="fa fa-check"></i>Future perspectives</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#references-10"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="gearfetting-yn-frysk.html"><a href="gearfetting-yn-frysk.html"><i class="fa fa-check"></i>Gearfetting yn Frysk</a></li>
<li class="chapter" data-level="" data-path="samenvatting-in-het-nederlands.html"><a href="samenvatting-in-het-nederlands.html"><i class="fa fa-check"></i>Samenvatting in het Nederlands</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung-auf-deutsch.html"><a href="zusammenfassung-auf-deutsch.html"><i class="fa fa-check"></i>Zusammenfassung auf Deutsch</a></li>
<li class="divider"></li>
<li class="copyright">© 2021 Matthijs Berends</li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A New Instrument for Microbial Epidemiology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch07-cons" class="section level1" number="7">
<h1><span class="header-section-number">7</span> Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019</h1>
<p>
In preparation
<note>
(as of date of PhD defence: 25 August 2021)
</note>
</p>
<p>Berends MS <sup>1,2</sup>, Luz CF <sup>2</sup>, Ott A <sup>1</sup>, Andriesse GI <sup>1</sup>, Becker K <sup>3,4</sup>, Glasner C <sup>2‡</sup>, Friedrich AW <sup>2‡</sup></p>
<ol style="list-style-type: decimal">
<li>Certe Medical Diagnostics and Advice Foundation, Groningen, the Netherlands</li>
<li>University of Groningen, University Medical Center Groningen, Department of Medical Microbiology and Infection Prevention, Groningen, the Netherlands</li>
<li>Institute of Medical Microbiology, University Hospital Münster, Münster, Germany</li>
<li>Friedrich Loeffler-Institute of Medical Microbiology, University Medicine Greifswald, Greifswald, Germany</li>
</ol>
<p><sup>‡</sup> These authors contributed equally</p>
<div id="abstract-5" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Abstract</h2>
<p>For years, coagulase-negative staphylococci (CoNS) were not considered a cause of bloodstream infections (BSIs) and were often regarded as contamination. However, the association of CoNS with nosocomial infections is increasingly recognised in research and clinical practice. At present, the CoNS group consists of 45 different species. Their identification has mainly been driven by the introduction of matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry. Yet, treatment guidelines consider CoNS as a whole group and rarely differentiate between species, despite increasing antibiotic resistance (ABR) in CoNS. Therefore, this retrospective study provides an in-depth analysis of CoNS isolates and their ABR profiles found in blood culture isolates between 2013 and 2019 in a novel full-region approach including the entire region of the Northern Netherlands. In total, 10,796 patients were included that were hospitalised in one of the 15 hospitals in the region leading to a sample of 14,992 first CoNS isolates for (ABR) data analysis. CoNS accounted for 27.6% of all available 71,632 blood culture isolates. EUCAST Expert rules were applied to correct for errors in antibiotic test results. A total of 27 different species were found. Major differences were observed in the occurrence and ABR profiles of the different species. The top five species covered 97.1% of all included isolates: <em>S. epidermidis</em> (48.4%), <em>S. hominis</em> (33.6%), <em>S. capitis</em> (9.3%), <em>S. haemolyticus</em> (4.1%), and <em>S. warneri</em> (1.7%). Regarding ABR, <em>S. epidermidis</em> and <em>S. haemolyticus</em> showed 50-80% resistance to teicoplanin and macrolides while resistance to these agents remained lower than 10% in most other CoNS species. Yet, such differences are neglected in national guideline development causing a focus on ‘ABR-safe’ agents such as glycopeptides. Nonetheless, other agents could be considered viable options for some species where ABR never surpassed 10%. In conclusion, a multi-year, full-region approach to extensively assess the trends in both the occurrence and AMR of CoNS species was carried out which could be used for evaluating treatment policies and understanding more about these important but still too often neglected pathogens.</p>
</div>
<div id="introduction-4" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Introduction</h2>
<p>Sepsis is a syndrome of physiologic, pathologic, and biochemical abnormalities induced by bloodstream infections (BSIs). It is the most frequent cause of death in hospitalised patients and has been recognised by the WHO as a global health priority <sup>[1,2]</sup>. For years, coagulase-negative staphylococci (CoNS) were not considered a cause of BSIs and were often regarded as contamination <sup>[3]</sup>. Yet, it has been shown that CoNS can cause BSIs and a high mortality rate <sup>[4,5]</sup>, especially in immunocompromised patients and newborns <sup>[6,7]</sup>. Moreover, CoNS have become increasingly associated with nosocomial infections <sup>[8]</sup>. This is attributed to (i) an increase of multimorbid and immunocompromised patients that are more prone to infections, (ii) the increased use of inserted foreign body material in modern medicine, and (iii) the property of CoNS to adapt molecularly to the hospital environment by diverging into new strains <sup>[8,9]</sup>. Specifically, <em>S. epidermidis</em> and <em>S. haemolyticus</em> are associated with sepsis caused by foreign-body-related infections (FBRIs), such as central line-associated BSIs and prosthetic joint infections <sup>[10]</sup>.</p>
<p>At present, the CoNS group consists of 45 different species <sup>[11]</sup>. This group is highly heterogeneous in its prevalence in humans and, more importantly, its antibiotic resistance (ABR) patterns. Zooming in on CoNS at the species level is therefore useful to evaluate treatment options for CoNS causing BSI. The clinical interpretation and relevance of BSIs caused by CoNS are dependent on the determination at the species level, since not all species in the CoNS group are pathogenic and associated with sepsis or (other) nosocomial infections <sup>[8,12]</sup>. While the microbiological diagnosis of BSIs has for decades been based on blood samples cultivated in automated blood-culture systems, molecular and mass spectrometry (MS) approaches enable more reliable microbiological diagnosis <sup>[13,14]</sup>. Since 2012, matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) MS has become a standard for the identification of bacterial species and has, together with sequencing approaches, led to a rapid discovery of new species compared to formerly used techniques <sup>[15,16]</sup>. Prior to the use of MALDI-TOF MS, identification of CoNS was primarily performed with biochemical and physiological tests, which yielded variable results, particularly in less prevalent species <sup>[16]</sup>. Examples include <em>S. warneri</em>, <em>S. auricularis</em>, <em>S. capitis</em>, and other CoNS species that primarily colonise the skin of animals or are found on food products <sup>[17]</sup>. Due to less specific traditional test techniques, previously reported prevalences and ABR patterns of specific species in the CoNS group may have been unreliable or under-evaluated. Consequently, identification using MALDI-TOF MS has become crucial to analyse species-specific ABR.</p>
<p>ABR is a global healthcare problem and of great concern in the antibiotic therapy of BSIs. This also applies to the CoNS group where multi-drug resistance is common in species circulating in hospitals <sup>[18]</sup>. The rise of beta-lactam resistance in CoNS species has led to vancomycin as a first-line therapy against CoNS-mediated BSI in many countries, even though information about the pharmacokinetics and pharmacodynamics (PK/PD) of vancomycin against CoNS is limited <sup>[5,19–21]</sup>. To assess the constant change of ABR in CoNS, geo-spatial and temporal analyses of ABR are required.</p>
<p>In the Netherlands, country-wide ABR analyses are used to develop antibiotic treatment guidelines by the Dutch Working Party on Antibiotic Policy (Stichting Werkgroep Antibiotica Beleid, SWAB) <sup>[21,22]</sup>. Their recommendations are based on NethMap, an annually released national report about ABR and antibiotic consumption by the Dutch National Institute for Public Health and the Environment (Rijksinstituut voor Volksgezondheid en Milieu, RIVM) <sup>[21]</sup>. However, this national report does not specify nor address ABR on a patient, hospital, or regional level.</p>
<p>Therefore, to inform clinical decision-makers this cross-sectional retrospective study provides an in-depth ABR analysis of all CoNS isolates found in blood cultures from 2013 until 2019 in the Northern Netherlands that were determined by MALDI-TOF MS. We aim to evaluate the differences in the occurrence of CoNS species and their ABR patterns and to assess their clinical microbiological relevance using a full-region approach.</p>
</div>
<div id="materials-methods" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Materials &amp; methods</h2>
<div id="study-setting-and-patient-cohort" class="section level3" number="7.3.1">
<h3><span class="header-section-number">7.3.1</span> Study setting and patient cohort</h3>
<p>This study was performed within the Northern Netherlands (Figure 1), a geographic region with 1.7 million inhabitants <sup>[23]</sup>. Its three provinces are similar in population density: Drenthe (492,167 inhabitants, 184/km2), Friesland (647,672 inhabitants, 183/km2) and Groningen (583,990 inhabitants, 243/km2) <sup>[23]</sup>. The study population consisted of 10,786 patients hospitalised with suspected BSI in 15 participating hospitals (14 secondary care, one tertiary care) located within this region between 1 January 2013 and 31 December 2019. All hospitals included at least one intensive care unit (ICU). There was no age restriction on including patients.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-1"></span>
<img src="images/07-01.jpg" alt="Locations of the fifteen hospitals in the three provinces in the North of the Netherlands. Between 2013 and June 2018, the region comprised fourteen hospitals; in July 2018, two hospitals merged into one new hospital, leaving a total of thirteen currently active hospitals." width="100%" />
<p class="caption">
Figure 7.1: Locations of the fifteen hospitals in the three provinces in the North of the Netherlands. Between 2013 and June 2018, the region comprised fourteen hospitals; in July 2018, two hospitals merged into one new hospital, leaving a total of thirteen currently active hospitals.
</p>
</div>
</div>
<div id="microbiological-and-demographic-data" class="section level3" number="7.3.2">
<h3><span class="header-section-number">7.3.2</span> Microbiological and demographic data</h3>
<p>All blood cultures were routinely drawn and analysed at one of the three medical microbiological laboratories in the region (Izore, Friesland; Certe, Groningen and Drenthe; University Medical Center Groningen). After routine processing, isolates were included in the study if the species was characterised as a member of the CoNS group and antibiotic test results were available. In the study period, CoNS species were the most prevalent microorganisms isolated from blood and accounted for 27.6% of all available 71,632 blood culture isolates. The following variables were available for all isolates: date, name of laboratory, name of the hospital, age, gender, and ID of the patient and type of ward (ICU, clinical, outward). Genotypic data was not available for this study, as genotyping was not part of routine analysis.</p>
</div>
<div id="species-determination-and-antibiotic-susceptibility-testing-ast" class="section level3" number="7.3.3">
<h3><span class="header-section-number">7.3.3</span> Species determination and antibiotic susceptibility testing (AST)</h3>
<p>Routine processing in the laboratories included the incubation of blood cultures allowing the colourimetric detection of CO2 produced by growing microorganisms. Determination of the taxonomic species level was done using MALDI-TOF MS. Two laboratories cultivated blood samples using the BacT/ALERT system (bioMérieux, France) and identified bacterial strains using the VITEK MS system (bioMérieux, France). One laboratory cultivated blood samples using the BACTEC (Becton Dickinson, UK) and identified bacterial strains using the Microflex System (Bruker Corporation, USA). Since the databases of these proprietary systems are not publicly available, a qualitative assessment could not be attained, nor was this available in public literature.</p>
<p>AST was performed using the VITEK 2 Advanced Expert System after isolates were incubated on blood agar plates containing 5% sheep blood (BA+5%SB). Two laboratories used the VITEK 2 P-586 cartridges and one laboratory used the VITEK 2 P-657 cartridge which are both developed specifically for Gram-positive bacteria such as staphylococci. All results were authorised and validated by at least two laboratory technicians and one clinical microbiologist. Since different VITEK 2 cartridges were used, not all isolates were tested for all antibiotics analysed in this study. Supplementary Material 2 contains a full list of all included isolates and their respective AST results.</p>
</div>
<div id="selection-of-bacterial-isolates" class="section level3" number="7.3.4">
<h3><span class="header-section-number">7.3.4</span> Selection of bacterial isolates</h3>
<p>First isolates were determined and selected using the AMR package for R to exclude duplicate findings following the M39-A4 guideline by the Clinical Laboratory Standards Institute (CLSI) <sup>[24,25]</sup>. This guideline defines first isolates based on the species level per patient episode, regardless of body site and other phenotypical characteristics. The episode length for this study was defined as 365 days, resulting in the inclusion of a unique species once a year per patient.</p>
<p>In this study, several additions were made in extension to the CLSI guideline. As the CLSI guideline only considers the genus/species per episode, we investigated the added value to include changes in the ABR profile per genus/species and episode. For this purpose, we weighted the ABR profile of six preselected antibiotics, which were specifically chosen based on clinical relevance for Gram-positive bacteria, such as CoNS: erythromycin, oxacillin, rifampicin, teicoplanin, tetracycline, and vancomycin. Any change in these antibiotics from susceptible to resistant or vice-versa within the same species in the same patient within one episode was considered a ‘first weighted isolate.’ ABR analysis results per species were included if at least 30 first isolates were available following the current CLSI guideline <sup>[24]</sup>.</p>
</div>
<div id="eucast-rules-and-antibiotic-resistance-analysis" class="section level3" number="7.3.5">
<h3><span class="header-section-number">7.3.5</span> EUCAST rules and antibiotic resistance analysis</h3>
<p>European Committee on Antimicrobial Susceptibility Testing (EUCAST) rules were applied to the AST results including EUCAST Expert Rules (v3.1, 2016), EUCAST Clinical Breakpoint Interpretations (v10.0, 2020), and EUCAST rules for Intrinsic Resistance and Unusual Phenotypes <sup>[26,27]</sup>. All applied changes can be found in Supplementary Table 1. Resistance was defined as the number of isolates with an antibiotic interpretation of R (resistant) divided by the total number of susceptible (S or I) isolates, following the latest EUCAST guideline <sup>[27]</sup>.</p>
</div>
<div id="statistical-analysis" class="section level3" number="7.3.6">
<h3><span class="header-section-number">7.3.6</span> Statistical analysis</h3>
<p>All statistical analyses were done using R v4.0.3, RStudio v1.4, and the AMR package v1.6.0 <sup>[25,28]</sup>. To test for linear trends, linear regression analyses were performed. Contingency tables were tested with Fisher’s exact test when the size was 2x2 and Chi-squared tests otherwise. For likelihood ratio tests exact binomial tests were used. Outcomes of statistical tests were considered significant when <em>p</em> &lt; 0.05.</p>
</div>
<div id="ethical-considerations" class="section level3" number="7.3.7">
<h3><span class="header-section-number">7.3.7</span> Ethical considerations</h3>
<p>Ethical approval and informed consent were not required according to the medical ethical committee of the University Medical Center Groningen (METc M21.277097). All data were anonymised at the associated laboratories before analysis.</p>
</div>
</div>
<div id="results-2" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Results</h2>
<div id="patients-and-included-isolates" class="section level3" number="7.4.1">
<h3><span class="header-section-number">7.4.1</span> Patients and included isolates</h3>
<p>A total of 10,796 patients were included in this seven-year study. The median age was 67 (IQR: 52-78) and 46.7% (n = 5,040) of the patients was female. A total of 19,803 CoNS isolates were included, of which 14,992 isolates were used for ABR analysis based on the “first weighted isolates” algorithm. A selection of first isolates using solely the CLSI guideline <sup>[24]</sup> would have yielded 12,971 isolates (-13.5%, <em>p</em> &lt; 0.001). On ICUs, 25.7% of the first weighted isolates was found in males compared to 17.0% in females (<em>p</em> &lt; 0.001). The number of ICU patients with CoNS compared to non-ICU patient with CoNS showed a significant difference between secondary care (17.5%, n = 1,403) and tertiary care (24.4%, n = 670, <em>p</em> &lt; 0.001). Yet, no significant difference was observed in the number of CoNS isolates found in ICU patients between secondary care (21.0%, n = 2,191) and tertiary care (22.8%, n = 1,034).</p>
<p class="tbl-caption">
Table 1. Numbers and characteristics per gender of included patients of the included CoNS isolates.
</p>
<p><img src="images/07-t01.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p>At total of 27 different species of the CoNS group were found within the isolate collection (Table 2). The top five species covered 97.1% (n = 14,560) of all first weighted isolates: <em>S. epidermidis</em> (n = 7,260, 48.4%), <em>S. hominis</em> (n = 5,033, 33.6%), <em>S. capitis</em> (n = 1,395, 9.3%), <em>S. haemolyticus</em> (n = 612, 4.1%), and <em>S. warneri</em> (n = 260, 1.7%).</p>
<p>The remaining 432 isolates (2,9%) consisted of: <em>S. lugdunensis</em> (n = 91, 0.6%), <em>S. saprophyticus</em> (n = 45, 0.3%), <em>S. pettenkoferi</em> (n = 44, 0.3%), <em>S. cohnii</em> (n = 43, 0.3%), <em>S. caprae</em> (n = 40, 0.2%), and 17 other species (n = 169, 1.1%).</p>
<p class="tbl-caption">
Table 2. Overview of the total number of isolated CoNS species (not only first isolates) found between 2013 and 2019 in the Northern Netherlands.
</p>
<p><img src="images/07-t02.jpg" width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="occurrence-of-cons-species" class="section level3" number="7.4.2">
<h3><span class="header-section-number">7.4.2</span> Occurrence of CoNS species</h3>
<p>The occurrence of CoNS species was stratified by type of care, type of hospital ward, geographic province, gender, and age (Figure 2). Age was grouped into five groups: 0-11, 12-24, 12-24, 25-54, 55-74, and 75 or more years. When stratifying by species level and the different types of care, the proportion of <em>S. epidermidis</em> among all CoNS isolates was 62.5% in tertiary care (n = 2,834) versus 42.3% in secondary care (n = 4,426; <em>p</em> = 0.049). Overall, <em>S. hominis</em> was less occurrent in tertiary care (20.3%, n = 919) than in secondary care (39.4%, n = 4,114, <em>p</em> = 0.013), while the occurrence of other CoNS species was comparable between secondary and tertiary care. Yet, major differences in relative occurrence were observed between ICU and non-ICU status in secondary care. On secondary care ICUs, <em>S. epidermidis</em> accounted for 55.9% of all first weighted CoNS isolates found while on non-ICU wards this was 39.1% (<em>p</em> &lt; 0.001). In contrast, <em>S. hominis</em> accounted for 25.7% on secondary care ICUs while on non-ICU wards this was 43.3% (<em>p</em> &lt; 0.001). Notably, <em>S. hominis</em> was found 105 times (7.53%) in children under the age of one.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-2"></span>
<img src="images/07-02.jpg" alt="The number of first weighted isolates of the top five CoNS species found in the study stratified by (A) type of care, (B) type of hospital ward, (C) province of the Netherlands, (D), gender, and (E) age group." width="100%" />
<p class="caption">
Figure 7.2: The number of first weighted isolates of the top five CoNS species found in the study stratified by (A) type of care, (B) type of hospital ward, (C) province of the Netherlands, (D), gender, and (E) age group.
</p>
</div>
<p>Although all three provinces in the study region are similar in population density and gender distribution <sup>[23]</sup>, major differences were observed in the occurrence of CoNS species between those provinces in secondary care. The occurrence of <em>S. epidermidis</em> among CoNS species in secondary care hospitals in Friesland was 38.7% in contrast to 43.7% and 45.9% in Drenthe and Groningen respectively (<em>p</em> &lt; 0.001). <em>S. hominis</em> was significantly more often found in secondary care hospitals in Friesland (45.9%) than in Drenthe (33.3%) and Groningen (36.0%) (<em>p</em> &lt; 0.001). Drenthe and Groningen did not differ significantly in the occurrence of CoNS species in secondary care.</p>
<p>Overall, there was no significant change in species distribution over the years. Stratified by gender, a linear increase of <em>S. hominis</em> over time (<em>p</em> = 0.001) and a decrease of <em>S. epidermidis</em> (<em>p</em> = 0.005) was found in males. In females, the occurrence of <em>S. hominis</em> also increased over time (<em>p</em> = 0.008), but no decrease of <em>S. epidermidis</em> or any other species was observed. In age groups, no significant trends in occurrence were observed.</p>
</div>
<div id="definition-of-cons-persistence" class="section level3" number="7.4.3">
<h3><span class="header-section-number">7.4.3</span> Definition of CoNS persistence</h3>
<p>In this retrospective study, it was impossible to differentiate between contaminated blood cultures and BSI-associated blood cultures, as clinical information was not available. Yet, to assess probable cases of BSIs caused by CoNS, we defined ‘CoNS persistence’ as a surrogate. CoNS persistence was defined by at least three positive blood cultures drawn on three different days within 60 days containing the same CoNS species within the same patient. In total, we identified 294 cases of CoNS persistence (Table 3). Aside from <em>S. massiliensis</em> that caused CoNS persistence in only one patient, the relatively most common causal agent of CoNS persistence was <em>S. haemolyticus</em> (5.8%, n = 32, <em>p</em> &lt; 0.001), followed by <em>S. epidermidis</em> (3.7%, n = 212, <em>p</em> &lt; 0.001), and <em>S. lugdunensis</em> (3.4%, n = 3, <em>p</em> = 0.46).</p>
<p class="tbl-caption">
Table 3. The number of patients with and without CoNS persistence per species.
</p>
<p><img src="images/07-t03.jpg" width="100%" style="display: block; margin: auto;" /></p>
</div>
<div id="antibiotic-resistance-analysis" class="section level3" number="7.4.4">
<h3><span class="header-section-number">7.4.4</span> Antibiotic resistance analysis</h3>
<p>Clinically relevant antibiotics and their respective ABR profiles were analysed and compared for the top five CoNS species. Figure 3 shows time trends regarding the ABR profiles to ten different clinically relevant antibiotics, while Table 4 contains resistance percentages of all applicable combinations of species and antibiotic agents. In the following subsections, more detail on occurrence and trends is provided per antibiotic class based on Figure 3 and Table 4. Comprehensive ABR analyses per species of all available variables can be found in Supplementary Table 3.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-3"></span>
<img src="images/07-03.jpg" alt="Antibiotic resistance of the five most occurrent CoNS (n = 14,560) over time between 2013 and 2019. Lines and points are missing where there were less than 30 isolates available for analysis." width="100%" />
<p class="caption">
Figure 7.3: Antibiotic resistance of the five most occurrent CoNS (n = 14,560) over time between 2013 and 2019. Lines and points are missing where there were less than 30 isolates available for analysis.
</p>
</div>
<p class="tbl-caption">
Table 4. Antibiotic resistance in all first weighted CoNS isolates in blood between 2013 and 2019 where at least 30 isolates were available for ABR analysis. Resistance of 100% denotes intrinsic resistance, as defined by EUCAST. Between parentheses are the number of resistant first weighted isolates and the total number of first weighted isolates for that bug-drug combination. The antibiotic names are followed by the official EARS-Net code (European Antimicrobial Resistance Surveillance Network) and ATC code (Anatomical Therapeutic Chemical).
</p>
<p><img src="images/07-t04.jpg" width="100%" style="display: block; margin: auto;" /><img src="images/07-t04b.jpg" width="100%" style="display: block; margin: auto;" /></p>
<div id="glycopeptides" class="section level4" number="7.4.4.1">
<h4><span class="header-section-number">7.4.4.1</span> Glycopeptides</h4>
<p>Vancomycin resistance was found in six <em>S. epidermidis</em> isolates (0.1%) and in one <em>S. hominis</em> isolate (0.0%). Half of all <em>S. epidermidis</em> isolates showed resistance to teicoplanin (50.5%, n = 2,752), which increased over the seven study years (min-max: 44.8%-54.5%, <em>p</em> = 0.001). An increase in teicoplanin resistance was observed in <em>S. haemolyticus</em> (min-max: 10.9%-44.0%, <em>p</em> &lt; 0.001). Teicoplanin resistance remained low in <em>S. capitis</em> (1.4%, n = 17), <em>S. hominis</em> (5.1%, n = 202), and <em>S. warneri</em> (9.6%, n = 22).</p>
</div>
<div id="macrolides" class="section level4" number="7.4.4.2">
<h4><span class="header-section-number">7.4.4.2</span> Macrolides</h4>
<p>Erythromycin resistance was highest in <em>S. haemolyticus</em> (77.6%, n = 437), followed by in <em>S. epidermidis</em> (51.5%, n = 3,471), <em>S. hominis</em> (45.7%, n = 2,086), <em>S. warneri</em> (17.5%, n = 40), and <em>S. capitis</em> (11.0%, n = 136). Resistance to azithromycin and clarithromycin was equal to erythromycin resistance, due to EUCAST expert rules. However, resistance to clindamycin remained lower than resistance to erythromycin in all species: 45.6% (n = 253) in <em>S. haemolyticus</em> and 43.4% (n = 2,910) in <em>S. epidermidis</em>, 29.6% (n = 1,347) in <em>S. hominis</em>, 4.4% (n = 10) in <em>S. warneri</em> ,and 10.8% (n = 132) in <em>S. capitis</em>.</p>
</div>
<div id="fluoroquinolones" class="section level4" number="7.4.4.3">
<h4><span class="header-section-number">7.4.4.3</span> Fluoroquinolones</h4>
<p>The highest ciprofloxacin resistance was found in <em>S. haemolyticus</em> (66.4%; n = 374) and <em>S. epidermidis</em> (51.5%; n = 3,468). Resistance to moxifloxacin was 26.4% (n = 24) in <em>S. haemolyticus</em> and less than 10% in all other species.</p>
</div>
<div id="beta-lactamspenicillins" class="section level4" number="7.4.4.4">
<h4><span class="header-section-number">7.4.4.4</span> Beta-lactams/penicillins</h4>
<p>Oxacillin resistance was as high as 61.9% (n = 4,135) in <em>S. epidermidis</em>, which was thus the proportion of MRSE (methicillin-resistant <em>S. epidermidis</em>) among all <em>S. epidermidis</em> isolates in this study. Oxacillin resistance in <em>S. haemolyticus</em> was even higher (72.1%, n = 403) but considerably lower in all other CoNS species (13.4%-38.6%). Almost all <em>S. epidermidis</em>, <em>S. haemolyticus</em>, and <em>S. hominis</em> were resistant to amoxicillin (95.4%, 93.6%, and 92.8% respectively), while all other species showed amoxicillin resistance ranging between 64.8% and 73.5%. Resistance to amoxicillin/clavulanic acid was 72.9% (n = 3,026) in <em>S. epidermidis</em>. <em>S. haemolyticus</em> showed a strong linear increase in amoxicillin/clavulanic acid resistance (<em>p</em> &lt; 0.001) since 2013 with 87% resistance in 2019 (n = 61).</p>
</div>
<div id="other-antibiotics" class="section level4" number="7.4.4.5">
<h4><span class="header-section-number">7.4.4.5</span> Other antibiotics</h4>
<p>Resistance remained low to rifampicin in <em>S. haemolyticus</em> (5.0%; n = 28) and <em>S. epidermidis</em> (4.5%; n = 300) and remained less than 0.6% in all other species. Linezolid resistance was 0.4% (n = 5) in <em>S. capitis</em>, 0.4% (n = 17) in <em>S. hominis</em>, 0.2% (n = 5) in <em>S. haemolyticus</em>, 0.1% (n = 5) in <em>S. epidermidis</em>, and absent in <em>S. warneri</em>. Mupirocin resistance was 14.8% in <em>S. epidermidis</em> (n = 987, of note: 166 additional isolates tested as “I”) and between 1.7% and 6.5% in other species.</p>
</div>
<div id="other-relevant-species" class="section level4" number="7.4.4.6">
<h4><span class="header-section-number">7.4.4.6</span> Other relevant species</h4>
<p>Resistance in <em>S. lugdunensis</em> (n = 82, sixth most occurrent species) remained generally low: 11.9% (n = 5) to amoxicillin/clavulanic acid, 7.3% (n = 6) to oxacillin, 4.8% (n = 4) to ciprofloxacin, 15.4% (n = 10) to tetracycline, 3.7% (n = 3) to teicoplanin, and no resistance was observed to rifampicin, linezolid, and vancomycin.</p>
<p><em>S. saprophyticus</em> (n = 45, seventh-most occurrent species) showed no resistance to ciprofloxacin, teicoplanin, rifampicin, and vancomycin. Resistance to erythromycin was 15.4% (n = 6), to linezolid 7.9% (n = 3), and to oxacillin 16.2% (n = 6).</p>
<p><em>S. pettenkoferi</em> (n = 44, eighth-most occurrent species) showed no resistance to gentamicin, tobramycin, linezolid, teicoplanin, or vancomycin but resistance to oxacillin was 40.4% (n = 14). Resistance to ciprofloxacin (8.1%, n = 3) and trimethoprim/sulfamethoxazole (2.7%, n = 1) remained low.</p>
</div>
<div id="effect-of-patient-age-groups-on-antibiotic-resistance-in-cons" class="section level4" number="7.4.4.7">
<h4><span class="header-section-number">7.4.4.7</span> Effect of patient age groups on antibiotic resistance in CoNS</h4>
<p>Thirty bug-drug combinations were analysed of which 13 showed a significant linear trend associated with age groups (Figure 4). In <em>S. epidermidis</em>, resistance to beta-lactam antibiotics was found to be lower in older patients (amoxicillin/clavulanic acid: <em>p</em> = 0.002; cefuroxime: <em>p</em> = 0.014). This was also observed in all aminoglycosides (e.g., gentamicin: <em>p</em> = 0.017; tobramycin: <em>p</em> = 0.009), except for kanamycin where higher age was associated with increasing resistance (<em>p</em> = 0.011). <em>S. epidermidis</em> was also less resistant to carbapenems in older patients (imipenem: <em>p</em> = 0.046; meropenem: <em>p</em> = 0.047). In <em>S. hominis</em>, similar trends were observed, although the effect of resistance to kanamycin was stronger (<em>p</em> = 0.006). <em>S. capitis</em> showed significantly more resistance to tetracycline (<em>p</em> = 0.022) in older patients.</p>
<div class="figure" style="text-align: center"><span id="fig:fig7-4"></span>
<img src="images/07-04.jpg" alt="Age group comparison of ABR per antibiotic. Only bug-drug combinations are shown where at least 30 isolates were available for each age group and where results for all age groups were available." width="100%" />
<p class="caption">
Figure 7.4: Age group comparison of ABR per antibiotic. Only bug-drug combinations are shown where at least 30 isolates were available for each age group and where results for all age groups were available.
</p>
</div>
</div>
</div>
</div>
<div id="discussion-3" class="section level2" number="7.5">
<h2><span class="header-section-number">7.5</span> Discussion</h2>
<p>The present study provides a comprehensive analysis of species in the CoNS group and their associated ABR patterns in a full-region approach using solely MALDI-TOF MS for discriminating CoNS species. We selected and analysed a total of 14,992 first weighted CoNS isolates from 10,786 patients over seven years and identified significant differences in the trends of occurrence of the different CoNS species as well as in their ABR patterns.</p>
<p>Before MALDI-TOF MS, CoNS were often reported without the species name as formerly used techniques were not able to reliably discriminate species <sup>[16]</sup>. The ratio of all CoNS species presented in the current study (Table 2) shows that five species accounted for 97.1% of all 27 found CoNS species with <em>S. epidermidis</em> accounting for the largest subgroup (48.4%, n = 7,260). This distribution of species largely confirms results by previous reports <sup>[9,29]</sup>.</p>
<p>For most CoNS species, pathogenicity has not been studied widely due to the lack of data. For this reason, we defined CoNS persistence as at least three positive blood cultures drawn on three different days within 60 days, containing the same CoNS species. This definition was applied for two reasons. Firstly, it rules out contamination since the chance of finding the same contaminating species three times on three different days is expected to be low. Secondly, it prevents underestimating the possible pathogenicity of CoNS species since three sequential findings indicate CoNS persistence. In total, 294 different cases of CoNS persistence were identified (Table 3) among the 10,786 included patients. <em>S. haemolyticus</em> was found to be proportionally more associated with CoNS persistence (5.8%) than <em>S. epidermidis</em> (3.7%) and <em>S. hominis</em> (0.9%), although the latter two were eight to ten times more prevalent than <em>S. haemolyticus</em>. <em>S. epidermidis</em> has widely been recognised as a pathogen and an important cause of BSIs <sup>[5,30]</sup>. It was probably found more often than <em>S. haemolyticus</em> due to its stronger association with skin colonisation <sup>[8]</sup> although we could not confirm this finding. It has been reported that <em>S. haemolyticus</em> is an emerging threat and one of the most frequent aetiological factors of staphylococcal infections <sup>[9,31]</sup>. Adding to this worrisome trend is the great concern of ABR in <em>S. haemolyticus</em> which was reported with 75% of analysed <em>S. haemolyticus</em> isolates to be multi-resistant <sup>[32]</sup>. We confirmed this in the present study in which the ABR analysis showed that 72.1% of <em>S. haemolyticus</em> isolates were resistant to oxacillin and 77.6% resistant to macrolides.</p>
<p>ABR analysis also showed substantial differences between CoNS species (Figure 3, Figure 4, Table 4). This observation could be supported by a recent study that showed strong heterogeneity in the resistance genes for CoNS species <sup>[33]</sup>. For example, the <em>blaZ</em> and <em>aac-aphD</em> genes that can lead to penicillin and aminoglycoside resistance, respectively, were found to be up to four times more common in <em>S. haemolyticus</em> than in other CoNS species <sup>[33]</sup>. The level of resistance to oxacillin and consequent amount of methicillin-resistant <em>S. epidermidis</em> (MRSE) identified in the present study (61.9%) could also be supported by the mentioned study, that reported high prevalence of <em>blaZ</em> in <em>S. epidermidis</em> (64.2%). Although differences in occurrence and ABR within CoNS species are known, they are often neglected, both in studies and in clinical practice. As an example, the Dutch national report on ABR and antibiotic consumption, NethMap, combines all CoNS species into one category making it impossible to distinguish between species. Nonetheless, Dutch treatment guidelines are based on NethMap <sup>[34]</sup>. As an example, in 2019 NethMap reported for isolates found on ICUs 0% linezolid resistance in CoNS, 8% rifampicin resistance, and more than 20% resistance in all other antibiotic classes in 2019. These results could be confirmed in the present study on the group level but not on the species level. The lack of acknowledging ABR differences within species might cause the development of treatment guidelines – and the subsequent future treatment of BSI caused by CoNS – to focus on ‘ABR-safe’ agents for treating CoNS, such as vancomycin or linezolid. Still, agents such as tetracycline, co-trimoxazole, and erythromycin could be considered viable options for some species where, according to our results, ABR never surpassed 10%. Furthermore, as age showed to have a significant effect on ABR (Figure 4), treatment guidelines could also be improved by incorporating age-specific recommendations. We could not find the correlation between ABR in CoNS species and age in previous literature.</p>
<p>In the present study, some CoNS species are noteworthy to be highlighted. For instance, <em>S. pettenkoferi</em> was found only two to three times per year between 2013 and 2017 while this increased to 13 and 22 times per year in 2018 and 2019, respectively. Although recently named, multiple case studies showed that <em>S. pettenkoferi</em> was found to be the causative agent of septic shock, bacteraemia, and wound infections and has also shown resistance to linezolid <sup>[35–37]</sup>. Opposingly, no linezolid resistance was found in the present study. Cases of BSI caused by <em>S. pettenkoferi</em> could incorrectly be assigned to <em>S. capitis</em> that greatly resembles <em>S. pettenkoferi</em> <sup>[38]</sup>. The emerging neonatal pathogen <em>S. capitis</em> is another noteworthy species causing sepsis and manifesting as a multidrug-resistant microorganism <sup>[39]</sup>. In this study, 7.53% of all first weighted <em>S. capitis</em> isolates was found in one-year old children. Clinically relevant ABR (e.g., to chloramphenicol or vancomycin) was not found in these children in this study. This implies that the internationally emerging <em>S. capitis</em> NRCS-A clone <sup>[39]</sup> has not been found in the Northern Netherlands between 2013 and 2019.</p>
<p>Our study has limitations, mostly due to its sole source of routine diagnostic data. Firstly, it was not known which isolates were causal to BSI. This hinders the assessment of contamination as well as the determination of clinical importance. Secondly, the VITEK 2 systems between laboratories used different cartridges with different antibiotics which could lead to an incorporation bias towards some laboratories or hospitals. Additionally, the MALDI-TOF MS systems of all laboratories keep their taxonomic reference data, which is proprietary, and the recency could not be assessed. Thirdly, no genotyping was available for any of the included isolates since genotyping was not considered common practice for routine diagnostic workflows at the time of the study. For this reason, no assessment could be made about a hospital-associated cluster of strains. Lastly, vancomycin resistance might have been underdiagnosed in this study since Vitek2 AST is not optimal for testing glycopeptide resistance <sup>[40]</sup>.</p>
<p>For the first time, a multi-year, full-region approach to comprehensively assess both the occurrence and ABR patterns of CoNS species based on MALDI-TOF MS results was carried out. Although CoNS often lack aggressive virulence properties, evaluating the occurrence and ABR patterns remains highly relevant <sup>[9]</sup>. Stratification by region and demography unveiled a large heterogeneity in ABR between species, settings, and age groups which could be used for (re-)evaluating treatment policies and understanding more about these important but still too often neglected pathogens.</p>
</div>
<div id="supplementary-tables" class="section level2 unnumbered">
<h2>Supplementary tables</h2>
<ol style="list-style-type: decimal">
<li>Supplementary Table 1 (file “supp_tbl1.xlsx”): Extensive output of EUCAST changes to the original data set.</li>
<li>Supplementary Table 2 (file “supp_tbl2.xlsx”): List of species and all available AST test results. This file contains a SHA2 hash (256-bit) of the patient IDs, to be able to reproduce some part of the Results section on the patient level. The hash contains irretrievable information, rendering the data set strictly anonymous.</li>
<li>Supplementary Table 3 (file “supp_tbl3.xlsx”): ABR analysis per species for all available variables.</li>
</ol>
</div>
<div id="references-6" class="section level2 unnumbered">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li>Reinhart K, Daniels R, Kissoon N, Machado FR, Schachter RD, Finfer S. Recognizing Sepsis as a Global Health Priority — A WHO Resolution. N Engl J Med 2017;377:414–7. <a href="doi:10.1056/NEJMp1707170" class="uri">doi:10.1056/NEJMp1707170</a>.</li>
<li>Singer M, Deutschman CS, Seymour CW, Shankar-Hari M, Annane D, Bauer M, <em>et al.</em> The Third International Consensus Definitions for Sepsis and Septic Shock (Sepsis-3). JAMA 2016;315:801–10. <a href="doi:10.1001/jama.2016.0287" class="uri">doi:10.1001/jama.2016.0287</a>.</li>
<li>Cheung GYC, Otto M. Understanding the significance of <em>S. epidermidis</em> bacteremia in babies and children. Curr Opin Infect Dis 2010;23:208–16. <a href="doi:10.1097/QCO.0b013e328337fecb" class="uri">doi:10.1097/QCO.0b013e328337fecb</a>.</li>
<li>Stearns-Kurosawa DJ, Osuchowski MF, Valentine C, Kurosawa S, Remick DG. The pathogenesis of sepsis. Annu Rev Pathol 2011;6:19–48. <a href="doi:10.1146/annurev-pathol-011110-130327" class="uri">doi:10.1146/annurev-pathol-011110-130327</a>.</li>
<li>Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. <em>S. epidermidis</em> as a cause of bacteremia. Future Microbiol 2015;10:1859–79. <a href="doi:10.2217/fmb.15.98" class="uri">doi:10.2217/fmb.15.98</a>.</li>
<li>Rogers KL, Fey PD, Rupp ME. Coagulase-Negative Staphylococcal Infections. Infect Dis Clin North Am 2009;23:73–98. <a href="doi:10.1016/j.idc.2008.10.001" class="uri">doi:10.1016/j.idc.2008.10.001</a>.</li>
<li>Dong Y, Speer CP. The role of <em>S. epidermidis</em> in neonatal sepsis: Guarding angel or pathogenic devil? Int J Med Microbiol 2014;304:513–20. <a href="doi:10.1016/j.ijmm.2014.04.013" class="uri">doi:10.1016/j.ijmm.2014.04.013</a>.</li>
<li>Heilmann C, Ziebuhr W, Becker K. Are coagulase-negative staphylococci virulent? Clin Microbiol Infect 2019;25:1071–80. <a href="doi:10.1016/j.cmi.2018.11.012" class="uri">doi:10.1016/j.cmi.2018.11.012</a>.</li>
<li>Becker K, Heilmann C, Peters G. Coagulase-Negative staphylococci. Clin Microbiol Rev 2014;27:870–926. <a href="doi:10.1128/CMR.00109-13" class="uri">doi:10.1128/CMR.00109-13</a>.</li>
<li>Sievert DM, Ricks P, Edwards JR, Schneider A, Patel J, Srinivasan A, <em>et al.</em> Antimicrobial-Resistant Pathogens Associated with Healthcare-Associated Infections Summary of Data Reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2009–2010. Infect Control Hosp Epidemiol 2013;34:1–14. <a href="doi:10.1086/668770" class="uri">doi:10.1086/668770</a>.</li>
<li>Becker K, Both A, Weißelberg S, Heilmann C, Rohde H. Emergence of coagulase-negative staphylococci. Expert Rev Anti Infect Ther 2020;18:349–66. <a href="doi:10.1080/14787210.2020.1730813" class="uri">doi:10.1080/14787210.2020.1730813</a>.</li>
<li>Rosenstein R, Götz F. What distinguishes highly pathogenic staphylococci from medium- and non-pathogenic? Curr Top Microbiol Immunol 2013;358:33–89. <a href="doi:10.1007/82_2012_286" class="uri">doi:10.1007/82_2012_286</a>.</li>
<li>Idelevich EA, Reischl U, Becker K. New Microbiological Techniques in the Diagnosis of Bloodstream Infections. Dtsch Arztebl Int 2018;115:822–32. <a href="doi:10.3238/arztebl.2018.0822" class="uri">doi:10.3238/arztebl.2018.0822</a>.</li>
<li>Eubank TA, Long SW, Perez KK. Role of Rapid Diagnostics in Diagnosis and Management of Patients With Sepsis. J Infect Dis 2020;222:S103–9. <a href="doi:10.1093/infdis/jiaa263" class="uri">doi:10.1093/infdis/jiaa263</a>.</li>
<li>Pantůček R, Sedláček I, Indráková A, Vrbovská V, Mašlaňová I, Kovařovic V, <em>et al.</em> <em>S. edaphicus</em> sp. nov., Isolated in Antarctica, Harbors the mecC Gene and Genomic Islands with a Suspected Role in Adaptation to Extreme Environments. Appl Environ Microbiol 2018;84:e01746-17. <a href="doi:10.1128/AEM.01746-17" class="uri">doi:10.1128/AEM.01746-17</a>.</li>
<li>Kassim A, Pflüger V, Premji Z, Daubenberger C, Revathi G. Comparison of biomarker based Matrix Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS) and conventional methods in the identification of clinically relevant bacteria and yeast. BMC Microbiol 2017;17:128. <a href="doi:10.1186/s12866-017-1037-z" class="uri">doi:10.1186/s12866-017-1037-z</a>.</li>
<li>Spanu T, Sanguinetti M, Ciccaglione D, D’Inzeo T, Romano L, Leone F, <em>et al.</em> Use of the VITEK 2 system for rapid identification of clinical isolates of Staphylococci from bloodstream infections. J Clin Microbiol 2003;41:4259–63. <a href="doi:10.1128/JCM.41.9.4259-4263.2003" class="uri">doi:10.1128/JCM.41.9.4259-4263.2003</a>.</li>
<li>Lane MJ, Roy AF, Kearney MT, Pucheu-Haston CM. Characterization, distribution, antimicrobial resistance and resistance risk factors in staphylococci isolated from cats from 2001 to 2014. Vet Med Sci 2018;4:315–25. <a href="doi:10.1002/vms3.122" class="uri">doi:10.1002/vms3.122</a>.</li>
<li>Wisplinghoff H, Rosato AE, Enright MC, Noto M, Craig W, Archer GL. Related clones containing SCCmec type IV predominate among clinically significant <em>S. epidermidis</em> isolates. Antimicrob Agents Chemother 2003;47:3574–9. <a href="doi:10.1128/aac.47.11.3574-3579.2003" class="uri">doi:10.1128/aac.47.11.3574-3579.2003</a>.</li>
<li>Ramos-Martín V, Johnson A, Livermore J, McEntee L, Goodwin J, Whalley S, <em>et al.</em> Pharmacodynamics of vancomycin for CoNS infection: experimental basis for optimal use of vancomycin in neonates. J Antimicrob Chemother 2016;71:992–1002. <a href="doi:10.1093/jac/dkv451" class="uri">doi:10.1093/jac/dkv451</a>.</li>
<li>Greeff S, Mouton J. NethMap 2018: Consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in the Netherlands / MARAN 2018: Monitoring of Antimicrobial Resistance and Antibiotic Usage in Animals in the Netherlands in 2017. Bilthoven: Rijksinstituut voor Volksgezondheid en Milieu RIVM Stichting Werkgroep Antibiotica Beleid SWAB; 2018.</li>
<li>Stichting Werkgroep Antibiotica Beleid (SWAB), 2010. Antibacterial therapy of adult patients with sepsis. Retrieved from <a href="https://www.swab.nl/english" class="uri">https://www.swab.nl/english</a>.</li>
<li>Statistics Netherlands [Centraal Bureau voor de Statistiek; CBS]. Regionale kerncijfers Nederland. 2018.</li>
<li>Clinical and Laboratory Standards Institute (CLSI). M39-A4 Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 2014.</li>
<li>Berends MS, Luz CF, Friedrich AW, Sinha BNM, Albers CJ, Glasner C. AMR - An R Package for Working with Antimicrobial Resistance Data. J Stat Softw 2021:Accepted for publication. <a href="doi:10.1101/810622" class="uri">doi:10.1101/810622</a>.</li>
<li>EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Intrinsic Resistance and Exceptional Phenotypes. Version 3.1. 2016.</li>
<li>EUCAST. The European Committee on Antimicrobial Susceptibility Testing. Breakpoint tables for interpretation of MICs and zone diameters. Version 10.0. 2020.</li>
<li>R Core Team. R: A language and environment for statistical computing. 2021.</li>
<li>Otto M. <em>S. epidermidis</em>: a major player in bacterial sepsis? Future Microbiol 2017;12:1031–3. <a href="doi:10.2217/fmb-2017-0143" class="uri">doi:10.2217/fmb-2017-0143</a>.</li>
<li>Ahlstrand E, Hellmark B, Svensson K, Söderquist B. Long-term molecular epidemiology of <em>S. epidermidis</em> blood culture isolates from patients with hematological malignancies. PLoS One 2014;9:e99045. <a href="doi:10.1371/journal.pone.0099045" class="uri">doi:10.1371/journal.pone.0099045</a>.</li>
<li>Czekaj T, Ciszewski M, Szewczyk EM. <em>S. haemolyticus</em> – an emerging threat in the twilight of the antibiotics age. Microbiology 2015;161:2061–8. <a href="doi:10.1099/mic.0.000178" class="uri">doi:10.1099/mic.0.000178</a>.</li>
<li>Barros EM, Ceotto H, Bastos MCF, Dos Santos KRN, Giambiagi-Demarval M. <em>S. haemolyticus</em> as an important hospital pathogen and carrier of methicillin resistance genes. J Clin Microbiol 2012;50:166–8. <a href="doi:10.1128/JCM.05563-11" class="uri">doi:10.1128/JCM.05563-11</a>.</li>
<li>Pedroso SHSP, Sandes SHC, Filho RAT, Nunes AC, Serufo JC, Farias LM, <em>et al.</em> Coagulase-Negative Staphylococci Isolated from Human Bloodstream Infections Showed Multidrug Resistance Profile. Microb Drug Resist 2018;24:635–47. <a href="doi:10.1089/mdr.2017.0309" class="uri">doi:10.1089/mdr.2017.0309</a>.</li>
<li>RIVM/SWAB. NethMap 2019 2019:87.</li>
<li>Vecchia M, Lombardi A, Zuccaro V, Mariani B, Morea A, Asperges E, <em>et al.</em> A clinical case of sepsis due to staphylococcus pettenkoferi. Infez Med 2018;26:369–73.</li>
<li>Mihaila L, Defrance G, Levesque E, Ichai P, Garnier F, Derouin V, <em>et al.</em> A dual outbreak of bloodstream infections with linezolid-resistant <em>S. epidermidis</em> and <em>S. pettenkoferi</em> in a liver Intensive Care Unit. Int J Antimicrob Agents 2012;40:472–4. <a href="doi:10.1016/j.ijantimicag.2012.06.014" class="uri">doi:10.1016/j.ijantimicag.2012.06.014</a>.</li>
<li>Hashi AA, Delport JA, Elsayed S, Silverman MS. <em>S. pettenkoferi</em> Bacteremia: A Case Report and Review of the Literature. Can J Infect Dis Med Microbiol 2015;26:319–22. <a href="doi:10.1155/2015/748154" class="uri">doi:10.1155/2015/748154</a>.</li>
<li>Trülzsch K, Grabein B, Schumann P, Mellmann A, Antonenka U, Heesemann J, <em>et al.</em> <em>S. pettenkoferi</em> sp. nov., a novel coagulase-negative staphylococcal species isolated from human clinical specimens. Int J Syst Evol Microbiol 2007;57:1543–8. <a href="doi:10.1099/ijs.0.64381-0" class="uri">doi:10.1099/ijs.0.64381-0</a>.</li>
<li>Carter GP, Ussher JE, Da Silva AG, Baines SL, Heffernan H, Riley T V., <em>et al.</em> Genomic Analysis of Multiresistant <em>S. capitis</em> Associated with Neonatal Sepsis. Antimicrob Agents Chemother 2018;62. <a href="doi:10.1128/AAC.00898-18" class="uri">doi:10.1128/AAC.00898-18</a>.</li>
<li>Rybak MJ, Vidaillac C, Sader HS, Rhomberg PR, Salimnia H, Briski LE, <em>et al.</em> Evaluation of vancomycin susceptibility testing for methicillin-resistant <em>S. aureus</em>: comparison of Etest and three automated testing methods. J Clin Microbiol 2013;51:2077–81. <a href="doi:10.1128/JCM.00448-13" class="uri">doi:10.1128/JCM.00448-13</a>.</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch06-radar2.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch08-defining-mdr.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["a_new_instrument_for_microbial_epidemiology.pdf", "a_new_instrument_for_microbial_epidemiology.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
