<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>8 Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines | A New Instrument for Microbial Epidemiology</title>
  <meta name="description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="8 Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines | A New Instrument for Microbial Epidemiology" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology" />
  <meta property="og:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />
  <meta property="og:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="github-repo" content="msberends/a_new_instrument_for_microbial_epidemiology" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="8 Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines | A New Instrument for Microbial Epidemiology" />
  
  <meta name="twitter:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="twitter:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />

<meta name="author" content="Matthijs S. Berends" />


<meta name="date" content="2021-08-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ch07-cons.html"/>
<link rel="next" href="ch09-changing-epidemiology.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://doi.org/10.33612/diss.177417131" target="blank"><img src="images/cover.jpg" style="width:90%; margin-left:5%;"></a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preamble</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#short-summary-250-words"><i class="fa fa-check"></i>Short summary (250 words)</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="colophon.html"><a href="colophon.html"><i class="fa fa-check"></i>Colophon</a></li>
<li class="chapter" data-level="" data-path="contents.html"><a href="contents.html"><i class="fa fa-check"></i>Contents</a></li>
<li class="chapter" data-level="1" data-path="ch01-introduction.html"><a href="ch01-introduction.html"><i class="fa fa-check"></i><b>1</b> General Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#microbial-epidemiology"><i class="fa fa-check"></i><b>1.1</b> Microbial epidemiology</a></li>
<li class="chapter" data-level="1.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#antimicrobial-resistance-in-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Antimicrobial resistance in microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#laboratory-diagnostics"><i class="fa fa-check"></i><b>1.2.1</b> Laboratory diagnostics</a></li>
<li class="chapter" data-level="1.2.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#interpretation-of-raw-results"><i class="fa fa-check"></i><b>1.2.2</b> Interpretation of raw results</a></li>
<li class="chapter" data-level="1.2.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#multi-drug-resistant-organisms"><i class="fa fa-check"></i><b>1.2.3</b> Multi-drug resistant organisms</a></li>
<li class="chapter" data-level="1.2.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#surveillance-programs"><i class="fa fa-check"></i><b>1.2.4</b> Surveillance programs</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#data-analysis-using-r"><i class="fa fa-check"></i><b>1.3</b> Data analysis using R</a></li>
<li class="chapter" data-level="1.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#setting-for-this-thesis"><i class="fa fa-check"></i><b>1.4</b> Setting for this thesis</a></li>
<li class="chapter" data-level="1.5" data-path="ch01-introduction.html"><a href="ch01-introduction.html#aim-of-this-thesis-and-introduction-to-its-chapters"><i class="fa fa-check"></i><b>1.5</b> Aim of this thesis and introduction to its chapters</a></li>
<li class="chapter" data-level="" data-path="ch01-introduction.html"><a href="ch01-introduction.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html"><i class="fa fa-check"></i><b>2</b> Diagnostic Stewardship: Sense or Nonsense?!</a>
<ul>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-1"><i class="fa fa-check"></i><b>2.1.1</b> Case 1</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-2"><i class="fa fa-check"></i><b>2.1.2</b> Case 2</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#the-general-concept"><i class="fa fa-check"></i><b>2.2</b> The general concept</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#diagnostics"><i class="fa fa-check"></i><b>2.2.1</b> ‘Diagnostics’</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-in-the-microbiological-laboratory"><i class="fa fa-check"></i><b>2.2.2</b> DSP in the microbiological laboratory</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-as-process-optimisation"><i class="fa fa-check"></i><b>2.2.3</b> DSP as process optimisation</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#multidisciplinary-aspects-of-dsp-and-infection-management"><i class="fa fa-check"></i><b>2.2.4</b> Multidisciplinary aspects of DSP and infection management</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#conclusion"><i class="fa fa-check"></i><b>2.3</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#financing"><i class="fa fa-check"></i>Financing</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html"><i class="fa fa-check"></i><b>3</b> Introducing a New, Free, and Independent Method for Standardised, Reproducible and Reliable Analyses of Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#background"><i class="fa fa-check"></i><b>3.1</b> Background</a></li>
<li class="chapter" data-level="3.2" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#standardising-amr-data-analysis"><i class="fa fa-check"></i><b>3.2</b> Standardising AMR data analysis</a></li>
<li class="chapter" data-level="3.3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#comparison-with-existing-software-methods"><i class="fa fa-check"></i><b>3.3</b> Comparison with existing software methods</a></li>
<li class="chapter" data-level="3.4" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#user-feedback"><i class="fa fa-check"></i><b>3.4</b> User feedback</a></li>
<li class="chapter" data-level="3.5" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#conclusion-1"><i class="fa fa-check"></i><b>3.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html"><i class="fa fa-check"></i><b>4</b> <code>AMR</code> - An <code>R</code> Package for Working with Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.2</b> Antimicrobial resistance data</a></li>
<li class="chapter" data-level="4.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data-transformation"><i class="fa fa-check"></i><b>4.3</b> Antimicrobial resistance data transformation</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-taxonomically-valid-microorganism-names"><i class="fa fa-check"></i><b>4.3.1</b> Working with taxonomically valid microorganism names</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-names-or-codes"><i class="fa fa-check"></i><b>4.3.2</b> Working with antimicrobial names or codes</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-susceptibility-test-results"><i class="fa fa-check"></i><b>4.3.3</b> Working with antimicrobial susceptibility test results</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#interpretative-rules-by-eucast"><i class="fa fa-check"></i><b>4.3.4</b> Interpretative rules by EUCAST</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-defined-daily-doses-ddd"><i class="fa fa-check"></i><b>4.3.5</b> Working with defined daily doses (DDD)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#enhancing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.4</b> Enhancing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-first-isolates"><i class="fa fa-check"></i><b>4.4.1</b> Determining first isolates</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-multi-drug-resistant-organisms-mdro"><i class="fa fa-check"></i><b>4.4.2</b> Determining multi-drug resistant organisms (MDRO)</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#analysing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.5</b> Analysing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#calculation-of-antimicrobial-resistance"><i class="fa fa-check"></i><b>4.5.1</b> Calculation of antimicrobial resistance</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#design-decisions"><i class="fa fa-check"></i><b>4.6</b> Design decisions</a></li>
<li class="chapter" data-level="4.7" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#reproducible-example"><i class="fa fa-check"></i><b>4.7</b> Reproducible example</a></li>
<li class="chapter" data-level="4.8" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#discussion"><i class="fa fa-check"></i><b>4.8</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#computational-details"><i class="fa fa-check"></i>Computational Details</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#references-3"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#appendix-a-included-data-sets"><i class="fa fa-check"></i>Appendix A: Included Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch05-radar.html"><a href="ch05-radar.html"><i class="fa fa-check"></i><b>5</b> Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship</a>
<ul>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="ch05-radar.html"><a href="ch05-radar.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch05-radar.html"><a href="ch05-radar.html#background-1"><i class="fa fa-check"></i><b>5.1.1</b> Background</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch05-radar.html"><a href="ch05-radar.html#objectives"><i class="fa fa-check"></i><b>5.1.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch05-radar.html"><a href="ch05-radar.html#methods"><i class="fa fa-check"></i><b>5.2</b> Methods</a></li>
<li class="chapter" data-level="5.3" data-path="ch05-radar.html"><a href="ch05-radar.html#results"><i class="fa fa-check"></i><b>5.3</b> Results</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch05-radar.html"><a href="ch05-radar.html#overview"><i class="fa fa-check"></i><b>5.3.1</b> Overview</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch05-radar.html"><a href="ch05-radar.html#application-design"><i class="fa fa-check"></i><b>5.3.2</b> Application Design</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch05-radar.html"><a href="ch05-radar.html#development-process"><i class="fa fa-check"></i><b>5.3.3</b> Development Process</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch05-radar.html"><a href="ch05-radar.html#workflow"><i class="fa fa-check"></i><b>5.3.4</b> Workflow</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch05-radar.html"><a href="ch05-radar.html#customisation"><i class="fa fa-check"></i><b>5.3.5</b> Customisation</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch05-radar.html"><a href="ch05-radar.html#discussion-1"><i class="fa fa-check"></i><b>5.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch05-radar.html"><a href="ch05-radar.html#principal-findings"><i class="fa fa-check"></i><b>5.4.1</b> Principal Findings</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch05-radar.html"><a href="ch05-radar.html#conclusions"><i class="fa fa-check"></i><b>5.4.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#conflicts-of-interests"><i class="fa fa-check"></i>Conflicts of interests</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#references-4"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch06-radar2.html"><a href="ch06-radar2.html"><i class="fa fa-check"></i><b>6</b> Better Antimicrobial Resistance Data Analysis and Reporting in Less Time</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="6.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#methods-1"><i class="fa fa-check"></i><b>6.2</b> Methods</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-setup"><i class="fa fa-check"></i><b>6.2.1</b> Study setup</a></li>
<li class="chapter" data-level="6.2.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data"><i class="fa fa-check"></i><b>6.2.2</b> AMR data</a></li>
<li class="chapter" data-level="6.2.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data-analysis-and-reporting"><i class="fa fa-check"></i><b>6.2.3</b> AMR data analysis and reporting</a></li>
<li class="chapter" data-level="6.2.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants"><i class="fa fa-check"></i><b>6.2.4</b> Study participants</a></li>
<li class="chapter" data-level="6.2.5" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-execution-and-data"><i class="fa fa-check"></i><b>6.2.5</b> Study execution and data</a></li>
<li class="chapter" data-level="6.2.6" data-path="ch06-radar2.html"><a href="ch06-radar2.html#evaluation-and-study-data-analysis"><i class="fa fa-check"></i><b>6.2.6</b> Evaluation and study data analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#results-1"><i class="fa fa-check"></i><b>6.3</b> Results</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants-1"><i class="fa fa-check"></i><b>6.3.1</b> Study participants</a></li>
<li class="chapter" data-level="6.3.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#effectiveness-and-accuracy"><i class="fa fa-check"></i><b>6.3.2</b> Effectiveness and accuracy</a></li>
<li class="chapter" data-level="6.3.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#efficiency"><i class="fa fa-check"></i><b>6.3.3</b> Efficiency</a></li>
<li class="chapter" data-level="6.3.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#satisfaction"><i class="fa fa-check"></i><b>6.3.4</b> Satisfaction</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#discussion-2"><i class="fa fa-check"></i><b>6.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#acknowledgements-2"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#funding"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#conflict-of-interest"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#references-5"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix"><i class="fa fa-check"></i>Appendix</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a1-task-lists-including-correct-results"><i class="fa fa-check"></i>Appendix A1: Task lists including correct results</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a2-system-usability-scale-sus"><i class="fa fa-check"></i>Appendix A2: System Usability Scale (SUS)</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a3-task-3-sub-analysis"><i class="fa fa-check"></i>Appendix A3: Task 3 sub-analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ch07-cons.html"><a href="ch07-cons.html"><i class="fa fa-check"></i><b>7</b> Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019</a>
<ul>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#abstract-5"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="7.1" data-path="ch07-cons.html"><a href="ch07-cons.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="ch07-cons.html"><a href="ch07-cons.html#materials-methods"><i class="fa fa-check"></i><b>7.2</b> Materials &amp; methods</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="ch07-cons.html"><a href="ch07-cons.html#study-setting-and-patient-cohort"><i class="fa fa-check"></i><b>7.2.1</b> Study setting and patient cohort</a></li>
<li class="chapter" data-level="7.2.2" data-path="ch07-cons.html"><a href="ch07-cons.html#microbiological-and-demographic-data"><i class="fa fa-check"></i><b>7.2.2</b> Microbiological and demographic data</a></li>
<li class="chapter" data-level="7.2.3" data-path="ch07-cons.html"><a href="ch07-cons.html#species-determination-and-antibiotic-susceptibility-testing-ast"><i class="fa fa-check"></i><b>7.2.3</b> Species determination and antibiotic susceptibility testing (AST)</a></li>
<li class="chapter" data-level="7.2.4" data-path="ch07-cons.html"><a href="ch07-cons.html#selection-of-bacterial-isolates"><i class="fa fa-check"></i><b>7.2.4</b> Selection of bacterial isolates</a></li>
<li class="chapter" data-level="7.2.5" data-path="ch07-cons.html"><a href="ch07-cons.html#eucast-rules-and-antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.2.5</b> EUCAST rules and antibiotic resistance analysis</a></li>
<li class="chapter" data-level="7.2.6" data-path="ch07-cons.html"><a href="ch07-cons.html#statistical-analysis"><i class="fa fa-check"></i><b>7.2.6</b> Statistical analysis</a></li>
<li class="chapter" data-level="7.2.7" data-path="ch07-cons.html"><a href="ch07-cons.html#ethical-considerations"><i class="fa fa-check"></i><b>7.2.7</b> Ethical considerations</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="ch07-cons.html"><a href="ch07-cons.html#results-2"><i class="fa fa-check"></i><b>7.3</b> Results</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="ch07-cons.html"><a href="ch07-cons.html#patients-and-included-isolates"><i class="fa fa-check"></i><b>7.3.1</b> Patients and included isolates</a></li>
<li class="chapter" data-level="7.3.2" data-path="ch07-cons.html"><a href="ch07-cons.html#occurrence-of-cons-species"><i class="fa fa-check"></i><b>7.3.2</b> Occurrence of CoNS species</a></li>
<li class="chapter" data-level="7.3.3" data-path="ch07-cons.html"><a href="ch07-cons.html#definition-of-cons-persistence"><i class="fa fa-check"></i><b>7.3.3</b> Definition of CoNS persistence</a></li>
<li class="chapter" data-level="7.3.4" data-path="ch07-cons.html"><a href="ch07-cons.html#antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.3.4</b> Antibiotic resistance analysis</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="ch07-cons.html"><a href="ch07-cons.html#discussion-3"><i class="fa fa-check"></i><b>7.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#supplementary-tables"><i class="fa fa-check"></i>Supplementary tables</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#references-6"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html"><i class="fa fa-check"></i><b>8</b> Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines</a>
<ul>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#abstract-6"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="8.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#introduction-5"><i class="fa fa-check"></i><b>8.1</b> Introduction</a></li>
<li class="chapter" data-level="8.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#methods-2"><i class="fa fa-check"></i><b>8.2</b> Methods</a></li>
<li class="chapter" data-level="8.3" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-3"><i class="fa fa-check"></i><b>8.3</b> Results</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#number-of-antibiograms-and-patients"><i class="fa fa-check"></i><b>8.3.1</b> Number of Antibiograms and Patients</a></li>
<li class="chapter" data-level="8.3.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-of-mrgn-and-brmo-classification"><i class="fa fa-check"></i><b>8.3.2</b> Results of MRGN and BRMO Classification</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#discussion-4"><i class="fa fa-check"></i><b>8.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#acknowledgements-3"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#conflicts-of-interest"><i class="fa fa-check"></i>Conflicts of interest</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#references-7"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html"><i class="fa fa-check"></i><b>9</b> Changing Epidemiology of Methicillin-Resistant <em>Staphylococcus aureus</em> in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow Policy</a>
<ul>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#abstract-7"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="9.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#introduction-6"><i class="fa fa-check"></i><b>9.1</b> Introduction</a></li>
<li class="chapter" data-level="9.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#methods-3"><i class="fa fa-check"></i><b>9.2</b> Methods</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#setting"><i class="fa fa-check"></i><b>9.2.1</b> Setting</a></li>
<li class="chapter" data-level="9.2.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#guidelines-for-patients-at-risk-for-mrsa-and-infection-prevention-and-control-measures"><i class="fa fa-check"></i><b>9.2.2</b> Guidelines for patients at risk for MRSA and infection prevention and control measures</a></li>
<li class="chapter" data-level="9.2.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-collection"><i class="fa fa-check"></i><b>9.2.3</b> Data collection</a></li>
<li class="chapter" data-level="9.2.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#ethical-statement"><i class="fa fa-check"></i><b>9.2.4</b> Ethical statement</a></li>
<li class="chapter" data-level="9.2.5" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-analysis"><i class="fa fa-check"></i><b>9.2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#results-4"><i class="fa fa-check"></i><b>9.3</b> Results</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#trend-and-cross-border-comparison-of-mrsa-rates"><i class="fa fa-check"></i><b>9.3.1</b> Trend and cross-border comparison of MRSA rates</a></li>
<li class="chapter" data-level="9.3.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#cross-border-comparison-of-healthcare-utilisation"><i class="fa fa-check"></i><b>9.3.2</b> Cross-border comparison of healthcare utilisation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#discussion-5"><i class="fa fa-check"></i><b>9.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#acknowledgements-4"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#funding-1"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#references-8"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html"><i class="fa fa-check"></i><b>10</b> A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures</a>
<ul>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#abstract-8"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="10.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#introduction-7"><i class="fa fa-check"></i><b>10.1</b> Introduction</a></li>
<li class="chapter" data-level="10.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#methods-4"><i class="fa fa-check"></i><b>10.2</b> Methods</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-design"><i class="fa fa-check"></i><b>10.2.1</b> Study Design</a></li>
<li class="chapter" data-level="10.2.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#statistical-analysis-software"><i class="fa fa-check"></i><b>10.2.2</b> Statistical Analysis &amp; Software</a></li>
<li class="chapter" data-level="10.2.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#ethics"><i class="fa fa-check"></i><b>10.2.3</b> Ethics</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#results-5"><i class="fa fa-check"></i><b>10.3</b> Results</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#healthcare-structure-of-the-participating-hospitals"><i class="fa fa-check"></i><b>10.3.1</b> Healthcare structure of the participating hospitals</a></li>
<li class="chapter" data-level="10.3.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-population-and-screening-samples-from-icus"><i class="fa fa-check"></i><b>10.3.2</b> Study population and screening samples from ICUs</a></li>
<li class="chapter" data-level="10.3.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-positive-mdros-mrsa-and-vre"><i class="fa fa-check"></i><b>10.3.3</b> Prevalence of Gram-positive MDROs: MRSA and VRE</a></li>
<li class="chapter" data-level="10.3.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdro-3gcre-and-cre"><i class="fa fa-check"></i><b>10.3.4</b> Prevalence of Gram-negative MDRO: 3GCRE and CRE</a></li>
<li class="chapter" data-level="10.3.5" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdros-based-on-dutch-and-german-definitions"><i class="fa fa-check"></i><b>10.3.5</b> Prevalence of Gram-negative MDROs based on Dutch and German definitions</a></li>
<li class="chapter" data-level="10.3.6" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#comparison-of-mdro-prevalence-between-nl-br-and-de-br-icus-in-university-and-non-university-hospitals"><i class="fa fa-check"></i><b>10.3.6</b> Comparison of MDRO prevalence between NL-BR and DE-BR ICUs in university and non-university hospitals</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#discussion-6"><i class="fa fa-check"></i><b>10.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#supplementary-files"><i class="fa fa-check"></i>Supplementary files</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#acknowledgements-5"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#conflict-of-interest-1"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#references-9"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="ch11-summary.html"><a href="ch11-summary.html"><i class="fa fa-check"></i><b>11</b> Summary and Future Perspectives</a>
<ul>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-i"><i class="fa fa-check"></i>Section I</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-ii"><i class="fa fa-check"></i>Section II</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-iii"><i class="fa fa-check"></i>Section III</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#future-perspectives"><i class="fa fa-check"></i>Future perspectives</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#references-10"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="gearfetting-yn-frysk.html"><a href="gearfetting-yn-frysk.html"><i class="fa fa-check"></i>Gearfetting yn Frysk</a></li>
<li class="chapter" data-level="" data-path="samenvatting-in-het-nederlands.html"><a href="samenvatting-in-het-nederlands.html"><i class="fa fa-check"></i>Samenvatting in het Nederlands</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung-auf-deutsch.html"><a href="zusammenfassung-auf-deutsch.html"><i class="fa fa-check"></i>Zusammenfassung auf Deutsch</a></li>
<li class="divider"></li>
<li class="copyright">© 2021 Matthijs Berends</li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A New Instrument for Microbial Epidemiology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch08-defining-mdr" class="section level1" number="8">
<h1><span class="header-section-number">8</span> Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines</h1>
<p>Published in Microorganisms, 2018 Jan 26;6(1):11</p>
<p>Köck R <sup>1,2,3</sup>, Siemer P <sup>4</sup>, Esser J <sup>5</sup>, Kampmeier S <sup>2</sup>, Berends MS <sup>6,7</sup>, Glasner C <sup>7</sup>, Arends JP <sup>7</sup>, Becker K <sup>3</sup>, Friedrich AW <sup>7</sup></p>
<ol style="list-style-type: decimal">
<li>Institute of Hospital Hygiene Oldenburg, Oldenburg, Germany</li>
<li>Institute of Hygiene, University Hospital Münster, Münster, Germany</li>
<li>Institute of Medical Microbiology, University Hospital Münster, Münster, Germany</li>
<li>European Medical School Oldenburg-Groningen, Oldenburg, Germany</li>
<li>Laborarztpraxis Osnabrück, Georgsmarienhütte, Germany</li>
<li>Certe Medical Diagnostics &amp; Advice Foundation, Groningen, the Netherlands</li>
<li>University of Groningen, University Medical Center Groningen, Department of Medical Microbiology, Groningen, the Netherlands</li>
</ol>
<div id="abstract-6" class="section level2 unnumbered">
<h2>Abstract</h2>
<p>Preventing the spread of multidrug-resistant Gram-negative bacteria (MDRGNB) is a public health priority. However, the definition of MDRGNB applied for planning infection prevention measures such as barrier precautions differs depending on national guidelines. This is particularly relevant in the Dutch-German border region, where patients are transferred between healthcare facilities located in the two different countries, because clinicians and infection control personnel must understand antibiograms indicating MDRGNB from both sides of the border and using both national guidelines. This retrospective study aimed to compare antibiograms of Gram-negative bacteria and classify them using the Dutch and German national standards for MDRGNB definition. A total of 31,787 antibiograms from six Dutch and four German hospitals were classified. Overall, 73.7% were no MDRGNB according to both guidelines. According to the Dutch and German guideline, 7772/31,787 (24.5%) and 4586/31,787 (12.9%) were MDRGNB, respectively (<em>p</em> &lt; 0.0001). Major divergent classifications were observed for extended-spectrum β-lactamase (ESBL) producing <em>Enterobacteriaceae</em>, non-carbapenemase-producing carbapenem-resistant <em>Enterobacteriaceae</em>, <em>Pseudomonas aeruginosa</em> and <em>Stenotrophomonas maltophilia</em>. The observed differences show that medical staff must carefully check previous diagnostic findings when patients are transferred across the Dutch-German border, as it cannot be assumed that MDRGNB requiring special hygiene precautions are marked in the transferred antibiograms in accordance with both national guidelines.</p>
</div>
<div id="introduction-5" class="section level2" number="8.1">
<h2><span class="header-section-number">8.1</span> Introduction</h2>
<p>Antimicrobial multidrug-resistant Gram-negative bacteria (MDRGNB) globally challenge clinicians and infection control personnel due to limited treatment options and the need to implement barrier precautions for preventing MDRGNB transmission <sup>[1]</sup>. Comparing this challenge is particularly interesting in neighbouring regions characterised by highly developed but structurally different healthcare systems. An example for such a region is the Dutch-German border area, which is inhabited by 12 million people and comprises &gt;100 hospitals.</p>
<p>In the Netherlands and Germany, surveillance systems currently indicate that 7.0% and 11.8% of all <em>Escherichia coli</em> and 10.8% and 14.3% of all <em>Klebsiella pneumoniae</em> isolated from blood cultures are non-susceptible to third-generation-cephalosporins indicative for production of extended-spectrum β-lactamases (ESBL) <sup>[2]</sup>. Moreover, carbapenemase-producing <em>Enterobacteriaceae</em> (CPE) occur in both countries, although the overall meropenem or imipenem resistance rates of <em>Enterobacteriaceae</em> (e.g., <em>Klebsiella spp.</em>) are still &lt;1% <sup>[2]</sup>. Thirdly, carbapenem resistance in <em>Acinetobacter baumannii</em>, which is often due to carbapenem-hydrolysing oxacillinase (OXA) production, affects 1.9% and 5.4% of all invasive isolates in The Netherlands and Germany respectively <sup>[2]</sup>. A fourth clinically relevant species is <em>Pseudomonas aeruginosa.</em> For this bacterium, 11% and 18% of all isolates from bloodstream infections were non-susceptible to ceftazidime and meropenem in Germany, respectively. In contrast, resistance rates were 3.5% and 6.1% in The Netherlands <sup>[2]</sup>.</p>
<p>Nosocomial transmission is a major reason why the incidence of MDRGNB increases. Hence, infection control guidelines describing measures to prevent MDRGNB dissemination are implemented in many countries including The Netherlands and Germany. However, it should be noted that, according to data from the European Centre for Disease Prevention and Control (ECDC), Germany is currently considered as a country, where CPE are regionally endemic indicating inter-institutional spread, while their occurrence is more limited in The Netherlands. The same is observed for carbapenem-resistant <em>A. baumannii</em> <sup>[3]</sup>. This highlights the need to critically evaluate and compare infection control guidelines, as well as different risks for MDRGNB spread in these two countries.</p>
<p>In this context, one aspect is the definition of MDRGNB. Although definitions for multidrug resistance in epidemiological studies are available <sup>[4]</sup>, and although theoretically CPE or ESBL-producing <em>Enterobacteriaceae</em> are clearly defined by harbouring respective resistance encoding genes, the questions concerning what MDRGNB are in clinical routine and for which MDRGNB special barrier precautions should be implemented, are not universally defined. Moreover, it is important to differentiate between MDRGNB definitions established for therapeutic decisions and those created for epidemiological purposes and infection prevention <sup>[4]</sup>. Recently, Müller <em>et al.</em> have pointed out differences between the Dutch and German guidelines regarding the advice they give to laboratories and infection control personnel, which Gram-negative bacteria and antimicrobial resistance patterns should be considered as MDRGNB <sup>[5]</sup>. As patient movement across the Dutch-German border is not infrequent, such divergent definitions could result in reduced patient safety, because isolates requiring isolation in the hospital abroad are not flagged as being multidrug-resistant on the microbiological reports.</p>
<p>Hence, in this article, we collected antibiograms of Gram-negative bacteria from Dutch and German hospitals located in the border region and applied both national MDRGNB definitions for infection prevention measures on this dataset. The results of this comparison shall clarify the differences between the two countries and estimate the impact of these differences for daily infection control practice.</p>
</div>
<div id="methods-2" class="section level2" number="8.2">
<h2><span class="header-section-number">8.2</span> Methods</h2>
<p>We retrospectively extracted antibiograms of Gram-negative bacteria from laboratory information systems. Data about phenotypic and genotypic ESBL and carbapenemase tests performed for the respective bacterial isolates were also extracted, if available. All isolates included originate from patients treated in six Dutch and four German hospitals. All hospitals are located in the Dutch-German border region including the Northern part (Ems Dollart region) and the central part (EUREGIO). Five of six Dutch hospitals provided datasets from 1 January 2015 to 31 December 2016, because only a small number of samples was tested in these facilities; the sixth Dutch hospital and the German hospitals provided data for 2016 only. Antimicrobial susceptibility testing was done using guidelines of the European Committee on Antimicrobial Susceptibility Testing (EUCAST) guidelines and clinical breakpoints in all laboratories.</p>
<p>Anonymisation of patient-related and hospital-related data was done before analysis. We initially included all Gram-negative bacterial species and then restricted the dataset to <em>Enterobacteriaceae</em>, P. aeruginosa, <em>A. baumannii</em> complex, and <em>Stenotrophomonas maltophilia</em>, as these are the species for which recommendations regarding MDRGNB definitions and special hygiene precautions are included in Dutch and German infection control guidelines <sup>[6,7]</sup>. We included all isolates; duplicate isolates from the same patient were not removed. Classification of MDRGNB was done according the German national guideline (MDRGNB classified according to this guideline are henceforth designated “Multiresistente Gramnegative Stäbchen,” MRGN, with the subtypes 3MRGN and 4MRGN) summarised in Table 1 <sup>[6]</sup> and according to the Dutch national guideline (MDRGNB according to this guideline are henceforth designated “Bijzonder Resistente Micro-Organismen,” BRMO) shown in Table 2 <sup>[7]</sup>, for all isolates including complete phenotypic susceptibility test data for the antibiotics mentioned. Incomplete antibiograms were deleted from the dataset.</p>
<p class="tbl-caption">
Table 1. Classification according to German guideline into 3MRGN and 4MRGN.
</p>
<p><img src="images/08-t01.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p class="tbl-caption">
Table 2. Classification according to Dutch guideline into BRMO.
</p>
<p><img src="images/08-t02.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p>Statistical analysis was done by Epi Info (version 7.0, CDC Atlanta, Atlanta, GA, USA) using Chi-Square or (where appropriate) Fisher’s exact test; <em>p</em> &lt; 0.05 was considered significant. The final dataset does not allow for conclusions about the epidemiology or the prevalence of MDRGNB, as it contains both isolates obtained from screening asymptomatic patients and clinical specimens. Moreover, the diagnostic procedures and indications for screening and clinical diagnostics were not harmonised in the participating laboratories and hospitals.</p>
</div>
<div id="results-3" class="section level2" number="8.3">
<h2><span class="header-section-number">8.3</span> Results</h2>
<div id="number-of-antibiograms-and-patients" class="section level3" number="8.3.1">
<h3><span class="header-section-number">8.3.1</span> Number of Antibiograms and Patients</h3>
<p>Initially, 35,619 antibiograms were included of which 12,616 were from Dutch and 23,003 from German hospitals. The 12,616 isolates were from five Dutch secondary-care hospitals (n = 4,377; from 2015 to 2016) and one Dutch university-hospital (n = 8,239, 2016), and the 23,003 isolates were from three German secondary-care hospitals (n = 6,914, 2016) and one German university-hospital (n = 16,089, 2016). Overall, 80.9% of all isolates were <em>Enterobacteriaceae</em> and 19.1% non-fermenting Gram-negative bacteria.</p>
<p>When analysing the data, two major limitations occurred: (i) For <em>Enterobacteriaceae</em>, the Dutch classification system could not be applied for 3,832 isolates, because they were not tested for the presence of ESBLs or test results were unclear (n = 3,720 isolates from the German hospitals and n = 112 from Dutch hospitals). This is because testing for the presence of ESBL is not required by the German MRGN classification system (and is often not performed in German laboratories except for <em>E. coli</em> and <em>Klebsiella spp.</em>, where this test is routinely implemented in automated systems used for antimicrobial susceptibility testing). These isolates were therefore excluded from further analysis, which reduced the total number of isolates analysed to 31,787. (ii) Overall, we lacked data for the results of carbapenemase PCRs for non-fermenting bacteria. As carbapenemase PCRs are not required for classification in the German system, these results were not available for 4,651 <em>P. aeruginosa</em> isolates from German hospitals. Since no VIM-carbapenemase was reported for the 1,205 <em>P. aeruginosa</em> isolates from Dutch hospitals, we coped with this problem by assuming that the German <em>P. aeruginosa</em> isolates were also VIM-negative and classified these isolates accordingly when applying the Dutch guideline. In contrast, for <em>A. baumannii</em>, we considered all carbapenem-resistant isolates as carbapenemase producers when applying the Dutch guidelines. For <em>Enterobacteriaceae</em>, test results were available, because German laboratories test the isolates in line with quality management measures.</p>
</div>
<div id="results-of-mrgn-and-brmo-classification" class="section level3" number="8.3.2">
<h3><span class="header-section-number">8.3.2</span> Results of MRGN and BRMO Classification</h3>
<p>According to the Dutch classification system, 7,772/31,787 (24.5%) isolates were BRMO. Applying the German classification system on the same antibiograms resulted in the identification of 4,586/31l,787 (12.9%) MRGN (<em>p</em> &lt; 0.0001). Table 3 shows where the two classification systems had the most divergent results on genus or species level.</p>
<p class="tbl-caption">
Table 3. Differences in using Dutch and German multidrug resistance classification systems.
</p>
<p><img src="images/08-t03.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p>The distribution of 3MRGN and 4MRGN among the 4,586 MRGN isolates is shown in Figure 1. Among all 152 carbapenem-resistant <em>Enterobacteriaceae</em> isolates, carbapenemases were detected in 42 isolates (27.6%) with OXA-48-like genes being predominant. The remaining 110 isolates were negative for carbapenemases (n = 87, 79.1%) or not tested (n = 23, 20.9%) and were meropenem-non-susceptible Morganella, Proteus, Providencia, and Serratia (n = 45), as well as <em>Klebsiella spp.</em> (n = 31), Enterobacter (n = 24), <em>E. coli</em> (n = 7), and Citrobacter (n = 3).</p>
<div class="figure" style="text-align: center"><span id="fig:fig8-1"></span>
<img src="images/08-01.jpg" alt="Species distribution among isolates classified as 3MRGN and 4MRGN according to the German guideline." width="100%" />
<p class="caption">
Figure 8.1: Species distribution among isolates classified as 3MRGN and 4MRGN according to the German guideline.
</p>
</div>
<p>Of all 6,882 isolates classified as BRMO-<em>Enterobacteriaceae</em>, 34 harboured carbapenemase-encoding genes (0.5%), 4,953 were ESBL-producers (80.0%), and 3,037 (44.1%) isolates were simultaneously resistant to fluoroquinolones and aminoglycosides. A total of 788 <em>P. aeruginosa</em> isolates were classified as BRMO, because they had a resistance pattern in accordance with Table 2. Among the remaining 5,058 <em>P. aeruginosa</em> isolates (3,961 from German and 1,107 from Dutch laboratories) not classified as BRMO, 1,107 (21.9%) were carbapenem-resistant (981 and 126 from German and Dutch laboratories, respectively). A total of 72 BRMO-A. baumannii isolates were classified as such, because they were carbapenem-resistant (n = 70), quinolone/aminoglycoside-resistant (n = 2) or both (n = 59).</p>
<p>However, of all isolates 23,433 (73.7%) were neither classified as MRGN, nor as BRMO. Among 3,780 and 806 isolates classified as 3MRGN and 4MRGN according to the German guideline, 3,271 (86.5%) and 733 (90.9%) were also classified as BRMO. In contrast, of the 7,772 isolates classified as BRMO, 3,768 were not classified MRGN (48.5%). An agreement matrix between the Dutch and German guidelines for MDRGNB classification is shown in Table 4.</p>
<p class="tbl-caption">
Table 4. Correlation matrix between the Dutch BRMO-classification and the German MRGN-classification system to define multidrug-resistant Gram-negative bacteria (MDRGNB) for 31,787 isolates of different bacterial species.
</p>
<p><img src="images/08-t04.jpg" width="100%" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="discussion-4" class="section level2" number="8.4">
<h2><span class="header-section-number">8.4</span> Discussion</h2>
<p>When patients are transferred between hospitals, information regarding MDRGNB colonisation or infection must also be transferred to ensure continuous implementation of infection control measures. This is usually supported by indicating on antibiograms, which are included in the records of a transferred patient, whether the respective bacteria are multidrug-resistant according to the national guideline. For cross-border healthcare, this implies that clinicians or infection control staff can interpret antibiograms according to guidelines from both countries or understand foreign ‘MDRGNB languages.’ The aim of this study was to describe different classifications used in The Netherlands and Germany in order to estimate the risk, which might be caused when patients infected or colonised with MDRGNB are transferred across the border without recognizing the respective bacteria as multidrug-resistant.</p>
<p>When planning the data analysis, a first hurdle occurred when the authors tried to actually understand the respective classification guidelines in detail. We learned that parts of the practical applicability of the guidelines (from both sides of the border) are rather locally defined. For example, in the Dutch guideline, it is not explicitly mentioned for <em>Enterobacteriaceae</em>, which fluoroquinolones (e.g., ciprofloxacin, levofloxacin, norfloxacin, moxifloxacin) and aminoglycosides (e.g., gentamicin, tobramycin, amikacin) should be considered for the classification of which bacterial species and how to categorise, if one quinolone is resistant and the other susceptible. In the German guideline, some exceptional rules, such as ignoring imipenem non-susceptibility in <em>Serratia</em> or <em>Proteus</em> for the classification (due to unreliable test results) are not mentioned and can only be concluded from other guidance papers or publications of German reference laboratories. This might cause problems if microbiological laboratories are working across the border and might be perceived as a lack of transparency. This issue could be improved when national policy makers published more detailed standard operating procedures for laboratories where the problems occurring in daily routine are more accurately described.</p>
<p>Overall, the Dutch guideline makes it more laborious for a microbiological laboratory to actually classify an isolate as MDRGNB (tests for phenotypic ESBL-production and VIM-carbapenemase encoding genes). This might reflect structural differences in the organisation of microbiological diagnostics between the two countries, as more laborious confirmation testing requires using more financial resources.</p>
<p>When comparing the Dutch and the German classification systems for MDRGNB (Table 4), we found very divergent results. The bulk of isolates, which were classified differently, were <em>E. coli</em> and <em>Klebsiella</em> isolates characterised by ESBL-production, but being susceptible to fluoroquinolones. In German hospitals, no other than basic hygiene measures are taken for patients colonised or infected with these strains. This can be criticised, because spread of ESBL-producers might increase carbapenem use. Moreover, ESBLs are usually encoded on plasmids, which can be transferred independently from the bacterial clone even to other bacterial species. However, recent investigations have shown, that clonal spread of ESBL-<em>E. coli</em> in healthcare settings is rarely observed <sup>[8,9]</sup>. A second reason for divergent classifications was that the Dutch guideline uses combined fluoroquinolone and aminoglycoside resistance as a criterion for multidrug resistance. Aminoglycosides are not considered in the German guideline, maybe due to their limited and decreasing use in German hospitals compared with the Netherlands (&lt;0.5 vs. 3.7 daily defined doses (DDD)/100 patient-days) <sup>[10,11]</sup>. Thirdly, major differences were also found for <em>P. aeruginosa</em>. Many of the very broadly resistant <em>P. aeruginosa</em> isolates, for which colistin, tobramycin, or new β-lactams (such as ceftolozane/tazobactam) were the only remaining treatment options, were not classified as MDRGNB by the Dutch guideline, because VIM-carbapenemases were not detected. In this context, we clearly overestimated the disagreement between the Dutch and German guideline (Table 4), because we considered all 1,107 carbapenem-resistant <em>P. aeruginosa</em> isolates (of 5,058 not classified as BRMO) as VIM-negative. This is not probable as it is well known that in Germany up to 24% of carbapenem-resistant <em>P. aeruginosa</em> isolates harbour carbapenemases among which VIM is predominant <sup>[12]</sup>. This points towards a major limitation of this study. Since the analysis was not prospectively planned, we had to cope with missing data. Of course, excluding 3,832 antibiograms (which is &gt;10% of the antibiograms collected in the participating hospitals) due to a lack of information about phenotypic ESBL-test results for non-<em>E. coli</em>/non-<em>Klebsiella</em> isolates might have caused significant impact on the results. However, the total numbers <em>Enterobacter</em>, <em>Citrobacter</em>, or <em>Hafnia</em> isolates included from both sides of the border was comparable. Overall, the results of this study demonstrate that in contrast to other multidrug-resistant bacteria such as methicillin-resistant <em>Staphylococcus aureus</em> or vancomycin-resistant enterococci, those resistance pheno- or genotypes that define Gram-negative bacteria as MDRGNB markedly differ between the Netherlands and Germany. For cross-border care, the easiest solution would be to harmonise the classification rules of both countries. As long as this is not done, the full antibiogram data of Gram-negative bacteria should be transferred together with the patient in order to enable classification by local infection control staff.</p>
</div>
<div id="acknowledgements-3" class="section level2 unnumbered">
<h2>Acknowledgements</h2>
<p>This study was supported by the INTERREG V A (202085) funded project EurHealth-1Health, part of a Dutch-German cross-border network supported by the European Commission, the Dutch Ministry of Health, Welfare and Sport (VWS), the Ministry of Economy, Innovation, Digitalisation and Energy of the German Federal State of North Rhine-Westphalia and the German Federal State of Lower Saxony.</p>
</div>
<div id="conflicts-of-interest" class="section level2 unnumbered">
<h2>Conflicts of interest</h2>
<p>The authors declare no conflict of interest. The founding sponsors had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.</p>
</div>
<div id="references-7" class="section level2 unnumbered">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li>Magiorakos, A.P.; Burns, K.; Rodríguez Baño, J.; Borg, M.; Daikos, G.; Dumpis, U.; Lucet, J.C.; Moro, M.L.; Tacconelli, E.; Simonsen, G.S.; <em>et al.</em> Infection prevention and control measures and tools for the prevention of entry of carbapenem-resistant <em>Enterobacteriaceae</em> into healthcare settings: Guidance from the European Centre for Disease Prevention and Control. Antimicrob. Resist. Infect. Control 2017, 6, 113, <a href="doi:10.1186/s13756-017-0259-z" class="uri">doi:10.1186/s13756-017-0259-z</a>.</li>
<li>European Centre for Disease Prevention and Control. EARS-Net Resistance Data for 2016. Available online: <a href="http://atlas.ecdc.europa.eu/public/index.aspx" class="uri">http://atlas.ecdc.europa.eu/public/index.aspx</a> (accessed on 15 December 2017).</li>
<li>Albiger, B.; Glasner, C.; Struelens, M.J.; Grundmann, H.; Monnet, D.L.; European Survey of Carbapenemase-Producing <em>Enterobacteriaceae</em> (EuSCAPE) Working Group. Carbapenemase-producing <em>Enterobacteriaceae</em> in Europe: Assessment by national experts from 38 countries, May 2015. Euro Surveill. 2015, 20, 30062, <a href="doi:10.2807/1560-7917.ES.2015.20.45.30062" class="uri">doi:10.2807/1560-7917.ES.2015.20.45.30062</a>.</li>
<li>Magiorakos, A.-P.; Srinivasan, A.; Carey, R.B.; Carmeli, Y.; Falagas, M.E.; Giske, C.G.; Harbarth, S.; Hindler, J.F.; Kahlmeter, G.; Olsson-Liljequist, B.; <em>et al.</em> Multidrug-resistant, extensively drug-resistant and pandrug-resistant bacteria: An international expert proposal for interim standard definitions for acquired resistance. Clin. Microbiol. Infect. 2012, 18, 268–281, <a href="doi:10.1111/j.1469-0691.2011.03570.x" class="uri">doi:10.1111/j.1469-0691.2011.03570.x</a>.</li>
<li>Müller, J.; Voss, A.; Köck, R.; Sinha, B.; Rossen, J.W.; Kaase, M.; Mielke, M.; Daniels-Haardt, I.; Jurke, A.; Hendrix, R.; <em>et al.</em> Cross-border comparison of the Dutch and German guidelines on multidrug-resistant Gram-negative microorganisms. Antimicrob. Resist. Infect. Control 2015, 4, 7, <a href="doi:10.1186/s13756-015-0047-6" class="uri">doi:10.1186/s13756-015-0047-6</a>.</li>
<li>Kommission für Krankenhaushygiene und Infektionsprävention (KRINKO). Hygienemaßnahmen bei Infektionen oder Besiedlung mit multiresistenten gramnegativen Stäbchen. Bundesgesundheitsbl 2012, 55, 1311–1354.</li>
<li>Werkgroep Infectiepreventie. BRMO (Bijzonder Resistente Micro-Organismen)—Ziekenhuizen. Available online: <a href="http://www.rivm.nl/dsresource?objectid=b6b99580-44e2-4b9c-8183-52871e61764f&amp;type=org&amp;disposition=inline" class="uri">http://www.rivm.nl/dsresource?objectid=b6b99580-44e2-4b9c-8183-52871e61764f&amp;type=org&amp;disposition=inline</a> (accessed on 17 December 2017).</li>
<li>Mellmann, A.; Bletz, S.; Böking, T.; Kipp, F.; Becker, K.; Schultes, A.; Prior, K.; Harmsen, D. Real-Time Genome Sequencing of Resistant Bacteria Provides Precision Infection Control in an Institutional Setting. J. Clin. Microbiol. 2016, 54, 2874–2881, <a href="doi:10.1128/JCM.00790-16" class="uri">doi:10.1128/JCM.00790-16</a>.</li>
<li>Kluytmans-van den Bergh, M.F.Q.; van Mens, S.P.; Haverkate, M.R.; Bootsma, M.C.J.; Kluytmans, J.A.J.W.; Bonten, M.J.M.; SoM Study Group. Quantifying Hospital-Acquired Carriage of Extended-Spectrum Beta-Lactamase-Producing <em>Enterobacteriaceae</em> Among Patients in Dutch Hospitals. Infect. Control Hosp. Epidemiol. 2018, 39, 32–39, <a href="doi:10.1017/ice.2017.241" class="uri">doi:10.1017/ice.2017.241</a>.</li>
<li>SWAB. NethMap 2017 - Consumption of Antimicrobial Agents and Antimicrobial Resistance among Medically Important Bacteria in The Netherlands in 2016. Available online: <a href="http://www.swab.nl" class="uri">http://www.swab.nl</a> (accessed on 17 December 2017).</li>
<li>Paul-Ehrlich Gesellschaft (PEG). Bundesamt für Verbraucherschutz und Lebensmittelsicherheit, GERMAP 2015—Antibiotika-Resistenz und –Verbrauch. Available online: <a href="http://www.p-e-g.org/econtext/germap" class="uri">http://www.p-e-g.org/econtext/germap</a> (accessed on 17 December 2017).</li>
<li>Kaase, M. Bericht des Nationalen Referenzzentrums (NRZ) für gramnegative Krankenhauserreger. Epidemiol. Bull. 2016, 2, 11–14.</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch07-cons.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="ch09-changing-epidemiology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["a_new_instrument_for_microbial_epidemiology.pdf", "a_new_instrument_for_microbial_epidemiology.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
