<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>11 Summary and Future Perspectives | A New Instrument for Microbial Epidemiology</title>
  <meta name="description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="generator" content="bookdown 0.22 and GitBook 2.6.7" />

  <meta property="og:title" content="11 Summary and Future Perspectives | A New Instrument for Microbial Epidemiology" />
  <meta property="og:type" content="book" />
  <meta property="og:url" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology" />
  <meta property="og:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />
  <meta property="og:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="github-repo" content="msberends/a_new_instrument_for_microbial_epidemiology" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="11 Summary and Future Perspectives | A New Instrument for Microbial Epidemiology" />
  
  <meta name="twitter:description" content="This is the integral PhD thesis ‘A New Instrument for Microbial Epidemiology’ (DOI 10.33612/diss.177417131) by Matthijs S. Berends, which was defended publicly at the University of Groningen, the Netherlands, on 25 August 2021." />
  <meta name="twitter:image" content="https://msberends.github.io/a_new_instrument_for_microbial_epidemiology/images/cover.jpg" />

<meta name="author" content="Matthijs S. Berends" />


<meta name="date" content="2021-08-25" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ch10-multi-mdro-screening.html"/>
<link rel="next" href="gearfetting-yn-frysk.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="https://doi.org/10.33612/diss.177417131" target="blank"><img src="images/cover.jpg" style="width:90%; margin-left:5%;"></a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preamble</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#short-summary-250-words"><i class="fa fa-check"></i>Short summary (250 words)</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="colophon.html"><a href="colophon.html"><i class="fa fa-check"></i>Colophon</a></li>
<li class="chapter" data-level="" data-path="contents.html"><a href="contents.html"><i class="fa fa-check"></i>Contents</a></li>
<li class="chapter" data-level="1" data-path="ch01-introduction.html"><a href="ch01-introduction.html"><i class="fa fa-check"></i><b>1</b> General Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#microbial-epidemiology"><i class="fa fa-check"></i><b>1.1</b> Microbial epidemiology</a></li>
<li class="chapter" data-level="1.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#antimicrobial-resistance-in-microorganisms"><i class="fa fa-check"></i><b>1.2</b> Antimicrobial resistance in microorganisms</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="ch01-introduction.html"><a href="ch01-introduction.html#laboratory-diagnostics"><i class="fa fa-check"></i><b>1.2.1</b> Laboratory diagnostics</a></li>
<li class="chapter" data-level="1.2.2" data-path="ch01-introduction.html"><a href="ch01-introduction.html#interpretation-of-raw-results"><i class="fa fa-check"></i><b>1.2.2</b> Interpretation of raw results</a></li>
<li class="chapter" data-level="1.2.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#multi-drug-resistant-organisms"><i class="fa fa-check"></i><b>1.2.3</b> Multi-drug resistant organisms</a></li>
<li class="chapter" data-level="1.2.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#surveillance-programs"><i class="fa fa-check"></i><b>1.2.4</b> Surveillance programs</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="ch01-introduction.html"><a href="ch01-introduction.html#data-analysis-using-r"><i class="fa fa-check"></i><b>1.3</b> Data analysis using R</a></li>
<li class="chapter" data-level="1.4" data-path="ch01-introduction.html"><a href="ch01-introduction.html#setting-for-this-thesis"><i class="fa fa-check"></i><b>1.4</b> Setting for this thesis</a></li>
<li class="chapter" data-level="1.5" data-path="ch01-introduction.html"><a href="ch01-introduction.html#aim-of-this-thesis-and-introduction-to-its-chapters"><i class="fa fa-check"></i><b>1.5</b> Aim of this thesis and introduction to its chapters</a></li>
<li class="chapter" data-level="" data-path="ch01-introduction.html"><a href="ch01-introduction.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html"><i class="fa fa-check"></i><b>2</b> Diagnostic Stewardship: Sense or Nonsense?!</a>
<ul>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#abstract"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#introduction"><i class="fa fa-check"></i><b>2.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="2.1.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-1"><i class="fa fa-check"></i><b>2.1.1</b> Case 1</a></li>
<li class="chapter" data-level="2.1.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#case-2"><i class="fa fa-check"></i><b>2.1.2</b> Case 2</a></li>
</ul></li>
<li class="chapter" data-level="2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#the-general-concept"><i class="fa fa-check"></i><b>2.2</b> The general concept</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#diagnostics"><i class="fa fa-check"></i><b>2.2.1</b> ‘Diagnostics’</a></li>
<li class="chapter" data-level="2.2.2" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-in-the-microbiological-laboratory"><i class="fa fa-check"></i><b>2.2.2</b> DSP in the microbiological laboratory</a></li>
<li class="chapter" data-level="2.2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#dsp-as-process-optimisation"><i class="fa fa-check"></i><b>2.2.3</b> DSP as process optimisation</a></li>
<li class="chapter" data-level="2.2.4" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#multidisciplinary-aspects-of-dsp-and-infection-management"><i class="fa fa-check"></i><b>2.2.4</b> Multidisciplinary aspects of DSP and infection management</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#conclusion"><i class="fa fa-check"></i><b>2.3</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#financing"><i class="fa fa-check"></i>Financing</a></li>
<li class="chapter" data-level="" data-path="ch02-diagnostic-stewardship.html"><a href="ch02-diagnostic-stewardship.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html"><i class="fa fa-check"></i><b>3</b> Introducing a New, Free, and Independent Method for Standardised, Reproducible and Reliable Analyses of Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#abstract-1"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="3.1" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#background"><i class="fa fa-check"></i><b>3.1</b> Background</a></li>
<li class="chapter" data-level="3.2" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#standardising-amr-data-analysis"><i class="fa fa-check"></i><b>3.2</b> Standardising AMR data analysis</a></li>
<li class="chapter" data-level="3.3" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#comparison-with-existing-software-methods"><i class="fa fa-check"></i><b>3.3</b> Comparison with existing software methods</a></li>
<li class="chapter" data-level="3.4" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#user-feedback"><i class="fa fa-check"></i><b>3.4</b> User feedback</a></li>
<li class="chapter" data-level="3.5" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#conclusion-1"><i class="fa fa-check"></i><b>3.5</b> Conclusion</a></li>
<li class="chapter" data-level="" data-path="ch03-introducing-new-method.html"><a href="ch03-introducing-new-method.html#references-2"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html"><i class="fa fa-check"></i><b>4</b> <code>AMR</code> - An <code>R</code> Package for Working with Antimicrobial Resistance Data</a>
<ul>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#abstract-2"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#introduction-1"><i class="fa fa-check"></i><b>4.1</b> Introduction</a></li>
<li class="chapter" data-level="4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.2</b> Antimicrobial resistance data</a></li>
<li class="chapter" data-level="4.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#antimicrobial-resistance-data-transformation"><i class="fa fa-check"></i><b>4.3</b> Antimicrobial resistance data transformation</a>
<ul>
<li class="chapter" data-level="4.3.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-taxonomically-valid-microorganism-names"><i class="fa fa-check"></i><b>4.3.1</b> Working with taxonomically valid microorganism names</a></li>
<li class="chapter" data-level="4.3.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-names-or-codes"><i class="fa fa-check"></i><b>4.3.2</b> Working with antimicrobial names or codes</a></li>
<li class="chapter" data-level="4.3.3" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-antimicrobial-susceptibility-test-results"><i class="fa fa-check"></i><b>4.3.3</b> Working with antimicrobial susceptibility test results</a></li>
<li class="chapter" data-level="4.3.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#interpretative-rules-by-eucast"><i class="fa fa-check"></i><b>4.3.4</b> Interpretative rules by EUCAST</a></li>
<li class="chapter" data-level="4.3.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#working-with-defined-daily-doses-ddd"><i class="fa fa-check"></i><b>4.3.5</b> Working with defined daily doses (DDD)</a></li>
</ul></li>
<li class="chapter" data-level="4.4" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#enhancing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.4</b> Enhancing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.4.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-first-isolates"><i class="fa fa-check"></i><b>4.4.1</b> Determining first isolates</a></li>
<li class="chapter" data-level="4.4.2" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#determining-multi-drug-resistant-organisms-mdro"><i class="fa fa-check"></i><b>4.4.2</b> Determining multi-drug resistant organisms (MDRO)</a></li>
</ul></li>
<li class="chapter" data-level="4.5" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#analysing-antimicrobial-resistance-data"><i class="fa fa-check"></i><b>4.5</b> Analysing antimicrobial resistance data</a>
<ul>
<li class="chapter" data-level="4.5.1" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#calculation-of-antimicrobial-resistance"><i class="fa fa-check"></i><b>4.5.1</b> Calculation of antimicrobial resistance</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#design-decisions"><i class="fa fa-check"></i><b>4.6</b> Design decisions</a></li>
<li class="chapter" data-level="4.7" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#reproducible-example"><i class="fa fa-check"></i><b>4.7</b> Reproducible example</a></li>
<li class="chapter" data-level="4.8" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#discussion"><i class="fa fa-check"></i><b>4.8</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#computational-details"><i class="fa fa-check"></i>Computational Details</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#acknowledgements"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#references-3"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch04-amr-r-package.html"><a href="ch04-amr-r-package.html#appendix-a-included-data-sets"><i class="fa fa-check"></i>Appendix A: Included Data Sets</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="ch05-radar.html"><a href="ch05-radar.html"><i class="fa fa-check"></i><b>5</b> Rapid Analysis of Diagnostic and Antimicrobial Patterns in R (RadaR): Interactive Open-Source Software App for Infection Management and Antimicrobial Stewardship</a>
<ul>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#abstract-3"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="5.1" data-path="ch05-radar.html"><a href="ch05-radar.html#introduction-2"><i class="fa fa-check"></i><b>5.1</b> Introduction</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="ch05-radar.html"><a href="ch05-radar.html#background-1"><i class="fa fa-check"></i><b>5.1.1</b> Background</a></li>
<li class="chapter" data-level="5.1.2" data-path="ch05-radar.html"><a href="ch05-radar.html#objectives"><i class="fa fa-check"></i><b>5.1.2</b> Objectives</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="ch05-radar.html"><a href="ch05-radar.html#methods"><i class="fa fa-check"></i><b>5.2</b> Methods</a></li>
<li class="chapter" data-level="5.3" data-path="ch05-radar.html"><a href="ch05-radar.html#results"><i class="fa fa-check"></i><b>5.3</b> Results</a>
<ul>
<li class="chapter" data-level="5.3.1" data-path="ch05-radar.html"><a href="ch05-radar.html#overview"><i class="fa fa-check"></i><b>5.3.1</b> Overview</a></li>
<li class="chapter" data-level="5.3.2" data-path="ch05-radar.html"><a href="ch05-radar.html#application-design"><i class="fa fa-check"></i><b>5.3.2</b> Application Design</a></li>
<li class="chapter" data-level="5.3.3" data-path="ch05-radar.html"><a href="ch05-radar.html#development-process"><i class="fa fa-check"></i><b>5.3.3</b> Development Process</a></li>
<li class="chapter" data-level="5.3.4" data-path="ch05-radar.html"><a href="ch05-radar.html#workflow"><i class="fa fa-check"></i><b>5.3.4</b> Workflow</a></li>
<li class="chapter" data-level="5.3.5" data-path="ch05-radar.html"><a href="ch05-radar.html#customisation"><i class="fa fa-check"></i><b>5.3.5</b> Customisation</a></li>
</ul></li>
<li class="chapter" data-level="5.4" data-path="ch05-radar.html"><a href="ch05-radar.html#discussion-1"><i class="fa fa-check"></i><b>5.4</b> Discussion</a>
<ul>
<li class="chapter" data-level="5.4.1" data-path="ch05-radar.html"><a href="ch05-radar.html#principal-findings"><i class="fa fa-check"></i><b>5.4.1</b> Principal Findings</a></li>
<li class="chapter" data-level="5.4.2" data-path="ch05-radar.html"><a href="ch05-radar.html#conclusions"><i class="fa fa-check"></i><b>5.4.2</b> Conclusions</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#acknowledgements-1"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#conflicts-of-interests"><i class="fa fa-check"></i>Conflicts of interests</a></li>
<li class="chapter" data-level="" data-path="ch05-radar.html"><a href="ch05-radar.html#references-4"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="ch06-radar2.html"><a href="ch06-radar2.html"><i class="fa fa-check"></i><b>6</b> Better Antimicrobial Resistance Data Analysis and Reporting in Less Time</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#abstract-4"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="6.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#introduction-3"><i class="fa fa-check"></i><b>6.1</b> Introduction</a></li>
<li class="chapter" data-level="6.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#methods-1"><i class="fa fa-check"></i><b>6.2</b> Methods</a>
<ul>
<li class="chapter" data-level="6.2.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-setup"><i class="fa fa-check"></i><b>6.2.1</b> Study setup</a></li>
<li class="chapter" data-level="6.2.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data"><i class="fa fa-check"></i><b>6.2.2</b> AMR data</a></li>
<li class="chapter" data-level="6.2.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#amr-data-analysis-and-reporting"><i class="fa fa-check"></i><b>6.2.3</b> AMR data analysis and reporting</a></li>
<li class="chapter" data-level="6.2.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants"><i class="fa fa-check"></i><b>6.2.4</b> Study participants</a></li>
<li class="chapter" data-level="6.2.5" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-execution-and-data"><i class="fa fa-check"></i><b>6.2.5</b> Study execution and data</a></li>
<li class="chapter" data-level="6.2.6" data-path="ch06-radar2.html"><a href="ch06-radar2.html#evaluation-and-study-data-analysis"><i class="fa fa-check"></i><b>6.2.6</b> Evaluation and study data analysis</a></li>
</ul></li>
<li class="chapter" data-level="6.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#results-1"><i class="fa fa-check"></i><b>6.3</b> Results</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="ch06-radar2.html"><a href="ch06-radar2.html#study-participants-1"><i class="fa fa-check"></i><b>6.3.1</b> Study participants</a></li>
<li class="chapter" data-level="6.3.2" data-path="ch06-radar2.html"><a href="ch06-radar2.html#effectiveness-and-accuracy"><i class="fa fa-check"></i><b>6.3.2</b> Effectiveness and accuracy</a></li>
<li class="chapter" data-level="6.3.3" data-path="ch06-radar2.html"><a href="ch06-radar2.html#efficiency"><i class="fa fa-check"></i><b>6.3.3</b> Efficiency</a></li>
<li class="chapter" data-level="6.3.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#satisfaction"><i class="fa fa-check"></i><b>6.3.4</b> Satisfaction</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="ch06-radar2.html"><a href="ch06-radar2.html#discussion-2"><i class="fa fa-check"></i><b>6.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#acknowledgements-2"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#funding"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#conflict-of-interest"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#references-5"><i class="fa fa-check"></i>References</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix"><i class="fa fa-check"></i>Appendix</a>
<ul>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a1-task-lists-including-correct-results"><i class="fa fa-check"></i>Appendix A1: Task lists including correct results</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a2-system-usability-scale-sus"><i class="fa fa-check"></i>Appendix A2: System Usability Scale (SUS)</a></li>
<li class="chapter" data-level="" data-path="ch06-radar2.html"><a href="ch06-radar2.html#appendix-a3-task-3-sub-analysis"><i class="fa fa-check"></i>Appendix A3: Task 3 sub-analysis</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="ch07-cons.html"><a href="ch07-cons.html"><i class="fa fa-check"></i><b>7</b> Trends in Occurrence and Phenotypic Resistance of Coagulase-Negative Staphylococci (CoNS) Found in Blood in the Northern Netherlands between 2013 and 2019</a>
<ul>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#abstract-5"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="7.1" data-path="ch07-cons.html"><a href="ch07-cons.html#introduction-4"><i class="fa fa-check"></i><b>7.1</b> Introduction</a></li>
<li class="chapter" data-level="7.2" data-path="ch07-cons.html"><a href="ch07-cons.html#materials-methods"><i class="fa fa-check"></i><b>7.2</b> Materials &amp; methods</a>
<ul>
<li class="chapter" data-level="7.2.1" data-path="ch07-cons.html"><a href="ch07-cons.html#study-setting-and-patient-cohort"><i class="fa fa-check"></i><b>7.2.1</b> Study setting and patient cohort</a></li>
<li class="chapter" data-level="7.2.2" data-path="ch07-cons.html"><a href="ch07-cons.html#microbiological-and-demographic-data"><i class="fa fa-check"></i><b>7.2.2</b> Microbiological and demographic data</a></li>
<li class="chapter" data-level="7.2.3" data-path="ch07-cons.html"><a href="ch07-cons.html#species-determination-and-antibiotic-susceptibility-testing-ast"><i class="fa fa-check"></i><b>7.2.3</b> Species determination and antibiotic susceptibility testing (AST)</a></li>
<li class="chapter" data-level="7.2.4" data-path="ch07-cons.html"><a href="ch07-cons.html#selection-of-bacterial-isolates"><i class="fa fa-check"></i><b>7.2.4</b> Selection of bacterial isolates</a></li>
<li class="chapter" data-level="7.2.5" data-path="ch07-cons.html"><a href="ch07-cons.html#eucast-rules-and-antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.2.5</b> EUCAST rules and antibiotic resistance analysis</a></li>
<li class="chapter" data-level="7.2.6" data-path="ch07-cons.html"><a href="ch07-cons.html#statistical-analysis"><i class="fa fa-check"></i><b>7.2.6</b> Statistical analysis</a></li>
<li class="chapter" data-level="7.2.7" data-path="ch07-cons.html"><a href="ch07-cons.html#ethical-considerations"><i class="fa fa-check"></i><b>7.2.7</b> Ethical considerations</a></li>
</ul></li>
<li class="chapter" data-level="7.3" data-path="ch07-cons.html"><a href="ch07-cons.html#results-2"><i class="fa fa-check"></i><b>7.3</b> Results</a>
<ul>
<li class="chapter" data-level="7.3.1" data-path="ch07-cons.html"><a href="ch07-cons.html#patients-and-included-isolates"><i class="fa fa-check"></i><b>7.3.1</b> Patients and included isolates</a></li>
<li class="chapter" data-level="7.3.2" data-path="ch07-cons.html"><a href="ch07-cons.html#occurrence-of-cons-species"><i class="fa fa-check"></i><b>7.3.2</b> Occurrence of CoNS species</a></li>
<li class="chapter" data-level="7.3.3" data-path="ch07-cons.html"><a href="ch07-cons.html#definition-of-cons-persistence"><i class="fa fa-check"></i><b>7.3.3</b> Definition of CoNS persistence</a></li>
<li class="chapter" data-level="7.3.4" data-path="ch07-cons.html"><a href="ch07-cons.html#antibiotic-resistance-analysis"><i class="fa fa-check"></i><b>7.3.4</b> Antibiotic resistance analysis</a></li>
</ul></li>
<li class="chapter" data-level="7.4" data-path="ch07-cons.html"><a href="ch07-cons.html#discussion-3"><i class="fa fa-check"></i><b>7.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#supplementary-tables"><i class="fa fa-check"></i>Supplementary tables</a></li>
<li class="chapter" data-level="" data-path="ch07-cons.html"><a href="ch07-cons.html#references-6"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html"><i class="fa fa-check"></i><b>8</b> Defining Multidrug Resistance of Gram-Negative Bacteria in the Dutch-German Border Region: Impact of National Guidelines</a>
<ul>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#abstract-6"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="8.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#introduction-5"><i class="fa fa-check"></i><b>8.1</b> Introduction</a></li>
<li class="chapter" data-level="8.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#methods-2"><i class="fa fa-check"></i><b>8.2</b> Methods</a></li>
<li class="chapter" data-level="8.3" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-3"><i class="fa fa-check"></i><b>8.3</b> Results</a>
<ul>
<li class="chapter" data-level="8.3.1" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#number-of-antibiograms-and-patients"><i class="fa fa-check"></i><b>8.3.1</b> Number of Antibiograms and Patients</a></li>
<li class="chapter" data-level="8.3.2" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#results-of-mrgn-and-brmo-classification"><i class="fa fa-check"></i><b>8.3.2</b> Results of MRGN and BRMO Classification</a></li>
</ul></li>
<li class="chapter" data-level="8.4" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#discussion-4"><i class="fa fa-check"></i><b>8.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#acknowledgements-3"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#conflicts-of-interest"><i class="fa fa-check"></i>Conflicts of interest</a></li>
<li class="chapter" data-level="" data-path="ch08-defining-mdr.html"><a href="ch08-defining-mdr.html#references-7"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html"><i class="fa fa-check"></i><b>9</b> Changing Epidemiology of Methicillin-Resistant <em>Staphylococcus aureus</em> in 42 Hospitals in the Dutch-German Border Region, 2012 to 2016: Results of the Search-and-Follow Policy</a>
<ul>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#abstract-7"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="9.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#introduction-6"><i class="fa fa-check"></i><b>9.1</b> Introduction</a></li>
<li class="chapter" data-level="9.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#methods-3"><i class="fa fa-check"></i><b>9.2</b> Methods</a>
<ul>
<li class="chapter" data-level="9.2.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#setting"><i class="fa fa-check"></i><b>9.2.1</b> Setting</a></li>
<li class="chapter" data-level="9.2.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#guidelines-for-patients-at-risk-for-mrsa-and-infection-prevention-and-control-measures"><i class="fa fa-check"></i><b>9.2.2</b> Guidelines for patients at risk for MRSA and infection prevention and control measures</a></li>
<li class="chapter" data-level="9.2.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-collection"><i class="fa fa-check"></i><b>9.2.3</b> Data collection</a></li>
<li class="chapter" data-level="9.2.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#ethical-statement"><i class="fa fa-check"></i><b>9.2.4</b> Ethical statement</a></li>
<li class="chapter" data-level="9.2.5" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#data-analysis"><i class="fa fa-check"></i><b>9.2.5</b> Data analysis</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#results-4"><i class="fa fa-check"></i><b>9.3</b> Results</a>
<ul>
<li class="chapter" data-level="9.3.1" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#trend-and-cross-border-comparison-of-mrsa-rates"><i class="fa fa-check"></i><b>9.3.1</b> Trend and cross-border comparison of MRSA rates</a></li>
<li class="chapter" data-level="9.3.2" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#cross-border-comparison-of-healthcare-utilisation"><i class="fa fa-check"></i><b>9.3.2</b> Cross-border comparison of healthcare utilisation</a></li>
</ul></li>
<li class="chapter" data-level="9.4" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#discussion-5"><i class="fa fa-check"></i><b>9.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#acknowledgements-4"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#funding-1"><i class="fa fa-check"></i>Funding</a></li>
<li class="chapter" data-level="" data-path="ch09-changing-epidemiology.html"><a href="ch09-changing-epidemiology.html#references-8"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html"><i class="fa fa-check"></i><b>10</b> A Prospective Multicentre MDRO Screening Study on ICUs in the Dutch-German Cross-Border Region (2017-2018): The Importance of Healthcare Structures</a>
<ul>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#abstract-8"><i class="fa fa-check"></i>Abstract</a></li>
<li class="chapter" data-level="10.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#introduction-7"><i class="fa fa-check"></i><b>10.1</b> Introduction</a></li>
<li class="chapter" data-level="10.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#methods-4"><i class="fa fa-check"></i><b>10.2</b> Methods</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-design"><i class="fa fa-check"></i><b>10.2.1</b> Study Design</a></li>
<li class="chapter" data-level="10.2.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#statistical-analysis-software"><i class="fa fa-check"></i><b>10.2.2</b> Statistical Analysis &amp; Software</a></li>
<li class="chapter" data-level="10.2.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#ethics"><i class="fa fa-check"></i><b>10.2.3</b> Ethics</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#results-5"><i class="fa fa-check"></i><b>10.3</b> Results</a>
<ul>
<li class="chapter" data-level="10.3.1" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#healthcare-structure-of-the-participating-hospitals"><i class="fa fa-check"></i><b>10.3.1</b> Healthcare structure of the participating hospitals</a></li>
<li class="chapter" data-level="10.3.2" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#study-population-and-screening-samples-from-icus"><i class="fa fa-check"></i><b>10.3.2</b> Study population and screening samples from ICUs</a></li>
<li class="chapter" data-level="10.3.3" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-positive-mdros-mrsa-and-vre"><i class="fa fa-check"></i><b>10.3.3</b> Prevalence of Gram-positive MDROs: MRSA and VRE</a></li>
<li class="chapter" data-level="10.3.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdro-3gcre-and-cre"><i class="fa fa-check"></i><b>10.3.4</b> Prevalence of Gram-negative MDRO: 3GCRE and CRE</a></li>
<li class="chapter" data-level="10.3.5" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#prevalence-of-gram-negative-mdros-based-on-dutch-and-german-definitions"><i class="fa fa-check"></i><b>10.3.5</b> Prevalence of Gram-negative MDROs based on Dutch and German definitions</a></li>
<li class="chapter" data-level="10.3.6" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#comparison-of-mdro-prevalence-between-nl-br-and-de-br-icus-in-university-and-non-university-hospitals"><i class="fa fa-check"></i><b>10.3.6</b> Comparison of MDRO prevalence between NL-BR and DE-BR ICUs in university and non-university hospitals</a></li>
</ul></li>
<li class="chapter" data-level="10.4" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#discussion-6"><i class="fa fa-check"></i><b>10.4</b> Discussion</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#supplementary-files"><i class="fa fa-check"></i>Supplementary files</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#acknowledgements-5"><i class="fa fa-check"></i>Acknowledgements</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#conflict-of-interest-1"><i class="fa fa-check"></i>Conflict of interest</a></li>
<li class="chapter" data-level="" data-path="ch10-multi-mdro-screening.html"><a href="ch10-multi-mdro-screening.html#references-9"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="ch11-summary.html"><a href="ch11-summary.html"><i class="fa fa-check"></i><b>11</b> Summary and Future Perspectives</a>
<ul>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-i"><i class="fa fa-check"></i>Section I</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-ii"><i class="fa fa-check"></i>Section II</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#section-iii"><i class="fa fa-check"></i>Section III</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#future-perspectives"><i class="fa fa-check"></i>Future perspectives</a></li>
<li class="chapter" data-level="" data-path="ch11-summary.html"><a href="ch11-summary.html#references-10"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="gearfetting-yn-frysk.html"><a href="gearfetting-yn-frysk.html"><i class="fa fa-check"></i>Gearfetting yn Frysk</a></li>
<li class="chapter" data-level="" data-path="samenvatting-in-het-nederlands.html"><a href="samenvatting-in-het-nederlands.html"><i class="fa fa-check"></i>Samenvatting in het Nederlands</a></li>
<li class="chapter" data-level="" data-path="zusammenfassung-auf-deutsch.html"><a href="zusammenfassung-auf-deutsch.html"><i class="fa fa-check"></i>Zusammenfassung auf Deutsch</a></li>
<li class="divider"></li>
<li class="copyright">© 2021 Matthijs Berends</li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A New Instrument for Microbial Epidemiology</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="ch11-summary" class="section level1" number="11">
<h1><span class="header-section-number">11</span> Summary and Future Perspectives</h1>
<div id="section-i" class="section level2 unnumbered">
<h2>Section I</h2>
<p>Where did microbial epidemiology start? How did it originate? And how does it add to the holistic approach of infection management? These questions are answered in this first section. It is subsequently outlined which important current limitations exist when applying microbial epidemiology in practice and how they could be overcome.</p>
<p>The general introduction of this thesis outlines in <strong>chapter 1</strong> that microbial epidemiology is a part of infectious disease epidemiology, which in turn is a part of clinical microbiology. Microbial epidemiology can be seen, among other things, as the scientific field for acquiring new insights about spreading microorganisms and their respective antimicrobial resistance (AMR) patterns. The advancements in information technology have brought us not only the possibilities to look beyond regional, national, and international borders to get an understanding of the spread of microorganisms and AMR, but even to observe, analyse and understand pandemics in real-time. Methods we develop and use today can be implemented on the other side of the world tomorrow. This is an important advantage in modern microbial epidemiology, which focus is increasingly becoming more data-driven.</p>
<p>To expedite this focus, data are the primary requirement. The data used as input for microbial epidemiological analyses are often obtained from laboratory information systems (LIS). These data consist of routine diagnostic results from laboratory tests. <strong>Chapter 2</strong> brings an opinionated view that diagnostics might lead to raw results, but not to a direct answer to the clinical question that a physician treating a patient might have. Providing physicians with answers requires the approach of a multidisciplinary, intertwined stewardship concept with a focus on diagnostics <sup>[1,2]</sup>. This demands medical specialists in general and microbiologists, in particular, to closely interact for optimal quality of care and patient safety in successful infection management: diagnostic stewardship (DSP). The concept of stewardships, in general, has been widely used to facilitate communication and clinical decision-making, while it proved challenging to establish a clear definition of ‘stewardship’ <sup>[3,4]</sup>. Moreover, diagnostics in clinical microbiology laboratories are currently advancing fast with regards to improved workflows and new technologies, such as matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry <sup>[5,6]</sup>. Yet, diagnostics in infection management is broader than this and covers many clinical areas where communication and interaction are fundamental to make the best use of knowledge and expertise, leading to all specialisms contributing to patient care. The right test at the right time for the right patient to answer the right questions and start the right treatment – this is what DSP in clinical microbiology is about. Microbial epidemiology can be utilised for a small aspect of this diagnostic entirety, by recycling the test results and subsequently bringing enrichments to the answer-generating process that DSP embodies.</p>
<p>This is where <strong>chapter 3</strong> continues, by highlighting important current limitations when applying microbial epidemiology, in particular AMR data analysis. Specifically, AMR data analysis has to be conducted in a clinically and epidemiologically sensible way <sup>[7]</sup>, but is challenging since it requires expertise in (clinical) epidemiology and (clinical) microbiology, and tools to handle the AMR data analysis itself. This is further hindered by the common lack of accessibility of data stored in LIS-es, as most LIS-es are not designed with a focus on epidemiology. As an example, every LIS keeps its own taxonomic data and laboratories are responsible for their regular update. Given that AMR guidelines are strongly based on the microbial taxonomy (some rules only apply to a specific genus, other rules apply to a specific family), this information must be correct and up to date <sup>[8–10]</sup>. Unfortunately, from studying seven clinical microbiology laboratories in the Netherlands, it became apparent that all their LIS-es contained severely outdated taxonomic names. This can impact both routine result reporting and (future) epidemiological analyses. For these reasons, the <code>AMR</code> package for <code>R</code> was introduced in this chapter as a new epidemiological instrument for AMR data analysis that is free, independent, open-source, and publicly available. Developed with a team from twelve different public health organisations in seven different countries, it provides tools to simplify AMR data cleaning, transformation and analysis, as well as methods to easily incorporate (inter)national guidelines, and scientifically reliable reference data. As of May 2021, it has been downloaded at least 50,000 times from 162 different countries since its first release in 2018 <sup>[11]</sup>. The results of a survey among users presented in this chapter showed that its use leads to more reproducibility of analysis results, more reliable outcomes of AMR data analyses, and new or improved insights in AMR for the users’ institutions and regions. Users also stated that the <code>AMR</code> package was used to support clinical decision-making. The package solves the inconvenience of being dependent on (inter)national guidelines and reliable (reference) data, while also providing a comprehensive toolbox for the analysis itself. The <code>AMR</code> package for <code>R</code> can therefore empower any specialist in the field working with AMR data.</p>
</div>
<div id="section-ii" class="section level2 unnumbered">
<h2>Section II</h2>
<p>Following the challenges outlined in the previous section, this section introduces the <code>AMR</code> package for <code>R</code> as a new instrument to cope with these challenges. From multiple viewpoints, the <code>AMR</code> package and its advantages are put into perspective: from a technical viewpoint, from an infection management viewpoint and from a clinical viewpoint. These combined provide a common ground for understanding the explications that the <code>AMR</code> package can yield in the field and how it can set a new empowered starting point for future applications of microbial epidemiology.</p>
<p>The technical functionalities of the <code>AMR</code> package for <code>R</code> have been described in <strong>chapter 4</strong>, where it is described how the <code>AMR</code> package has been developed to standardise clean and reproducible AMR data analyses using international standardised recommendations <sup>[9,12]</sup>. To facilitate this, scientifically reliable reference data are incorporated regarding valid laboratory results (as opposed to e.g., non-existing MIC values), antimicrobial agents, and the complete biological taxonomy of microorganisms. Source data should be analysed in the most reliable way, especially when for example the outcome will be used to evaluate patient treatment options. This requires reproducible and field-specific, specialised data cleaning and transforming. The <code>AMR</code> package provides a standardised and automated way of cleaning, transforming, and enhancing common LIS data, independent of the underlying data source and data accuracy. For this reason, general algorithms were developed to clean AMR test results and to validate the names of microorganisms and antimicrobial agents. The equation for taxonomic name validation takes into account the human pathogenic prevalence of microorganisms and is context-aware about other taxonomic properties such as the kingdom, phylum, order and family. To exemplify, a data value “E. coli” will be translated to the bacterium <em>Escherichia coli</em>, while informing the user that the parasite <em>Entamoeba coli</em> is also eligible but has a lower likelihood. Using convenient functions, users can quickly retrieve consistent microbial properties, such as the taxonomic kingdom, phylum, class, order, family, genus, species, subspecies, previously accepted names and even the Gram stain. Aside from information about microorganisms, the package also includes reference data about antibiotics, which comprises common laboratory information system codes, official names, ATC (Anatomical Therapeutic Chemical) codes, ATC group names, defined daily doses (DDD) and more than 5,000 trade names of 456 antimicrobial agents. Using these reference data, users can translate raw data and retrieve properties about any microorganism or antimicrobial drug. Furthermore, the <code>AMR</code> package is capable of determining multi-drug resistant organisms (MDROs) based on national and international guidelines, interpreting raw minimum inhibitory concentrations (MICs) and can determine first isolates to be used for calculating AMR of both monotherapy and combination therapies. The <code>AMR</code> package itself was meant as a comprehensive instrument for data-technical staff working in the field of AMR, although its use is not limited to this group.</p>
<p>To exemplify this, <strong>chapter 5</strong> shows that the <code>AMR</code> package was used as a backbone in an interactive open-source software app for infection management and antimicrobial stewardship, called RadaR (rapid analysis of diagnostic and antimicrobial patterns in R). Infection management in the form of antimicrobial stewardship (AMS) programs has emerged as an effective solution to address this global health problem in hospitals <sup>[3]</sup>. Connecting to <strong>chapter 2</strong>, stewardship interventions and activities focus on individual patients (personalised medicine and consulting) as well as patient groups or clinical syndromes (guidelines, protocols, information technology infrastructure, and clinical decision support systems) while prioritising improvement in quality of care and patient safety for any intervention <sup>[13,14]</sup>. However, easy access to analyse patient groups (e.g., stratified by departments or wards, specific antimicrobials, or diagnostic procedures used) is difficult to implement in daily practice. It is even more challenging to rapidly analyse larger patient populations (e.g., spread over multiple specialities) even though this information might be available in the data. Therefore, the development of RadaR was intended to serve AMS teams with a user-friendly and time-saving data analysis resource, without the need for profound technical expertise. RadaR was developed for graphical exploratory (AMR) data analysis. Among others, it provides Kaplan-Meier curves about lengths of hospitals stays, time trends for the number of admissions, antimicrobial consumption, and an automated AMR data analysis for which the <code>AMR</code> package for <code>R</code> was used. RadaR was validated by 12 ESGAP members (European Society of Clinical Microbiology and Infectious Diseases Study Group for Antimicrobial Stewardship) from 9 different countries. It has the potential to be a highly useful tool for infection management and AMS teams in daily practice. Additionally, this chapter shows that the <code>AMR</code> package can be used as part of another software solution to empower integrated infection management.</p>
<p>Following this insight, <strong>Chapter 6</strong> demonstrates the effectiveness of the <code>AMR</code> package among users, by evaluating its usability and impact on clinicians’ workflows in a typical hospital scenario. Although the use of the <code>AMR</code> package in research has been demonstrated in multiple studies from different countries already <sup>[15–18]</sup>, the impact on workflows for AMR data analysis and reporting in clinical settings was still pending. AMR data analysis and reporting, unfortunately, require specifically skilled personnel. Moreover, thorough and in-depth analyses can be time-consuming and sufficient resources need to be allocated for consistent and repeated reporting. To determine the impact of these facts in a clinical setting, common questions about blood culture data were formulated that had to be answered by routine clinical personnel, including clinical microbiologists, paediatricians and intensivists. In total, ten clinicians participated in the study. Additionally, participants were asked to fill in an online questionnaire capturing their backgrounds, demographics, software experience, and experience in AMR data analysis and reporting. All participants had to answer the study questions twice: the first time with their software of choice (round 1) and the second time using a newly developed web application built around the <code>AMR</code> package for <code>R</code> (round 2). The development of this web application was utilised in a highly efficient and agile workflow. The answers to the list of questions served as the basis to compare the effectiveness (solvability of each task for every user) and efficiency (time spent solving each task) between the two rounds. Not all participants were able to complete the tasks within the given time frame. Average task completion between the first and second round increased from 56% (SD: 23%) to 96% (SD: 6%). The proportion of correct answers between the first and second round increased from 38% to 98%. The mean time spent per round was reduced from 94 minutes (SD: 22 minutes) to 22 minutes (SD: 14 minutes). This chapter demonstrates the increased effectiveness, efficiency, and accuracy of using the <code>AMR</code> package for <code>R</code> for AMR data analysis compared to traditional software applications such as Microsoft Excel and SPSS.</p>
</div>
<div id="section-iii" class="section level2 unnumbered">
<h2>Section III</h2>
<p>Many clinical studies in the field of infectious diseases and microbiology rely on some form of (microbial) epidemiology. While the <code>AMR</code> package was presented in the previous section and its use in different settings was showcased, this section starts with an epidemiological research projects in the Northern Dutch region, and then extends to the Dutch-German cross-border region to better understand the occurrence and AMR patterns of pathogens on a (eu)regional level. Focusing on the regions on each side of a national border allows comparisons between two different nations on the micro level. And different nations ultimately mean different healthcare systems. What is left of ‘One Health?’ What are the implications on comparison of having differences between countries in AMR test methodologies, MDRO interpretations and screening policies? This section provides answers to these questions.</p>
<p><strong>Chapter 7</strong> zooms in on coagulase-negative staphylococci (CoNS), which are known to cause bloodstream infection (BSI) and a high mortality rate, although for years they had often been regarded as contamination <sup>[19–23]</sup>. Moreover, CoNS have become increasingly associated with nosocomial infections <sup>[24]</sup>. At present, the CoNS group consists of 45 different species, although determining the species level has only recently been made possible for routine diagnostic laboratories <sup>[25–27]</sup>. Since 2012, MALDI-TOF mass spectrometry has become the standard for the identification of bacterial species such as CoNS. Before that, identification of CoNS was primarily done with biochemical and physiological tests, which yielded generally variable results, in particular in less prevalent species <sup>[27]</sup>. AMR, and especially multi-drug resistance, is also an increasing problem in CoNS <sup>[28]</sup>. Nonetheless, treatment guidelines and national surveillance programs (such as the Dutch NethMap) still gather CoNS as a whole group, lacking differentiation between species <sup>[29]</sup>. Consequently, little is known about trends in occurrence and AMR in CoNS on the local and regional level. Therefore, this retrospective study shows an in-depth AMR analysis of 19,803 CoNS isolates found in all available 71,632 blood culture isolates between 2013 and 2019 in the Northern Netherlands that were determined by MALDI-TOF MS. This study followed a full-region approach by covering the whole Northern Netherlands. Through this analysis, we aimed to evaluate the differences in the occurrence of CoNS species and their AMR patterns and to assess their clinical microbiological relevance to this end. A total of 27 different species of the CoNS group were found. Major differences were observed in the occurrence of the different species: the top five species covered 97.1% of all included isolates. These were: <em>S. epidermidis</em> (48.4%), <em>S. hominis</em> (33.6%), <em>S. capitis</em> (9.3%), <em>S. haemolyticus</em> (4.1%) and <em>S. warneri</em> (1.7%), meaning that the remaining 2.9% of isolates consisted of 22 different CoNS species. The proportion of CoNS in intensive care units (ICUs) compared to other departments was also found to be significantly different between secondary care (17.5% of isolates from ICU) and tertiary care (24.4%% of isolates from ICU). As it was unknown which patients had BSI, ‘CoNS persistence’ was defined as a surrogate having at least three positive blood cultures drawn on three different days within 60 days, containing the same CoNS species, within the same patient. The relatively most common causal agent of CoNS persistence was <em>S. haemolyticus</em> (5.8% of all patients with <em>S. haemolyticus</em>), followed by <em>S. epidermidis</em> (3.7%,) and <em>S. lugdunensis</em> (3.4%). AMR analysis has shown substantial differences between CoNS species and was presented thoroughly per antibiotic class in tables and text. For example, <em>S. epidermidis</em> and <em>S. haemolyticus</em> showed 50% to 80% resistance to teicoplanin, erythromycin, ciprofloxacin, and oxacillin, while resistance to these agents remained lower than 10% in most other CoNS species. Yet, these differences are neglected on the national level such as in NethMap, which might cause the development of treatment guidelines to focus on ‘AMR-safe’ agents for treating CoNS, such as vancomycin or linezolid. Nonetheless, agents such as tetracycline, co-trimoxazole, and erythromycin could be considered viable options for some species, where according to the study results, AMR never surpassed 10%. In conclusion, a multi-year full-region approach to extensively assess the trends in both the occurrence and AMR of CoNS species was carried out, which could be used for evaluating treatment policies and understanding more about these important but still too often neglected pathogens. Furthermore, this study served as a practical research example of how the <code>AMR</code> package for <code>R</code> can be used to gain new AMR insights using epidemiologically sounds methods.</p>
<p>Following new insights by studying AMR test results in the Northern Netherlands, <strong>chapter 8</strong> provides a comparison of AMR test results and their national interpretations of MDROs in the Dutch-German cross-border region, especially concerning the practical impact on cross-border healthcare workers. Comparing AMR in general, not only MDROs, in this cross-border region is particularly interesting since both countries are characterised by highly developed but structurally different healthcare systems. AMR interpretations in patient records are transferred between healthcare facilities located in these two different countries, while the underlying definitions differ. This causes the need for clinicians and infection control personnel to understand AMR results from both sides of the border and to be able to comprehend both national MDRO interpretation guidelines. By comparing antibiograms of Gram-negative bacteria from both sides of the border, the degree of impact of these challenges was sought to determine. To this end, 35,619 antibiograms from six Dutch and four German hospitals were analysed between 2015 and 2016 of all species of <em>Enterobacteriaceae</em>, and P. aeruginosa, the A. baumannii complex and Stenotrophomonas maltophilia. MDRO recommendations and special hygiene precautions exist in this region for all of these species. On the Dutch side of the border, isolate selection was carried out using the <code>AMR</code> package. From the Dutch hospitals, 12,616 antibiograms were selected using the <code>AMR</code> package for <code>R</code> applying the Dutch MDRO interpretation guideline. Of note, other national and international guidelines, such as the German MDRO interpretation guideline, are also included in the <code>AMR</code> package for <code>R</code>. From German hospitals, 23,003 antibiograms were selected using other methods. According to the Dutch guideline, 24.5% of all isolates were an MDRO. According to the German guideline, 12.9% of all isolates were an MDRO. However, of all isolates, 73.7% were not classified as an MDRO according to either guideline. Among all carbapenem-resistant <em>Enterobacteriaceae</em> isolates, carbapenemases were detected in 27.6% with OXA-48-like genes being predominant. The remaining isolates were negative for carbapenemases (79.1%) or not tested (20.9%). When patients are transferred between hospitals, information regarding MDRO colonisation or infection must also be transferred to ensure continuous implementation of infection control measures. For cross-border healthcare, this implies that clinicians or infection control staff should be able to determine MDROs based on antibiograms according to guidelines from either of the two countries. For cross-border healthcare, the easiest solution would be to harmonise the classification rules of both countries. This would likewise solve the understandable confusion patients might experience if infection control measures are imposed in one country, but relieved after transfer to another country. As long as the harmonisation is not done, the full AMR data of Gram-negative bacteria should be transferred together with the patient to enable classification by local infection control staff.</p>
<p>Other AMR-related cross-border challenges and differences are illustrated in <strong>chapter 9</strong>, which comprises a comprehensive microbial epidemiological analysis of MRSA occurrence, policies, and healthcare effects in the Dutch-German border region. MRSA is still one of the major causes of healthcare-associated infections due to AMR pathogens <sup>[30]</sup>. In this study, MRSA surveillance data of five years (2012-2016) from Dutch and German cross-border region hospitals were analysed to describe temporal and spatial trends of MRSA rates and find differences between these groups of hospitals. The research setting comprised 42 hospitals located in the Dutch-German cross-border region treating approximately 620,000 admitted patients (68.0% in the German part of the study region) with 3.9 million patient days per year. All hospitals had implemented MRSA-related infection prevention control measures according to their national guidelines and recommendations, and the guideline differences between the two countries were compared. On both sides of the border, the median nasopharyngeal MRSA screening rate increased significantly between 2012 and 2016, although the median MRSA incidence remained stable over time at both sides of the border. Overall, the median screening rate was 14 times higher in the German border region (DE-BR) than in the Dutch border region (NL-BR). The median percentage of MRSA in <em>S. aureus</em> blood culture isolates decreased from 12.5% in 2012 to 5.0% in 2016 in DE-BR, while it remained stable at 0% to 1.9% in NL-BR. Nonetheless, MRSA among <em>S. aureus</em> isolates was 34 times higher in DE-BR. The in-hospital length of stay of MRSA patients was similar in both regions, while the general length of stay differed significantly. Furthermore, the number of nasopharyngeal MRSA screening swabs before or at admission to hospital per 100 inhabitants was 12.2 in DE-BR and 0.36 in NL-BR, also 34 times higher in DE-BR. The number of inpatient MRSA cases per 1,000 inhabitants was 2.52 in DE-BR and 0.14 in NL-BR. Thus, this study revealed significant differences between Dutch and German hospitals. The median MRSA incidence in DE-BR hospitals was more than seven times higher than in NL-BR hospitals. According to the European Centre of Disease Prevention and Control (ECDC), differences in the occurrence of AMR pathogens between European countries are most likely caused by differences in healthcare utilisation, antimicrobial use and infection prevention control practices <sup>[31]</sup>. Concerning healthcare utilisation in our context, we found that inhabitants in the German part of the study region were almost three times as often hospitalised and had a significantly longer length of stay than patients on the Dutch part. This may be due to socioeconomic factors or a different organisation of ambulatory healthcare. This comprehensive study on MRSA covering hospitals across a European border demonstrated that routine MRSA surveillance may be helpful to monitor trends of MRSA parameters, to enable (inter)national comparisons.</p>
<p>The discussion of this study concluded with “cross-border surveillance should be extended to other multidrug-resistant organisms,” which is where <strong>chapter 10</strong> continues. Given that not only MRSA but MDROs, in general, pose a risk for healthcare, both in the community and hospitals, the study aimed to determine the prevalence of multiple MDROs in this cross-border region to understand differences and improve infection prevention based on real-time routine data and workflows. To this end, 23 hospitals in the Dutch-German cross-border region (NL-BR and DE-BR) participated between 2017 and 2018 in this prospective study by screening all patients upon admission to intensive care units (ICUs). All hospitals (8 in NL-BR, 15 in DE-BR) enrolled for eight consecutive weeks and screened patients for nasal carriage of MRSA and rectal carriage of vancomycin-resistant <em>Enterococcus faecium</em>/<em>E. faecalis</em> (VRE), third-generation cephalosporin-resistant <em>Enterobacteriaceae</em> (3GCRE) and carbapenem-resistant <em>Enterobacteriaceae</em> (CRE). A total of 3,365 patients were screened: 35.7% on NL-BR ICUs and 64.3% on DE-BR ICUs. The median age of all screened patients was 68 years (IQR: 57-77), while patients in DE-BR were significantly older than patients in the NL-BR. A total of 6,462 swabs were processed. The overall screening compliance (screened for at least one MDRO group) was 60.4%, in NL-BR 56.9% and in DE-BR 62.9%. All AMR data analyses were carried out and automated using the <code>AMR</code> package for <code>R</code>. The prevalence of MRSA was 1.7% in DE-BR ICUs and 0.6% in NL-BR ICUs. The prevalence of VRE was 2.7% in DE-BR ICUs and 0.1% in NL-BR ICUs. Notably, this prevalence ranged from 0% to 4.1% in DE-BR. All 56 cases of VRE were caused by E. faecium. The prevalence of 3GCRE was 6.6% in DE-BR ICUs and 3.6% in NL-BR ICUs, whereas the prevalence for CRE was practically non-existent on both sides of the border. The prevalence for Gram-negative MDROs differed within both countries between hospitals, ranging from 0% to 5.0% in NL-BR and from 1.2% to 10.9% in DE-BR ICUs. For NL-BR ICUs, the prevalence of all MDRO groups was not significantly different between the non-university and the university hospital. For the DE-BR ICUs however, the prevalence of Gram-negative MDROs was significantly higher in the non-university hospitals. In the NL-BR, 4.8 per 100 hospital admissions led to ICU admission. In contrast, in the DE-BR this was 7.7 per 100 hospital admissions. This difference can be explained by the higher ICU capacity in DE-BR hospitals (4.8% of all hospital beds) compared to NL-BR hospitals (2.4% of all hospital beds). The overall carriage prevalence for the different MDROs was higher in the DE-BR ICUs, although some differences were marginal. Specifically, the prevalence of MRSA carriage was three times higher in the DE-BR (1.7%) than in the NL-BR (0.6%). These prevalences are consistent with the study mentioned in <strong>chapter 9</strong>. The difference in MDRO prevalence between NL-BR and DE-BR was observed for all MDROs groups. Yet, the study findings were not all comparable with (inter)national averages. For example, the 3GCRE carriage prevalence in the DE-BR was almost twice as high (6.6%) as in the NL-BR (3.6%), but both were still lower than national averages. The ECDC reported 3GCRE proportions among blood culture isolates of <em>E. coli</em> and <em>K. pneumoniae</em> as 12.2% to 12.9% for Germany and 7.3% to 11.1% for the Netherlands. This highlights that there are important differences to be found when studying carriage in specified populations versus looking at the proportion of (probably) invasive isolates. Thus, this study highlights the importance of a regional and cross-border approach in any European cross-border region, to illustrate the difference in AMR prevalence between the regions and to highlight potential differences with country-wide reports. Attaining a deeper level of detail is required to be able to elaborate on this further, for example by collecting information about staff on the wards and infection control staff, MDRO outbreaks, infections, antibiotic use and risk factors of patients. In conclusion, geographical and political borders do not seem to be “respected” by MDROs, although healthcare systems, geographic nature and guidelines are very different between countries. Proportions of MDROs of certain pathogens, as reported on the national and international level, do not reflect MDRO prevalence in the patient or general population. This should be taken into serious consideration when interpreting reports on the country or even continental level.</p>
</div>
<div id="future-perspectives" class="section level2 unnumbered">
<h2>Future perspectives</h2>
<blockquote>
<p>After hearing for several decades that computers will soon be able to assist with difficult diagnoses, the practising physician may well wonder why the revolution has not occurred. Scepticism at this point is understandable. Few, if any, programs currently have active roles as consultants to physicians. The story behind these unfulfilled expectations is instructive and, we believe, offers hope for the future.</p>
</blockquote>
<p>These words are from Schwartz <em>et al.</em> and, unfortunately, not very recent. It was published 34 years ago in The New England Journal of Medicine in 1987 <sup>[36]</sup>. Many might find it quite disappointing that this exact quote can still apply to current times. Yet, this is not due to a lack of technological advancements – computational power and software capabilities have increased significantly over the last decades. And with them, the enablement of making optimal use of existing data to aid clinical decision-making and to support medicine as a whole. Hence, if it is not due to lack of technological advancements, what is then inhibiting the use of these advancements for clinical use? Others pointed out that the answer might be the gap in culture between the clinicians, biomedical scientists, and those skilled in computer programming <sup>[37,38]</sup>. To this end, one might contemplate whether multi-disciplinarity was imbedded well enough into our integral medical field, since the differences are not only cultural. While both clinicians and biomedical scientists endure more than a decade of specialised training and education in a similar field, they often (1) do not speak each other’s language, (2) lack a common value system, even regarding knowledge and ignorance, and (3) have different sources of passion and emotional intensity <sup>[37]</sup>. Scientists have to focus on asking “why?” and “how?” whereas clinicians have to focus on acquiring practical answers to “how?” and “what?” From a clinician’s perspective, asking “why?” distracts from the sense of mastery that comes from accumulating information and applying it in a clinical setting. Neither perspectives are wrong, they are just inherently different, and this results in a cultural gap. Unfortunately, this cultural gap hinders the translation of scientific discoveries into medical advances and may even hinder scientific progress <sup>[37]</sup>.</p>
<p>While this gap may be existent, this thesis aims to narrow this gap for clinicians and scientists working in the fields of clinical microbiology and microbial epidemiology, by providing an instrument that can be beneficial and usable for clinicians and scientists alike. Ultimately, it could yield more collaboration, communication, and efficacy between scientists and clinicians. The <code>AMR</code> package for <code>R</code> has empowered the four studies mentioned in <strong>SECTION III</strong>, which were conducted in the Northern Netherlands as well as in the Dutch-German cross-border region. In these studies, the <code>AMR</code> package affected the selection of isolates, determination of MDROs, or the entire AMR data analysis. Combined with the user survey results in <strong>chapter 3</strong> (that also included the use of the <code>AMR</code> package by both clinicians and scientists), the proof of concept of an integrated design in <strong>chapter 5</strong>, and the positive effects on clinical staff working with AMR data in <strong>chapter 6</strong>, this indicates that this new instrument can be deployed and used in a multi-disciplinarily fashion. Many others have pointed out the challenges in AMR data analysis on (cumulative) antibiograms and, inter alia, the necessity for correcting duplicate isolates <sup>[7,18,39–45]</sup>. Still, all these are theoretical and did not provide a pragmatic solution for those conducting microbial epidemiology. Hindler <em>et al.</em> presented a practical example of a data set that might require a correction for duplicate isolates (Table 1) <sup>[7]</sup>. The algorithm of choice could be isolate-based, patient-based, episode-based, or phenotype-based. This choice is dependent on the type of analysis and desired outcome. Table 2 illustrates the scope of the isolates that should be included based on a chosen algorithm and, more importantly, shows how the <code>AMR</code> package for <code>R</code> can be used to accomplish this in one simple command, underlining its approachability. Some of those functions to apply the respective algorithm using the <code>AMR</code> package for <code>R</code> have been used by others <sup>[15–18]</sup>.</p>
<p class="tbl-caption">
Table 1. Example AMR test results of four <em>Staphylococcus aureus</em> isolates from a single patient.
</p>
<p><img src="images/11-t01.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p class="tbl-caption">
Table 2. Algorithms for including isolates and the accompanying function in the <code>AMR</code> package for <code>R</code> for use in the AMR data analysis. The ‘x’ in the last column denotes any data set in a similar structure as Table 1.
</p>
<p><img src="images/11-t02.jpg" width="100%" style="display: block; margin: auto;" /></p>
<p>User feedback as presented in <strong>chapter 3</strong> implies that usage of the <code>AMR</code> package has led to higher reproducibility, higher reliability, new AMR insights and improved clinical decision-making. From <strong>chapter 5</strong> until <strong>chapter 10</strong>, it is shown that the <code>AMR</code> package can be a sensible and reliable tool for microbial isolate selection and conducting AMR data analysis. These examples indicate that the <code>AMR</code> package for <code>R</code> has the potential to become a centrepiece in AMR data analyses, which is further supported by its use in other scientific publications <sup>[15–18]</sup>. One of its most important features – enabling users to transform raw data into valuable new insights – allows data sets from any clinical source to be used. For example, data sets from different regions could be analysed every year in the same manner by reusing an automated AMR script, comparing trends in the occurrence of MDROs. This uniformity is an important advantage for gaining new AMR insights on the local, regional or national level and should be exploited to the fullest.</p>
<p>From an international point of view, it could be viable to achieve a common workflow with AMR interpretation guideline suppliers such as the Clinical and Laboratory Standards Institute (CLSI) and the European Committee on Antimicrobial Susceptibility Testing (EUCAST) <sup>[8,10]</sup>. These organisations provide clinical microbiology laboratories around the globe with static and manually formatted Microsoft Excel and Portable Document Format (PDF) files, requiring laboratory staff to manually apply guideline updated into their LIS. Since the <code>AMR</code> package for <code>R</code> contains machine-readable files of these (often yearly) guidelines, a collaborative workflow could lead to a more seamless implementation and update process in clinical laboratories worldwide, increasing reliability and reducing the workload on laboratory staff. These possible effects have yet to be studied.</p>
<p>Similarly, LIS manufacturers could benefit from the freely available comprehensive reference data about antimicrobial agents and the taxonomy of microorganisms that the <code>AMR</code> package provides. The data provided in the <code>AMR</code> package are automatically updated using services from the World Health Organization Collaborating Centre for Drug Statistics Methodology, PubChem, the Catalogue of Life, the List of Prokaryotic names with Standing in Nomenclature and SNOMED CT. LIS manufacturers could provide this same automated process or the data in the <code>AMR</code> package directly to their end-users (the clinical microbiology laboratories), to ensure a continuously up to date version of the reference data about antimicrobial agents and the taxonomy of microorganisms. This would mean that laboratories could be unburdened by losing the necessity of keeping their local data up to date. Maintaining these local data is of paramount importance, as all AMR interpretation guidelines are based on these data. It would strongly optimise the quality of the output of clinical routine laboratories. Aside from this optimisation, with less manual and tedious work to conduct microbial epidemiology for data-technical staff, using this presented new instrument hopefully also leads to faster availability of higher-quality research in the field of AMR, as well as a better patient outcome in clinical settings.</p>
<p>A more clinical example of the possibilities of the <code>AMR</code> package is to analyse microbiological data from urinary tract infections in comparison with blood culture data. Some patients suffer from a urinary tract infection but are admitted to the hospital with urosepsis sometime after. As these are the kind of clinical complications we should thrive to prevent, a full-region analysis of these data might shed light on the reasons why these clinical complications were not or could not be prevented. Fortunately, it is not difficult anymore to select patients who had a laboratory-confirmed urinary tract infection in primary care and a positive blood culture with the same pathogen in the weeks after. The <code>AMR</code> package can be used to do so, and to calculate suggestions for more specific and probably effective antibiotic treatment. This puzzle may not be easily solved, but it is at least now possible to get the data into the right format and have them generate the answers to back our hypotheses. Research initiatives to study this clinical example have recently commenced in both our Northern Dutch region and in the Dutch-German cross-border region.</p>
<p>Yet, the <code>AMR</code> package could be used for even more sophisticated outcomes by combining microbial epidemiology with computational intelligence, and this is where the real potential lies. For example, empirical sepsis therapy could become more personalised or, as others call it, become precision medicine by performing in-depth analyses of blood cultures isolates <sup>[46]</sup>. Blood cultures are namely the most reliable diagnostic measure for analysing microbes and their AMR, even if they are drawn from e.g. arterial catheters <sup>[47–49]</sup>. Combining AMR test results from blood culture isolates with patient demographics and hospital-specific traits might enable a comprehensive and multi-angle view on the patient’s disease. To specify, by stratifying patient demographics (such as age, gender, comorbidities, history of antibiotic consumption) and comparing them with hospital-specific traits (such as geographic location, common microbial findings, infection control measures, allowed number of patients per room), AMR data analysis could show major differences between all these patient stratifications. The subsequent results could be used to calculate the likelihood of finding similar pathogens and AMR in similar cases, leading to predictive modelling for upcoming septic patients. For example, a septic 60-year-old male patient with a long antibiotic consumption history due to chronic obstructive pulmonary disease (COPD) might require different empiric antiseptic treatment than a septic 60-year-old male patient without COPD and no antibiotic consumption history. In other words, this modelling could lead to personalised empiric treatment guidelines, increasing the chance of therapeutic success. For a study to investigate this, the <code>AMR</code> package for <code>R</code> could be used to identify eligible patients, compare the antibiotic consumption histories, and calculate the AMR rates for pre-defined groups. The <code>AMR</code> package can also calculate the empiric chance of success of different monotherapies and combination therapies using different algorithms. The output of the models will most probably be different between regions and could perhaps even differ per hospital, although the model itself could be universally implementable. Using predictive modelling for treating patients opens a new way of how we make the best use of our data; data we already have and have had for many years. Better yet, new data are generated each day and their quality is constantly improving, due to technical laboratory advancements. Although this specific example of predicting therapeutic success would have been impossible to study twenty years ago, it is highly feasible now. Others have already shown similar approaches recently to predict sepsis using neutrophil-to-lymphocyte ratios, neutrophil dysregulation, or high-resolution vital signs time series <sup>[50–52]</sup>. Yet, these modern approaches predict occurrence of sepsis and do not predict the likelihoods of the most effective empiric treatment if patients are already septic. Microbial epidemiology could pose an effective perspective to this end when collaborating with specialities such as acute care medicine and pharmacy, which links back to <strong>chapter 2</strong> about DSP: the right prediction at the right time for the right patient to (answer the right questions and) start the right predicted treatment. Still, improving empiric antiseptic treatment may feel like extensively training the goalkeeper. This may be necessary, but we should also realise that when the ball enters the penalty area, a lot has gone wrong already.</p>
<p>One might deduce that microbial epidemiology is not yet utilised to the fullest within clinical microbiology and this has a clear explanation. Advancements in information technology have progressed fast over the last decades, even more so over the last years. These advancements have led to improved LIS systems, enhanced software to apply complicated statistics and advanced mathematics, and even to this thesis. Thus, these advancements are quite novel, which means that they can bring new input to existing scientific fields such as clinical microbiology. Training and education are key in accelerating the required knowledge to apply these new advancements. This in turn will lead to the effect that e.g., clinical microbiologists and researchers in the field of clinical microbiology are urged to collaboratively think, develop and learn to work with these advancements. Yet, the cultural gap between clinicians and scientists as outlined earlier might inhibit progress to this end. Still, only collaborations and multi-disciplinary approaches will make sure that we can utilise the advancements in information technology up to their full potential, so patients will benefit most from our future scientific developments. For this reason, we should all strive to narrow and bridge the cultural gap. With regard to the practical labour concerning (predictive) modelling, it should perhaps become more common for research groups within our field, and probably many other research fields, to include (more) modellers and other data-technical staff.</p>
</div>
<div id="references-10" class="section level2 unnumbered">
<h2>References</h2>
<ol style="list-style-type: decimal">
<li>World Health Organization. Diagnostic stewardship: A guide to implementation in antimicrobial resistance surveillance sites. 2016.</li>
<li>Morgan DJ, Malani P, Diekema DJ. Diagnostic Stewardship-Leveraging the Laboratory to Improve Antimicrobial Use. JAMA 2017;171:157–64. <a href="doi:10.1001/jama.2017.8531" class="uri">doi:10.1001/jama.2017.8531</a>.</li>
<li>Dyar OJ, Huttner B, Schouten J, Pulcini C, ESGAP (ESCMID Study Group for Antimicrobial stewardshiP). What is antimicrobial stewardship? Clin Microbiol Infect 2017;23:793–8. <a href="doi:10.1016/j.cmi.2017.08.026" class="uri">doi:10.1016/j.cmi.2017.08.026</a>.</li>
<li>Mendelson M, Balasegaram M, Jinks T, Pulcini C, Sharland M. Antibiotic resistance has a language problem. Nature 2017;545:23–5. <a href="doi:10.1038/545023a" class="uri">doi:10.1038/545023a</a>.</li>
<li>Greub G, Moran-Gilad J, Rossen J, Egli A, ESCMID Study Group for Genomic and Molecular Diagnostics (ESGMD). ESCMID postgraduate education course: applications of MALDI-TOF mass spectrometry in clinical microbiology. Microbes Infect 2017;19:433–42. <a href="doi:10.1016/j.micinf.2017.06.004" class="uri">doi:10.1016/j.micinf.2017.06.004</a>.</li>
<li>Didelot X, Bowden R, Wilson DJ, Peto TEA, Crook DW. Transforming clinical microbiology with bacterial genome sequencing. Nat Rev Genet 2012;13:601–12. <a href="doi:10.1038/nrg3226" class="uri">doi:10.1038/nrg3226</a>.</li>
<li>Hindler JF, Stelling J. Analysis and Presentation of Cumulative Antibiograms: A New Consensus Guideline from the Clinical and Laboratory Standards Institute. Clin Infect Dis 2007;44:867–73. <a href="doi:10.1086/511864" class="uri">doi:10.1086/511864</a>.</li>
<li>European Committee on Antimicrobial Susceptibility Testing (EUCAST) 2021. <a href="https://eucast.org" class="uri">https://eucast.org</a> (accessed March 21, 2021).</li>
<li>Clinical and Laboratory Standards Institute (CLSI). M39-A4 Analysis and Presentation of Cumulative Antimicrobial Susceptibility Test Data, 4th Edition. 2014.</li>
<li>Clinical and Laboratory Standards Institute (CLSI) 2021. <a href="https://clsi.org" class="uri">https://clsi.org</a> (accessed March 21, 2021).</li>
<li>RStudio PBC. CRAN package download logs 2021. <a href="http://cran-logs.rstudio.com" class="uri">http://cran-logs.rstudio.com</a> (accessed March 31, 2021).</li>
<li>Leclercq R, Cantón R, Brown DFJ, Giske CG, Heisig P, MacGowan AP, <em>et al.</em> EUCAST expert rules in antimicrobial susceptibility testing. Clin Microbiol Infect 2013;19:141–60. <a href="doi:10.1111/j.1469-0691.2011.03703.x" class="uri">doi:10.1111/j.1469-0691.2011.03703.x</a>.</li>
<li>Barlam TF, Cosgrove SE, Abbo LM, MacDougall C, Schuetz AN, Septimus EJ, <em>et al.</em> Implementing an Antibiotic Stewardship Program: Guidelines by the Infectious Diseases Society of America and the Society for Healthcare Epidemiology of America. Clin Infect Dis 2016;62:e51-77. <a href="doi:10.1093/cid/ciw118" class="uri">doi:10.1093/cid/ciw118</a>.</li>
<li>Davey P, Marwick CA, Scott CL, Charani E, McNeil K, Brown E, <em>et al.</em> Interventions to improve antibiotic prescribing practices for hospital inpatients. Cochrane Database Syst Rev 2017;2:CD003543. <a href="doi:10.1002/14651858.CD003543.pub4" class="uri">doi:10.1002/14651858.CD003543.pub4</a>.</li>
<li>Le Guern R, Titécat M, Loïez C, Duployez C, Wallet F, Dessein R. Comparison of time-to-positivity between two blood culture systems: a detailed analysis down to the genus-level. Eur J Clin Microbiol Infect Dis 2021. <a href="doi:10.1007/s10096-021-04175-9" class="uri">doi:10.1007/s10096-021-04175-9</a>.</li>
<li>Dutey-Magni PF, Gill MJ, McNulty D, Sohal G, Hayward A, Shallcross L, <em>et al.</em> Feasibility study of hospital antimicrobial stewardship analytics using electronic health records. JAC-Antimicrobial Resist 2021;3. <a href="doi:10.1093/jacamr/dlab018" class="uri">doi:10.1093/jacamr/dlab018</a>.</li>
<li>N. Tenea G, Jarrin-V P, Yepez L. Microbiota of Wild Fruits from the Amazon Region of Ecuador: Linking Diversity and Functional Potential of Lactic Acid Bacteria with Their Origin. Ecosyst. Biodivers. Amaz., IntechOpen; 2021. <a href="doi:10.5772/intechopen.94179" class="uri">doi:10.5772/intechopen.94179</a>.</li>
<li>Kim S, Yoo SJ, Chang J. Importance of Susceptibility Rate of ‘the First’ Isolate: Evidence of Real-World Data. Medicina (B Aires) 2020;56:507. <a href="doi:10.3390/medicina56100507" class="uri">doi:10.3390/medicina56100507</a>.</li>
<li>Rogers KL, Fey PD, Rupp ME. Coagulase-Negative Staphylococcal Infections. Infect Dis Clin North Am 2009;23:73–98. <a href="doi:10.1016/j.idc.2008.10.001" class="uri">doi:10.1016/j.idc.2008.10.001</a>.</li>
<li>Dong Y, Speer CP. The role of <em>Staphylococcus epidermidis</em> in neonatal sepsis: Guarding angel or pathogenic devil? Int J Med Microbiol 2014;304:513–20. <a href="doi:10.1016/j.ijmm.2014.04.013" class="uri">doi:10.1016/j.ijmm.2014.04.013</a>.</li>
<li>Cheung GYC, Otto M. Understanding the significance of <em>Staphylococcus epidermidis</em> bacteremia in babies and children. Curr Opin Infect Dis 2010;23:208–16. <a href="doi:10.1097/QCO.0b013e328337fecb" class="uri">doi:10.1097/QCO.0b013e328337fecb</a>.</li>
<li>Stearns-Kurosawa DJ, Osuchowski MF, Valentine C, Kurosawa S, Remick DG. The pathogenesis of sepsis. Annu Rev Pathol 2011;6:19–48. <a href="doi:10.1146/annurev-pathol-011110-130327" class="uri">doi:10.1146/annurev-pathol-011110-130327</a>.</li>
<li>Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. <em>Staphylococcus epidermidis</em> as a cause of bacteremia. Future Microbiol 2015;10:1859–79. <a href="doi:10.2217/fmb.15.98" class="uri">doi:10.2217/fmb.15.98</a>.</li>
<li>Heilmann C, Ziebuhr W, Becker K. Are coagulase-negative staphylococci virulent? Clin Microbiol Infect 2018;0. <a href="doi:10.1016/j.cmi.2018.11.012" class="uri">doi:10.1016/j.cmi.2018.11.012</a>.</li>
<li>Becker K, Both A, Weißelberg S, Heilmann C, Rohde H. Emergence of coagulase-negative staphylococci. Expert Rev Anti Infect Ther 2020;18:349–66. <a href="doi:10.1080/14787210.2020.1730813" class="uri">doi:10.1080/14787210.2020.1730813</a>.</li>
<li>Pantůček R, Sedláček I, Indráková A, Vrbovská V, Mašlaňová I, Kovařovic V, <em>et al.</em> <em>Staphylococcus edaphicus</em> sp. nov., Isolated in Antarctica, Harbors the mecC Gene and Genomic Islands with a Suspected Role in Adaptation to Extreme Environments. Appl Environ Microbiol 2018;84:e01746-17. <a href="doi:10.1128/AEM.01746-17" class="uri">doi:10.1128/AEM.01746-17</a>.</li>
<li>Kassim A, Pflüger V, Premji Z, Daubenberger C, Revathi G. Comparison of biomarker based Matrix Assisted Laser Desorption Ionization-Time of Flight Mass Spectrometry (MALDI-TOF MS) and conventional methods in the identification of clinically relevant bacteria and yeast. BMC Microbiol 2017;17:128. <a href="doi:10.1186/s12866-017-1037-z" class="uri">doi:10.1186/s12866-017-1037-z</a>.</li>
<li>Lane MJ, Roy AF, Kearney MT, Pucheu-Haston CM. Characterization, distribution, antimicrobial resistance and resistance risk factors in staphylococci isolated from cats from 2001 to 2014. Vet Med Sci 2018;4:315–25. <a href="doi:10.1002/vms3.122" class="uri">doi:10.1002/vms3.122</a>.</li>
<li>Greeff S, Mouton J. NethMap 2018: Consumption of antimicrobial agents and antimicrobial resistance among medically important bacteria in the Netherlands / MARAN 2018: Monitoring of Antimicrobial Resistance and Antibiotic Usage in Animals in the Netherlands in 2017. Bilthoven: Rijksinstituut voor Volksgezondheid en Milieu RIVM Stichting Werkgroep Antibiotica Beleid SWAB; 2018.</li>
<li>European Centre for Disease Prevention and Control. European Antimicrobial Resistance Surveillance Network (EARS-Net) 2021. <a href="https://www.ecdc.europa.eu/en/about-us/partnerships-and-networks/disease-and-laboratory-networks/ears-net" class="uri">https://www.ecdc.europa.eu/en/about-us/partnerships-and-networks/disease-and-laboratory-networks/ears-net</a> (accessed March 25, 2021).</li>
<li>European Centre for Disease Prevention and Control. Antimicrobial resistance surveillance in Europe 2016. Annual Report of the European Antimicrobial Resistance Surveillance Network (EARS-Net). Stockholm: 2017.</li>
<li>van Cleef BAGL, Kluytmans JAJW, van Benthem BHB, Haenen A, Monen J, Daniels-Haardt I, <em>et al.</em> Cross border comparison of MRSA bacteraemia between The Netherlands and North Rhine-Westphalia (Germany): a cross-sectional study. PLoS One 2012;7:e42787. <a href="doi:10.1371/journal.pone.0042787" class="uri">doi:10.1371/journal.pone.0042787</a>.</li>
<li>Teicher CL, Ronat J-B, Fakhri RM, Basel M, Labar AS, Herard P, <em>et al.</em> Antimicrobial drug-resistant bacteria isolated from Syrian war-injured patients, August 2011-March 2013. Emerg Infect Dis 2014;20:1949–51. <a href="doi:10.3201/eid2011.140835" class="uri">doi:10.3201/eid2011.140835</a>.</li>
<li>Balkhair A, Al-Farsi YM, Al-Muharrmi Z, Al-Rashdi R, Al-Jabri M, Neilson F, <em>et al.</em> Epidemiology of multi-drug resistant organisms in a teaching hospital in oman: a one-year hospital-based study. ScientificWorldJournal 2014;2014:157102. <a href="doi:10.1155/2014/157102" class="uri">doi:10.1155/2014/157102</a>.</li>
<li>Szmigielski S, Prévost G, Monteil H, Colin DA, Jeljaszewicz J. Leukocidal toxins of staphylococci. Zentralbl Bakteriol 1999;289:185–201. <a href="doi:10.1016/s0934-8840(99)80105-4" class="uri">doi:10.1016/s0934-8840(99)80105-4</a>.</li>
<li>Schwartz WB, Patil RS, Szolovits P. Artificial intelligence in medicine. Where do we stand? N Engl J Med 1987;316:685–8. <a href="doi:10.1056/NEJM198703123161109" class="uri">doi:10.1056/NEJM198703123161109</a>.</li>
<li>Restifo LL, Phelan GR. The cultural divide: exploring communication barriers between scientists and clinicians. Dis Model Mech 2011;4:423–6. <a href="doi:10.1242/dmm.008177" class="uri">doi:10.1242/dmm.008177</a>.</li>
<li>Bargiela D, Verkerk MM. The clinician-programmer: designing the future of medical practice. BMJ 2013;347. <a href="doi:10.1136/bmj.f4563" class="uri">doi:10.1136/bmj.f4563</a>.</li>
<li>Bax R, Bywater R, Cornaglia G, Goossens H, Hunter P, Isham V, <em>et al.</em> Surveillance of antimicrobial resistance–what, how and whither? Clin Microbiol Infect 2001;7:316–25. <a href="doi:10.1046/j.1198-743x.2001.00239.x" class="uri">doi:10.1046/j.1198-743x.2001.00239.x</a>.</li>
<li>den Heijer CDJ, Penders J, Donker GA, Bruggeman CA, Stobberingh EE. The importance of gender-stratified antibiotic resistance surveillance of unselected uropathogens: a Dutch Nationwide Extramural Surveillance study. PLoS One 2013;8:e60497. <a href="doi:10.1371/journal.pone.0060497" class="uri">doi:10.1371/journal.pone.0060497</a>.</li>
<li>Chopra I, Roberts M. Tetracycline antibiotics: mode of action, applications, molecular biology, and epidemiology of bacterial resistance. Microbiol Mol Biol Rev 2001;65:232-60 ; second page, table of contents. <a href="doi:10.1128/MMBR.65.2.232-260.2001" class="uri">doi:10.1128/MMBR.65.2.232-260.2001</a>.</li>
<li>Patel D, Nimbalkar A, Sethi A, Kungwani A, Nimbalkar S. Blood culture isolates in neonatal sepsis and their sensitivity in Anand District of India. Indian J Pediatr 2014;81:785–90. <a href="doi:10.1007/s12098-013-1314-2" class="uri">doi:10.1007/s12098-013-1314-2</a>.</li>
<li>Critchley IA, Karlowsky JA. Optimal use of antibiotic resistance surveillance systems. Clin Microbiol Infect 2004;10:502–11. <a href="doi:10.1111/j.1469-0691.2004.00911.x" class="uri">doi:10.1111/j.1469-0691.2004.00911.x</a>.</li>
<li>Bosso JA, Mauldin PD, Steed LL. Consequences of combining cystic fibrosis- and non-cystic fibrosis-derived Pseudomonas aeruginosa antibiotic susceptibility results in hospital antibiograms. Ann Pharmacother 2006;40:1946–9. <a href="doi:10.1345/aph.1H377" class="uri">doi:10.1345/aph.1H377</a>.</li>
<li>Horvat RT, Klutman NE, Lacy MK, Grauer D, Wilson M. Effect of duplicate isolates of methicillin-susceptible and methicillin-resistant <em>Staphylococcus aureus</em> on antibiogram data. J Clin Microbiol 2003;41:4611–6. <a href="doi:10.1128/jcm.41.10.4611-4616.2003" class="uri">doi:10.1128/jcm.41.10.4611-4616.2003</a>.</li>
<li>Atreya MR, Wong HR. Precision medicine in pediatric sepsis. Curr Opin Pediatr 2019;31:322–7. <a href="doi:10.1097/MOP.0000000000000753" class="uri">doi:10.1097/MOP.0000000000000753</a>.</li>
<li>Ombelet S, Barbé B, Affolabi D, Ronat J-B, Lompo P, Lunguya O, <em>et al.</em> Best Practices of Blood Cultures in Low- and Middle-Income Countries. Front Med 2019;6:131. <a href="doi:10.3389/fmed.2019.00131" class="uri">doi:10.3389/fmed.2019.00131</a>.</li>
<li>Kleinschmidt S, Huygens F, Faoagali J, Rathnayake IU, Hafner LM. <em>Staphylococcus epidermidis</em> as a cause of bacteremia. Future Microbiol 2015;10:1859–79. <a href="doi:10.2217/fmb.15.98" class="uri">doi:10.2217/fmb.15.98</a>.</li>
<li>Berger I, Gil Margolis M, Nahum E, Dagan O, Levy I, Kaplan E, <em>et al.</em> Blood Cultures Drawn From Arterial Catheters Are Reliable for the Detection of Bloodstream Infection in Critically Ill Children. Pediatr Crit Care Med 2018;19:e213–8. <a href="doi:10.1097/PCC.0000000000001462" class="uri">doi:10.1097/PCC.0000000000001462</a>.</li>
<li>Huang Z, Fu Z, Huang W, Huang K. Prognostic value of neutrophil-to-lymphocyte ratio in sepsis: A meta-analysis. Am J Emerg Med 2020;38:641–7. <a href="doi:10.1016/j.ajem.2019.10.023" class="uri">doi:10.1016/j.ajem.2019.10.023</a>.</li>
<li>Shen X-F, Cao K, Jiang J-P, Guan W-X, Du J-F. Neutrophil dysregulation during sepsis: an overview and update. J Cell Mol Med 2017;21:1687–97. <a href="doi:10.1111/jcmm.13112" class="uri">doi:10.1111/jcmm.13112</a>.</li>
<li>Nemati S, Holder A, Razmi F, Stanley MD, Clifford GD, Buchman TG. An Interpretable Machine Learning Model for Accurate Prediction of Sepsis in the ICU. Crit Care Med 2018;46:547–53. <a href="doi:10.1097/CCM.0000000000002936" class="uri">doi:10.1097/CCM.0000000000002936</a>.</li>
</ol>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ch10-multi-mdro-screening.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="gearfetting-yn-frysk.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["a_new_instrument_for_microbial_epidemiology.pdf", "a_new_instrument_for_microbial_epidemiology.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
